{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "fabcaa23",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import numpy as np\n",
    "import requests\n",
    "import json\n",
    "import urllib, urllib.request"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "63d0a567",
   "metadata": {},
   "source": [
    "**Biorxiv API** information is found [here](https://api.biorxiv.org/details/medrxiv/help). "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 75,
   "id": "29687220",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "{'messages': [{'status': 'ok',\n",
       "   'interval': '2018-08-21:2018-08-28',\n",
       "   'cursor': '101',\n",
       "   'count': 100,\n",
       "   'count_new_papers': 450,\n",
       "   'total': 630}],\n",
       " 'collection': [{'doi': '10.1101/367060',\n",
       "   'title': 'Model-based assessment of public health impact and cost-effectiveness of dengue vaccination following screening for prior exposure',\n",
       "   'authors': 'Espana, G.; Yao, Y.; Anderson, K. B.; Fitzpatrick, M. C.; Smith, D. L.; Morrison, A. C.; Wilder-Smith, A.; Scott, T. W.; Perkins, A.',\n",
       "   'author_corresponding': 'Alex  Perkins',\n",
       "   'author_corresponding_institution': 'University of Notre Dame',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'epidemiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/367060.source.xml',\n",
       "   'abstract': 'The tetravalent dengue vaccine CYD-TDV (Dengvaxia(R)) is the first licensed vaccine against dengue, but recent findings indicate an elevated risk of severe disease among vaccinees without prior dengue virus (DENV) exposure. The World Health Organization currently recommends CYD-TDV only for individuals with serological confirmation of past DENV exposure. Our objective was to evaluate the potential impact and cost-effectiveness of vaccination following serological screening. To do so, we used an agent-based model to simulate DENV transmission with and without vaccination over a 30-year timeframe. Under a range of values for the proportion of vaccinees with prior DENV exposure, we projected the proportion of symptomatic and hospitalized cases averted as a function of the sensitivity and specificity of serological screening. Scenarios about the cost-effectiveness of screening and vaccination were chosen to be representative of Brazil and the Philippines. We found that public health impact depended primarily on the sensitivity of serological screening in high-transmission settings and on a combination of sensitivity and specificity in low-transmission settings. Reducing risk from an individual perspective required high specificity, no less than 0.8. Cost-effectiveness could be achievable from the perspective of a public payer provided that screening sensitivity for prior DENV exposure and the value of a disability-adjusted life-year were both high. Cost-effectiveness was also achievable from an individual perspective, particularly with high screening specificity and low coverage. Whereas the results of this analysis offer general guidelines about CYD-TDV vaccination, decisions in specific contexts would benefit from additional, more context-specific modeling analyses. In conclusion, vaccination with CYD-TDV following serological screening could have a positive impact in certain epidemiological settings, provided that screening is highly specific, at least moderately sensitive, and inexpensive.',\n",
       "   'published': '10.1371/journal.pntd.0007482',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/369553',\n",
       "   'title': 'The regulatory control of Cebpa enhancers and silencers in the myeloid and red-blood cell lineages',\n",
       "   'authors': 'Repele, A.; Krueger, S.; Tuineau, M. Y.; Manu, M.',\n",
       "   'author_corresponding': 'Manu  Manu',\n",
       "   'author_corresponding_institution': 'University of North Dakota',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/369553.source.xml',\n",
       "   'abstract': 'During development, cell identity is determined by lineage-specific transcriptional programs en-coded in the cis-regulatory DNA sequence of developmental genes. The sequence-level regulatory logic--identities of bound transcription factors (TFs), TF binding sites, and TF regulatory roles--of most developmental cis-regulatory modules (CRMs) is yet to be determined. We had previously developed an approach for inferring regulatory logic de novo by training sequence-based thermo-dynamic models on comprehensive reporter activity and gene expression datasets and applied it to Cebpa, an important hematopoietic gene. Here, we experimentally test thermodynamic models to decode the cis-regulatory logic of 4 enhancers and 3 silencers neighboring Cebpa at the resolution of individual binding sites. Cebpa is expressed at high and intermediate levels in neutrophils and macrophages respectively and downregulated in non-myeloid lineages. We tested the binding sites and functional roles of inferred TFs by designing and constructing mutated CRMs and comparing theoretical predictions of their activity against empirical measurements. Reporter activity was measured in PUER cells, which can be induced to differentiate into macrophages or neutrophils. All four enhancers were found to be simultaneously active in undifferentiated PUER cells and early-stage macrophages and neutrophils, and activated by combinations of PU.1, C/EBP family TFs, Egr1, and Gfi1. We show that silencers repress the activity of the proximal promoter in a dominant manner in G1ME cells, which are derived from the red-blood cell lineage. Dominant repression in G1ME cells can be traced to binding sites for GATA and Myb, a motif shared by all of the silencers. Finally, we demonstrate that GATA and Myb act redundantly to silence the proximal promoter. The result that silencers quench the promoter selectively in non-myeloid cells indicates that dominant repression is a novel mechanism for resolving hematopoietic lineages. Furthermore, Cebpa has a fail-safe cis-regulatory architecture, featuring several functionally similar CRMs, each of which contains redundant binding sites for multiple TFs. Lastly, by experimentally demonstrating the predictive ability of our sequence-based thermodynamic models, this work highlights the utility of this computational approach for decoding the logic of mammalian gene regulation.',\n",
       "   'published': '10.1371/journal.pone.0217580',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/369926',\n",
       "   'title': 'Seeing versus Knowing: The Temporal Dynamics of Real and Implied Colour Processing in the Human Brain',\n",
       "   'authors': 'Teichmann, L.; Grootswagers, T.; Carlson, T.; Rich, A.',\n",
       "   'author_corresponding': 'Lina  Teichmann',\n",
       "   'author_corresponding_institution': 'Macquarie University',\n",
       "   'date': '2018-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/369926.source.xml',\n",
       "   'abstract': 'Colour is a defining feature of many objects, playing a crucial role in our ability to rapidly recognise things in the world around us and make categorical distinctions. For example, colour is a useful cue when distinguishing lemons from limes or blackberries from raspberries. That means our representation of many objects includes key colour-related information. The question addressed here is whether the neural representation activated by knowing that something is red is the same as that activated when we actually see something red, particularly in regard to timing. We addressed this question using neural timeseries (magnetoencephalography, MEG) data to contrast real colour perception and implied object colour activation. We applied multivariate pattern analysis (MVPA) to analyse the brain activation patterns evoked by colour accessed via real colour perception and implied colour activation. Applying MVPA to MEG data allows us here to focus on the temporal dynamics of these processes. Male and female human participants (N=18) viewed isoluminant red and green shapes and grey-scale, luminance-matched pictures of fruits and vegetables that are red (e.g., tomato) or green (e.g., kiwifruit) in nature. We show that the brain activation pattern evoked by real colour perception is similar to implied colour activation, but that this pattern is instantiated at a later time. These results suggest that a common colour representation can be triggered by activating object representations from memory and perceiving colours.',\n",
       "   'published': '10.1016/j.neuroimage.2019.06.062',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/370874',\n",
       "   'title': 'A randomised controlled trial of an Intervention to Improve Compliance with the ARRIVE guidelines (IICARus)',\n",
       "   'authors': 'Hair, K.; Macleod, M. R.; Sena, E. S.; The IICARus Collaboration,  ',\n",
       "   'author_corresponding': 'Emily S Sena',\n",
       "   'author_corresponding_institution': 'University of Edinburgh',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'scientific communication and education',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/370874.source.xml',\n",
       "   'abstract': 'The ARRIVE (Animal Research: Reporting of In Vivo Experiments) guidelines are widely endorsed but compliance is limited. We sought to determine whether journal-requested completion of an ARRIVE checklist improves full compliance with the guidelines. In a randomised controlled trial, manuscripts reporting in vivo animal research submitted to PLOS ONE (March-June 2015) were allocated to either requested completion of an ARRIVE checklist or current standard practice. We measured the change in proportion of manuscripts meeting all ARRIVE guideline checklist items between groups. We randomised 1,689 manuscripts, 1,269 were sent for peer review and 762 accepted for publication. The request to complete an ARRIVE checklist had no effect on full compliance with the ARRIVE guidelines. Details of animal husbandry (ARRIVE sub-item 9a) was the only item to show improved reporting, from 52.1% to 74.1% (X2=34.0, df=1, p=2.1x10-7). These results suggest that other approaches are required to secure greater implementation of the ARRIVE guidelines.\\\\n\\\\nBackgroundThere are widespread failures across in vivo animal research to adequately describe and report research methods, including critical measures to reduce the risk of experimental bias (Kilkenny et al., 2009, Macleod et al., 2015). Such omissions have been shown to be associated with overestimation of effect sizes (Macleod et al., 2015, Hirst et al., 2014) and are likely to contribute, in part, to translational failure. In an effort to improve reporting standards, an expert working group coordinated by the National Centre for the Replacement, Refinement and Reduction of Animals in Research (NC3Rs) developed the Animal Research: Reporting of In Vivo Experiments (ARRIVE) guidelines (Kilkenny et al., 2010), published in 2010.\\\\n\\\\nSince the ARRIVE guidelines were first published, they have been endorsed by many journals in their instructions to authors, but this has not been accompanied by substantial improvements in reporting (Baker et al., 2014, McGrath and Lilley, 2015, Gulin et al., 2015a, Avey et al., 2016). Simply endorsing the guidelines does not appear to be sufficient to encourage compliance. Recent findings suggest that following the introduction of mandated completion of a distinct reporting checklist at ten Nature Journals at the stage of first revision significantly improved the quality in reporting versus that of comparator journals (Han et al., 2017, Macleod and The NPQIP Collaborative Group, 2017)\\\\n\\\\nPLOS ONE is an open access online only journal which at the time this study began published around 32,000 research articles per year. Of these, some 5,000 described in vivo research. At present, PLOS ONE instructions to authors encourage compliance with the ARRIVE guidelines, but do not mandate checklist completion. Journals have an important role to play in ensuring that the quality of reporting in the research they publish is robust, yet the most effective mechanism by which they can achieve this remains unclear.\\\\n\\\\nOur aim was to test the impact on the quality of published reports of an intervention which would request, at the time of manuscript submission, that authors complete a checklist detailing where in the manuscript the various components of the ARRIVE checklist were met. This study, to our knowledge, is the first randomised controlled trial of requested ARRIVE guideline completion.',\n",
       "   'published': '10.1186/s41073-019-0069-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/372383',\n",
       "   'title': 'Recurrent PD-L1 Structural Rearrangements in Natural Killer/T Cell Lymphoma Patients with Complete Response to PD-1 Blockade Therapy',\n",
       "   'authors': 'Lim, J. Q.; Tang, T.; Cai, Q.-q.; Tan, D.; Nairismagi, M.-L.; Laurensia, Y.; Chia, B. K. H.; Peng, R.-J.; Iqbal, J.; Huang, D.; Song, T.; Pang, W. L.; Cheah, D. M. Z.; Ng, C. C. Y.; Rajasegaran, V.; Hong, H.; Tse, E.; Mow, B.; Cai, Q.-C.; Poon, L.-M.; Tan, J.; Grigoropoulos, N. F.; Goh, Y. T.; Phipps, C.; Rotzschke, O.; Cheng, C. L.; Lee, Y. S.; Loh, Y.; Tao, M.; Farid, M.; Au-Yeung, R.; Chan, T. S.-Y.; Ng, S.-B.; Kwong, Y.-L.; Hwang, W.; Chng, W.-J.; Tousseyn, T.; Tan, P.; Teh, B. T.; Khor, C. C.; Rozen, S.; Bei, J.-X.; Lin, T.; Lim, S. T.; Ong, C. K.',\n",
       "   'author_corresponding': 'Choon Kiat  Ong',\n",
       "   'author_corresponding_institution': 'National Cancer Centre Singapore',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/372383.source.xml',\n",
       "   'abstract': 'AbstractThis study aims to identify recurrent genetic alterations in relapsed or refractory (RR) natural-killer/T-cell lymphoma (NKTL) patients who have achieved complete response (CR) with programmed cell death 1 (PD-1) blockade therapy. Seven of the eleven patients treated with pembrolizumab achieved CR while the remaining four had progressive disease (PD). Using whole genome sequencing (WGS), we found recurrent clonal structural rearrangements (SR) of the PD-L1 gene in four of the seven (57%) CR patients pretreated tumors. These PD-L1 SRs consist of inter-chromosomal translocations, tandem duplication and micro-inversion that disrupted the suppressive function of PD-L1 3UTR. Interestingly, recurrent JAK3-activating (p.A573V) mutations were also validated in two CR patients tumors that did not harbor the PD-L1 SR. Importantly, these mutations were absent in the four PD cases. With immunohistochemistry (IHC), PD-L1 positivity could not discriminate patients who archived CR (range: 6%-100%) from patients who had PD (range: 35%-90%). PD-1 blockade with pembrolizumab is a potent strategy for RR NKTL patients and genomic screening could potentially accompany PD-L1 IHC positivity to better select patients for anti-PD-1 therapy.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/371799',\n",
       "   'title': 'Implementation of the Water Framework Directive: Lessons learned and future perspectives for an ecologically meaningful classification of the status of Greek lakes, Mediterranean region',\n",
       "   'authors': 'Moustaka-Gouni, M.; Sommer, U.; Economou-Amilli, A.; Arhonditsis, G. B.; Katsiapi, M.; Papastergiadou, E.; Kormas, K. A.; Vardaka, E.; Karayanni, H.; Papadimitriou, T.',\n",
       "   'author_corresponding': 'Maria  Moustaka-Gouni',\n",
       "   'author_corresponding_institution': 'Aristotle University of Thessaloniki',\n",
       "   'date': '2018-08-27',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/371799.source.xml',\n",
       "   'abstract': 'The enactment of the Water Framework Directive (WFD) initiated scientific efforts to develop reliable methods for comparing prevailing lake conditions against reference (or non-impaired) states, using the state of a set biological elements. Drawing a distinction between impaired and natural conditions can be a challenging exercise, as it stipulates the robust delineation of reference conditions along with the establishment of threshold values for key environmental variables used as proxies for the degree of system impairment. Another important aspect is to ensure that water quality assessment is comparable among the different Member States. In this context, the present paper offers a constructive critique of the practices followed during the WFD implementation in Greece by pinpointing methodological weaknesses and knowledge gaps that undermine our ability to classify the ecological status of Greek lakes. One of the pillars of WDF is a valid lake typology that sets ecological standards transcending geographic regions and national boundaries. The national typology of Greek lakes has failed to take into account essential components (e.g. surface area, altitude, salinity). WFD compliance assessments based on descriptions of phytoplankton communities are oversimplified and as such should be revisited. Exclusion of most chroococcal species from the analysis of cyanobacteria biovolume in Greek lakes and most reservoirs in the Mediterranean Geographical Intercalibration Group (Greece, Spain, Portugal and Cyprus) is not consistent with the distribution of those taxa in lakes. Similarly, the total biovolume reference values and the indices used in their classification schemes reflect misunderstandings of WFD core principles. This hampers the comparability of ecological status across Europe and leads to quality standards that are too relaxed to provide an efficient target especially for the protection and management of Greek/transboundary lakes such as Lake Megali Prespa, one of the oldest lakes in Europe.',\n",
       "   'published': '10.1007/s00267-019-01226-y',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/373084',\n",
       "   'title': 'A biophysical model of molecular clusters: interplay of multivalency, membrane localization, steric hindrance, molecular flexibility and intracellular crowding',\n",
       "   'authors': 'Chattaraj, A.; Youngstrom, M.; Loew, L. M.',\n",
       "   'author_corresponding': 'Leslie M. Loew',\n",
       "   'author_corresponding_institution': 'University of Connecticut School of Medicine',\n",
       "   'date': '2018-08-26',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/373084.source.xml',\n",
       "   'abstract': 'Dynamic molecular clusters are assembled through weak multivalent interactions and are platforms for cellular functions, especially receptor-mediated signaling. Clustering is also a prerequisite for liquid-liquid phase separation. But it is not well understood how molecular structure and cellular organization control clustering. Using coarse-grain kinetic Langevin dynamics, we performed computational experiments on a prototypical ternary system modeled after membrane-bound nephrin, the adaptor Nck1 and the actin nucleation promoting factor NWASP. Steady state cluster size distributions favored stoichiometries that optimized binding (stoichiometry matching), but still were quite broad. At high concentrations, the system can be driven beyond the saturation boundary such that cluster size is limited only by the number of available molecules. This behavior would be predictive of phase separation. Domains close to binding sites sterically inhibited clustering much less than terminal domains because the latter effectively restrict access to the cluster interior. Increased flexibility of interacting molecules diminished clustering by shielding binding sites within compact conformations. Membrane association of nephrin increased the cluster size distribution in a density-dependent manner. These properties provide insights into how molecular ensembles function to localize and amplify cell signaling.',\n",
       "   'published': '10.1016/j.bpj.2019.01.001',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/375816',\n",
       "   'title': 'Lysosome exocytosis is required for mitosis',\n",
       "   'authors': 'Nugues, C.; Helassa, N.; Burgoyne, R.; Haynes, L.',\n",
       "   'author_corresponding': 'Lee  Haynes',\n",
       "   'author_corresponding_institution': 'University of Liverpool',\n",
       "   'date': '2018-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/375816.source.xml',\n",
       "   'abstract': 'Mitosis, the accurate segregation of duplicated genetic material into what will become two new daughter cells, is accompanied by extensive membrane remodelling and membrane trafficking activities. Early in mitosis, adherent cells partially detach from the substratum, round up and their surface area decreases. This likely results from an endocytic uptake of plasma membrane material. As cells enter cytokinesis they re-adhere, flatten and exhibit an associated increase in surface area. The identity of the membrane donor for this phase of mitosis remains unclear. Here we show by biochemical and imaging approaches that lysosomes undergo exocytosis at telophase and that this requires the activity of phosphatidylinositol 4-kinase-III{beta}. Inhibition of lysosome exocytosis resulted in mitotic failure in a significant proportion of cells suggesting that this facet of lysosome physiology is essential and represents a new regulatory mechanism in mitosis.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/377119',\n",
       "   'title': 'Compensatory mechanisms render Tcf7l1a dispensable for eye formation despite its requirement in eye field specification',\n",
       "   'authors': 'Young, R. M.; Cavodeassi, F.; Hawkins, T. A.; Stickney, H. L.; Schwarz, Q.; Lawrence, L. M.; Wierzbicki, C.; Gestri, G.; Ambrosio, E. M.; Klosner, A.; Rowell, J.; Bianco, I. H.; Allende, M. L.; Wilson, S. W.',\n",
       "   'author_corresponding': 'Rodrigo M  Young',\n",
       "   'author_corresponding_institution': 'University College London',\n",
       "   'date': '2018-08-25',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/377119.source.xml',\n",
       "   'abstract': 'The vertebrate eye originates from the eyefield, a domain of cells specified by a small number of transcription factors. In this study, we show that Tcf7la is one such transcription factor that acts cell-autonomously to specify the eye field in zebrafish. Despite the much reduced eyefield in tcf7l1a mutants, these fish develop normal eyes revealing a striking ability of the eye to recover from a severe early phenotype. This robustness is not mediated through compensation by paralogous genes; instead, the smaller optic vesicle of tcf7l1a mutants shows delayed neurogenesis and continues to grow until it achieves approximately normal size. Although the developing eye is robust to the lack of Tcf7l1a function, it is sensitised to the effects of additional mutations. In support of this, a forward genetic screen identified mutations in hesx1, cct5 and gdf6a, which give synthetically enhanced eye specification or growth phenotypes when in combination with the tcf7l1a mutation.',\n",
       "   'published': '10.7554/elife.40093',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/378109',\n",
       "   'title': 'Autoimmunity to hypocretin and molecular mimicry to flu antigens in Type 1 narcolepsy',\n",
       "   'authors': 'Luo, G.; Ambati, A.; Lin, L.; Bonvalet, M.; Partinen, M.; Ji, X.; Maecker, H. T.; Mignot, E.',\n",
       "   'author_corresponding': 'Emmanuel  Mignot',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2018-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'immunology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/378109.source.xml',\n",
       "   'abstract': 'Type 1 narcolepsy (T1N) is caused by hypocretin (HCRT) neuronal loss. Association with the Human Leukocyte Antigen (HLA)-DQB1*06:02/DQA1*01:02 (98% vs 25%) heterodimer (DQ0602), T cell receptor (TCR) and other immune loci suggest autoimmunity but autoantigen(s) are unknown. Onset is seasonal and associated with influenza A, notably pandemic 2009 H1N1 (pH1N1). An extensive unbiased DQ0602 binding peptide screen was performed encompassing peptides derived from Pandemrix(R) X-179-A pH1N1 influenza-A vaccine, a known T1N trigger, other H1N1 strains, and potential human autoantigens HCRT and RFX4, identifying 109 binders. The presence of cognate tetramer-peptide specific CD4+ T cells was studied in 35 narcolepsy cases and 22 DQ0602 controls after expansion of antigen-specific cells in Peripheral Blood Monocytes Cell (PBMC) cultures. Higher reactivity to influenza epitopes pHA273-287 (pH1N1 specific) and PR8 (H1N1 pre 2009)-specific NP17-31 were observed in T1N. Extensive reactivity to C-amidated but not native version of HCRT54-66 and HCRT86-97, which are two highly homologous peptides (HCRTNH2) was observed with higher frequencies of specific T cells in T1N. TCR/{beta} CDR3 sequences found in pHA273-287, NP17-31 and HCRTNH2 tetramer positive CD4+ cells were also retrieved in single INF{gamma}-secreting CD4+ sorted cells stimulated with Pandemrix(R), confirming immunodominance and functional significance in DQ0602-mediated responses and molecular mimicry. TCR/{beta} CDR3 motifs of HCRT54-66 and HCRT86-97 tetramers were extensively shared. Particularly notable was sharing across subjects of an CDR3, CAVETDSWGKLQF (in association with various CDR3{beta} that used TRAJ24, a chain modulated by Single Nucleotide Polymorphism (SNPs) rs1154155 and rs1483979 associated with T1N. Sharing of CDR3{beta} CASSQETQGRNYGYTF (in association with various CDR3 was also observed with HCRTNH2 and pHA273-287-tetramers across subjects. This segment uses TRBV4-2, a segment modulated by narcolepsy-associated SNP rs1008599. Higher HCRTNH2 positive CD4+ T cell numbers in T1N together with sharing of J24 CAVETDSWGKLQF in HCRTNH2 autoimmune responses, indicates causal DQ0602-mediated CD4+ autoreactivity to HCRT in T1N. Our results provide evidence for autoimmunity and molecular mimicry with flu antigens modulated by genetic components in the pathophysiology of T1N.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/379859',\n",
       "   'title': 'Overexpression of Arabidopsis ubiquitin ligase AtPUB46 enhances tolerance to drought and oxidative stress',\n",
       "   'authors': 'Adler, G.; Mishra, A. K.; Maymon, T.; Raveh, D.; Bar-Zvi, D.',\n",
       "   'author_corresponding': 'Dudy  Bar-Zvi',\n",
       "   'author_corresponding_institution': 'Ben-Gurion University of the Negev',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'plant biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/379859.source.xml',\n",
       "   'abstract': 'The U-Box E3 ubiquitin ligase, AtPUB46, functions in the drought response: T-DNA insertion mutants of this single paralogous gene are hypersensitive to water- and oxidative stress (Adler et al. BMC Plant Biology 17:8, 2017). Here we analyze the phenotype of AtPUB46 overexpressing (OE) plants. AtPUB46-OE show increased tolerance to water stress and have smaller leaf blades and reduced stomatal pore area and stomatal index compared with wild type (WT). Despite this, the rate of water loss from detached rosettes is similar in AtPUB46-OE and WT plants. Germination of AtPUB46-OE seeds was less sensitive to salt than WT whereas seedling greening was more sensitive. We observed a complex response to oxidative stress applied by different agents: AtPUB46-OE plants were hypersensitive to H2O2 but hyposensitive to methyl viologen. AtPUB46-GFP fusion protein is cytoplasmic, however, in response to H2O2 a considerable proportion translocates to the nucleus. We conclude that the differential stress phenotype of the AtPUB46-OE does not result from its smaller leaf size but from a change in the activity of a stress pathway(s) regulated by a degradation substrate of the AtPUB46 E3 and also from a reduction in stomatal pore size and index.\\\\n\\\\nAccession NumbersSequence data for this article can be found in: The Arabidopsis Information Resource database (http://www.arabidopsis.org) under accession numbers At5G18320 (PUB46).',\n",
       "   'published': '10.1016/j.plantsci.2018.08.018',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/380998',\n",
       "   'title': 'Effect of de novo transcriptome assembly on transcript quantification',\n",
       "   'authors': 'Hsieh, P.-H.; Oyang, Y.-J.; Chen, C.-Y.',\n",
       "   'author_corresponding': 'CHIEN-YU  CHEN',\n",
       "   'author_corresponding_institution': 'Department of Bio-Industrial Mechatronics Engineering, National Taiwan University',\n",
       "   'date': '2018-08-22',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/380998.source.xml',\n",
       "   'abstract': 'BackgroundCorrect quantification of transcript expression is essential to understand the functional products of the genome in different physiological conditions and developmental stages. Recently, the development of high-throughput RNA sequencing (RNA-Seq) allows the researchers to perform transcriptome analysis for the organisms without the reference genome and transcriptome. For such projects, de novo transcriptome assembly must be carried out prior to quantification. However, a large number of erroneous contigs produced by the assemblers might result in unreliable estimation on the abundance of transcripts. In this regard, this study comprehensively investigates how assembly quality affects the performance of quantification for RNA-Seq analysis based on de novo transcriptome assembly.\\\\n\\\\nResultsSeveral important factors that might seriously affect the accuracy of the RNA-Seq analysis were thoroughly discussed. First, we found that the assemblers perform comparatively well for the transcriptomes with lower biological complexity. Second, we examined the over-extended and incomplete contigs, and then demonstrated that assembly completeness has a strong impact on the estimation of contig abundance. Lastly, we investigated the behavior of the quantifiers with respect to sequence ambiguity which might be originally present in the transcriptome or accidentally produced by assemblers. The results suggest that the quantifiers often over-estimate the expression of family-collapse contigs and under-estimate the expression of duplicated contigs. For organisms without reference transcriptome, it remained challenging to detect the inaccurate abundance estimation on family-collapse contigs. On the contrary, we observed that the situation of under-estimation on duplicated contigs can be warned through analyzing the read distribution of the duplicated contigs.\\\\n\\\\nConclusionsIn summary, we explicated the behavior of quantifiers when erroneous contigs are present and we outlined the potential problems that the assemblers might cause for the downstream analysis of RNA-Seq. We anticipate the analytic results conducted in this study provides valuable insights for future development of transcriptome assembly and quantification.\\\\n\\\\nAvailabilitywe proposed an open-source Python based package QuantEval that builds connected components for the assembled contigs based on sequence similarity and evaluates the quantification results for each connected component. The package can be downloaded from https://github.com/dn070017/QuantEval.',\n",
       "   'published': '10.1038/s41598-019-44499-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/381855',\n",
       "   'title': 'Lesion Attributes Segmentation for Melanoma Detection with Deep Learning',\n",
       "   'authors': 'Chen, E. Z.; Dong, X.; Wu, J.; Jiang, H.; Li, X.; Rong, R.',\n",
       "   'author_corresponding': 'Eric Z. Chen',\n",
       "   'author_corresponding_institution': 'Dana-Farber Cancer Institute',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/381855.source.xml',\n",
       "   'abstract': 'Melanoma is the most deadly form of skin cancer worldwide. Many efforts have been made for early detection of melanoma. The International Skin Imaging Collaboration (ISIC) hosted the 2018 Challenges to help the diagnosis of melanoma based on dermoscopic images. In this paper, we describe our solutions for the task 2 of ISIC 2018 Challenges. We present two deep learning approaches to automatically detect lesion attributes of melanoma, one is a multi-task U-Net model and the other is a Mask R-CNN based model. Our multi-task U-Net model achieved a Jaccard index of 0.433 on official test data, which ranks the 5th place on the final leaderboard. The code for our solutions is publicly available.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/380733',\n",
       "   'title': 'Genomic response to selection for diurnality and nocturnality in Drosophila',\n",
       "   'authors': 'Pegoraro, M.; Flavell, L. M.; Menegazzi, P.; Colombi, P.; Dao, P.; Charlotte Helfrich-Förster, C.; Tauber, E.',\n",
       "   'author_corresponding': 'Eran  Tauber',\n",
       "   'author_corresponding_institution': 'University of Haifa',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/380733.source.xml',\n",
       "   'abstract': 'Most animals restrict their activity to a specific part of the day, being diurnal, nocturnal or crepuscular. The genetic basis underlying diurnal preference is largely unknown. Under laboratory conditions, Drosophila melanogaster is crepuscular, showing a bi-modal activity profile. However, a survey of strains derived from wild populations indicated that high variability among individuals exists, with diurnal and nocturnal flies being observed. Using a highly diverse population, we have carried out an artificial selection experiment, selecting flies with extreme diurnal or nocturnal preference. After 10 generations, we obtained highly diurnal and nocturnal strains. We used whole-genome expression analysis to identify differentially expressed genes in diurnal, nocturnal and crepuscular (control) flies. Other than one circadian clock gene (pdp1), most differentially expressed genes were associated with either clock output (pdf, to) or input (Rh3, Rh2, msn). This finding was congruent with behavioural experiments indicating that both light masking and the circadian pacemaker are involved in driving nocturnality. The diurnal and nocturnal selection strains provide us with a unique opportunity to understand the genetic architecture of diurnal preference.',\n",
       "   'published': '10.1186/s12864-020-07020-z',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/380808',\n",
       "   'title': 'SciPipe - A workflow library for agile development of complex and dynamic bioinformatics pipelines',\n",
       "   'authors': 'Lampa, S.; Dahlö, M.; Alvarsson, J.; Spjuth, O.',\n",
       "   'author_corresponding': 'Samuel  Lampa',\n",
       "   'author_corresponding_institution': 'Department of Pharmaceutical Biosciences, Uppsala University, Sweden',\n",
       "   'date': '2018-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/380808.source.xml',\n",
       "   'abstract': 'BackgroundThe complex nature of biological data has driven the development of specialized software tools. Scientific workflow management systems simplify the assembly of such tools into pipelines, assist with job automation and aid reproducibility of analyses. Many contemporary workflow tools are specialized and not designed for highly complex workflows, such as with nested loops, dynamic scheduling and parametrization, which is common in e.g. machine learning.\\\\n\\\\nFindingsSciPipe is a workflow programming library implemented in the programming language Go, for managing complex and dynamic pipelines in bioinformatics, cheminformatics and other fields. SciPipe helps in particular with workflow constructs common in machine learning, such as extensive branching, parameter sweeps and dynamic scheduling and parametrization of downstream tasks. SciPipe builds on Flow-based programming principles to support agile development of workflows based on a library of self-contained, re-usable components. It supports running subsets of workflows for improved iterative development, and provides a data-centric audit logging feature that saves a full audit trace for every output file of a workflow, which can be converted to other formats such as HTML, TeX and PDF on-demand. The utility of SciPipe is demonstrated with a machine learning pipeline, a genomics, and a transcriptomics pipeline.\\\\n\\\\nConclusionsSciPipe provides a solution for agile development of complex and dynamic pipelines, especially in machine leaning, through a flexible programming API suitable for scientists used to programming or scripting.',\n",
       "   'published': '10.1093/gigascience/giz044',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/378919',\n",
       "   'title': 'JAK2 is highly expressed by cyst-lining cells and regulates cystogenesis.',\n",
       "   'authors': 'Patera, F.; Cudzuch-Mardy, A.; Huang, Z.; Fragiadaki, M.',\n",
       "   'author_corresponding': 'Maria  Fragiadaki',\n",
       "   'author_corresponding_institution': 'University of Sheffield',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/378919.source.xml',\n",
       "   'abstract': 'Dysregulated JAK/STAT signalling is implicated in polycystic kidney disease, which is a common genetic disease leading to renal failure. However, the mechanisms underlying JAK/STAT-mediated cystogenesis arepoorlyunderstood.TheroleofJAK2wasinvestigated immunohistochemically in a murine model of cystic disease (Pkd1nl/nl). In the normal kidney, JAK2 is restricted to tubular epithelial and vascular cells with lesser staining in bowmans capsule and is undetectable in the interstitium. By contrast, in the diseased kidney JAK2 appears stronger in cyst-lining cells when compared to normal tubules and appears mislocalised in the interstitium. Given that JAK2 is a major tyrosine kinase activating JAK/STAT, we considered whether its inhibition can attenuate cystic growth in vitro. To assess this we used curcumin, a natural phytochemical, which significantly reduced JAK2 levels and STAT3 activity. Consistently, reduced JAK2/STAT3 activity was correlated with reduced cystic growth of cystic cells in three-dimensional cyst assays. Taken together, our results suggest that JAK2 is a key signalling molecule that functions to inhibit cystic growth in cystic tubular epithelial cells, thus providing the foundation for its development as a novel therapeutic in polycystic kidney disease.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/383240',\n",
       "   'title': 'The fitness consequences of genetic variation in wild populations of mice',\n",
       "   'authors': 'Barrett, R. D. H.; Laurent, S.; Mallarino, R.; Pfeifer, S. P.; Xu, C. C. Y.; Foll, M.; Wakamatsu, K.; Duke-Cohan, J. S.; Jensen, J. D.; Hoekstra, H. E.',\n",
       "   'author_corresponding': 'Hopi E. Hoekstra',\n",
       "   'author_corresponding_institution': 'Howard Hughes Medical Institute and Harvard University, USA',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/383240.source.xml',\n",
       "   'abstract': 'Adaptive evolution can occur when genetic change affects traits subject to natural selection. Although selection is a deterministic process, adaptation can be difficult to predict in finite populations because the functional connections between genotype, phenotype, and fitness are complex. Here, we make these connections using a combination of field and laboratory experiments. We conduct a large-scale manipulative field experiment with wild populations of deer mice in distinct habitats to directly estimate natural selection on pigmentation traits and next test whether this selection drives changes in allele frequency at an underlying pigment locus. We find that divergent cryptic phenotypes are repeatedly favoured in each habitat, leaving footprints of selection in the Agouti gene. Next, using transgenic experiments in Mus, we functionally test one of the Agouti mutations associated with survival, a Serine deletion in exon 2, and find that it causes lighter coat colour via changes in its protein binding properties. Finally, we show significant change in the frequency of this mutation in our field experiment. Together, our findings demonstrate how a sequence variant alters phenotype and show the ensuing ecological consequences that drive changes in population allele frequency, thereby revealing the full process of evolution by natural selection.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/379412',\n",
       "   'title': 'Where do animals covered by the US Animal Welfare Act live? Analysis of USDA licensed and registered entities',\n",
       "   'authors': 'Bennett, A. J.; Schoenbeck, E. G.',\n",
       "   'author_corresponding': 'Allyson J. Bennett',\n",
       "   'author_corresponding_institution': 'University of Wisconsin-Madison',\n",
       "   'date': '2018-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'zoology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/379412.source.xml',\n",
       "   'abstract': 'Nonhuman animals are housed in captivity for a range of purposes in the US and other countries. The regulation, oversight, and public transparency of decisions related to the care and use of those animals varies by species and activity. The goals of this paper are 1) to provide a concise and accessible summary of the number and types of facilities either registered or licensed by the USDA and overseen by the federal agency; and 2) to provide concise comparisons of the relative proportion of different types of use of nonhuman animals that fall under the AWA (Animal Welfare Act). Analysis of the publicly-available list of USDA certificate holders produced descriptive data for each state and the US overall. Licensed exhibitors (N=2,640; 33% of total) and breeders (N=2,701; 34% of total) comprise two-thirds of the 8,002 USDA certificate holders. Registered research facilities (N=1,110) account for 14% of the total USDA certificates. The final 19% consists of licensed dealers (N=763; 9%) and registered carriers or interim handlers (N=788; 10%). The number and distribution of types of certificates varies across states. The largest number of exhibitors and dealers are in Florida, while the largest number of breeders are in Missouri. California has the largest number of research registered facilities. Finally, a comparison of the estimated number of animals in the US that are pets, used in agriculture, exhibition, and research suggests that animals in research are between 0.07-2.3% of the total. The comparison also highlights areas in which public information about captive animals in the US is uneven or missing altogether. Together, the current report provides a concise view of basic information and data relevant to public consideration and policy decisions about animals housed in the set of captive settings to which the AWA applies.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/383679',\n",
       "   'title': 'Torpor duration, more than temperature, is key to hummingbird nighttime energy savings',\n",
       "   'authors': 'Shankar, A.; Schroeder, R. J.; Wethington, S. M.; Graham, C. H.; Powers, D. R.',\n",
       "   'author_corresponding': 'Anusha  Shankar',\n",
       "   'author_corresponding_institution': 'Stony Brook University',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'ecology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/383679.source.xml',\n",
       "   'abstract': 'O_LITorpor is an important energy saving strategy in small endotherms, but it has been insufficiently studied in natural field conditions. Building on what we know from laboratory studies, we compared torpor use across hummingbird species and different natural temperature regimes to explore predominant hypotheses about torpor use and evaluate the possible effects of environmental variation on energy management.\\\\nC_LIO_LIWe found that the probability of an individual entering torpor was correlated with mass and unrelated to nighttime temperature, and that hummingbirds at both warm tropical and cooler temperate sites used torpor.\\\\nC_LIO_LIEnergy savings in torpor were maximized as ambient temperatures approached a species minimum body temperature consistent with laboratory studies; energy savings ranged between 65-92% of energy per hour in torpor compared to normothermy.\\\\nC_LIO_LIHowever, variation in total nighttime energy expenditure was more significantly influenced by torpor bout duration than by the variation in energy savings in torpor.\\\\nC_LIO_LIOur results show that a small endotherms nighttime energy management in its natural habitat is more affected by torpor bout duration, which is linked to photoperiod, than by temperature. This result suggests that in their natural environments, hummingbirds are able to save energy in torpor across a range nighttime temperature, indicating that they may have sufficient physiological flexibility to tolerate climatic variation.\\\\nC_LI',\n",
       "   'published': '10.1111/jav.02305',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/384032',\n",
       "   'title': 'Transcriptomic and functional analysis of the oosome, a unique form of germ plasm in the wasp Nasonia vitripennis',\n",
       "   'authors': 'Quan, H.; Lynch, J. A.',\n",
       "   'author_corresponding': 'Jeremy A Lynch',\n",
       "   'author_corresponding_institution': 'University of Illinois at Chicago',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/384032.source.xml',\n",
       "   'abstract': 'BackgroundThe oosome is the germline determinant in the wasp Nasonia vitripennis and is homologous to the polar granules of Drosophila. Despite a common evolutionary origin and developmental role, the oosome is morphologically quite distinct from polar granules. It is a solid sphere that migrates within the cytoplasm before budding out and forming pole cells.\\\\n\\\\nResultsTo gain an understanding of both the molecular basis of the novel form of the oosome, and the conserved essential features of germ plasm, we quantified and compared transcript levels between embryo fragments that contained the oosome, and those that did not. The identity of the localized transcripts indicated that Nasonia uses different molecules to carry out conserved germ plasm functions. In addition, functional testing of a sample of localized transcripts revealed potentially novel mechanisms of ribonucleoprotein assembly and pole cell cellularization in the wasp.\\\\n\\\\nConclusionsOur results demonstrate that numerous novel and unexpected molecules have been recruited in order produce the unique characteristics of the oosome and pole cell formation in Nasonia. This work will serve as the basis for further investigation into the patterns of germline determinant evolution among insects, the molecular basis of extreme morphology of ribonucleoproteins, and the incorporation of novel components into developmental networks.',\n",
       "   'published': '10.1186/s12915-019-0696-7',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/384032',\n",
       "   'title': 'Transcriptomic and functional analysis of the oosome, a unique form of germ plasm in the wasp Nasonia vitripennis',\n",
       "   'authors': 'Quan, H.; Lynch, J. A.',\n",
       "   'author_corresponding': 'Jeremy A Lynch',\n",
       "   'author_corresponding_institution': 'University of Illinois at Chicago',\n",
       "   'date': '2018-08-25',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/384032.source.xml',\n",
       "   'abstract': 'BackgroundThe oosome is the germline determinant in the wasp Nasonia vitripennis and is homologous to the polar granules of Drosophila. Despite a common evolutionary origin and developmental role, the oosome is morphologically quite distinct from polar granules. It is a solid sphere that migrates within the cytoplasm before budding out and forming pole cells.\\\\n\\\\nResultsTo gain an understanding of both the molecular basis of the novel form of the oosome, and the conserved essential features of germ plasm, we quantified and compared transcript levels between embryo fragments that contained the oosome, and those that did not. The identity of the localized transcripts indicated that Nasonia uses different molecules to carry out conserved germ plasm functions. In addition, functional testing of a sample of localized transcripts revealed potentially novel mechanisms of ribonucleoprotein assembly and pole cell cellularization in the wasp.\\\\n\\\\nConclusionsOur results demonstrate that numerous novel and unexpected molecules have been recruited in order produce the unique characteristics of the oosome and pole cell formation in Nasonia. This work will serve as the basis for further investigation into the patterns of germline determinant evolution among insects, the molecular basis of extreme morphology of ribonucleoproteins, and the incorporation of novel components into developmental networks.',\n",
       "   'published': '10.1186/s12915-019-0696-7',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/384586',\n",
       "   'title': 'bayNorm: Bayesian gene expression recovery, imputation and normalisation for single cell RNA-sequencing data',\n",
       "   'authors': 'Tang, W.; Bertaux, F.; Thomas, P.; Stefanelli, C.; Saint, M.; Marguerat, S. B.; Shahrezaei, V.',\n",
       "   'author_corresponding': 'Vahid  Shahrezaei',\n",
       "   'author_corresponding_institution': 'Imperial College London',\n",
       "   'date': '2018-08-26',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/384586.source.xml',\n",
       "   'abstract': 'Normalisation of single cell RNA sequencing (scRNA-seq) data is a prerequisite to their interpretation. The marked technical variability and high amounts of missing observations typical of scRNA-seq datasets make this task particularly challenging. Here, we introduce bayNorm, a novel Bayesian approach for scaling and inference of scRNA-seq counts. The methods likelihood function follows a binomial model of mRNA capture, while priors are estimated from expression values across cells using an empirical Bayes approach. We demonstrate using publicly-available scRNA-seq datasets and simulated expression data that bayNorm allows robust imputation of missing values generating realistic transcript distributions that match single molecule FISH measurements. Moreover, by using priors informed by dataset structures, bayNorm improves accuracy and sensitivity of differential expression analysis and reduces batch effect compared to other existing methods. Altogether, bayNorm provides an efficient, integrated solution for global scaling normalisation, imputation and true count recovery of gene expression measurements from scRNA-seq data.',\n",
       "   'published': '10.1093/bioinformatics/btz726',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/386458',\n",
       "   'title': 'The crossover function of MutSγ is activated via Cdc7-dependent stabilization of Msh4',\n",
       "   'authors': 'He, W.; Rao, H. B. D. P.; Tang, S.; Bhagwat, N.; Kulkarni, D.; Ma, Y.; Chang, M. A. W.; Hall, C.; Wang, J.; Manacsa, H. S.; Baker, C.; Verhees, G. F.; Singh, L.; Chen, X.; Hollingsworth, N. M.; Cejka, P.; Hunter, N.',\n",
       "   'author_corresponding': 'Neil  Hunter',\n",
       "   'author_corresponding_institution': 'University of California, Davis',\n",
       "   'date': '2018-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genetics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/386458.source.xml',\n",
       "   'abstract': 'The MutS{gamma} complex, Msh4-Msh5, binds DNA joint-molecule (JM) intermediates during homologous recombination to promote crossing over and accurate chromosome segregation at the first division of meiosis. MutS{gamma} facilitates the formation and biased resolution of crossover-specific JM intermediates called double Holliday junctions. Here we show that these activities are governed by regulated proteasomal degradation. MutS{gamma} is initially inactive for crossing over due to an N-terminal degron on Msh4 that renders it unstable. Activation of MutS{gamma} requires the Dbf4-dependent kinase, Cdc7 (DDK), which directly phosphorylates and thereby neutralizes the Msh4 degron. Phosphorylated Msh4 is chromatin bound and requires DNA strand exchange and chromosome synapsis, implying that DDK specifically targets MutS{gamma} that has already bound nascent JMs. Our study establishes regulated protein degradation as a fundamental mechanism underlying meiotic crossover control.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/386466',\n",
       "   'title': 'Can patients with cerebellar disease switch learning mechanisms to reduce their adaptation deficits?',\n",
       "   'authors': 'Wong, A. L.; Marvel, C. L.; Taylor, J. A.; Krakauer, J. W.',\n",
       "   'author_corresponding': 'Aaron L Wong',\n",
       "   'author_corresponding_institution': 'Moss Rehabilitation Research Institute',\n",
       "   'date': '2018-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/386466.source.xml',\n",
       "   'abstract': 'Systematic perturbations in motor adaptation tasks are primarily countered by learning from sensory-prediction errors, with secondary contributions from other learning processes. Despite the availability of these additional processes, particularly the use of explicit re-aiming to counteract observed target errors, patients with cerebellar degeneration are surprisingly unable to compensate for their sensory-prediction-error deficits by spontaneously switching to another learning mechanism. We hypothesized that if the nature of the task was changed - by allowing vision of the hand, which eliminates sensory-prediction errors - patients could be induced to preferentially adopt aiming strategies to solve visuomotor rotations. To test this, we first developed a novel visuomotor rotation paradigm that provides participants with vision of their hand in addition to the cursor, effectively setting the sensory-prediction-error signal to zero. We demonstrated in younger healthy controls that this promotes a switch to strategic re-aiming based on target errors. We then showed that with vision of the hand, patients with spinocerebellar ataxia could also switch to an aiming strategy in response to visuomotor rotations, performing similarly to age-matched participants (older controls). Moreover, patients could retrieve their learned aiming solution after vision of the hand was removed, and retain it for at least one year. Both patients and older controls, however, exhibited impaired overall adaptation performance compared to younger healthy controls (age, 18-33), likely due to age-related reductions in spatial and working memory. Moreover, patients failed to generalize, i.e., they were unable to adopt analogous aiming strategies in response to novel rotations, nor could they further improve their performance without vision of the hand. Hence, there appears to be an inescapable obligatory dependence on sensory-prediction-error-based learning - even when this system is impaired in patients with cerebellar degeneration. The persistence of sensory-prediction-error-based learning effectively suppresses a switch to target-error-based learning, which perhaps explains the unexpectedly poor performance by patients with spinocerebellar ataxia in visuomotor adaptation tasks.',\n",
       "   'published': '10.1093/brain/awy334',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/385807',\n",
       "   'title': 'Molecular Heterogeneity and Early Metastatic Clone Selection in Testicular Germ Cell Cancer Development',\n",
       "   'authors': 'Dorssers, L. C.; Gillis, A. J.; Stoop, H.; van Marion, R.; Nieboer, M. M.; van Riet, J.; van de Werken, H. J.; Oosterhuis, J. W.; de Ridder, J.; Looijenga, L. H.',\n",
       "   'author_corresponding': 'Lambert CJ Dorssers',\n",
       "   'author_corresponding_institution': 'Erasmus MC Cancer Institute, University Medical Center Rotterdam',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cancer biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/385807.source.xml',\n",
       "   'abstract': 'BackgroundTesticular germ cell cancer (TGCC), being the most frequent malignancy in young Caucasian males, is initiated from an embryonic germ cell. This study determines intratumor heterogeneity to unravel tumor progression from initiation till metastasis.\\\\n\\\\nMethodsIn total 42 purified samples of four treatment-resistant nonseminomatous TGCC (NS) were investigated, including the precursor germ cell neoplasia in situ (GCNIS) and metastatic specimens, using whole genome- and targeted sequencing. Their evolution was reconstructed.\\\\n\\\\nResultsIntratumor molecular heterogeneity did not correspond to the supposed primary tumor histological evolution. Metastases after systemic treatment could be derived from cancer stem cells not identified in the primary cancer. GCNIS mostly lacked the molecular marks of the primary NS and comprised dominant clones that failed to progress. A BRCA-like mutational signature was observed without evidence for direct involvement of BRCA1 and BRCA2 genes.\\\\n\\\\nConclusionsOur data strongly support the hypothesis that NS is initiated by whole genome duplication, followed by chromosome copy number alterations in the cancer stem cell population, and accumulation of low numbers of somatic mutations. These observations of heterogeneity at all stages of tumorigenesis should be considered when treating patients with GCNIS-only disease, or with clinically overt NS.',\n",
       "   'published': '10.1038/s41416-019-0381-1',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/388074',\n",
       "   'title': 'Abnormalities of placental development and function are associated with the different fetal growth patterns of hypoplastic left heart syndrome and transposition of the great arteries.',\n",
       "   'authors': 'Troja, W.; Owens, K. J.; Courtney, J.; Hinton, A. C.; Hinton, R. B.; Cnota, J. F.; Jones, H. N.',\n",
       "   'author_corresponding': 'Helen N Jones',\n",
       "   'author_corresponding_institution': \"Cincinnati Children's Hospital Medical Center\",\n",
       "   'date': '2018-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'physiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/388074.source.xml',\n",
       "   'abstract': 'BackgroundBirthweight is a critical predictor of congenital heart disease (CHD) surgical outcomes. Hypoplastic left heart syndrome (HLHS) is cyanotic CHD with known fetal growth restriction and placental abnormalities. Transposition of the great arteries (TGA) is cyanotic CHD with normal fetal growth. Comparison of the placenta in these diagnoses may provide insights on the fetal growth abnormality of CHD.\\\\n\\\\nMethodsClinical data and placental histology from placentas associated with Transposition of the Great Arteries (TGA) were analyzed for gross pathology, morphology, maturity and vascularity and compared to both control and previously analyzed HLHS placentas [1]. RNA was isolated from HLHS, TGA and control placentas and sequenced by Illumina HiSeq.Gene, analysis was performed using TopHat, R and MSigDB. Cluster analysis was performed using GoElite and Pathway analysis performed using PANTHERdb Overrepresentation Test. Immunohistochemistry was utilized to assess placental nutrient transporter expression in all three groups.\\\\n\\\\nResultsPlacental weight was reduced in TGA cases, and demonstrated reduced villous vasculature, immature terminal villi, and increased fibrin deposition in the parenchyma compared to controls and reflected our previous data from HLHS placentas. However, birth weight was not reduced in TGA cases compared to controls in contrast to the HLHS cohort and birthweight:placental weight ratio was significantly increased in TGA cases but not HLHS compared to control. Need to include RNA and IHC.\\\\n\\\\nConclusionsDespite common vascular disturbances in placentas from HLHAs and TGA, these do not account for the',\n",
       "   'published': '10.1016/j.placenta.2020.09.007',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/389296',\n",
       "   'title': 'One read per cell per gene is optimal for single-cell RNA-Seq',\n",
       "   'authors': 'Zhang, M. J.; Ntranos, V.; Tse, D.',\n",
       "   'author_corresponding': 'David  Tse',\n",
       "   'author_corresponding_institution': 'Stanford University',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/389296.source.xml',\n",
       "   'abstract': 'An underlying question for virtually all single-cell RNA sequencing experiments is how to allocate the limited sequencing budget: deep sequencing of a few cells or shallow sequencing of many cells? A mathematical framework reveals that, for estimating many important gene properties, the optimal allocation is to sequence at the depth of one read per cell per gene. Interestingly, the corresponding optimal estimator is not the widely-used plugin estimator but one developed via empirical Bayes.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/389460',\n",
       "   'title': 'The C-terminal tail of the bacterial translocation ATPase SecA modulates its activity',\n",
       "   'authors': 'Jamshad, M.; Knowles, T.; White, S.; Ward, D.; Mohammed, F.; Rahman, K. F.; Hughes, G. W.; Kramer, G.; Bukau, B.; Huber, D.',\n",
       "   'author_corresponding': 'Damon  Huber',\n",
       "   'author_corresponding_institution': 'University of Birmingham',\n",
       "   'date': '2018-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'biochemistry',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/389460.source.xml',\n",
       "   'abstract': 'In bacteria, the translocation of a subset of proteins across the cytoplasmic membrane by the Sec machinery requires SecA. Although SecA can recognise nascent polypeptides, the mechanism of cotranslational substrate protein recognition is not known. Here, we investigated the role of the C-terminal tail (CTT) of SecA, which consists of a flexible linker (FLD) and a small metal-binding domain (MBD), in its interaction with nascent polypeptides. Phylogenetic analysis and ribosome binding experiments indicated that the MBD interacts with 70S ribosomes. Disruption of the entire CTT or the MBD alone had opposing effects on ribosome binding, substrate-protein binding, ATPase activity and in vivo function. Autophotocrosslinking, mass spectrometry, x-ray crystallography and small-angle x-ray scattering experiments provided insight into the CTT-mediated conformational changes in SecA. Finally, photocrosslinking experiments indicated that binding of SecA to substrate protein affected its interaction with the ribosome. Taken together, our results suggest a mechanism for substrate protein recognition.\\\\n\\\\nImpact StatementSecA is an evolutionarily conserved ATPase that is required for the translocation of a subset of proteins across the cytoplasmic membrane in bacteria. We investigated how SecA recognises its substrate proteins at the ribosome as they are still being synthesised (i.e. cotranslationally).',\n",
       "   'published': '10.7554/eLife.48385',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/389890',\n",
       "   'title': 'Latent Structure of Risk Perception',\n",
       "   'authors': 'Pabon, E.; MacKillop, J.; Palmer, A.; de Wit, H.',\n",
       "   'author_corresponding': 'Elisa  Pabon',\n",
       "   'author_corresponding_institution': 'University of Chicago',\n",
       "   'date': '2018-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/389890.source.xml',\n",
       "   'abstract': 'Risk-taking behavior affects many aspects of life, including maladaptive behaviors such as illicit substance use, unsafe driving, and risky sexual behavior. Risk-taking has been measured using both self-report measures and behavioral tasks designed for the purpose, but there is little consensus in the associations among measures and our understanding of the latent constructs underlying different forms of risk is limited. In the present study we examined the construct of risk using data from over 1000 young adults who completed measures of risk-taking, including self-reports of perception of risk, propensity to engage in risky behaviors and performance on behavioral tasks designed to measure risk. To examine the latent structure of risk preferences, we conducted a principal component analysis (PCA). The PCA revealed a latent structure of three distinct components of risk-taking behavior: \\\\\"Lifestyle Risk Sensitivity\\\\\", \\\\\"Financial Risk Sensitivity\\\\\", and \\\\\"Behavioral Risk Sensitivity\\\\\", which consisted only of the Balloon Analogue Risk Task (BART; Lejuez et al., 2002). As expected, risk-taking and perception of risk differed in men and women. Yet, the PCA components were similar in men and women. Future work utilizing additional measures of risk-taking behavior in more heterogeneous samples will help to identify the true biobehavioral constructs underlying these behaviors.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/390419',\n",
       "   'title': 'Complex genetic and epigenetic regulation deviates gene expression from a unifying global transcriptional program',\n",
       "   'authors': 'Chagoyen, M.; Poyatos, J. F.',\n",
       "   'author_corresponding': 'Juan F Poyatos',\n",
       "   'author_corresponding_institution': 'Spanish National Biotechnology Centre (CNB-CSIC)',\n",
       "   'date': '2018-08-25',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'systems biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/390419.source.xml',\n",
       "   'abstract': 'Environmental or genetic perturbations lead to gene expression changes. While most analyses of these changes emphasize the presence of qualitative differences on just a few genes, we now know that changes are widespread. This large-scale variation has been linked to the exclusive influence of a global transcriptional program determined by the new physiological state of the cell. However, given the sophistication of eukaryotic regulation, we expect to have a complex architecture of specific control affecting this program. Here, we examine this architecture. Using data of Saccharomyces cerevisiae expression in different nutrient conditions, we first propose a five-sector genome partition, which integrates earlier models of resource allocation, as a framework to examine the deviations from the global control. In this scheme, we recognize invariant genes, whose regulation is dominated by physiology, specific genes, which substantially depart from it, and two additional classes that contain the frequently assumed growth-dependent genes. Whereas the invariant class shows a considerable absence of specific regulation, the rest is enriched by regulation at the level of transcription factors (TFs) and epigenetic modulators. We nevertheless find markedly different strategies in how these classes deviate. On the one hand, there are TFs that act in a unique way between partition constituents, and on the other, the action of chromatin modifiers is significantly diverse. The balance between regulatory strategies ultimately modulates the action of the general transcription machinery and therefore limits the possibility of establishing a unifying program of expression change at a genomic scale.',\n",
       "   'published': '10.1371/journal.pcbi.1007353',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/390773',\n",
       "   'title': 'Quantifying the potential for future gene therapy to lower lifetime risk of polygenic late-onset diseases',\n",
       "   'authors': 'Oliynyk, R. T.',\n",
       "   'author_corresponding': 'Roman Teo Oliynyk',\n",
       "   'author_corresponding_institution': 'University of Auckland Faculty of Science',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '4',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/390773.source.xml',\n",
       "   'abstract': 'BackgroundWithin the next few decades, gene therapy techniques and genetic knowledge may sufficiently advance to support prophylactic gene therapy to prevent polygenic late-onset diseases. A combination of a large number of common low effect gene variants offers the most likely explanation for the heritability of the majority of these diseases, and their risk may be lowered by correcting the effect of a subset of such gene variants.\\\\n\\\\nMethodsComputer simulations quantified the correlation between the aging process, polygenic risk score, and hazard ratio change with age, using as inputs clinical incidence rates and familial heritability, and estimated the outcomes of hypothetical future prophylactic gene therapy on the lifetime risk and age of onset for eight highly prevalent late-onset diseases.\\\\n\\\\nResultsThe simulation results confirmed that gene therapy would be beneficial in both delaying the age of onset and lowering the lifetime risk of the analyzed lateonset diseases. Longer life expectancy is associated with a higher lifetime risk of these diseases, and this tendency, while delayed, will continue after the therapy.\\\\n\\\\nConclusionsWhen the gene therapy as hypothesized here becomes possible, and if the incidences of the treated diseases follow the proportional hazards model with multiplicative genetic architecture composed of a sufficient number of common small effect gene variants, then (a) late-onset diseases with the highest familial heritability will have the largest number of variants available for editing, (b) diseases that currently have the highest lifetime risk, and particularly those with the highest incidence rate continuing into older ages, will prove the most challenging cases in which to lower lifetime risk and delay the age of onset at the populational level, and (c) diseases that are characterized by the lowest lifetime risk will show the strongest and longest-lasting response to such therapy.',\n",
       "   'published': '10.3390/ijms20133352',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/392373',\n",
       "   'title': 'The N-end Rule Pathway and Ubr1 enforces the compartmentalization of mitochodrial and secretory proteins through the recognition of P2-encoded cellular location signals',\n",
       "   'authors': 'Tran, A.',\n",
       "   'author_corresponding': 'Anthony  Tran',\n",
       "   'author_corresponding_institution': 'National University of Singapore',\n",
       "   'date': '2018-08-25',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/392373.source.xml',\n",
       "   'abstract': 'The Arg/N-end rule pathway, a mechanism of protein degradation conserved from yeast to humans, is involved in cellular protein quality control, but its role has only been vaguely understood. Through systematic examination of single residue mutants of model misfolded substrates, and global analyses of yeast proteins, we discovered that Ubr1, an E3 ligase of the Arg/N-end rule, degrades organellar proteins that fail to reach their intended subcellular compartments. We determined that recognition by Ubr1 is dependent on location signals that are naturally embedded into the 2nd amino acid residue of the majority of proteins. The N-end rule pathway is thus likely to have been critical to the evolution of endosymbiotic relationships which paved the way for advanced eukaryotic cellular life.\\n\\nSignificance StatementThis work elucidates a novel role for the N-end Rule Pathway, a protein degradation pathway highly conserved from yeast to humans. We demonstrate that the N-end rule pathway enforces the cellular compartmentalization of ER and mitochondrial proteins by degrading them when they fail to successfully translocate into their intended destinations and thus become mislocalized to the cytosol. This mechanism prevents the accumulation of toxic foreign proteins within the cytosol. Recognition of the displaced proteins is dependent on cellular location signals programmed into the 2nd residue of the target proteins, as well as the tendency for the proteins to misfold in a foreign environment. These findings have significant relevance to research on the mechanisms causing human diseases involving protein misfolding.',\n",
       "   'published': '10.1242/jcs.231662',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/393330',\n",
       "   'title': 'Putative pore-forming subunits of the mechano-electrical transduction channel, Tmc1/2b, require Tmie to localize to the site of mechanotransduction in zebrafish sensory hair cells',\n",
       "   'authors': 'Pacentine, I.; Nicolson, T.',\n",
       "   'author_corresponding': 'Teresa  Nicolson',\n",
       "   'author_corresponding_institution': 'OHSU',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/393330.source.xml',\n",
       "   'abstract': 'Mutations in transmembrane inner ear (TMIE) cause deafness in humans; previous studies suggest involvement in the mechano-electrical transduction (MET) complex in sensory hair cells, but TMIEs precise role is unclear. In tmie zebrafish mutants, we observed that GFP-tagged Tmc1 and Tmc2b, which are putative subunits of the MET channel, fail to target to the hair bundle. In contrast, overexpression of Tmie strongly enhances the targeting of Tmc2b-GFP to stereocilia. To identify the motifs of Tmie underlying the regulation of the Tmcs, we systematically deleted or replaced peptide segments. We then assessed localization and functional rescue of each mutated/chimeric form of Tmie in tmie mutants. We determined that the first putative helix was dispensable and identified a novel critical region of Tmie, the extracellular region and transmembrane domain, which mediates both mechanosensitivity and Tmc2b-GFP expression in bundles. Collectively, our results suggest that Tmies role in sensory hair cells is to target and stabilize Tmc subunits to the site of MET.\\\\n\\\\nAuthor summaryHair cells mediate hearing and balance through the activity of a pore-forming channel in the cell membrane. The transmembrane inner ear (TMIE) protein is an essential component of the protein complex that gates this so-called mechanotransduction channel. While it is known that loss of TMIE results in deafness, the function of TMIE within the complex is unclear. Using zebrafish as a deafness model, Pacentine and Nicolson demonstrate that Tmie is required for the localization of other essential complex members, the transmembrane channel-like (Tmc) proteins, Tmc1/2b. They then evaluate twelve unique versions of Tmie, each containing mutations to different domains of Tmie. This analysis reveals that some mutations in Tmie cause dysfunctional gating of the channel as demonstrated through reduced hair cell activity, and that these same dysfunctional versions also display reduced Tmc expression at the normal site of the channel. These findings link hair cell activity with the levels of Tmc in the bundle, reinforcing the currently-debated notion that the Tmcs are the pore-forming subunits of the mechanotransduction channel. The authors conclude that Tmie, through distinct regions, is involved in both trafficking and stabilizing the Tmcs at the site of mechanotransduction.',\n",
       "   'published': '10.1371/journal.pgen.1007635',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/392837',\n",
       "   'title': 'Cell Diameter in Bacillus subtilis is Determined by the Opposing Actions of Two Distinct Cell Wall Synthetic Systems',\n",
       "   'authors': 'Dion, M.; Kapoor, M.; Sun, Y.; Wilson, S.; Ryan, J.; Vigouroux, A.; van Teeffelen, S.; Oldenbourg, R.; Garner, E. C.',\n",
       "   'author_corresponding': 'Ethan C Garner',\n",
       "   'author_corresponding_institution': 'Harvard',\n",
       "   'date': '2018-08-22',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/392837.source.xml',\n",
       "   'abstract': 'Rod shaped bacteria grow by adding material into their cell wall via the action of two spatially distinct enzymatic systems: The Rod system moves around the cell circumference, while the class A penicillin-binding proteins (aPBPs) are unorganized. To understand how the combined action of these two systems defines bacterial dimensions, we examined how each system affects the growth and width of Bacillus subtilis, as well as the mechanical anisotropy and orientation of material within their sacculi. We find that rod diameter is not determined by MreB, rather it depends on the balance between the systems: The Rod system reduces diameter, while aPBPs increase it. RodA/PBP2A can both thin or widen cells, depending on its levels relative to MreBCD. Increased Rod system activity correlates with an increased density of directional MreB filaments, and a greater fraction of directionally moving PBP2A molecules. This increased circumferential synthesis increases the amount of oriented material within the sacculi, increasing their mechanical anisotropy and reinforcing rod shape. Together, these experiments explain how the combined action of the two main cell wall synthetic systems build rods of different widths, a model that appears generalizable: Escherichia coli containing Rod system mutants show the same relationship between the density of directionally moving MreB filaments and cell width.',\n",
       "   'published': '10.1038/s41564-019-0439-0',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/392928',\n",
       "   'title': 'Intact mast cell content during mild head injury is required for the development of latent pain sensitization - implications for mechanisms underlying posttraumatic headache',\n",
       "   'authors': 'Bree, D.; Levy, D.',\n",
       "   'author_corresponding': 'Dan  Levy',\n",
       "   'author_corresponding_institution': 'BIDMC/Harvard Medical School',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/392928.source.xml',\n",
       "   'abstract': 'Posttraumatic headache (PTH) is one of the most common, debilitating and difficult symptoms to manage after a traumatic head injury. While the mechanisms underlying PTH remain elusive, recent studies in rodent models suggest the potential involvement of calcitonin gene-related peptide (CGRP), a mediator of neurogenic inflammation, and the ensuing activation of meningeal mast cells (MCs), pro-algesic resident immune cells that can lead to the activation of the headache pain pathway. Here, we investigated the relative contribution of MCs to the development of PTH-like pain behaviors in a model of mild closed head injury (mCHI) in male rats. We initially tested the relative contribution of peripheral CGRP signaling to the activation of meningeal MCs following mCHI using an anti-CGRP monoclonal antibody. We then employed a prophylactic MC granule depletion approach to address the hypotheses that intact meningeal MC granule content is necessary for the development of PTH-related pain-like behaviors. The data suggest that following mCHI, ongoing activation of meningeal MCs is not mediated by peripheral CGRP signaling, and does not contribute to development of the mCHI-evoked cephalic mechanical pain hypersensitivity. Our data, however, also reveals that the development of latent sensitization, manifested as persistent hypersensitivity upon the recovery from mCHI-evoked acute cranial hyperalgesia to the headache trigger glyceryl trinitrate requires intact MC content during and immediately after mCHI. Collectively, our data implicate the acute activation of meningeal MCs as mediator of chronic pain hypersensitivity following a concussion or mCHI. Targeting MCs may be explored for early prophylactic treatment of PTH.',\n",
       "   'published': '10.1097/j.pain.0000000000001481',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/391292',\n",
       "   'title': 'The saturation gap: a simple transformation of oxygen saturation using virtual shunt',\n",
       "   'authors': 'Zhou, G.; Karlen, W.; Brant, R.; Wiens, M.; Ansermino, J. M.',\n",
       "   'author_corresponding': 'Guohai  Zhou',\n",
       "   'author_corresponding_institution': 'University of British Columbia',\n",
       "   'date': '2018-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioengineering',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/391292.source.xml',\n",
       "   'abstract': 'ObjectivePeripheral oxygen saturation (SpO2) obtained from pulse oximetry is a widely used physiological measurement. Clinical interpretation is limited by the nonlinear relationship between SpO2, degree of impairment in gas exchange, and effect of altitude. SpO2 is frequently dichotomized to overcome these limitations during prediction modelling. Using the known physiological relationship between virtual shunt and SpO2, we propose the saturation gap as a transformation of SpO2.\\\\n\\\\nApproachWe computed the theoretical virtual shunt corresponding to various SpO2 values and derived an accurate approximation formula between virtual shunt and SpO2. The approximation was based on previously described empiric observations and known physiological relationships. We evaluated the utility of the saturation gap in a clinical study predicting the need for facility admission in children in a rural health-care setting.\\\\n\\\\nMain ResultsThe transformation was saturation gap = 49.314*log10(103.711 - SpO2) -37.315. The ability to predict hospital admission based on a continuous variable SpO2 or saturation gap produced an identical area under the curve of 0.71 (95% CI: 0.69-0.73), compared to only 0.57 (CI: 0.56-0.58) based on diagnosis of hypoxemia (defined as SpO2<90%). However, SpO2 demonstrated a lack of fit compared to saturation gap (goodness-of-fit test p-value <0.0001 versus 0.098). The observed admission rates varied linearly with saturation gap but varied nonlinearly with SpO2.\\\\n\\\\nSignificanceThe saturation gap estimates a continuous linearly interpretable disease severity from SpO2 and improves clinical prediction models. The saturation gap will also allow for straightforward incorporation of altitude in interpretation of measurements of SpO2.',\n",
       "   'published': '10.1038/s41390-019-0525-2',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/393140',\n",
       "   'title': 'FastqCleaner: an interactive Bioconductor application for quality-control, filtering and trimming of FASTQ files',\n",
       "   'authors': 'Roser, L. G.; Aügero, F.; Sanchez, D.',\n",
       "   'author_corresponding': 'Fernén  Agüero',\n",
       "   'author_corresponding_institution': 'Instituto de Investigaciones Biotecnológicas, UNSAM, IIB-INTECH-CONICET',\n",
       "   'date': '2018-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/393140.source.xml',\n",
       "   'abstract': 'BackgroundExploration and processing of FASTQ files are the first steps in state-of-the-art data analysis workflows of Next Generation Sequencing (NGS) platforms. The large amount of data generated by these technologies has put a challenge in terms of rapid analysis and visualization of sequencing information. Recent integration of the R data analysis platform with web visual frameworks has stimulated the development of user-friendly, powerful, and dynamic NGS data analysis applications.\\\\n\\\\nResultsThis paper presents FastqCleaner, a Bioconductor visual application for both quality-control (QC) and pre-processing of FASTQ files. The interface shows diagnostic information for the input and output data and allows to select a series of filtering and trimming operations in an interactive framework. FastqCleaner combines the technology of Bioconductor for NGS data analysis with the data visualization advantages of a web environment.\\\\n\\\\nConclusionsFastqCleaner is an user-friendly, offline-capable tool that enables access to advanced Bioconductor infrastructure. The novel concept of a Bioconductor interactive application that can be used without the need for programming skills, makes FastqCleaner a valuable resource for NGS data analysis.',\n",
       "   'published': '10.1186/s12859-019-2961-8',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/393140',\n",
       "   'title': 'FastqCleaner: an interactive Bioconductor application for quality-control, filtering and trimming of FASTQ files',\n",
       "   'authors': 'Roser, L. G.; Agüero, F.; Sanchez, D.',\n",
       "   'author_corresponding': 'Fernén  Agüero',\n",
       "   'author_corresponding_institution': 'Instituto de Investigaciones Biotecnológicas, UNSAM, IIB-INTECH-CONICET',\n",
       "   'date': '2018-08-28',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/393140.source.xml',\n",
       "   'abstract': 'BackgroundExploration and processing of FASTQ files are the first steps in state-of-the-art data analysis workflows of Next Generation Sequencing (NGS) platforms. The large amount of data generated by these technologies has put a challenge in terms of rapid analysis and visualization of sequencing information. Recent integration of the R data analysis platform with web visual frameworks has stimulated the development of user-friendly, powerful, and dynamic NGS data analysis applications.\\\\n\\\\nResultsThis paper presents FastqCleaner, a Bioconductor visual application for both quality-control (QC) and pre-processing of FASTQ files. The interface shows diagnostic information for the input and output data and allows to select a series of filtering and trimming operations in an interactive framework. FastqCleaner combines the technology of Bioconductor for NGS data analysis with the data visualization advantages of a web environment.\\\\n\\\\nConclusionsFastqCleaner is an user-friendly, offline-capable tool that enables access to advanced Bioconductor infrastructure. The novel concept of a Bioconductor interactive application that can be used without the need for programming skills, makes FastqCleaner a valuable resource for NGS data analysis.',\n",
       "   'published': '10.1186/s12859-019-2961-8',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/393884',\n",
       "   'title': 'Adaptive evolution of feed-forward loops versus diamonds to filter out short spurious signals',\n",
       "   'authors': 'Xiong, K.; Lancaster, A. K.; Siegal, M. L.; Masel, J.',\n",
       "   'author_corresponding': 'Joanna  Masel',\n",
       "   'author_corresponding_institution': 'University of Arizona',\n",
       "   'date': '2018-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'systems biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/393884.source.xml',\n",
       "   'abstract': 'We develop a null model of the evolution of transcriptional regulatory networks, and use it to support an adaptive origin for a canonical \"motif\", a 3-node feed-forward loop (FFL) hypothesized to filter out short spurious signals by integrating information from a fast and a slow pathway. Our mutational model captures the intrinsically high prevalence of weak affinity transcription factor binding sites. We also capture stochasticity and delays in gene expression that distort external signals and intrinsically generate noise. Functional FFLs evolve readily under selection for the hypothesized function, but not in negative controls. Interestingly, a 4-node \"diamond\" motif also emerged as a short spurious signal filter. The diamond uses expression dynamics rather than path length to provide fast and slow pathways. When there is no external spurious signal to filter out, but only internally generated noise, only the diamond and not the FFL evolves.',\n",
       "   'published': '10.1038/s41467-019-10388-6',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395657',\n",
       "   'title': 'Bariatric surgery changes gut microbiota composition independent of operation type',\n",
       "   'authors': 'Paganelli, F. L.; Luyer, M.; Hazelbag, C. M.; Uh, H.-W.; Rogers, M. R. C.; Adriaans, D.; Berbers, R.-M.; Hendrickx, A. P. A.; Viveen, M. C.; Groot, J. A.; Bonten, M. J. M.; Fluit, A. C.; Willems, R. J. L.; Leavis, H.',\n",
       "   'author_corresponding': 'Helen L. Leavis',\n",
       "   'author_corresponding_institution': 'University Medical Center Utrecht',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/395657.source.xml',\n",
       "   'abstract': 'BackgroundBariatric surgery in patients with morbid obesity, either through gastric sleeve gastrectomy or Roux-Y gastric bypass surgery, leads to sustainable weight loss, improvement of metabolic disorders and changes in the intestinal microbiota. Yet, the relationship between changes in gut microbiota, weight loss and the surgical procedure remains incompletely understood.\\\\n\\\\nSubjects/MethodsWe determined temporal changes in microbiota composition in 45 obese patients undergoing a crash diet followed by gastric sleeve gastrectomy (n= 22) or Roux-Y gastric bypass (n= 23). Intestinal microbiota composition was determined before intervention (baseline, S1), 2 weeks after a crash diet (S2), and 1 week (S3), 3 months (S4) and 6 months (S5) after surgery.\\\\n\\\\nResultsRelative to S1, the microbial diversity index declined at S2 and S3 (p< 0.05), and gradually returned to baseline levels at S5. The crash diet was associated with an increased abundance of Rikenellaceae and decreased abundances of Ruminococcaceae and Streptococcaceae (p< 0.05). After surgery, at S3, the relative abundance of Bifidobacteriaceae had decreased (compared to the moment directly after the crash diet), whereas those of Streptococcaceae and Enterobacteriaceae had increased (p< 0.05). Increased weight loss during the next 6 months was not associated without major changes in microbiota composition. Significant differences between both surgical procedures were not observed at any of the time points.\\\\n\\\\nConclusionsIn conclusion, undergoing a crash diet and bariatric surgery were associated with an immediate but temporary decline in the microbial diversity, with immediate and permanent changes in microbiota composition, with no differences between patients undergoing gastric sleeve gastrectomy or Roux-Y gastric bypass surgery.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395657',\n",
       "   'title': 'Bariatric surgery changes gut microbiota composition independent of operation type',\n",
       "   'authors': 'Paganelli, F. L.; Luyer, M.; Hazelbag, C. M.; Uh, H.-W.; Rogers, M. R. C.; Adriaans, D.; Berbers, R.-M.; Hendrickx, A. P. A.; Viveen, M. C.; Groot, J. A.; Bonten, M. J. M.; Fluit, A. C.; Willems, R. J. L.; Leavis, H.',\n",
       "   'author_corresponding': 'Helen L. Leavis',\n",
       "   'author_corresponding_institution': 'University Medical Center Utrecht',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/395657.source.xml',\n",
       "   'abstract': 'BackgroundBariatric surgery in patients with morbid obesity, either through gastric sleeve gastrectomy or Roux-Y gastric bypass surgery, leads to sustainable weight loss, improvement of metabolic disorders and changes in the intestinal microbiota. Yet, the relationship between changes in gut microbiota, weight loss and the surgical procedure remains incompletely understood.\\\\n\\\\nSubjects/MethodsWe determined temporal changes in microbiota composition in 45 obese patients undergoing a crash diet followed by gastric sleeve gastrectomy (n= 22) or Roux-Y gastric bypass (n= 23). Intestinal microbiota composition was determined before intervention (baseline, S1), 2 weeks after a crash diet (S2), and 1 week (S3), 3 months (S4) and 6 months (S5) after surgery.\\\\n\\\\nResultsRelative to S1, the microbial diversity index declined at S2 and S3 (p< 0.05), and gradually returned to baseline levels at S5. The crash diet was associated with an increased abundance of Rikenellaceae and decreased abundances of Ruminococcaceae and Streptococcaceae (p< 0.05). After surgery, at S3, the relative abundance of Bifidobacteriaceae had decreased (compared to the moment directly after the crash diet), whereas those of Streptococcaceae and Enterobacteriaceae had increased (p< 0.05). Increased weight loss during the next 6 months was not associated without major changes in microbiota composition. Significant differences between both surgical procedures were not observed at any of the time points.\\\\n\\\\nConclusionsIn conclusion, undergoing a crash diet and bariatric surgery were associated with an immediate but temporary decline in the microbial diversity, with immediate and permanent changes in microbiota composition, with no differences between patients undergoing gastric sleeve gastrectomy or Roux-Y gastric bypass surgery.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395392',\n",
       "   'title': 'A note on stochastic modeling of biological systems: automatic generation of an optimized Gillespie algorithm',\n",
       "   'authors': 'Vanhaelen, Q.',\n",
       "   'author_corresponding': 'Quentin  Vanhaelen',\n",
       "   'author_corresponding_institution': 'Insilico Medicine Inc.',\n",
       "   'date': '2018-08-27',\n",
       "   'version': '2',\n",
       "   'type': 'confirmatory results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'systems biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/395392.source.xml',\n",
       "   'abstract': 'Signaling pathways and gene regulatory networks (GRNs) play a central role in the signal trans-duction and regulation of biochemical processes occurring within the cellular environment. Under-standing their mechanisms and dynamics is of major interest in various areas of life sciences and biological sciences. For example controlling stem cell fate decision requires a comprehension of the dynamical behavior of the networks involved in stem cell differentiation and pluripotency mainte-nance. In addition to analytical mathematical methods which are applicable for small or medium sized systems, there are many computational approaches to model and analyze the behavior of larger systems. However, from a dynamical point of view, modeling a combination of signaling pathways and GRNs present several challenges. Indeed, in addition to being of large dimensionality, these systems have specific dynamical features. Among the most commonly encountered is that the signal transduction controlled by the signaling pathways occurs at a different time scale than the transcription and translation processes. Also, stochasticity is known to strongly impact the regulation of gene expression. In this paper, we describe a simple implementation of an optimized version of the Gille-spie algorithm for simulating relatively large biological networks which include delayed reactions. The implementation presented herein comes with a script for automatically generating the different data structures and source files of the algorithm using standardized input files.\\\\n\\\\nCode availabilityThe Fortran90 implementation of the code and the R script described here as well as the tutorial with practical instructions are stored on the following github repository qvhaelen/ typhon',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395285',\n",
       "   'title': 'Occurrence of non-western magic in the European brain',\n",
       "   'authors': 'Koten, J. W.; Schueppen, A.; Kumar, V.; Wood, G.',\n",
       "   'author_corresponding': 'Jan Willem  Koten',\n",
       "   'author_corresponding_institution': 'University of Graz',\n",
       "   'date': '2018-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/395285.source.xml',\n",
       "   'abstract': 'Timecourses that exhibit identical behaviour at distinct measurement occasions are reliable. Voodoo connectivity occurs when connectivity among brain regions exceeds within subject timecourse reliability. Thus, timecourse reliability limits the true detectable connectivity. We reproduced a working memory related connectome consisting of 561 paths obtained from 67 individuals. We tested >100000 fc-MRI pipelines and show that Savitzky Golay (SG) filters maximize true connectivity while conserving cognitively relevant changes of signals. This is noteworthy for approaches that focus on rapidly changing aspects of connectomes. Furthermore, SG filters detect zombie activity. These \\\\\"resting state oscillations\\\\\" are not under human control and contaminate working state signals. SPM pipelines exhibit more voodoo connectivity than SG pipelines. With the SPM pipeline, we observed a connectivity of r=0.44 and a poor true connectivity of r=0.23, but with the SG pipeline we observed a connectivity of r=0.59 and a fair true connectivity of r=0.43. The number of paths detected with fair true connectivity (r >0.4) was 4 for the SPM pipeline but 352 for the SG based pipeline. However, superior statistical properties of SG pipelines may not reflect neural reality. Hence, causal external validation of fc-MRI pipelines is crucial. Without such studies, different pipelines produce at best \\\\\"alternative maps\\\\\".',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395285',\n",
       "   'title': 'Occurrence of non-western magic in the European brain',\n",
       "   'authors': 'Koten, J. W.; Schueppen, A.; Kumar, V.; Wood, G.',\n",
       "   'author_corresponding': 'Jan Willem  Koten',\n",
       "   'author_corresponding_institution': 'University of Graz',\n",
       "   'date': '2018-08-24',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/395285.source.xml',\n",
       "   'abstract': 'Timecourses that exhibit identical behaviour at distinct measurement occasions are reliable. Voodoo connectivity occurs when connectivity among brain regions exceeds within subject timecourse reliability. Thus, timecourse reliability limits the true detectable connectivity. We reproduced a working memory related connectome consisting of 561 paths obtained from 67 individuals. We tested >100000 fc-MRI pipelines and show that Savitzky Golay (SG) filters maximize true connectivity while conserving cognitively relevant changes of signals. This is noteworthy for approaches that focus on rapidly changing aspects of connectomes. Furthermore, SG filters detect zombie activity. These \\\\\"resting state oscillations\\\\\" are not under human control and contaminate working state signals. SPM pipelines exhibit more voodoo connectivity than SG pipelines. With the SPM pipeline, we observed a connectivity of r=0.44 and a poor true connectivity of r=0.23, but with the SG pipeline we observed a connectivity of r=0.59 and a fair true connectivity of r=0.43. The number of paths detected with fair true connectivity (r >0.4) was 4 for the SPM pipeline but 352 for the SG based pipeline. However, superior statistical properties of SG pipelines may not reflect neural reality. Hence, causal external validation of fc-MRI pipelines is crucial. Without such studies, different pipelines produce at best \\\\\"alternative maps\\\\\".',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395863',\n",
       "   'title': 'The RNA helicase UPF1 associates with mRNAs co-transcriptionally and is required for the release of mRNAs from transcription sites',\n",
       "   'authors': 'Singh, A. K.; Choudhury, S. R.; De, S.; Zhang, J.; Kissane, S.; Dwivedi, V.; Ramanathan, P.; Orsini, L.; Hebenstreit, D.; Brogna, S.',\n",
       "   'author_corresponding': 'Saverio  Brogna',\n",
       "   'author_corresponding_institution': 'University of Birmingham',\n",
       "   'date': '2018-08-25',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/395863.source.xml',\n",
       "   'abstract': 'UPF1 is an RNA helicase that is required for efficient nonsense-mediated mRNA decay (NMD) in eukaryotes, and the predominant view is that UPF1 mainly operates on the 3UTRs of mRNAs that are directed for NMD in the cytoplasm. Here we offer evidence, obtained from Drosophila, that UPF1 constantly moves between the nucleus and cytoplasm and that it has multiple functions in the nucleus. It is associated, genome-wide, with nascent RNAs at most of the active Pol II transcription sites and at some Pol III-transcribed genes, as demonstrated microscopically on the polytene chromosomes of salivary gland and by ChIP-seq analysis in S2 cells. Intron recognition seems to interfere with association and translocation of UPF1 on nascent pre-mRNA transcripts, and cells depleted of UPF1 show defects in several nuclear processes essential to correct gene expression - most strikingly, the release of mRNAs from transcription sites and mRNA export from the nucleus.',\n",
       "   'published': '10.7554/eLife.41444',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395863',\n",
       "   'title': 'The RNA helicase UPF1 associates with mRNAs co-transcriptionally and is required for the release of mRNAs from transcription sites',\n",
       "   'authors': 'Singh, A. K.; Choudhury, S. R.; De, S.; Zhang, J.; Kissane, S.; Dwivedi, V.; Ramanathan, P.; Orsini, L.; Hebenstreit, D.; Brogna, S.',\n",
       "   'author_corresponding': 'Saverio  Brogna',\n",
       "   'author_corresponding_institution': 'University of Birmingham',\n",
       "   'date': '2018-08-26',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/395863.source.xml',\n",
       "   'abstract': 'UPF1 is an RNA helicase that is required for efficient nonsense-mediated mRNA decay (NMD) in eukaryotes, and the predominant view is that UPF1 mainly operates on the 3UTRs of mRNAs that are directed for NMD in the cytoplasm. Here we offer evidence, obtained from Drosophila, that UPF1 constantly moves between the nucleus and cytoplasm and that it has multiple functions in the nucleus. It is associated, genome-wide, with nascent RNAs at most of the active Pol II transcription sites and at some Pol III-transcribed genes, as demonstrated microscopically on the polytene chromosomes of salivary gland and by ChIP-seq analysis in S2 cells. Intron recognition seems to interfere with association and translocation of UPF1 on nascent pre-mRNA transcripts, and cells depleted of UPF1 show defects in several nuclear processes essential to correct gene expression - most strikingly, the release of mRNAs from transcription sites and mRNA export from the nucleus.',\n",
       "   'published': '10.7554/eLife.41444',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396465',\n",
       "   'title': 'PARylation regulates stress granule dynamics, phase separation, and neurotoxicity of disease-related RNA-binding proteins',\n",
       "   'authors': 'Duan, Y.; Du, A.; Gu, J.; Duan, G.; Wang, C.; Ma, Z.; Sun, L.; Qian, B.; Deng, X.; Zhang, K.; Tian, K.; Zhang, Y.; Jiang, H.; Liu, C.; Fang, Y.',\n",
       "   'author_corresponding': 'Yanshan  Fang',\n",
       "   'author_corresponding_institution': 'IRCBC, SIOC, Chinese Academy of Sciences',\n",
       "   'date': '2018-08-26',\n",
       "   'version': '3',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/396465.source.xml',\n",
       "   'abstract': 'Mutations in RNA-binding proteins localized in ribonucleoprotein (RNP) granules, such as hnRNP A1 and TDP-43, promote aberrant protein aggregations, which are pathological hallmarks in neurodegenerative diseases including amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD). Protein posttranslational modifications (PTMs) are known to regulate RNP granules. In this study, we investigate the function of PARylation, an important PTM involved in DNA damage repair and cell death, in RNP-related neurodegeneration. We reveal that PARylation levels are a major regulator of the dynamic assembly-disassembly of RNP granules, and the disease-related RNPs such as hnRNP A1 and TDP-43 can both be PARylated and bind to PARylated proteins. We further identify the PARylation site of hnRNP A1 at K298, which controls the cytoplasmic translocation of hnRNP A1 in response to stress, as well as the PAR-binding motif (PBM) of hnRNP A1, which is required for the delivery and association of hnRNP A1 to stress granules. Moreover, we show that PAR not only dramatically enhances the liquid-liquid phase separation of hnRNP A1, but also promotes the co-phase separation of hnRNP A1 and TDP-43 in vitro and their interaction in vivo. Finally, we establish that both genetic and pharmacological inhibition of PARP mitigates hnRNP A1 and TDP-43-mediated neurotoxicity in cell and Drosophila models of ALS. Together, our findings indicate a novel and crucial role of PARylation in regulating the assembly and the dynamics of RNP granules, and dysregulation of PARylation may contribute to ALS disease pathogenesis.',\n",
       "   'published': '10.1038/s41422-019-0141-z',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396705',\n",
       "   'title': 'Orientation processing by synaptic integration across first-order tactile neurons',\n",
       "   'authors': 'Hay, E.; Pruszynski, J. A.',\n",
       "   'author_corresponding': 'Etay  Hay',\n",
       "   'author_corresponding_institution': 'Western University',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/396705.source.xml',\n",
       "   'abstract': 'Our ability to manipulate objects relies on tactile inputs from first-order tactile neurons that innervate the glabrous skin of the hand. The distal axon of these neurons branches in the skin and innervates many mechanoreceptors, yielding spatially-complex receptive fields. Here we show that synaptic integration across the complex signals from the first-order neuronal population could underlie human ability to accurately (< 3{degrees}) and rapidly process the orientation of edges moving across the fingertip. We first derive spiking models of human first-order tactile neurons that fit and predict responses to moving edges with high accuracy. We then use the model neurons in simulating the peripheral neuronal population that innervates a fingertip. We use machine learning to train classifiers performing synaptic integration across the neuronal population activity, and show that synaptic integration across first-order neurons can process edge orientations with high acuity and speed. In particular, our models suggest that integration of fast-decaying (AMPA-like) synaptic inputs within short timescales is critical for discriminating fine orientations, whereas integration of slow-decaying (NMDA-like) synaptic inputs refine discrimination and maintain robustness over longer timescales. Taken together, our results provide new insight into the computations occurring in the earliest stages of the human tactile processing pathway and how they may be critical for supporting hand function.\\\\n\\\\nAuthor SummaryOur ability to manipulate objects relies on tactile inputs signaled by first-order neurons that innervate mechanoreceptors in the skin of the hand and have spatially-complex receptive fields. Here we show how synaptic integration across the rich inputs from first-order neurons can rapidly and accurately process the orientation of edges moving across the fingertip. We derive spiking models of human first-order tactile neurons, then use the models to simulate the peripheral neuronal population that innervates a fingertip. We use machine learning to train classifiers performing synaptic integration across the neuronal population activity, and show that synaptic integration across first-order neurons could underlie human ability to process edge orientations with high acuity and speed.',\n",
       "   'published': '10.1371/journal.pcbi.1008303',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397067',\n",
       "   'title': 'Cleaner wrasse pass the mark test. What are the implications for consciousness and self-awareness testing in animals?',\n",
       "   'authors': 'Kohda, M.; Takashi, H.; Takeyama, T.; Awata, S.; Tanaka, H.; Asai, J.-y.; Jordan, A.',\n",
       "   'author_corresponding': 'Alex  Jordan',\n",
       "   'author_corresponding_institution': 'Max-Planck-Institut fur Ornithologie',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'animal behavior and cognition',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397067.source.xml',\n",
       "   'abstract': 'The ability to perceive and recognise a reflected mirror image as self (mirror self-recognition, MSR) is considered a hallmark of cognition across species. Although MSR has been reported in mammals and birds, it is not known to occur in any other major taxon. A factor potentially limiting the ability to test for MSR is that the established assay for MSR, the mark test, shows an interpretation bias towards animals with the dexterity (or limbs) required to touch a mark. Here, we show that the cleaner wrasse fish, Labroides dimidiatus, passes through all phases of the mark test: (i) social reactions towards the reflection, (ii) repeated idiosyncratic behaviours towards the mirror (contingency testing), and (iii) frequent observation of their reflection. When subsequently provided with a coloured tag, individuals attempt to remove the mark in the presence of a mirror but show no response towards transparent marks, or to coloured marks in the absence of a mirror. This remarkable finding presents a challenge to our interpretation of the mark test - do we accept that these behavioural responses in the mark test, which are taken as evidence of self-recognition in other species, mean that fish are self-aware? Or do we conclude that these behavioural patterns have a basis in a cognitive process other than self-recognition? If the former, what does this mean for our understanding of animal intelligence? If the latter, what does this mean for our application and interpretation of the mark test as a metric for animal cognitive abilities?',\n",
       "   'published': '10.1371/journal.pbio.3000021',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397059',\n",
       "   'title': 'In silico REPOSITIONING OF APPROVED DRUGS AGAINST Schistosoma mansoni ENERGY METABOLISM TARGETS',\n",
       "   'authors': 'SILVA, L. D. A.; Bezerra, C. B.; dos Santos, D. B.; Calixto, N. M.',\n",
       "   'author_corresponding': 'LOURIVAL DE  ALMEIDA  SILVA',\n",
       "   'author_corresponding_institution': 'Instituto Federal Goiano',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'confirmatory results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397059.source.xml',\n",
       "   'abstract': 'Schistosomiasis is a neglected parasitosis caused by Schistosoma spp. Praziquantel is used for the chemoprophylaxis and treatment of this disease. Although this monotherapy is effective, the risk of resistance and its low efficiency against immature worms compromises its effectiveness. Therefore, it is necessary to develop new schistosomicide drugs. However, the development of new drugs is a long and expensive process. The repositioning of approved drugs has been proposed as a quick, cheap, and effective alternative to solve this problem. This study employs chemogenomic analysis with use of bioinformatics tools to search, identify, and analyze data on approved drugs with the potential to inhibit Schistosoma mansoni energy metabolism enzymes. The TDR Targets Database, Gene DB, Protein, DrugBank, Therapeutic Targets Database (TTD), Promiscuous, and PubMed databases were used. Fifty-nine target proteins were identified, of which 18 had one or more approved drugs. The results identified 20 potential drugs for schistosomiasis treatment, all approved for use in humans.',\n",
       "   'published': '10.1371/journal.pone.0203340',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397075',\n",
       "   'title': 'A neuromechanistic model for rhythmic beat generation',\n",
       "   'authors': 'Bose, A.; Byrne, A.; Rinzel, J.',\n",
       "   'author_corresponding': 'Aine  Byrne',\n",
       "   'author_corresponding_institution': 'New York University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397075.source.xml',\n",
       "   'abstract': 'When listening to music, humans can easily identify and move to the beat. Numerous experimental studies have identified brain regions that may be involved with beat perception and representation. Several theoretical and algorithmic approaches have been proposed to account for this ability. Related to, but different from the issue of how we perceive a beat, is the question of how we learn to generate and hold a beat. In this paper, we introduce a neuronal framework for a beat generator that is capable of learning isochronous rhythms over a range of frequencies that are relevant to music and speech. Our approach combines ideas from error-correction and entrainment models to investigate the dynamics of how a biophysically-based neuronal network model synchronizes its period and phase to match that of an external stimulus. The model makes novel use of on-going faster gamma rhythms to form a set of discrete clocks that provide estimates, but not exact information, of how well the beat generator spike times match those of a stimulus sequence. The beat generator is endowed with plasticity allowing it to quickly learn and thereby adjust its spike times to achieve synchronization. Our model makes generalizable predictions about the existence of asymmetries in the synchronization process, as well as specific predictions about resynchronization times after changes in stimulus tempo or phase. Analysis of the model demonstrates that accurate rhythmic time keeping can be achieved over a range of frequencies relevant to music, in a manner that is robust to changes in parameters and to the presence of noise.\\n\\nAuthor summaryMusic is integral to human experience and is appreciated across a wide range of cultures. Although many features distinguish different musical traditions, rhythm is central to nearly all. Most humans can detect and move along to the beat through finger or foot tapping, hand clapping or other bodily movements. But many people have a hard time \"keeping a beat\", or say they have \"no sense of rhythm\". There appears to be a disconnect between our ability to perceive a beat versus our ability to produce a beat, as a drummer would do as part of a musical group. Producing a beat requires beat generation, the process by which we learn how to keep track of the specific time intervals between beats, as well as executing the motor movement needed to produce the sound associated with a beat. In this paper, we begin to explore neural mechanisms that may be responsible for our ability to generate and keep a beat. We develop a computational model that includes different neurons and shows how they cooperate to learn a beat and keep it, even after the stimulus is removed, across a range of frequencies relevant to music. Our dynamical systems model leads to predictions for how the brain may react when learning a beat. Our findings and techniques should be widely applicable to those interested in understanding how the brain processes time, particularly in the context of music.',\n",
       "   'published': '10.1371/journal.pcbi.1006450',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396713',\n",
       "   'title': 'Single cell dynamics of embryonic muscle progenitor cells in zebrafish',\n",
       "   'authors': 'Sharma, P.; Ruel, T. D.; Kocha, K. M.; Liao, S.; Huang, P.',\n",
       "   'author_corresponding': 'Peng  Huang',\n",
       "   'author_corresponding_institution': 'University of Calgary',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396713.source.xml',\n",
       "   'abstract': 'Muscle stem cells hold a great therapeutic potential in regenerating damaged muscles. However, the in vivo behavior of muscle stem cells during muscle growth and regeneration is still poorly understood. Using zebrafish as a model, we describe the in vivo dynamics and function of dermomyotome cells, a population of embryonic muscle progenitor cells. Dermomyotome cells are located in a superficial layer external to muscle fibers and express many extracellular matrix (ECM) genes including col1a2. Utilizing a new col1a2 transgenic line, we show that dermomyotome cells display a ramified morphology with dynamic cellular processes. Cell lineage tracing demonstrates that col1a2+ dermomyotome cells contribute to normal muscle growth as well as muscle injury repair. Combination of live imaging and single cell clonal analysis reveals a highly-choreographed process of muscle regeneration. Activated dermomyotome cells change from the quiescent ramified morphology to a polarized and elongated morphology and generate daughter cells that fuse with existing muscle fibers. Ablation of the dermomyotome severely compromises muscle injury repair. Our work provides a dynamic view of embryonic muscle progenitor cells during zebrafish muscle regeneration.\\\\n\\\\nSummary statementLive imaging and single cell clonal analysis reveal dynamic behaviors of zebrafish embryonic muscle progenitor cells in quiescence and activation.',\n",
       "   'published': '10.1242/dev.178400',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396861',\n",
       "   'title': 'Dynamic regulation of HYL1 provides new insights into its multifaceted role in Arabidopsis',\n",
       "   'authors': 'Bhagat, P. K.; Verma, D.; Badmi, R.; Sinha, A. K.',\n",
       "   'author_corresponding': 'Alok Krishna Sinha',\n",
       "   'author_corresponding_institution': 'National Institute of Plant Genome Research',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'plant biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396861.source.xml',\n",
       "   'abstract': 'MicroRNAs (miRNAs) are 21 to 24 nucleotide non-coding RNAs that regulate gene expression. Biogenesis of miRNAs is fine-tuned by specialized microprocessor complex, the regulation of which is being continuously understood. Recruitment of HYL1 to the microprocessor complex is crucial for accurate primary-miRNA (pri-miRNA) processing and accumulation of mature miRNA in Arabidopsis thaliana. HYL1 is a double-stranded RNA binding protein also termed as DRB1, has two double-stranded RNA binding domain at N-terminal and a highly disordered C-terminal region. Also, the biological activity of HYL1 is dynamically regulated through transition from hyperphosphorylation to hypophosphorylation state. HYL1 is known to be phosphorylated by a MAP kinase MPK3 and SnRK2. However, the precise role of its phosphorylation are still unknown. Recently, the stability of HYL1 protein has been shown to be regulated by an unknown protease X. However, the identity of the protease and its molecular mechanisms are poorly understood. Here, we describe, three functionally important facets of HYL1, which provide a better picture of its association with molecular processes. First, we identified a conserved MPK3 phosphorylation site on HYL1 and its possible role in the miRNA biogenesis. Secondly, the C-terminal region of HYL1 displays tendencies to bind dsDNA. Lastly, the role of C-terminal region of HYL1 in the regulation of its protein stability and the regulation of miRNA biogenesis is documented. We show the unexplored role of C-terminal and hypothesize the novel functions of HYL1 in addition to miRNA biogenesis. We anticipate that the data presented in this study, will open a new dimension of understanding the role of double stranded RNA binding proteins in diverse biological processes of plants and animal.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396473',\n",
       "   'title': 'Function of the Pedicellate Spikelet and Awn in Sorghum and Related Grasses',\n",
       "   'authors': 'AuBuchon-Elder, T.; Coneva, V.; Goad, D. M.; Allen, D. K.; Kellogg., E. A.',\n",
       "   'author_corresponding': 'Elizabeth Anne Kellogg.',\n",
       "   'author_corresponding_institution': 'Donald Danforth Plant Science Center',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'plant biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396473.source.xml',\n",
       "   'abstract': 'Sorghum and its relatives in the grass tribe Andropogoneae bear their flowers in pairs of spikelets, in which one spikelet (seed-bearing, or SS) of the pair produces a seed and the other is sterile or male (staminate). This division of function does not occur in other major cereals such as wheat or rice. Additionally, one bract of the seed-bearing spikelet often produces a long extension, the awn, which is in the same position as but independently derived from that of wheat and rice. The function of the sterile spikelet is unknown and that of the awn has not been tested in Andropogoneae. We used radioactive and stable isotopes of carbon, as well as RNA-seq of metabolically important enzymes to show that the sterile spikelet assimilates carbon, which is translocated to the largely heterotrophic SS, thereby functioning as a nurse tissue. The awn shows no evidence of photosynthesis. These results apply to distantly related species of Andropogoneae. Thus, the sterile spikelet, but not the awn, could affect yield in the cultivated species and fitness in the wild ones.',\n",
       "   'published': '10.1105/tpc.20.00424',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396564',\n",
       "   'title': 'Enhanced resistance to bacterial and oomycete pathogens by short tandem target mimic RNAs in tomato',\n",
       "   'authors': 'Canto-Pastor, A.; Santos, B.; Valli, A.; Summers, W.; Schornack, S.; Baulcombe, D.',\n",
       "   'author_corresponding': 'David  Baulcombe',\n",
       "   'author_corresponding_institution': 'Department of Plant Sciences, University of Cambridge',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'plant biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396564.source.xml',\n",
       "   'abstract': 'Nucleotide binding site leucine-rich repeat (NLR) proteins of the plant innate immune system are negatively regulated by the miR482/2118 family microRNAs (miRNAs) that are in a distinct 22nt class of miRNAs with a double mode of action. First they cleave the target RNA, as with the canonical 21nt miR-NAs, and second they trigger secondary siRNA production using the target RNA as a template. Here we address the extent to which the miR482/2118 family affects expression of NLR mR-NAs and disease resistance. First we show that structural differences of miR482/2118 family members in tomato (Solanum lycopersicum) are functionally significant. The predicted target of the miR482 subfamily is conserved motif in multiple NLR mRNAs whereas, for miR2118b, it is a novel non-coding RNA target formed by rearrangement of several different NLR genes. From RNA sequencing and degradome data in lines expressing short tandem target mimic (STTM) RNAs of miR482/2118 we confirm the different targets of these miRNAs. The effect on NLR mRNA accumulation is slight but, nevertheless, the tomato STTM lines display enhanced resistance to infection with the oomycete and bacterial pathogens. These data implicate an RNA cascade of miRNAs and secondary siRNAs in the regulation of NLR RNAs and show that the encoded NLR proteins have a role in quantitative disease resistance in addition to dominant gene resistance that has been well characterized elsewhere. We also illustrate the use of STTM RNA in a biotechnological approach for enhancing quantitative disease resistance in highly bred cultivars.',\n",
       "   'published': '10.1073/pnas.1814380116',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396358',\n",
       "   'title': 'Probabilistic modeling of personalized drug combinations from integrated chemical screen and molecular data in sarcoma',\n",
       "   'authors': 'Berlow, N. E.; Rikhi, R.; Geltzeiler, M. N.; Abraham, J.; Svalina, M. N.; Davis, L. E.; Wise, E.; Mancini, M.; Noujaim, J.; Mansoor, A.; Quist, M. J.; Matlock, K. L.; Goros, M. W.; Hernandez, B. S.; Duong, Y. C.; Thway, K.; Tsukahara, T.; Nishio, J.; Huang, E. C.; Airhart, S.; Bult, C. J.; Gandour-Edwards, R.; Maki, R. G.; Jones, R. L.; Michalek, J. E.; Milovancev, M.; Ghosh, S.; Pal, R.; Keller, C.',\n",
       "   'author_corresponding': 'Charles  Keller',\n",
       "   'author_corresponding_institution': \"Children's Cancer Therapy Development Institute\",\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396358.source.xml',\n",
       "   'abstract': 'Cancer patients with advanced disease exhaust available clinical regimens and lack actionable genomic medicine results, leaving a large patient population without effective treatments options when their disease inevitably progresses. To address the unmet clinical need for evidence-based therapy assignment when standard clinical approaches have failed, we have developed a probabilistic computational modeling approach which integrates sequencing data with functional assay data to develop patient-specific combination cancer treatments. This computational modeling approach addresses three major challenges in personalized cancer therapy, which we validate across multiple species via computationally-designed personalized synergistic drug combination predictions, identification of unifying therapeutic targets to overcome intra-tumor heterogeneity, and mitigation of cancer cell resistance and rewiring mechanisms. These proof-of-concept studies support the use of an integrative functional approach to personalized combination therapy prediction for the population of high-risk cancer patients lacking viable clinical options and without actionable DNA sequencing-based therapy.',\n",
       "   'published': '10.1186/s12885-019-5681-6',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396333',\n",
       "   'title': 'Selecting causal risk factors from high-throughput experiments using multivariable Mendelian randomization',\n",
       "   'authors': 'Zuber, V.; Colijn, J. M.; Klaver, C.; Burgess, S.',\n",
       "   'author_corresponding': 'Stephen  Burgess',\n",
       "   'author_corresponding_institution': 'MRC Biostatistics Unit, University of Cambridge,',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'genetics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396333.source.xml',\n",
       "   'abstract': 'Modern high-throughput experiments provide a rich resource to investigate causal determinants of disease risk. Mendelian randomization (MR) is the use of genetic variants as instrumental variables to infer the causal effect of a specific risk factor on an outcome. Multivariable MR is an extension of the standard MR framework to consider multiple potential risk factors in a single model. However, current implementations of multivariable MR use standard linear regression and hence perform poorly with many risk factors.\\\\n\\\\nHere, we propose a novel approach to two-sample multivariable MR based on Bayesian model averaging (MR-BMA) that scales to high-throughput experiments. In a realistic simulation study, we show that MR-BMA can detect true causal risk factors even when the candidate risk factors are highly correlated. We illustrate MR-BMA by analysing publicly-available summarized data on metabolites to prioritise likely causal biomarkers for age-related macular degeneration.',\n",
       "   'published': '10.1038/s41467-019-13870-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/394031',\n",
       "   'title': 'Persistent hotspots of Schistosoma mansoni transmission in the Kenyan waters of Lake Victoria: a search for snail-related answers beneath the waves',\n",
       "   'authors': 'Mutuku, M.; Laidemitt, M.; Mwangi, I.; Otiato, F.; Agola, E.; Steinauer, M.; Ochanda, H.; Kamel, B.; Loker, E.; Mkoji, G.',\n",
       "   'author_corresponding': 'Martin  Mutuku',\n",
       "   'author_corresponding_institution': 'Centre for Biotech Research and Development, Kenya Medical Research Institute, Nairobi, Kenya',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'epidemiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/394031.source.xml',\n",
       "   'abstract': 'Following a four-year annual praziquantel treatment campaign the resulting prevalence of S. mansoni was seen to differ among individual villages along the Kenyan shore of Lake Victoria. We have investigated possible inherent differences in snail-related aspects of transmission among such 10 villages, including six persistent hotspot (PHS) villages ([&le;]30% reduction in prevalence following repeated treatments) located along the west-facing shore of the lake, and four PZQ-responding (RESP) villages (>30% prevalence reduction following repeated treatment) along Winam Gulf. When taking into account all sampling sites and times and water hyacinth presence/absence, shoreline-associated B. sudanica from PHS and RESP villages did not differ in relative abundance or prevalence of S. mansoni infection. Water hyacinth intrusions were associated with increased B. sudanica abundance. The deeper water snail Biomphalaria choanomphala was significantly more abundant in the PHS villages and prevalence of S. mansoni among villages both before and after control was positively correlated with B. choanomphala abundance. Worm recoveries from sentinel mice did not differ between PHS and RESP villages, and abundance of non-schistosome trematode species was not associated with S. mansoni abundance. Biomphalaria choanomphala provides an alternative, deepwater mode of transmission that may favor greater persistence of S. mansoni in PHS villages. As we found evidence for ongoing S. mansoni transmission in all 10 villages, we conclude conditions conducive for transmission and reinfection occur ubiquitously. This argues for an integrated, basin-wide plan for schistosomiasis control to counteract rapid reinfections facilitated by large snail populations and movements of infected people around the lake.',\n",
       "   'published': '10.4269/ajtmh.19-0089',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395293',\n",
       "   'title': 'Glenohumeral Injection Using Anatomic Landmark Versus Sonographic Needle Guidance',\n",
       "   'authors': 'Moore, T. S.; Paffett, C. F.; Sibbitt, W. L.; Hayward, W. A.; Gibb, J. I.; Kettwich, S. D.; Fields, R. A.; Emil, N. S.; Fangtham, M.; Bankhurst, A. D.',\n",
       "   'author_corresponding': 'Wilmer L Sibbitt',\n",
       "   'author_corresponding_institution': 'University of New Mexico Health Sciences Center',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'clinical trials',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/395293.source.xml',\n",
       "   'abstract': 'ObjectiveWe hypothesized ultrasound (US) guidance improves outcomes of corticosteroid injection of the painful shoulder.\\\\n\\\\nMethods30 patients with symptomatic shoulders due to osteoarthritis were randomized to glenohumeral injection with 3 milliliters of 1% lidocaine and 60 mg of triamcinolone acetonide using the anterior approach with 1) conventional anatomic landmark palpation-guidance or 2) US-guidance. Injection pain (visual analogue pain scale (VAS)), pain at outcome (2 weeks and 6 months), therapeutic duration, time-to-next-injection, and costs were determined.\\\\n\\\\nResultsInjection pain was less with US (VAS: 0.3{+/-}0.6 cm) vs. landmark-guidance (VAS: 1.4{+/-}2.4 cm, 95% CI of difference: 0.5<1.1<1.7, p=0.05). Pain scores were similar at 2 weeks: US: 2.2{+/-}2.4 cm; Landmark: 1.8{+/-}2.7 cm, 95% CI of difference: -2.2<-0.4<1.4, p=0.66 and 6 months: US: 5.8{+/-}2.8 cm; Landmark: 6.4{+/-}2.9 cm, 95% CI of difference: -0.4<0.6< 1.1, p =0.71. Therapeutic duration (US: 3.9{+/-}1.5 months; Landmark: 3.0{+/-}1.2 months, 95% CI of difference: - 1.4 <-0.9<-0.4, p=0.045) and time-to-next-injection (US: 8.1{+/-}3.5 months; Landmark: 5.7{+/-}2.9 months, 95% CI of difference: -3.6<-2.4<-1.3, p=0.025) were longer, and fewer injections per year (29% less) were required: US: 1.5{+/-}0.2 injections/year; Landmark: 2.1{+/-}0.2 injections/year (p<0.037; 95% CI of difference -0.9<-0.6<-0.3). However, cost/patient/year was modestly greater with US (US: $318{+/-}89, Landmark: $301{+/-}67; p=0.28).\\\\n\\\\nConclusionAnatomic landmark guidance in the short-term is equally effective as US for injection of the osteoarthritic shoulder and modestly less costly, however, US may reduce the need for repetitive injections by prolonging the therapeutic effect and thus time to next injection.\\\\n\\\\nIRB StatementThis project was in compliance with the Helsinki Declaration, was approved by the Institutional Review Board (IRB) as ultrasound subset of a syringe safety trial (Human Research Review Committee approval 04-347), and was registered at ClinicalTrials.gov (Clinical Trial Identifier NCT00651625). The subjects gave informed consent to participate prior to all studies and interventions. Patient confidentiality was protected according to the U.S. Health Insurance Portability and Accountability Act (HIPAA) and all data was de-identified.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395343',\n",
       "   'title': 'Prescription Opioid Analgesic Use and Mortality in Systemic Lupus Erythematosus',\n",
       "   'authors': 'Cabacungan, R. J.; Qualls, C. R.; Sibbitt, W. L.; Hayward, W. A.; Gibb, J. I.; Kettwich, S. D.; Fields, R. A.; Emil, N. S.; Fangtham, M.; Bankhurst, A. D.',\n",
       "   'author_corresponding': 'Wilmer L Sibbitt',\n",
       "   'author_corresponding_institution': 'University of New Mexico Health Sciences Center, Albuquerque, NM, USA',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'epidemiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/395343.source.xml',\n",
       "   'abstract': 'ObjectivesThis research investigated the prevalence of opioid analgesic use in patients with systemic lupus erythematosus (SLE).\\\\n\\\\nMethodsThis 5-year prospective cohort study of 275 SLE patients focused on prescription opioid use and 5-year outcome. Associations were determined with univariable regression analysis and then multivariable models were created to determine independent effects on dependent variables\\\\n\\\\nResultsPrescription opioid use was common in SLE with 24% using opioid analgesics chronically and 76% not using opioids. Opioid users had a higher rate of tobacco use (p<0.01), cocaine use (p<0.002), mean pain scores (p<0.001), disease activity (SLEDAI-2K) (p<0.001), disease damage (SLICC/ACRDI) (p<0.001), non-adherence to medical therapy (p<0.01), and total deaths at 5 years (opioids: 48.0%, no opioids 19.0%, p<0.001). Logistic regression analysis predicting death revealed opioid use (hazard ratio 2.6, p<0.001) and SLEDAI-2K (1.1, p<0.001) respectively; and opioid use (hazard ratio 2.5, p<0.002), SLEDAI-2K (hazard ratio 1.1, p<0.001), and non-adherence (hazard ratio 1.6, p=0.11), respectively. Multivariable Cox Model analysis estimating probability of death with covariates: opioid use (hazard ratio 2.6, p<0.001) and SLEDAI-2K (hazard ratio 1.1, p<0.001); opioid use (hazard ratios 3.0, p<0.001), and cocaine use (hazard ratio 3.2, p<0.001). The Kaplan-Meir survival analysis revealed a significantly higher probability of death for SLE patients using opioid analgesics.\\\\n\\\\nConclusionsPrescription opioid analgesic use is common in SLE and is associated with markedly increased mortality. Preferably, non-opioid approaches to treat chronic pain should be used in SLE patients.\\\\n\\\\nClinical trial registration numberThis was not a clinical trial.\\\\n\\\\nKEY MESSAGES1. Chronic opioid analgesic use is common in SLE (24%).\\\\n\\\\n2. Opioid use is associated with greater disease severity, tobacco use, non-adherence, and increased mortality.\\\\n\\\\n3. Opioids should be used cautiously in SLE; alternative non-opioid management of pain is recommended.\\\\n\\\\nACKNOWLEDGMENTS AND FUNDING INFORMATIONThis work was supported by US National Institutes of Health research grants to Dr. Sibbitt (R01 NS035708) and to the Clinical and Translational Research Center (UL1TR001449).',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396598',\n",
       "   'title': 'Macroevolutionary diversification rates show time-dependency',\n",
       "   'authors': 'Henao Diaz, L. F.; Harmon, L. J.; Sugawara, M. T. C.; Pennell, M. W.',\n",
       "   'author_corresponding': 'Matthew W. Pennell',\n",
       "   'author_corresponding_institution': 'University of British Columbia',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396598.source.xml',\n",
       "   'abstract': 'For centuries, biologists have been captivated by the vast disparity in species richness between different groups of organisms. Variation in diversity is widely attributed to differences between groups in how fast they speciate or go extinct. Such macroevolutionary rates have been estimated for thousands of groups and have been correlated with an incredible variety of organismal traits. Here we analyze a large collection of phylogenetic trees and fossil time series and report a hidden generality amongst these seemingly idiosyncratic results: speciation and extinction rates follow a scaling law where both depend strongly on the age of the group in which they are measured, with the fastest rates in the youngest clades. Using a series of simulations and sensitivity analyses, we demonstrate that the time-dependency is unlikely to be a result of simple statistical artifacts. As such, this time-scaling has profound implications for the interpretation of rate estimates and suggests there might be general laws governing macroevolutionary dynamics.',\n",
       "   'published': '10.1073/pnas.1818058116',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396556',\n",
       "   'title': 'Circadian control of lung inflammation in influenza infection',\n",
       "   'authors': 'Sengupta, S.; Tang, S. Y.; Devine, J.; Nayak, S.; Zhang, S.; Valenzuela, A.; Lopez, C. B.; Grant, G.; FitzGerald, G. A.',\n",
       "   'author_corresponding': 'Shaon  Sengupta',\n",
       "   'author_corresponding_institution': 'University of Pennsylvania',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'immunology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396556.source.xml',\n",
       "   'abstract': 'Influenza is a leading cause of respiratory mortality and morbidity. While inflammation is necessary for fighting infection, a fine balance of anti-viral defense and host tolerance is necessary for recovery. Circadian rhythms have been known to modulate inflammation. However, the importance of diurnal variability in the timing of influenza infection is not well understood. Here we demonstrate that endogenous rhythms influence the cellular response to infection in bronchoalveolar lavage (BAL), the pulmonary transcriptomic profile and lesional histology. This time dependent variability does not reflect alterations in viral replication. Rather, we found that better time-dependent outcomes were associated with a preponderance of NK and NKT cells and lower proportion of monocytes in the lung. Thus, host tolerance, rather than viral burden underlies the diurnal gating of influenza induced lung injury.\\\\n\\\\nSignificance statementOur work demonstrates the importance of circadian rhythms in influenza infection --a condition with significant public health implications. Our findings, which establish the role of the circadian rhythms in maintaining the balance between host tolerance pathways and anti-viral responses confers a new framework for evaluating the relevance of circadian influences on immunity.',\n",
       "   'published': '10.1038/s41467-019-11400-9',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/392936',\n",
       "   'title': 'Human Subthalamic Nucleus Selectively Decreases its Response to the Go-Signal in a Motor Inhibition Context',\n",
       "   'authors': 'Marmor, O.; Rapple, P.; Valsky, D.; Bick, A.; Arkadir, D.; Linetzky, E.; Peled, O.; Tamir, I.; Bergman, H.; Israel, Z.; Eitan, R.',\n",
       "   'author_corresponding': 'Odeya  Marmor',\n",
       "   'author_corresponding_institution': 'HUJI',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/392936.source.xml',\n",
       "   'abstract': 'To better understand the mechanism of movement facilitation and inhibition in the subthalamic nucleus (STN), we recorded subthalamic multiunit activity intra-operatively while parkinsonian patients (n=43 patients, 173 sites) performed increasingly complex oddball paradigms: auditory ( None-Go, n=7, 28), simple movement ( All-Go, n=7, 26) and movement inhibition ( Go-NoGo, n=29, 119) tasks. To enable physiological sampling of the different subthalamic nucleus domains in both hemispheres, each patient performed one of the oddball paradigms several times.\\\\n\\\\nThe human STN responded mainly to movement-involving tasks: movement execution at the motor STN and movement planning at the limbic-associative STN. In the limbic-associative STN, responses to the inhibitory cue (deviant tone) in the movement inhibition task were not significantly different from the simple movement task. However, responses to the go cue (frequent tone) were significantly reduced. The reduction was mainly in the negative component of the evoked response amplitude. Successful movement inhibition was correlated with higher baseline activity before the inhibitory cue.\\\\n\\\\nWe suggest that the STN adapts to movement inhibition context by selectively decreasing the amplitude of neuronal activity. Thus, the STN enables movement inhibition not by increasing responses to the inhibitory cue but by reducing responses to the release cue. The negative component of the evoked response probably facilitates movement and a higher baseline activity enables successful inhibition of movement. These discharge modulations were found in the ventromedial, non-motor domain of the STN and therefore suggest a significant role of the associative-limbic domains in movement planning and in global movement regulation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/395152',\n",
       "   'title': 'Genome-wide patterns of gene expression in a wild primate indicate species-specific mechanisms associated with tolerance to natural simian immunodeficiency virus infection',\n",
       "   'authors': 'Simons, N.; Eick, G. N.; Ruiz-Lopez, M. J.; Hyeroba, D.; Omeja, P. A.; Weny, G.; Chapman, C. A.; Goldberg, T. L.; Zheng, H.; Shankar, A.; Switzer, W. M.; Frost, S. D. W.; Jones, J. H.; Sterner, K. N.; Ting, N.',\n",
       "   'author_corresponding': 'Nelson  Ting',\n",
       "   'author_corresponding_institution': 'University of Oregon',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/395152.source.xml',\n",
       "   'abstract': 'Over 40 species of nonhuman primates host simian immunodeficiency viruses (SIVs). In natural hosts, infection is generally assumed to be nonpathogenic due to a long coevolutionary history between host and virus, although pathogenicity is difficult to study in wild nonhuman primates. We used whole-blood RNA-seq and SIV prevalence from 29 wild Ugandan red colobus (Piliocolobus tephrosceles) to assess the effects of SIV infection on host gene expression in wild, naturally SIV-infected primates. We found no evidence for chronic immune activation in infected individuals, suggesting that SIV is not immunocompromising in this species, in contrast to HIV in humans. Notably, an immunosuppressive gene, CD101, was upregulated in infected individuals. This gene has not been previously described in the context of nonpathogenic SIV infection. This expands the known variation associated with SIV infection in natural hosts, and may suggest a novel mechanism for tolerance of SIV infection in the Ugandan red colobus.',\n",
       "   'published': '10.1093/gbe/evz099',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396721',\n",
       "   'title': 'Centrosome Aurora A gradient ensures a single PAR-2 polarity axis by regulating RhoGEF ECT-2 localization in C. elegans embryos',\n",
       "   'authors': 'Kotak, S.; Kapoor, S.',\n",
       "   'author_corresponding': 'Sachin  Kotak',\n",
       "   'author_corresponding_institution': 'Indian Institute of Science',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'cell biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396721.source.xml',\n",
       "   'abstract': 'The proper establishment of the cell polarity is essential for development and morphogenesis. In the Caenorhabditis elegans one-cell embryo, a centrosome localized signal provides spatial information that is responsible for generating a single polarity axis. It is hypothesized that such a signal causes local inhibition of cortical actomyosin network in the vicinity of the centrosome. This pivotal event initiates symmetry breaking to direct partitioning of the partition defective proteins (PARs) in the one-cell embryo. However, the molecular nature of the centrosome regulated signal that impinges on the posterior cortex to bring upon cortical anisotropy in the actomyosin network and to promote polarity establishment remains elusive. Here, we discover that Aurora A kinase (AIR-1 in C. elegans) is essential for proper cortical contractility in the one-cell embryo. Loss of AIR-1 causes pronounced cortical contractions on the entire embryo surface during polarity establishment phase, and this creates more than one PAR-2 polarity axis. Moreover, we show that in the absence of AIR-1, centrosome positioning becomes dispensable in dictating the PAR-2 polarity axis. Interestingly, we identify that Rho Guanine Exchange Factor (GEF) ECT-2 acts downstream to AIR-1 to control excess contractility and notably AIR-1 loss affects ECT-2 cortical localization and thereby polarity establishment. Overall, our study unravels a novel insight whereby an evolutionarily conserved kinase Aurora A inhibits promiscuous PAR-2 domain formation and ensures singularity in the polarity establishment axis.',\n",
       "   'published': '10.1242/dev.174565',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396739',\n",
       "   'title': 'Somatically Mutated Genes Under Positive and Negative Selection Found by Transcriptome Sequence Analysis Include Oncogene and Tumor Suppressor Candidates',\n",
       "   'authors': 'Drubin, C. W.; Ramu, A.; Rockweiler, N. B.; Conrad, D. F.',\n",
       "   'author_corresponding': 'Donald F Conrad',\n",
       "   'author_corresponding_institution': 'Department of Genetics, Washington University School of Medicine, St. Louis, Missouri 63110',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396739.source.xml',\n",
       "   'abstract': 'IntroductionOncogenic somatic mutations confer proliferative advantage and undergo positive clonal selection. We developed software and applied new analytical approaches to identify: (1) somatic mutations in diverse tissues, (2) somatically mutated genes under positive and negative selection, (3) post-transcriptional modifications in the mitochondrial transcriptome, and (4) inherited germline alleles predisposing people to higher somatic mutation burden or higher levels of post-transcriptional modification.\\\\n\\\\nMethodsTranscriptome sequence data (Genotype Tissue Expression project) for 7051 tissue samples from 549 postmortem donors and representing 44 tissue types were used. Germline mutations were inferred from whole-exome DNA sequencing and SNP arrays. DNA somatic mutations were inferred from variant allele frequencies (VAF) in RNA-seq data. Post-transcriptional modifications were inferred from Polymorphism Information Content (PIC) at the p9 sites of mitochondrial tRNA sequences. Positive and negative clonal selection was evaluated using a nonsynonomous/synonomous mutation rate (dN/dS) model. Genome-wide association studies (GWAS) were assessed with mitochondrial PIC for post-transcriptional modification level, or using the total number of somatic mutations observed per donor for somatic mutation burden.\\\\n\\\\nResultsOur dN/dS model identified 78 genes under negative selection for somatic mutations (dN/dS < 1, padj < 0.05) and 14 under positive selection (dN/dS > 1, padj <0.05). Our GWAS identified 2 sites associated with post-transcriptional modification (1 approaching significance with p=5.99x10-8, 1 with p<5x10-8) and [~]20 sites associated with somatic mutation burden (p<5x10- 8).\\\\n\\\\nConclusionsTo our knowledge these are the first genome-wide association studies on normal somatic mutation burden. These studies were an attempt to increase understanding of the somatic mutation process. Our work identified somatic mutations at the global organismal level that may promote cell proliferation in a tissue-specific manner. By identifying tissue-specific mutations in actively expressed genes that appear before cancer phenotype is detected, this work also identifies gene candidates that might initiate tumorigenesis.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396994',\n",
       "   'title': 'All four double-stranded RNA binding domains of Staufen2 contribute to efficient mRNA recognition and transcript localization',\n",
       "   'authors': 'Heber, S.; Gaspar, I.; Tants, J.-N.; Günther, J.; Fernandez Moya, S. M.; Janowski, R.; Ephrussi, A.; Sattler, M.; Niessing, D.',\n",
       "   'author_corresponding': 'Dierk  Niessing',\n",
       "   'author_corresponding_institution': 'Institute of Pharmaceutical Biotechnology, Ulm University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'biochemistry',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396994.source.xml',\n",
       "   'abstract': 'Throughout metazoans, Staufen (Stau) proteins are core factors of mRNA localization particles. They consist of three to four double-stranded RNA binding domains (dsRBDs) and a C-terminal dsRBD-like domain. Mouse Staufen2 (mStau2) like Drosophila Stau (dmStau) contains four dsRBDs. Existing data suggest that only dsRBDs 3-4 are necessary and sufficient for mRNA binding. Here, we show that dsRBDs 1 and 2 of mStau2 bind RNA with similar affinities and kinetics as dsRBDs 3 and 4. While RNA binding by these tandem domains is transient, all four dsRBDs recognize their target RNAs with high stability. Rescue experiments in Drosophila oocytes demonstrate that mStau2 partially rescues dmStau-dependent mRNA localization. In contrast, a rescue with mStau2 bearing RNA-binding mutations in dsRBD1-2 fails, confirming the physiological relevance of our findings. In summary, our data show that the dsRBDs 1-2 play essential roles in the mRNA recognition and function of Stau- family proteins of different species.',\n",
       "   'published': '10.1038/s41467-019-09655-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397117',\n",
       "   'title': 'A large-scale systemic RNAi screen in the red flour beetle Tribolium castaneum identifies novel genes involved in arthropod muscle development',\n",
       "   'authors': 'Schultheis, D.; Weisskopf, M.; Schaub, C.; Ansari, S.; Dao, V.-A.; Grossmann, D.; Majumdar, U.; Din Muhammad, M. S.; Troelenberg, N.; Richter, T.; Schmitt-Engel, C.; Schwirz, J.; Ströhlein, N.; Teuscher, M.; Bucher, G.; Frasch, M.',\n",
       "   'author_corresponding': 'Manfred  Frasch',\n",
       "   'author_corresponding_institution': 'Division of Developmental Biology, Friedrich-Alexander University of Erlangen-Nürnberg',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'developmental biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397117.source.xml',\n",
       "   'abstract': 'Although muscle development has been widely studied in Drosophila melanogaster there are still many gaps in our knowledge, and it is not known to which extent this knowledge can be transferred to other insects. To help in closing these gaps we participated in a large-scale RNAi screen that used the red flour beetle, Tribolium castaneum, as a screening platform. The effects of systemic RNAi were screened upon double-stranded RNA injections into appropriate muscle-EGFP tester strains. Injections into pupae were followed by the analysis of the late embryonic/early larval muscle patterns, and injections into larvae by the analysis of the adult thoracic muscle patterns. Herein we describe the results of the first-pass screens with pupal and larval injections, which covered ~8,500 and ~5,000 genes, respectively, of a total of ~16,500 genes of the Tribolium genome. Apart from many genes known from Drosophila as regulators of muscle development, a collection of genes previously unconnected to muscle development yielded phenotypes in larval body wall and leg muscles as well as in indirect flight muscles. We then present the main candidates from the pupal injection screen that remained after being processed through a series of verification and selection steps. Further, we discuss why distinct though overlapping sets of genes are revealed by the Drosophila and Tribolium screening approaches.',\n",
       "   'published': '10.1534/g3.118.200995',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397141',\n",
       "   'title': 'Positive human health effects of sea spray aerosols: molecular evidence from exposed lung cell lines.',\n",
       "   'authors': 'Asselman, J.; Van Acker, E.; De Rijcke, M.; Tilleman, L.; Van Nieuwerburgh, F.; Mees, J.; De Schamphelaere, K.; Janssen, C.',\n",
       "   'author_corresponding': 'Jana  Asselman',\n",
       "   'author_corresponding_institution': 'Ghent University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'pharmacology and toxicology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397141.source.xml',\n",
       "   'abstract': 'Sea spray aerosols (SSAs) have profound effects on climate and ecosystems. Furthermore, the presence of microbiota and biogenic molecules, produced by among others marine phytoplankton, in SSAs could lead to potential human health effects. Yet the exposure and effects of SSAs on human health remain poorly studied. Here, we exposed human epithelial lung cells to different concentrations of extracts of a natural sea spray aerosol (SSA), a laboratory-generated SSA, the marine algal toxin homoyessotoxin and a chemical mTOR inhibitor. The mTOR inhibitor was included as it has been hypothesized that natural SSAs may influence the mTOR cell signaling pathway. We observed significant effects on the mTOR pathway and PCSK9 in all exposures. Based on these expression patterns, a clear dose response relationship was observed. Our results indicate a potential for positive health effects when lung cells are exposed to environmentally relevant concentrations of natural SSAs, whereas potential negative effects were observed at high levels of the laboratory SSA and the marine algal toxin. Overall, these results provide a substantial molecular evidence base for potential positive health effects of SSAs at environmentally relevant concentrations through the mTOR pathway. The results provided here suggest that SSAs contain biomolecules with significant pharmaceutical potential in targeting PCSK9.',\n",
       "   'published': '10.1038/s41598-018-36866-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397141',\n",
       "   'title': 'Positive human health effects of sea spray aerosols: molecular evidence from exposed lung cell lines.',\n",
       "   'authors': 'Asselman, J.; Van Acker, E.; De Rijcke, M.; Tilleman, L.; Van Nieuwerburgh, F.; Mees, J.; De Schamphelaere, K.; Janssen, C.',\n",
       "   'author_corresponding': 'Jana  Asselman',\n",
       "   'author_corresponding_institution': 'Ghent University',\n",
       "   'date': '2018-08-23',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'pharmacology and toxicology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/397141.source.xml',\n",
       "   'abstract': 'Sea spray aerosols (SSAs) have profound effects on climate and ecosystems. Furthermore, the presence of microbiota and biogenic molecules, produced by among others marine phytoplankton, in SSAs could lead to potential human health effects. Yet the exposure and effects of SSAs on human health remain poorly studied. Here, we exposed human epithelial lung cells to different concentrations of extracts of a natural sea spray aerosol (SSA), a laboratory-generated SSA, the marine algal toxin homoyessotoxin and a chemical mTOR inhibitor. The mTOR inhibitor was included as it has been hypothesized that natural SSAs may influence the mTOR cell signaling pathway. We observed significant effects on the mTOR pathway and PCSK9 in all exposures. Based on these expression patterns, a clear dose response relationship was observed. Our results indicate a potential for positive health effects when lung cells are exposed to environmentally relevant concentrations of natural SSAs, whereas potential negative effects were observed at high levels of the laboratory SSA and the marine algal toxin. Overall, these results provide a substantial molecular evidence base for potential positive health effects of SSAs at environmentally relevant concentrations through the mTOR pathway. The results provided here suggest that SSAs contain biomolecules with significant pharmaceutical potential in targeting PCSK9.',\n",
       "   'published': '10.1038/s41598-018-36866-3',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397000',\n",
       "   'title': 'CryoEM structures of open dimers of Gyrase A in complex with DNA illuminate mechanism of strand passage',\n",
       "   'authors': 'Soczek, K. M.; Grant, T.; Rosenthal, P. B.; Mondragon, A.',\n",
       "   'author_corresponding': 'Alfonso  Mondragon',\n",
       "   'author_corresponding_institution': 'Northwestern University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397000.source.xml',\n",
       "   'abstract': 'Gyrase is a unique type IIA topoisomerase that uses ATP hydrolysis to maintain the negatively supercoiled state of bacterial DNA. In order to perform its function, gyrase undergoes a sequence of conformational changes that consist of concerted gate openings, DNA cleavage, and DNA strand passage events. Structures where the transported DNA molecule (T-segment) is trapped by the A subunit have not been observed. Here we present the cryoEM structures of two oligomeric complexes of open gyrase A dimers and DNA. The protein subunits in these complexes were solved to 4 [A] and 5.16 [A] resolution. One of the complexes traps a linear DNA molecule, a putative T-segment, which interacts with the open gyrase A dimers in two states, representing steps either prior to or after passage through the DNA-gate. The structures locate the T-segment in important intermediate conformations of the catalytic cycle and provide insights into gyrase-DNA interactions and mechanism.',\n",
       "   'published': '10.7554/elife.41215',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396663',\n",
       "   'title': 'Large-Scale Annotation of Histopathology Images from Social Media',\n",
       "   'authors': 'Schaumberg, A. J.; Juarez, W.; Choudhury, S. J.; Pastrian, L. G.; Pritt, B. S.; Prieto Pozuelo, M.; Sotillo Sanchez, R.; Ho, K.; Zahra, N.; Sener, B. D.; Yip, S.; Xu, B.; Annavarapu, S. R.; Morini, A.; Jones, K. A.; Rosado-Orozco, K.; Sirintrapun, S. J.; Aly, M.; Fuchs, T. J.',\n",
       "   'author_corresponding': 'Andrew J. Schaumberg',\n",
       "   'author_corresponding_institution': 'Weill Cornell Graduate School of Medical Sciences, NY, USA',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'pathology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396663.source.xml',\n",
       "   'abstract': 'Pathologists are responsible for rapidly providing a diagnosis on critical health issues. Challenging cases benefit from additional opinions of pathologist colleagues. In addition to on-site colleagues, there is an active worldwide community of pathologists on social media for complementary opinions. Such access to pathologists worldwide has the capacity to improve diagnostic accuracy and generate broader consensus on next steps in patient care. From Twitter we curate 13,626 images from 6,351 tweets from 25 pathologists from 13 countries. We supplement the Twitter data with 113,161 images from 1,074,484 PubMed articles. We develop machine learning and deep learning models to (i) accurately identify histopathology stains, (ii) discriminate between tissues, and (iii) differentiate disease states. Area Under Receiver Operating Characteristic is 0.805-0.996 for these tasks. We repurpose the disease classifier to search for similar disease states given an image and clinical covariates. We report precision@k=1 = 0.7618{+/-}0.0018 (chance 0.397{+/-}0.004, mean{+/-}stdev). The classifiers find texture and tissue are important clinico-visual features of disease. Deep features trained only on natural images (e.g. cats and dogs) substantially improved search performance, while pathology-specific deep features and cell nuclei features further improved search to a lesser extent. We implement a social media bot (@pathobot on Twitter) to use the trained classifiers to aid pathologists in obtaining real-time feedback on challenging cases. If a social media post containing pathology text and images mentions the bot, the bot generates quantitative predictions of disease state (normal/artifact/infection/injury/nontumor, pre-neoplastic/benign/ low-grade-malignant-potential, or malignant) and lists similar cases across social media and PubMed. Our project has become a globally distributed expert system that facilitates pathological diagnosis and brings expertise to underserved regions or hospitals with less expertise in a particular disease. This is the first pan-tissue pan-disease (i.e. from infection to malignancy) method for prediction and search on social media, and the first pathology study prospectively tested in public on social media. We will share data through pathobotology.org. We expect our project to cultivate a more connected world of physicians and improve patient care worldwide.',\n",
       "   'published': '10.1038/s41379-020-0540-1',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396838',\n",
       "   'title': 'Survival of selected lactic acid bacteria isolated from pineapple waste in refrigerated pineapple juices',\n",
       "   'authors': 'Wen Fang Wu, J.; Uribe, L.; WingChing-Jones, R.; Usaga, J.; Barboza, N.',\n",
       "   'author_corresponding': 'Natalia  Barboza',\n",
       "   'author_corresponding_institution': 'Universidad de Costa Rica',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'bioengineering',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396838.source.xml',\n",
       "   'abstract': 'The aim of this research was to isolate and identify lactic acid bacteria (LAB) from pineapple waste. The survival in refrigerated pineapple juices, of a selected isolate with potential probiotic properties, was also studied. The 16S rRNA and pheS partial genes were used to identify LAB, multilocus sequence typing (MLST) genes were used in order to separate strains grouping with Lactobacillus casei and L. paracasei phylogenetically. Tests for survival at pH 2.0, resistance to lysozyme and tolerance to bile salts were used to screen the strains for potential probiotic properties. A L. fermentum isolate was used for the survival study. Three types of pineapple juice made from pulp, a blend of pulp and peel (80:20), and peel extract were inoculated to approximately 106 CFU/mL with L. fermentum and stored at 4 {degrees}C for up to six weeks. The physicochemical composition of juices, including concentration of fermentable sugars and organic acids, total solids content, soluble solids content, titratable acidity and pH, was determined during the survival study. Two genera and five species were identified. Pineapple juices supported the survival of L. fermentum during refrigerated storage but the population of the bacteria decreased over time regardless of the juice type. Juice made from pulp was a more suitable vehicle for the survival of the selected LAB. Some of the juice physicochemical properties, including sugars and organic acids content, pH and titratable acidity, varied significantly (P<0.05) during storage. Further sensory studies are necessary to evaluate consumer acceptance of juices containing the selected isolate.\\\\n\\\\nIMPORTANCELactic acid bacteria (LAB), isolated from pineapple waste, were phylogenetically analyzed and characterized in regards to their tolerance to pH 2.0, lysozyme and bile salts; showing their potential as probiotic strains, if health benefits associated to their ingestion are eventually confirmed. Moreover, pineapple juice supported the survival of Lactobacillus fermentum, isolated from the same food matrix, during refrigerated storage at 4 {degrees}C. Among the three pineapple juices tested (pulp, pulp + peel and peel), L. fermentum survived better in juice made from pulp. However, significant variations were observed overtime in some of the physicochemical properties of the juices including sugars and organic acids content, pH and total titratable acidity.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396606',\n",
       "   'title': 'Streptomyces volatile compounds influence exploration and microbial community dynamics by altering iron availability',\n",
       "   'authors': 'Jones, S. E.; Pham, C. A.; McKillip, J.; Zambri, M.; Carlson, E. E.; Elliot, M.',\n",
       "   'author_corresponding': 'Marie  Elliot',\n",
       "   'author_corresponding_institution': 'McMaster University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396606.source.xml',\n",
       "   'abstract': 'Bacteria and fungi produce a wide array of volatile organic compounds (VOCs), and these can act as infochemicals or as competitive tools. Recent work has shown that the VOC trimethylamine (TMA) can promote a new form of Streptomyces growth, termed  exploration. Here, we report that TMA also serves to alter nutrient availability in the area surrounding exploring cultures: TMA dramatically increases the environmental pH, and in doing so, reduces iron availability. This, in turn, compromised the growth of other soil bacteria and fungi. In contrast, Streptomyces thrives in these iron-depleted niches by secreting a suite of differentially modified siderophores, and by upregulating genes associated with siderophore uptake. Further reducing iron levels by siderophore piracy, limiting siderophore uptake, or growing cultures in the presence of iron chelators, unexpectedly enhanced exploration. Our work reveals a new role for VOCs in modulating iron levels in the environment, and implies a critical role for VOCs in modulating the behaviour of microbes and the makeup of their communities.',\n",
       "   'published': '10.1128/mbio.00171-19',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396820',\n",
       "   'title': 'Dynamin is required for efficient cytomegalovirus maturation and envelopment',\n",
       "   'authors': 'Hasan, M. H.; Davis, L.; Bollavarapu, R. K.; Mitra, D.; Parmar, R. C.; Tandon, R.',\n",
       "   'author_corresponding': 'Ritesh  Tandon',\n",
       "   'author_corresponding_institution': 'University of Mississippi Medical Center',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396820.source.xml',\n",
       "   'abstract': 'Cytomegalovirus secondary envelopment occurs in a virus-induced cytoplasmic assembly compartment (vAC) generated via a drastic reorganization of the membranes of the secretory and endocytic systems. Dynamin is a eukaryotic GTPase that is implicated in membrane remodeling and endocytic membrane fission events; however, the role of dynamin in cellular trafficking of viruses beyond virus entry is only partially understood. Mouse embryonic fibroblasts (MEF) engineered to excise all three isoforms of dynamin were infected with mouse cytomegalovirus (MCMV-K181). Immediate early (IE1; m123) viral protein was detected in these triple dynamin knockout (TKO) cells as well as in mock-induced parental MEF at early times post infection although levels were reduced in TKO cells, indicating that virus entry was affected but not eliminated. Levels of IE1 protein and another viral early protein (m04) were normalized by 48 hours post infection; however, late protein (m55; gB) expression was significantly reduced in infected TKO cells compared to parental MEF. Ultrastructural analysis revealed intact stages of nuclear virus maturation in both cases with equivalent numbers of nucleocapsids containing packaged viral DNA (C-capsids) indicating successful viral DNA replication, capsid assembly and genome packaging. Most importantly, severe defects in virus envelopment were visualized in TKO cells but not in parental cells. Dynamin inhibitor (dynasore) treated MEF showed a phenotype similar to TKO cells upon MCMV infection confirming the role of dynamin in late maturation processes. In summary, dynamin-mediated endocytic pathways are critical for the completion of cytoplasmic stages of cytomegalovirus maturation.\\\\n\\\\nImportanceViruses are known to exploit specific cellular functions at different stages of their life cycle in order to replicate, avoid immune recognition by the host and to establish a successful infection. Cytomegalovirus (CMV) infected cells are characterized by a prominent cytoplasmic inclusion (virus assembly compartment; vAC) that is the site of virus maturation and envelopment. While endocytic membranes are known to be the functional components of vAC, knowledge of specific endocytic pathways implicated in CMV maturation and envelopment is lacking. Here we show that dynamin, which is an integral part of host endocytic machinery, is largely dispensable for early stages of CMV infection but is required at a late stage of CMV maturation. Studies on dynamin function in CMV infection will help us understand the host-virus interaction pathways amenable to targeting by conventional small molecules as well as by newer generation nucleotide-based therapeutics (e.g. siRNA, CRISPR/CAS gRNA, etc.).',\n",
       "   'published': '10.1128/jvi.01418-18',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396846',\n",
       "   'title': 'Screening of antimicrobial activity and polyketide synthase gene identification from the Actinomycetes isolates',\n",
       "   'authors': 'SRIVASTAV, A. V.',\n",
       "   'author_corresponding': 'ASHU  Srivastav',\n",
       "   'author_corresponding_institution': 'INSTITUTE OF CHEMICAL TECHONOLOGY',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'confirmatory results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396846.source.xml',\n",
       "   'abstract': 'The Streptomyces genus is well studied owing to its capacity in producing more than 70% of antibiotics. This study was undertaken to characterize Streptomyces strains, occurring in soils of different districts of Tamilnadu, India as well as to evaluate their potential to produce antimicrobial compounds. Samples were collected from rice rhizosphere from different district of Tamilnadu agricultural zone. In primary screening by cross streak method, Streptomyces strains were assessed for antibiotic production and activity against different human bacterial. Then active isolates were selected for secondary screening by agar well diffusion method. Solvent extraction method was used to identify the best crude samples which are exhibiting better antibacterial activity. Then, 16S rRNA PCR was carried out for confirmation of isolates. In primary screening among all of the isolates, 50% isolates were active against at least one of the test organisms and 21.31% strains exhibited a broad-spectrum activity against almost all of the test bacteria. The minimum inhibitory concentrations (MICs) of the ethyl acetate extracts measured. Out of 26 positive strains two of the most active isolates SVG-07-15 and TK-01-05 were taken for further studies. These results highlight the importance of Streptomyces isolates in antibiotic production. Together with antibacterial activity, the PKS gene based approach can be applied for efficient screening of isolated strains of pharmaceutical value and related compounds.\\n\\nHighlightsO_LIIsolation of actinomycetes from soil samples collected from South India.\\nC_LIO_LIScreening of isolated actinomycetes for antibacterial activity\\nC_LIO_LIDetails study of potent strains against human pathogens\\nC_LIO_LIIdentification of PKS gene from the active strains\\nC_LI',\n",
       "   'published': '10.4172/1948-5948.1000404',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397182',\n",
       "   'title': 'A better way to define and describe Morlet wavelets for time-frequency analysis',\n",
       "   'authors': 'Cohen, M. X.',\n",
       "   'author_corresponding': 'Michael X Cohen',\n",
       "   'author_corresponding_institution': 'Radboud University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397182.source.xml',\n",
       "   'abstract': 'Morlet wavelets are frequently used for time-frequency analysis of non-stationary time series data, such as neuroelectrical signals recorded from the brain. The crucial parameter of Morlet wavelets is the width of the Gaussian that tapers the sine wave. This width parameter controls the trade-off between temporal precision and frequency precision. It is typically defined as the \\\\\"number of cycles,\\\\\" but this parameter is opaque, and often leads to uncertainty and suboptimal analysis choices, as well as being difficult to interpret and evaluate. The purpose of this paper is to present alternative formulations of Morlet wavelets in time and in frequency that allow parameterizing the wavelets directly in terms of the desired temporal and spectral smoothing (as full-width at half-maximum). This formulation provides clarity on an important data analysis parameter, and should facilitate proper analyses, reporting, and interpretation of results. MATLAB code is provided.',\n",
       "   'published': '10.1016/j.neuroimage.2019.05.048',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397166',\n",
       "   'title': 'Marker-based watershed transform method for fully automatic mandibular segmentation from low-dose CBCT images',\n",
       "   'authors': 'Fan, Y.; Beare, R.; Matthews, H.; Schneider, P.; Kilpatrick, N.; Clement, J.; Claes, P.; Penington, A.; Adamson, C.',\n",
       "   'author_corresponding': 'Christopher  Adamson',\n",
       "   'author_corresponding_institution': 'Developmental Imaging, Murdoch Childrens Research Institute, Melbourne, Australia',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397166.source.xml',\n",
       "   'abstract': 'Introduction Introduction Methods and materials Results Discussion Conclusions Funding References Three-dimensional mandibular models are useful for planning maxillofacial surgery and orthodontic treatment.1,2 In studies of growth, mandibular models are important for assessing morphological changes over time.3,4 Such models are typically obtained from conventional computed tomography (CT), using high radiation dose to capture fine detail of the bony structure. Cone beam computed tomography (CBCT) shows promise for oral and craniofacial imaging applications due to lower radiation dose, lower cost and shorter acquisition time compared to CT. However, CBCT images have lower contrast and higher levels of ...',\n",
       "   'published': '10.1259/dmfr.20180261',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397158',\n",
       "   'title': 'Long fragments achieve lower base quality in Illumina paired-end sequencing',\n",
       "   'authors': 'Tan, G.; Opitz, L.; Schlapbach, R.; Rehrauer, H.',\n",
       "   'author_corresponding': 'Hubert  Rehrauer',\n",
       "   'author_corresponding_institution': 'University of Zurich',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397158.source.xml',\n",
       "   'abstract': 'Illuminas technology provides high quality reads of DNA fragments with error rates below 1/1000 per base. Runs typically generate a millions of reads where the vast majority of the reads has also an average error rate below 1/1000. However, some paired-end sequencing data show the presence of a subpopulation of reads where the second read has lower average qualities. We show that the fragment length is a major driver of increased error rates in the R2 reads. Fragments above 500 nt tend to yield lower base qualities and higher error rates than shorter fragments. We demonstrate the fragment length dependency of the R2 read qualities using publicly available Illumina data generated by various library protocols, in different labs and using different sequencer models. Our finding extends the understanding of the Illumina read quality and has implications on error models for Illumina reads. It also sheds a light on the importance of the fragmentation during library preparation and the resulting fragment length distribution.',\n",
       "   'published': '10.1038/s41598-019-39076-7',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396655',\n",
       "   'title': 'Divine: Prioritizing Genes for Rare Mendelian Disease in Whole Exome Sequencing Data',\n",
       "   'authors': 'Hong, C.; Clemenceau, J. R.; Yeu, Y.; Hwang, T. H.',\n",
       "   'author_corresponding': 'Tae Hyun  Hwang',\n",
       "   'author_corresponding_institution': 'Cleveland Clinic',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396655.source.xml',\n",
       "   'abstract': 'MotivationRecent studies showed that a phenotype-driven analysis of whole exome sequencing (WES) could provide more accurate and clinically relevant genetic variants.\\\\n\\\\nResultsWe develop a computational tool called Divine that integrates patients phenotype(s) and WES data with 30 prior biological knowledge (e.g., human phenotype ontology, gene ontology, pathway database, protein-protein interaction networks, pathogenicity by the amino acid change due to polymorphism, and hot-spot protein domains) to prioritize potential disease-causing genes. In a retrospective study with 22 real and four simulated data set, Divine ranks the same pathogenic genes confirmed by the original studies 5th on average out of a thousand of mutated genes and outperforms existing state-of-the-art methods.\\\\n\\\\nAvailabilityhttps://github.com/hwanglab/divine\\\\n\\\\nContacthwangt@ccf.org\\\\n\\\\nSupplementary informationSupplementary Document is attached at the end of the page.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397042',\n",
       "   'title': 'Fast Batch Alignment of Single Cell Transcriptomes Unifies Multiple Mouse Cell Atlases into an Integrated Landscape',\n",
       "   'authors': 'Park, J.-E.; Polanski, K.; Meyer, K.; Teichmann, S. A.',\n",
       "   'author_corresponding': 'Sarah A Teichmann',\n",
       "   'author_corresponding_institution': 'Welcome Sanger Institute',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397042.source.xml',\n",
       "   'abstract': 'Increasing numbers of large scale single cell RNA-Seq projects are leading to a data explosion, which can only be fully exploited through data integration. Therefore, efficient computational tools for combining diverse datasets are crucial for biology in the single cell genomics era. A number of methods have been developed to assist data integration by removing technical batch effects, but most are computationally intensive. To overcome the challenge of enormous datasets, we have developed BBKNN, an extremely fast graph-based data integration method. We illustrate the power of BBKNN for dimensionalityreduced visualisation and clustering in multiple biological scenarios, including a massive integrative study over several murine atlases. BBKNN successfully connects cell populations across experimentally heterogeneous mouse scRNA-Seq datasets, which reveals global markers of cell type and organspecificity and provides the foundation for inferring the underlying transcription factor network. BBKNN is available at https://github.com/Teichlab/bbknn.',\n",
       "   'published': '10.1093/bioinformatics/btz625',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397042',\n",
       "   'title': 'Fast Batch Alignment of Single Cell Transcriptomes Unifies Multiple Mouse Cell Atlases into an Integrated Landscape',\n",
       "   'authors': 'Park, J.-E.; Polanski, K.; Meyer, K.; Teichmann, S. A.',\n",
       "   'author_corresponding': 'Sarah A Teichmann',\n",
       "   'author_corresponding_institution': 'Welcome Sanger Institute',\n",
       "   'date': '2018-08-22',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/397042.source.xml',\n",
       "   'abstract': 'Increasing numbers of large scale single cell RNA-Seq projects are leading to a data explosion, which can only be fully exploited through data integration. Therefore, efficient computational tools for combining diverse datasets are crucial for biology in the single cell genomics era. A number of methods have been developed to assist data integration by removing technical batch effects, but most are computationally intensive. To overcome the challenge of enormous datasets, we have developed BBKNN, an extremely fast graph-based data integration method. We illustrate the power of BBKNN for dimensionalityreduced visualisation and clustering in multiple biological scenarios, including a massive integrative study over several murine atlases. BBKNN successfully connects cell populations across experimentally heterogeneous mouse scRNA-Seq datasets, which reveals global markers of cell type and organspecificity and provides the foundation for inferring the underlying transcription factor network. BBKNN is available at https://github.com/Teichlab/bbknn.',\n",
       "   'published': '10.1093/bioinformatics/btz625',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397091',\n",
       "   'title': 'ENIGMA: An Enterotype-Like Unigram Mixture Model for Microbial Association Analysis',\n",
       "   'authors': 'Abe, K.; Hirayama, M.; Ohno, K.; Shimamura, T.',\n",
       "   'author_corresponding': 'Teppei  Shimamura',\n",
       "   'author_corresponding_institution': 'Division of Systems Biology, Nagoya University Graduate School of Medicine',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397091.source.xml',\n",
       "   'abstract': 'BackgroundOne of the major challenges in microbial studies is to discover associations between microbial communities and a specific disease. A specialized feature of microbiome count data is that intestinal bacterial communities have clusters reffered as enterotype characterized by differences in specific bacterial taxa, which makes it difficult to analyze these data under health and disease conditions. Traditional probabilistic modeling cannot distinguish dysbiosis of interest with the individual differences.\\\\n\\\\nResultsWe propose a new probabilistic model, called ENIGMA (Enterotype-like uNIGram mixture model for Microbial Association analysis), to address these problems. ENIGMA enables us to simultaneously estimate enterotype-like clusters characterized by the abundances of signature bacterial genera and environmental effects associated with the disease.\\\\n\\\\nConclusionWe illustrate the performance of the proposed method both through the simulation and clinical data analysis. ENIGMA is implemented with R and is available from GitHub (https://github.com/abikoushi/enigma).',\n",
       "   'published': '10.1186/s12864-019-5476-9',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397273',\n",
       "   'title': 'A Multimodal Adaptive Super-Resolution and Confocal Microscope',\n",
       "   'authors': 'Valiya Peedikakkal, L.; Furley, A.; Cadby, A. J.',\n",
       "   'author_corresponding': 'Ashley J Cadby',\n",
       "   'author_corresponding_institution': 'The University of Sheffield',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397273.source.xml',\n",
       "   'abstract': 'Existing optical microscopy techniques compromise between resolution, photodamage, speed of acquisition and imaging in to deep samples. This often confines a technique to a certain biological system or process. We present a versatile imaging system which can switch between imaging modalities with sub millisecond transition times to adapt to the needs of a wide range of sample types. The imaging modalities provide the minimally invasive but low-resolution epi-fluorescence though increasing invasive but higher resolution confocal and structured illumination until the highest resolution is achieved through the most intrusive, localisation microscopy. The ability of the system to overcome the limitations of conventional single mode microscopy is demonstrated by several biological investigations. The ideas presented in this work allow researchers to move away from the model of a single imaging modality to study a specific process and instead follow those processes using the most suitable method available during the lifetime of the investigation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397273',\n",
       "   'title': 'A Multimodal Adaptive Super-Resolution and Confocal Microscope',\n",
       "   'authors': 'Valiya Peedikakkal, L.; Furley, A.; Cadby, A. J.',\n",
       "   'author_corresponding': 'Ashley J Cadby',\n",
       "   'author_corresponding_institution': 'The University of Sheffield',\n",
       "   'date': '2018-08-24',\n",
       "   'version': '2',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'biophysics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/397273.source.xml',\n",
       "   'abstract': 'Existing optical microscopy techniques compromise between resolution, photodamage, speed of acquisition and imaging in to deep samples. This often confines a technique to a certain biological system or process. We present a versatile imaging system which can switch between imaging modalities with sub millisecond transition times to adapt to the needs of a wide range of sample types. The imaging modalities provide the minimally invasive but low-resolution epi-fluorescence though increasing invasive but higher resolution confocal and structured illumination until the highest resolution is achieved through the most intrusive, localisation microscopy. The ability of the system to overcome the limitations of conventional single mode microscopy is demonstrated by several biological investigations. The ideas presented in this work allow researchers to move away from the model of a single imaging modality to study a specific process and instead follow those processes using the most suitable method available during the lifetime of the investigation.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396911',\n",
       "   'title': 'Mitochondrial DNA and their nuclear copies in parasitic wasp Pteromalus puparum: A comparative analysis in Chalcidoidea',\n",
       "   'authors': 'Yan, Z.; Fang, Q.; Tian, Y.; Wang, F.; Chen, X.; Werren, J. H.; Ye, G.',\n",
       "   'author_corresponding': 'Gongyin  Ye',\n",
       "   'author_corresponding_institution': 'Zhejiang University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'genomics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396911.source.xml',\n",
       "   'abstract': 'Chalcidoidea (chalcidoid wasps) are an abundant and megadiverse insect group with both ecological and economical importance. Here we report a complete mitochondrial genome in Chalcidoidea from Pteromalus puparum (Pteromalidae). Eight tandem repeats followed by 6 reversed repeats were detected in its 3,308 bp control region. This long and complex control region may explain failures of amplifying and sequencing of complete mitochondrial genomes in some chalcidoids. In addition to 37 typical mitochondrial genes, an extra identical isoleucine tRNA (trnI) was detected. We speculate this recent mitochondrial gene duplication indicates that gene arrangements in chalcidoids are ongoing. A comparison among available chalcidoid mitochondrial genomes, reveals rapid gene order rearrangements overall, and high substitution rate in P. puparum. In addition, we identified 24 nuclear sequences of mitochondrial origin (NUMTs) in P. puparum, summing up to 9,989 bp, with 3,617 bp of these NUMTs originating from mitochondrial coding regions. NUMTs abundance in P. puparum is only one-twelfth of that in its relative, Nasonia vitripennis. Based on phylogenetic analysis, we provide evidence that a faster nuclear degradation rate contributes to the reduced NUMT numbers in P. puparum. Overall, our study shows unusually high rates of mitochondrial evolution and considerable variation in NUMT accumulation in Chalcidoidea.',\n",
       "   'published': '10.1016/j.ijbiomac.2018.10.039',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396804',\n",
       "   'title': 'A highly conserved complete accessory Escherichia coli type III secretion system 2 is widespread in bloodstream isolates of the ST69 lineage',\n",
       "   'authors': 'Evans, T. J.; Fox, S.; Goswami, C.; Holden, M. T.; Connolly, J. P.; Roe, A. J.; Connor, M.; Leanord, A.',\n",
       "   'author_corresponding': 'Tom J  Evans',\n",
       "   'author_corresponding_institution': 'University of Glasgow',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396804.source.xml',\n",
       "   'abstract': 'Bacterial type III secretion systems (T3SS) play an important role in pathogenesis of Gram-negative infections. Enteropathogenic and enterohemorrhagic Escherichia coli contain a well-defined T3SS but in addition a second T3SS termed E. coli T3SS 2 (ETT2) has been described in a number of strains of E. coli. The majority of E. coli contain elements of a genetic locus encoding ETT2, but which has undergone significant mutational attrition rendering it without predicted function. Only a very few strains have been reported to contain an intact ETT2 locus. To investigate the occurrence of the ETT2 locus in strains of human pathogenic E. coli, we carried out genomic sequencing of 162 isolates obtained from patient blood cultures in Scotland. We found that all 26 ST69 isolates from this collection contained an intact ETT2 together with an associated eip locus which encodes putative secreted ETT2 effectors as well as eilA, a gene encoding a putative transcriptional regulator of ETT2 associated genes. Using a reporter gene for eilA activation, we defined conditions under which this gene was differentially activated. However, comparison of secreted proteins from ST69 strains under high and low eilA activation failed to identify any ETT2 secreted substrates. The conservation of the genes encoding ETT2 in human pathogenic ST69 strains strongly suggests it has functional importance in infection, although its exact functional role remains obscure.\\\\n\\\\nImportanceOne of the commonest bacteria causing bloodstream infections in humans is Escherichia coli, which has a significant morbidity and mortality. Better understating of the mechanisms by which this microbe can invade blood could lead to more effective prevention and treatment. One mechanism by which some strains cause disease is by elaboration of a specialized secretion system, the type III secretion system (T3SS), encoded by the locus of enterocyte effacement (LEE). In addition to this well-defined T3SS, a second T3SS has been found in some E. coli strains termed E. coli type III secretion system 2 (ETT2). Most strains carry elements of the ETT2 locus, but with significant mutational attrition rendering it functionless. The significance of our work is that we have discovered that human bloodstream isolates of E. coli of sequence type 69 contain a fully intact ETT2 and associated genes, strongly suggesting its functional importance in human infection.',\n",
       "   'published': '10.1038/s41598-020-61026-x',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396978',\n",
       "   'title': 'Pyrite formation from FeS and H2S is mediated by a novel type of microbial energy metabolism',\n",
       "   'authors': 'Thiel, J.; Byrne, J.; Kappler, A.; Schink, B.; Pester, M.',\n",
       "   'author_corresponding': 'Michael  Pester',\n",
       "   'author_corresponding_institution': 'Leibniz Institute DSMZ - German Collection of Microorganisms and Cell Cultures',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396978.source.xml',\n",
       "   'abstract': 'The exergonic reaction of FeS with H2S to form FeS2 (pyrite) and H2 was postulated to have operated as an early form of energy metabolism on primordial Earth. Since the Archean, sedimentary pyrite formation played a major role in the global iron and sulfur cycles, with direct impact on the redox chemistry of the atmosphere. To date, pyrite formation was considered a purely geochemical reaction. Here, we present microbial enrichment cultures, which grew with FeS, H2S, and CO2 as their sole substrates to produce FeS2 and CH4. Cultures grew over periods of three to eight months to cell densities of up to 2-9x106 cells mL-1. Transformation of FeS with H2S to FeS2 was followed by 57Fe Mossbauer spectroscopy and showed a clear biological temperature profile with maximum activity at 28{degrees}C and decreasing activities towards 4{degrees}C and 60{degrees}C. CH4 was formed concomitantly with FeS2 and exhibited the same temperature dependence. Addition of either penicillin or 2-bromoethanesulfonate inhibited both FeS2 and CH4 production, indicating a syntrophic coupling of pyrite formation to methanogenesis. This hypothesis was supported by a 16S rRNA gene-based phylogenetic analysis, which identified at least one archaeal and five bacterial species. The archaeon was closely related to the hydrogenotrophic methanogen Methanospirillum stamsii while the bacteria were most closely related to sulfate-reducing Deltaproteobacteria, as well as uncultured Firmicutes and Actinobacteria. We identified a novel type of microbial metabolism able to conserve energy from FeS transformation to FeS2, which may serve as a model for a postulated primordial iron-sulfur world.\\\\n\\\\nSignificance statementPyrite is the most abundant iron-sulfur mineral in sediments. Over geological times, its burial controlled oxygen levels in the atmosphere and sulfate concentrations in seawater. Its formation in sediments is so far considered a purely geochemical process that is at most indirectly supported by microbial activity. We show that lithotrophic microorganisms can directly transform FeS and H2S to FeS2 and use this exergonic reaction as a novel form of energy metabolism that is syntrophically coupled to methanogenesis. Our results provide insights into a syntrophic relationship that could sustain part of the deep biosphere and lend support to the iron-sulfur-world theory that postulated FeS transformation to FeS2 as a key energy-delivering reaction for life to emerge.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/391888',\n",
       "   'title': 'The role of the fornix in human navigational learning',\n",
       "   'authors': 'Hodgetts, C. J.; Stefani, M.; Williams, A. N.; Kolarik, B. S.; Yonelinas, A. P.; Ekstrom, A. D.; Lawrence, A. D.; Zhang, J.; Graham, K. S.',\n",
       "   'author_corresponding': 'Carl J Hodgetts',\n",
       "   'author_corresponding_institution': 'Cardiff University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/391888.source.xml',\n",
       "   'abstract': 'Studies in rodents have demonstrated that transecting the white matter pathway linking the hippocampus and anterior thalamic nuclei - the fornix - impairs flexible navigational learning in the Morris Water Maze (MWM), as well as similar spatial learning tasks. While diffusion MRI studies in humans have linked fornix microstructure to scene discrimination and memory, its role in human navigation is currently unknown. We used high-angular resolution diffusion MRI to ask whether inter-individual differences in fornix microstructure would be associated with spatial learning in a virtual MWM task. To increase sensitivity to individual learning across trials, we adopted a novel curve fitting approach to estimate a single index of learning rate. We found a significant correlation between learning rate and the microstructure (mean diffusivity) of the fornix, but not that of a control tract linking occipital and anterior temporal cortices (the inferior longitudinal fasciculus, ILF). Further, this correlation remained significant when controlling for hippocampal volume. These findings extend previous animal studies by demonstrating the functional relevance of the fornix for human navigational learning, and highlight the importance of a distributed neuroanatomical network, underpinned by key white matter pathways, such as the fornix, in complex spatial behaviour.',\n",
       "   'published': '10.1016/j.cortex.2019.10.017',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396986',\n",
       "   'title': 'Altered dendritic spine function and integration in a mouse model of Fragile X Syndrome',\n",
       "   'authors': 'Booker, S. A.; Domanski, A. P.; Dando, O. R.; Jackson, A. D.; Isaac, J. T.; Hardingham, G. E.; Wyllie, D. J.; Kind, P. C.',\n",
       "   'author_corresponding': 'Peter C Kind',\n",
       "   'author_corresponding_institution': 'University of Edinburgh',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396986.source.xml',\n",
       "   'abstract': 'Cellular and circuit hyperexcitability are core features of Fragile X Syndrome and related autism spectrum disorder models. However, a synaptic basis for this hyperexcitability has proved elusive. We show in a mouse model of Fragile X Syndrome, glutamate uncaging onto individual dendritic spines yields stronger single-spine excitation than wild-type, with more silent spines. Furthermore, near-simultaneous uncaging at multiple spines revealed fewer spines are required to trigger an action potential. This arose, in part, from increased dendritic gain due to increased intrinsic excitability, resulting from reduced hyperpolarization-activated currents. Super-resolution microscopy revealed no change in dendritic spine morphology, pointing to an absence of a structure-function relationship. However, ultrastructural analysis revealed a 3-fold increase in multiply-innervated spines, accounting for the increased single-spine excitatory currents following glutamate uncaging. Thus, loss of FMRP causes abnormal synaptogenesis, leading to large numbers of poly-synaptic spines despite normal spine morphology, thus explaining the synaptic perturbations underlying circuit hyperexcitability.',\n",
       "   'published': '10.1038/s41467-019-11891-6',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397224',\n",
       "   'title': 'Regional low-frequency oscillations in human rapid-eye movement sleep',\n",
       "   'authors': 'Bernardi, G.; Betta, M.; Ricciardi, E.; Pietrini, P.; Tononi, G.; Siclari, F.',\n",
       "   'author_corresponding': 'Francesca  Siclari',\n",
       "   'author_corresponding_institution': 'Lausanne University Hospital',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'neuroscience',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397224.source.xml',\n",
       "   'abstract': 'Although the EEG slow wave of sleep is typically considered to be a hallmark of Non Rapid Eye Movement (NREM) sleep, recent work in mice has shown that slow waves can also occur in REM sleep. Here we investigated the presence and cortical distribution of low-frequency (1-4 Hz) oscillations in human REM sleep by analyzing high-density EEG sleep recordings obtained in 28 healthy subjects. We identified two clusters of low-frequency oscillations with distinctive properties: 1) a fronto-central cluster characterized by [~]2.5-3.0 Hz, relatively large, notched delta waves (so-called  sawtooth waves) that tended to occur in bursts, were associated with increased gamma activity and rapid eye movements, and upon source modeling, displayed an occipito-temporal and a fronto-central component; and 2) a medial occipital cluster characterized by more isolated, slower (<2 Hz) and smaller waves that were not associated with rapid eye movements, displayed a negative correlation with gamma activity and were also found in NREM sleep. Thus, low-frequency oscillations are an integral part of REM sleep in humans, and the two identified subtypes (sawtooth and medial-occipital slow waves) may reflect distinct generation mechanisms and functional roles. Sawtooth waves, which are exclusive to REM sleep, share many characteristics with ponto-geniculo-occipital (PGO) waves described in animals and may represent the human equivalent or a closely related event while medio-occipital slow waves appear similar to NREM sleep slow waves.',\n",
       "   'published': '10.1523/JNEUROSCI.2298-18.2019',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397018',\n",
       "   'title': 'Farmers perceptions on the causes of cassava root bitterness in konzo-affected areas of Mtwara Region, Tanzania',\n",
       "   'authors': 'Imakumbili, M. L. E.; Semu, E.; Semoka, J. M. R.; Abass, A.; Mkamilo, G.',\n",
       "   'author_corresponding': 'Matema L. E. Imakumbili',\n",
       "   'author_corresponding_institution': 'Sokoine University of Agriculture',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'pharmacology and toxicology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397018.source.xml',\n",
       "   'abstract': 'The agronomic factors influencing increased cyanogenic glucoside levels, particularly in bitter cassava varieties during periods without water stress, in areas where konzo (a cassava cyanide related paralytic disorder also called spastic paraparesis) persists, are hardly known. However, through their assessment of bitter taste, farmers may have noticed factors unrelated to water stress and variety type that additionally influence cassava root cyanogenic glucoside content in these environments. Bitterness in cassava is usually associated with high cyanogenic glucoside levels. Using some konzo-affected areas in Mtwara region of Tanzania as a case study, a survey was thus carried out to identify the factors, hitherto overlooked, that may additionally influence cyanogenic glucoside levels in cassava. A total of 120 farmers were interviewed. A number of factors unrelated to water stress and variety type that could be additionally influencing cyanogenic glucoside production in cassava plants were mentioned. The mentioned factors included nutrient poor soils, plant age at harvest, weeds, piecemeal harvesting, and branch pruning; the factors, respectively, constituted 14.2%, 7.5%, 0.8%, 0.8%, and 0.8% of the total responses given. The revealed factors constitute permanent environmental characteristics and commonly used crop management practices by farmers living in konzo-prone Mtwara region of Tanzania that could be additionally resulting in high cyanogenic glucoside levels in cassava, regardless of water stress.',\n",
       "   'published': '10.1371/journal.pone.0215527',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397208',\n",
       "   'title': 'Mutual synchronization pattern as a complementary indicator of the short-term blood pressure - heart rate feedback regulation activity',\n",
       "   'authors': 'Pyko, N. S.; Pyko, S. A.; Markelov, O. A.; Mamontov, O. V.; Bogachev, M.',\n",
       "   'author_corresponding': 'Mikhail  Bogachev',\n",
       "   'author_corresponding_institution': 'St. Petersburg Electrotechnical University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'physiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397208.source.xml',\n",
       "   'abstract': 'We suggest a complementary indicator of the blood pressure - heart rate feedback regulation based on their synchronization pattern assessed by Hilbert transform. We determine the synchronization coefficient Sync as the fraction of time fragments where the standard deviation of the differences between instantaneous phases of blood pressure and pulse intervals are below a certain threshold. While BRS characterizes the intensity of the pulse intervals response to the blood pressure changes during observed feedback responses, the Sync likely indicates how often such responses are activated in the first place. Data from 95 tilt test records indicate that in both healthy subjects and patients with moderate autonomic dysfunction BRS and Sync are typically reciprocal suggesting that low intensity of the feedback responses characterized by low BRS is rather compensated by their more frequent activation indicated by higher Sync. In contrast, in diabetes patients with autonomic neuropathy BRS and Sync are positively correlated likely indicating the breakdown of this compensation in some of the diabetic patients. Therefore we suggest that Sync could be used as an additional indicator of the blood pressure - heart rate feedback regulation activity that is complementary to the widely used baroreflex sensitivity (BRS).',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396697',\n",
       "   'title': 'Systems therapeutics analyses identify genomic signatures defining responsiveness to allopurinol and combination therapy for lung cancer',\n",
       "   'authors': 'Tavassoly, I.; Hu, Y.; Zhao, S.; Mariottini, C.; Boran, A.; Chen, Y.; Li, L.; Tolentino, R. E.; Jayaraman, G.; Goldfarb, J.; Gallo, J.; Iyengar, R.',\n",
       "   'author_corresponding': 'Ravi  Iyengar',\n",
       "   'author_corresponding_institution': 'Icahn School of Medicine at Mount Sinai',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'systems biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396697.source.xml',\n",
       "   'abstract': 'The ability to predict responsiveness to drugs in individual patients is limited. We hypothesized that integrating molecular information from databases would yield predictions that could be experimentally tested to develop genomic signatures for sensitivity or resistance to specific drugs. We analyzed TCGA data for lung adenocarcinoma (LUAD) patients and identified a subset where xanthine dehydrogenase expression correlated with decreased survival. We tested allopurinol, a FDA approved drug that inhibits xanthine dehydrogenase on a library of human Non Small Cell Lung Cancer (NSCLC) cell lines from CCLE and identified sensitive and resistant cell lines. We utilized the gene expression profiles of these cell lines to identify six-gene signatures for allopurinol sensitive and resistant cell lines. Network building and analyses identified JAK2 as an additional target in allopurinol-resistant lines. Treatment of resistant cell lines with allopurinol and CEP-33779 (a JAK2 inhibitor) resulted in cell death. The effectiveness of allopurinol alone or allopurinol and CEP-33779 were verified in vivo using tumor formation in NCR-nude mice. We utilized the six-gene signatures to predict five additional allopurinolsensitive NSCLC lines, and four allopurinol-resistant lines susceptible to combination therapy. We found that drug treatment of all cell lines yielded responses as predicted by the genomic signatures. We searched the library of patient derived NSCLC tumors from Jackson Laboratory to identify tumors that would be predicted to be sensitive or resistant to allopurinol treatment. Both patient derived tumors predicted to be allopurinol sensitive showed the predicted sensitivity, and the predicted resistant tumors were sensitive to combination therapy. These data indicate that we can use integrated molecular information from cancer databases to predict drug responsiveness in individual patients and thus enable precision medicine.',\n",
       "   'published': '10.1002/1878-0261.12521',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397133',\n",
       "   'title': 'DocClustering: An implementation of the novel PS-Document Clustering algorithm',\n",
       "   'authors': 'Doerpinghaus, J.; Schaaf, S.; Jacobs, M.',\n",
       "   'author_corresponding': 'Jens  Doerpinghaus',\n",
       "   'author_corresponding_institution': 'Fraunhofer Institute for Algorithms and Scientific Computing SCAI',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'confirmatory results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'bioinformatics',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397133.source.xml',\n",
       "   'abstract': 'AO_SCPCAPBSTRACTC_SCPCAPDocument clustering is widely used in science for data retrieval and organisation. DocClustering is developed to include and use a novel algorithm called PS-Document Clustering that has been first introduced in 2017. This method combines approaches of graph theory with state of the art NLP-technologies. This new heuristic has been shown to be superior to conventional algorithms and it provides - given a suiting similarity measure - a more accurate clustering on biological and medical data.\\\\n\\\\nSince the application is written for research on biomedical literature, interfaces for PubMed and SCAIView are available. In this brief report the source code as well as a short overview about the new features, novel heuristics and approaches are provided.\\\\n\\\\nThe software can be obtained from the authors or directly downloaded from GitHub, see https://github.com/jd-s/DocClustering.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397034',\n",
       "   'title': 'Little variation in the morphology of the atria across 13 orders of birds',\n",
       "   'authors': 'Kroneman, J.; Faber, J.; Wolschrijn, C.; Christoffels, V.; Jensen, B.',\n",
       "   'author_corresponding': 'Bjarke  Jensen',\n",
       "   'author_corresponding_institution': 'University of Amsterdam',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nd',\n",
       "   'category': 'evolutionary biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397034.source.xml',\n",
       "   'abstract': 'Mammals and birds acquired high performance hearts and endothermy during their independent evolution from amniotes with many reptile characters. A literature review shows that the variation in atrial morphology is greater in mammals than in ectothermic reptiles. We therefore hypothesized that the transition from ectothermy to endothermy associated with greater variation in cardiac structure. We tested the hypothesis in birds, by assessing the variation in 15 characters in hearts from 13 orders of birds. Hearts were assessed by gross morphology and histology, and we focused on the atria as they have multiple features that lend themselves to quantification. We found bird hearts to have multiple features in common with ectothermic reptiles (synapomorphies), for instance the presence of three sinus horns. Convergent features were shared with crocodylians and mammals, such as the cranial offset of the left atrioventricular junction. Other convergent features like the compact organization of the atrial walls were shared with mammals only. Sinus myocardium expressing Isl1 was node-like (Mallard), thickened (chicken), or anatomically indistinct from surrounding myocardium (Lesser redpoll). Some features were distinctively avian (apomorphies), including the presence of a left atrial antechamber, and the ventral merger of the left and right atrium, which was found in parrots and passerine birds. Most features, however, exhibited little variation. For instance, there were always three systemic veins and two pulmonary veins, whereas among mammals there are 2-3 and 1-7, respectively. Our findings suggest that the transition to high cardiac performance does not necessarily lead to greater variation in cardiac structure.',\n",
       "   'published': '10.1002/jmor.20952',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396689',\n",
       "   'title': 'Defining the core essential genome of Pseudomonas aeruginosa',\n",
       "   'authors': 'Poulsen, B. E.; Yang, R.; Clatworthy, A. E.; White, T.; Osmulski, S. J.; Li, L.; Penaranda, C.; Shoresh, N.; Hung, D. T.',\n",
       "   'author_corresponding': 'Deborah T Hung',\n",
       "   'author_corresponding_institution': 'Massachusetts General Hospital',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396689.source.xml',\n",
       "   'abstract': 'Genomics offered the promise of transforming antibiotic discovery by revealing many new essential genes as good targets, but the results fell short of the promise. It is becoming clear that a major limitation was that essential genes for a bacterial species were often defined based on a single or limited number of strains grown under a single or limited number of in vitro laboratory conditions. In fact, the essentiality of a gene can depend on both genetic background and growth condition. We thus developed a strategy for more rigorously defining the core essential genome of a bacterial species by studying many pathogen strains and growth conditions. We assessed how many strains must be examined to converge on a set of core essential genes for a species. We used transposon insertion sequencing (Tn-Seq) to define essential genes in nine strains of Pseudomonas aeruginosa on five different media and developed a novel statistical model, FiTnEss, to classify genes as essential versus non-essential across all strain-media combinations. We defined a set of 321 core essential genes, representing 6.6% of the genome. We determined that analysis of 4 strains was typically sufficient in P. aeruginosa to converge on a set of core essential genes likely to be essential across the species across a wide range of conditions relevant to in vivo infection, and thus to represent attractive targets for novel drug discovery.',\n",
       "   'published': '10.1073/pnas.1900570116',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/396812',\n",
       "   'title': 'Altered Bacteria-Fungi Inter-Kingdom Network in Gut of Ankylosing Spondylitis Patients',\n",
       "   'authors': 'li, M.; Dai, B.; Tang, Y.; Lei, L.; Li, N.; Liu, C.; Ge, T.; Zhang, L.; Xu, Y.; Hu, Y.; Li, P.; Zhang, Y.; Yuan, J.; Li, X.',\n",
       "   'author_corresponding': 'Xia  Li',\n",
       "   'author_corresponding_institution': 'Dalian Medical University',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_no',\n",
       "   'category': 'microbiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/396812.source.xml',\n",
       "   'abstract': 'Intestinal bacterial dysbiosis has been increasingly linked to Ankylosing Spondylitis (AS), which is a prototypic and best studied subtype of Spondyloarthritis (SpA). Fungi and bacteria coexist in human gut and interact with each other, although they have been shown to contribute actively to health or diseases, no studies have investigated whether fungal microbiota in AS patients is perturbed. In this study, fecal samples of 22 AS patients, with clinical and radiographic assessments, and 16 healthy controls (HCs) were collected to systematically characterize the gut microbiota and mycobiota in AS patients by 16S rDNA and ITS2-based DNA sequencing. The relationships between therapeutic regimens, disease activity, radiographic damage of AS and gut micro/mycobiome were investigated. Our results showed a distinct mycobiota pattern in AS in addition to microbiota dysbiosis. The gut mycobiome of AS patients was characterized by higher taxonomic levels of Ascomycota, especially the class of Dothideomycetes, and decreased abundance of Basidiomycota, which was mainly contributed by the decease of Agaricales. Compared to HCs, changing of the ITS2/16S biodiversity ratio, and bacteria-fungi interkingdom network were observed in AS patients. Alteration of gut mycobiota was associated with different therapeutic regimens, disease activity, as well as different degrees of radiographic damage. Moreover, we unraveled a disease-specific interkingdom network alteration in AS. Finally, we also identified some trends suggesting that different therapeutic regimens may induce changing of both bacterial and fungal microbiota in AS.\\\\n\\\\nIMPORTANCEHuman gut is colonized by diverse fungi (mycobiome), and they have long been suspected in the pathogenesis of Spondyloarthritis (SpA). Our study unraveled a disease-specific interkingdom network alteration in AS, suggesting that fungi, or the interkingdom interactions between bacteria and fungi, may play an essential role in AS development. However, limited by sample size and indeep mechanism studies, further large scale investigations on the characterization of gut mycobiome in AS patients are needed to form a foundation for research into the relationship between mycobiota dysbiosis and AS development.',\n",
       "   'published': 'NA',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397216',\n",
       "   'title': 'Senescent, dysfunctional human cardiac progenitor cells (CPCs) accumulate in the aged heart and elimination of senescent cells enhances CPC activation and cardiomyocyte proliferation in aged mice',\n",
       "   'authors': 'Lewis-McDougall, F. C.; Ruchaya, P. J.; Domenjo-Vila, E.; Teoh, T. S.; Prata, L.; Cottle, B. J.; Clark, J. E.; Punjabi, P. P.; Awad, W.; Torella, D.; Tchkonia, T.; Kirkland, J. L.; Ellison-Hughes, G. M.',\n",
       "   'author_corresponding': 'Georgina M Ellison-Hughes',\n",
       "   'author_corresponding_institution': 'Kings College London',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'physiology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397216.source.xml',\n",
       "   'abstract': 'Aging leads to increased cellular senescence and is associated with decreased potency of tissue-specific stem/progenitor cells. Here we have done an extensive analysis of cardiac progenitor cells (CPCs) isolated from human subjects with cardiovascular disease (n=119), aged 32-86 years. In aged subjects (>74 years old) over half of CPCs are senescent (p16INK4A, SA-{beta}-gal, DNA damage {gamma}H2AX, telomere length, Senescence-Associated Secretory Phenotype (SASP)), unable to replicate, differentiate, regenerate or restore cardiac function following transplantation into the infarcted heart. SASP factors secreted by senescent CPCs renders otherwise healthy CPCs to senescence. Elimination of senescent CPCs using senolytics abrogates the SASP and its debilitative effect in vitro. Global elimination of senescent cells in aged mice (INK-ATTAC or wildtype mice treated with D+Q senolytics) in vivo activates resident CPCs (0.23{+/-}0.06% vs. 0.01{+/-}0.01% vehicle; p<0.05) and increased the number of small, proliferating Ki67-, EdU-positive cardiomyocytes (0.25{+/-}0.07% vs. 0.03{+/-}0.03% vehicle; p<0.05). Therapeutic approaches that eliminate senescent cells may alleviate cardiac deterioration with aging and rejuvenate the regenerative capacity of the heart.',\n",
       "   'published': '10.1111/acel.12931',\n",
       "   'server': 'biorxiv'},\n",
       "  {'doi': '10.1101/397315',\n",
       "   'title': 'Plug-and-Play Metabolic Transducers Expand the Chemical Detection Space of Cell-Free Biosensors',\n",
       "   'authors': 'Voyvodic, P. L.; Pandi, A.; Koch, M.; Faulon, J.-L.; Bonnet, J.',\n",
       "   'author_corresponding': 'Jerome  Bonnet',\n",
       "   'author_corresponding_institution': 'Centre de Biochimie Structurale, INSERM U1054, CNRS UMR5048, University of Montpellier',\n",
       "   'date': '2018-08-21',\n",
       "   'version': '1',\n",
       "   'type': 'new results',\n",
       "   'license': 'cc_by_nc_nd',\n",
       "   'category': 'synthetic biology',\n",
       "   'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/397315.source.xml',\n",
       "   'abstract': 'Cell-free transcription-translation systems have great potential for biosensing, yet the range of detectable chemicals is limited. Here we provide a framework to expand the range of molecules detectable by cell-free biosensors by combining synthetic metabolic cascades with transcription factor-based networks. These hybrid cell-free biosensors are highly-sensitive and have a fast response and high-dynamic range. This work provides a foundation to engineer modular cell-free biosensors tailored for many applications.',\n",
       "   'published': '10.1038/s41467-019-09722-9',\n",
       "   'server': 'biorxiv'}]}"
      ]
     },
     "execution_count": 75,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "request_API = requests.get(\"https://api.biorxiv.org/details/biorxiv/2018-08-21/2018-08-28/101\")\n",
    "data = request_API.text\n",
    "json_info = json.loads(data)\n",
    "json_info"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "268bf080",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "630"
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "total = json_info[\"messages\"][0][\"total\"]\n",
    "total"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 85,
   "id": "68b9edbb",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "0\n",
      "100\n",
      "200\n",
      "300\n",
      "400\n",
      "500\n",
      "600\n"
     ]
    }
   ],
   "source": [
    "total = json_info[\"messages\"][0][\"total\"]\n",
    "current_cursor = 0\n",
    "journal_list = []\n",
    "\n",
    "while (current_cursor <= total):\n",
    "    request_API = requests.get(f\"https://api.biorxiv.org/details/biorxiv/2018-08-21/2018-08-28/{current_cursor}\")\n",
    "    print(current_cursor)\n",
    "        \n",
    "    data = request_API.text\n",
    "    json_info = json.loads(data)\n",
    "    \n",
    "    for journal in response_info[\"collection\"]:\n",
    "        journal_list.append([journal[\"doi\"], journal[\"title\"], journal[\"authors\"],\n",
    "                             journal[\"author_corresponding\"], \n",
    "                             journal[\"author_corresponding_institution\"],\n",
    "                             journal[\"date\"], journal[\"version\"], journal[\"type\"],\n",
    "                             journal[\"category\"], journal[\"jatsxml\"], journal[\"published\"]])\n",
    "        \n",
    "    current_cursor += 100"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 86,
   "id": "460f54dd",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Corresponding Authors</th>\n",
       "      <th>Institution</th>\n",
       "      <th>Date</th>\n",
       "      <th>Version</th>\n",
       "      <th>Type</th>\n",
       "      <th>Category</th>\n",
       "      <th>Xml</th>\n",
       "      <th>Published</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/066688</td>\n",
       "      <td>Cross-scale dynamics and the evolutionary emer...</td>\n",
       "      <td>Schreiber, S. J.; Ke, R.; Loverdo, C.; Park, M...</td>\n",
       "      <td>Sebastian J. Schreiber</td>\n",
       "      <td>University of California, Davis</td>\n",
       "      <td>2018-08-21</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>epidemiology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1093/ve/veaa105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/079954</td>\n",
       "      <td>iCARE: An R Package to Build, Validate and App...</td>\n",
       "      <td>Pal Choudhury, P.; Maas, P.; Wilcox, A.; Wheel...</td>\n",
       "      <td>Nilanjan  Chatterjee</td>\n",
       "      <td>Johns Hopkins University</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>bioinformatics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1371/journal.pone.0228198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/093120</td>\n",
       "      <td>Darwinian selection of host and bacteria suppo...</td>\n",
       "      <td>Osmanovic, D.; Kessler, D. A.; Rabin, Y.; Soen...</td>\n",
       "      <td>Yoav  Soen</td>\n",
       "      <td>Weizmann Institute of Science</td>\n",
       "      <td>2018-08-21</td>\n",
       "      <td>5</td>\n",
       "      <td>new results</td>\n",
       "      <td>evolutionary biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1186/s13062-018-0224-7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/097816</td>\n",
       "      <td>Voltage-gated Proton Channel Hv1 Supports Insu...</td>\n",
       "      <td>Pang, H.; Wang, X.; Xi, W.; Zhao, Q.; Zhang, S...</td>\n",
       "      <td>Shu Jie  Li</td>\n",
       "      <td>Nankai University</td>\n",
       "      <td>2018-08-24</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>cell biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/130989</td>\n",
       "      <td>On The Peopling Of The Americas: Molecular Evi...</td>\n",
       "      <td>Yuan, D.; Huang, S.</td>\n",
       "      <td>Shi  Huang</td>\n",
       "      <td>Central South University</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>evolutionary biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>695</th>\n",
       "      <td>10.1101/361840</td>\n",
       "      <td>Instability in 18S and 5.8S rDNA copy numbers ...</td>\n",
       "      <td>Ligerman, C. B.; Baird, L.; Leppert, M. F.; Ca...</td>\n",
       "      <td>Richard M Cawthon</td>\n",
       "      <td>University of Utah, Dept. of Human Genetics</td>\n",
       "      <td>2018-08-28</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>genomics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>696</th>\n",
       "      <td>10.1101/363622</td>\n",
       "      <td>New tools for diet analyses: nanopore sequenci...</td>\n",
       "      <td>Pearman, W.; Smith, A. N. H.; Breckell, G.; Da...</td>\n",
       "      <td>Olin K. Silander</td>\n",
       "      <td>Institute of Natural and Mathematical Sciences...</td>\n",
       "      <td>2018-08-28</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>ecology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>697</th>\n",
       "      <td>10.1101/363077</td>\n",
       "      <td>Systems Biology Understanding of the Effects o...</td>\n",
       "      <td>Ge, W.; Jakobsson, E.</td>\n",
       "      <td>Eric  Jakobsson</td>\n",
       "      <td>University of Illinois at Urbana-Champaign</td>\n",
       "      <td>2018-08-24</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>cancer biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.3389/fonc.2019.00296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>698</th>\n",
       "      <td>10.1101/364331</td>\n",
       "      <td>Resource limitation modulates the fate of diss...</td>\n",
       "      <td>Vuono, D. C.; Read, R. W.; Hemp, J.; Sullivan,...</td>\n",
       "      <td>Joseph J Grzymski</td>\n",
       "      <td>Desert Research Institute</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>4</td>\n",
       "      <td>new results</td>\n",
       "      <td>microbiology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.3389/fmicb.2019.00003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>699</th>\n",
       "      <td>10.1101/364190</td>\n",
       "      <td>Intragenomic Conflict over Bet-Hedging</td>\n",
       "      <td>Wilkins, J. F.; Bhattacharya, T.</td>\n",
       "      <td>Jon F Wilkins</td>\n",
       "      <td>Ronin Institute</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>evolutionary biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1098/rstb.2018.0142</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>700 rows × 11 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "                DOI                                              Title  \\\n",
       "0    10.1101/066688  Cross-scale dynamics and the evolutionary emer...   \n",
       "1    10.1101/079954  iCARE: An R Package to Build, Validate and App...   \n",
       "2    10.1101/093120  Darwinian selection of host and bacteria suppo...   \n",
       "3    10.1101/097816  Voltage-gated Proton Channel Hv1 Supports Insu...   \n",
       "4    10.1101/130989  On The Peopling Of The Americas: Molecular Evi...   \n",
       "..              ...                                                ...   \n",
       "695  10.1101/361840  Instability in 18S and 5.8S rDNA copy numbers ...   \n",
       "696  10.1101/363622  New tools for diet analyses: nanopore sequenci...   \n",
       "697  10.1101/363077  Systems Biology Understanding of the Effects o...   \n",
       "698  10.1101/364331  Resource limitation modulates the fate of diss...   \n",
       "699  10.1101/364190             Intragenomic Conflict over Bet-Hedging   \n",
       "\n",
       "                                               Authors  \\\n",
       "0    Schreiber, S. J.; Ke, R.; Loverdo, C.; Park, M...   \n",
       "1    Pal Choudhury, P.; Maas, P.; Wilcox, A.; Wheel...   \n",
       "2    Osmanovic, D.; Kessler, D. A.; Rabin, Y.; Soen...   \n",
       "3    Pang, H.; Wang, X.; Xi, W.; Zhao, Q.; Zhang, S...   \n",
       "4                                  Yuan, D.; Huang, S.   \n",
       "..                                                 ...   \n",
       "695  Ligerman, C. B.; Baird, L.; Leppert, M. F.; Ca...   \n",
       "696  Pearman, W.; Smith, A. N. H.; Breckell, G.; Da...   \n",
       "697                              Ge, W.; Jakobsson, E.   \n",
       "698  Vuono, D. C.; Read, R. W.; Hemp, J.; Sullivan,...   \n",
       "699                   Wilkins, J. F.; Bhattacharya, T.   \n",
       "\n",
       "      Corresponding Authors  \\\n",
       "0    Sebastian J. Schreiber   \n",
       "1      Nilanjan  Chatterjee   \n",
       "2                Yoav  Soen   \n",
       "3               Shu Jie  Li   \n",
       "4                Shi  Huang   \n",
       "..                      ...   \n",
       "695       Richard M Cawthon   \n",
       "696        Olin K. Silander   \n",
       "697         Eric  Jakobsson   \n",
       "698       Joseph J Grzymski   \n",
       "699           Jon F Wilkins   \n",
       "\n",
       "                                           Institution        Date Version  \\\n",
       "0                      University of California, Davis  2018-08-21       2   \n",
       "1                             Johns Hopkins University  2018-08-23       2   \n",
       "2                        Weizmann Institute of Science  2018-08-21       5   \n",
       "3                                    Nankai University  2018-08-24       3   \n",
       "4                             Central South University  2018-08-23       3   \n",
       "..                                                 ...         ...     ...   \n",
       "695        University of Utah, Dept. of Human Genetics  2018-08-28       3   \n",
       "696  Institute of Natural and Mathematical Sciences...  2018-08-28       2   \n",
       "697         University of Illinois at Urbana-Champaign  2018-08-24       2   \n",
       "698                          Desert Research Institute  2018-08-23       4   \n",
       "699                                    Ronin Institute  2018-08-23       3   \n",
       "\n",
       "            Type              Category  \\\n",
       "0    new results          epidemiology   \n",
       "1    new results        bioinformatics   \n",
       "2    new results  evolutionary biology   \n",
       "3    new results          cell biology   \n",
       "4    new results  evolutionary biology   \n",
       "..           ...                   ...   \n",
       "695  new results              genomics   \n",
       "696  new results               ecology   \n",
       "697  new results        cancer biology   \n",
       "698  new results          microbiology   \n",
       "699  new results  evolutionary biology   \n",
       "\n",
       "                                                   Xml  \\\n",
       "0    https://www.biorxiv.org/content/early/2018/08/...   \n",
       "1    https://www.biorxiv.org/content/early/2018/08/...   \n",
       "2    https://www.biorxiv.org/content/early/2018/08/...   \n",
       "3    https://www.biorxiv.org/content/early/2018/08/...   \n",
       "4    https://www.biorxiv.org/content/early/2018/08/...   \n",
       "..                                                 ...   \n",
       "695  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "696  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "697  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "698  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "699  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "\n",
       "                        Published  \n",
       "0              10.1093/ve/veaa105  \n",
       "1    10.1371/journal.pone.0228198  \n",
       "2       10.1186/s13062-018-0224-7  \n",
       "3                              NA  \n",
       "4                              NA  \n",
       "..                            ...  \n",
       "695                            NA  \n",
       "696                            NA  \n",
       "697       10.3389/fonc.2019.00296  \n",
       "698      10.3389/fmicb.2019.00003  \n",
       "699        10.1098/rstb.2018.0142  \n",
       "\n",
       "[700 rows x 11 columns]"
      ]
     },
     "execution_count": 86,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "journal_df = pd.DataFrame(data=journal_list,\n",
    "                         columns=[\"DOI\", \"Title\", \"Authors\", \"Corresponding Authors\",\n",
    "                                  \"Institution\",\n",
    "                                  \"Date\", \"Version\", \"Type\", \"Category\", \"Xml\", \"Published\"])\n",
    "journal_df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 66,
   "id": "cda8f700",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['new results', 'confirmatory results'], dtype=object)"
      ]
     },
     "execution_count": 66,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "journal_df[\"Type\"].unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 80,
   "id": "1d624740",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Corresponding Authors</th>\n",
       "      <th>Institution</th>\n",
       "      <th>Date</th>\n",
       "      <th>Version</th>\n",
       "      <th>Type</th>\n",
       "      <th>Category</th>\n",
       "      <th>Xml</th>\n",
       "      <th>Published</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [DOI, Title, Authors, Corresponding Authors, Institution, Date, Version, Type, Category, Xml, Published]\n",
       "Index: []"
      ]
     },
     "execution_count": 80,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "journal_df.loc[journal_df[\"DOI\"] == \"10.1101/367060\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 27,
   "id": "b5e7654b",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[{'doi': '10.1101/066688',\n",
       "  'title': 'Cross-scale dynamics and the evolutionary emergence of infectious diseases',\n",
       "  'authors': 'Schreiber, S. J.; Ke, R.; Loverdo, C.; Park, M.; Ahsan, P.; Lloyd-Smith, J. O.',\n",
       "  'author_corresponding': 'Sebastian J. Schreiber',\n",
       "  'author_corresponding_institution': 'University of California, Davis',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'epidemiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/066688.source.xml',\n",
       "  'abstract': 'AO_SCPLOWBSTRACTC_SCPLOWWhen emerging pathogens encounter new host species for which they are poorly adapted, they must evolve to escape extinction. Pathogens experience selection on traits at multiple scales, including replication rates within host individuals and transmissibility between hosts. We analyze a stochastic model linking pathogen growth and competition within individuals to transmission between individuals. Our analysis reveals a new factor, the cross-scale reproductive number of a mutant virion, that quantifies how quickly mutant strains increase in frequency when they initially appear in the infected host population. This cross-scale reproductive number combines with viral mutation rates, single-strain reproductive numbers, and transmission bottleneck width to determine the likelihood of evolutionary emergence, and whether evolution occurs swiftly or gradually within chains of transmission. We find that wider transmission bottlenecks facilitate emergence of pathogens with short-term infections, but hinder emergence of pathogens exhibiting cross-scale selective conflict and long-term infections. Our results provide a framework to advance the integration of laboratory, clinical and field data in the context of evolutionary theory, laying the foundation for a new generation of evidence-based risk assessment of emergence threats.',\n",
       "  'published': '10.1093/ve/veaa105',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/079954',\n",
       "  'title': 'iCARE: An R Package to Build, Validate and Apply Absolute Risk Models',\n",
       "  'authors': 'Pal Choudhury, P.; Maas, P.; Wilcox, A.; Wheeler, W.; Brook, M.; Check, D.; Garcia-Closas, M.; Chatterjee, N.',\n",
       "  'author_corresponding': 'Nilanjan  Chatterjee',\n",
       "  'author_corresponding_institution': 'Johns Hopkins University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/079954.source.xml',\n",
       "  'abstract': 'This report describes a R package, called the Individualized Coherent Absolute Risk Estimation (iCARE) tool, that allows researchers to build and evaluate models for absolute risk and apply them to estimate an individuals risk of developing disease during a specified time interval based on a set of user defined input parameters. An attractive feature of the software is that it gives users flexibility to update models rapidly based on new knowledge on risk factors and tailor models to different populations by specifying three input arguments: (1) a model for relative risk, (2) an age-specific disease incidence rate, (3) the distribution of risk factors for the population of interest. The tool can handle missing information on risk factors for individuals for whom risks are to be predicted using a coherent approach where all estimates are derived from a single model after appropriate model averaging. The software allows single nucleotide polymorphisms (SNPs) to be incorporated into the model using published odds ratios and allele frequencies. The validation component of the software implements the methods for evaluation of model calibration, discrimination and risk-stratification based on independent validation datasets. We provide an illustration of the utility of iCARE for building, validating and applying absolute risk models using breast cancer as an example.',\n",
       "  'published': '10.1371/journal.pone.0228198',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/093120',\n",
       "  'title': 'Darwinian selection of host and bacteria supports emergence of Lamarckian-like adaptation of the system as a whole',\n",
       "  'authors': 'Osmanovic, D.; Kessler, D. A.; Rabin, Y.; Soen, Y.',\n",
       "  'author_corresponding': 'Yoav  Soen',\n",
       "  'author_corresponding_institution': 'Weizmann Institute of Science',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '5',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/093120.source.xml',\n",
       "  'abstract': 'BackgroundThe relatively fast selection of symbiotic bacteria within hosts and the potential transmission of these bacteria across generations of hosts raise the question of whether interactions between host and bacteria support emergent adaptive capabilities beyond those of germ-free hosts.\\\\n\\\\nResultsTo investigate possibilities for emergent adaptations that may distinguish composite host-microbiome systems from germ-free hosts, we introduce a population genetics model of a host-microbiome system with vertical transmission of bacteria. The host and its bacteria are jointly exposed to a toxic agent, creating a toxic stress that can be alleviated by selection of resistant individuals and by secretion of a detoxification agent (\\\\\"detox\\\\\"). We show that toxic exposure in one generation of hosts leads to selection of resistant bacteria, which in turn, increases the toxic tolerance of the hosts offspring. Prolonged exposure to toxin over many host generations promotes additional form of emergent adaptation due to selection of hosts based on detox capabilities of their bacterial community as a whole (as opposed to properties of individual bacteria).\\\\n\\\\nConclusionsThese findings show that interactions between pure Darwinian selections of host and its bacteria can give rise to emergent adaptive capabilities, including Lamarckian-like adaptation of the host-microbiome system.',\n",
       "  'published': '10.1186/s13062-018-0224-7',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/097816',\n",
       "  'title': 'Voltage-gated Proton Channel Hv1 Supports Insulin Secretion in Pancreatic β Cells through calcium entry, depolarization and intracellular pH regulation',\n",
       "  'authors': 'Pang, H.; Wang, X.; Xi, W.; Zhao, Q.; Zhang, S.; Qin, J.; Lv, J.; Che, Y.; Zuo, W.; Li, S. J.',\n",
       "  'author_corresponding': 'Shu Jie  Li',\n",
       "  'author_corresponding_institution': 'Nankai University',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'cell biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/097816.source.xml',\n",
       "  'abstract': 'Here, we demonstrate that the voltage-gated proton channel Hv1 represents a regulatory mechanism for insulin secretion of pancreatic islet {beta} cell. In vivo, Hv1-de[fi]cient mice display hyperglycemia and glucose intolerance due to reduced insulin secretion, but normal peripheral insulin sensitivity. In vitro, islets of Hv1-de[fi]cient and heterozygous mice, INS-1 (832/13) cells with siRNA-mediated knockdown of Hv1 exhibit a marked defect in glucose- and K+-induced insulin secretion. Hv1 de[fi]ciency decreases both insulin and proinsulin contents, and limits glucose-induced Ca2+ entry and membrane depolarization. Furthermore, loss of Hv1 increases insulin-containing granular pH and decreases cytosolic pH. In addition, histologic studies show a decrease in {beta} cell mass in islets of Hv1-deficient mice. Collectively, our results indicate that Hv1 supports insulin secretion in the {beta} cell by calcium entry, membrane depolarization and intracellular pH regulation.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/130989',\n",
       "  'title': 'On The Peopling Of The Americas: Molecular Evidence For The Paleoamerican And The Solutrean Models',\n",
       "  'authors': 'Yuan, D.; Huang, S.',\n",
       "  'author_corresponding': 'Shi  Huang',\n",
       "  'author_corresponding_institution': 'Central South University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/130989.source.xml',\n",
       "  'abstract': 'Morphological and archaeological studies suggest that the Americas were first occupied by non-Mongoloids with Australo-Melanesian traits (the Paleoamerican model), which was subsequently followed by Southwest Europeans coming in along the pack ice of the North Atlantic Ocean (the Solutrean model) and by East Asians and Siberians arriving by way of the Bering Strait. Past DNA studies, however, have produced different accounts. With a better understanding of genetic diversity, we have now reinterpreted public DNA data. Consistent with our recent finding of a close relationship between South Pacific populations and Denisovans or Neanderthals who were archaic Africans with Eurasian admixtures, the [~]9500 year old Kennewick Man skeleton with Australo-Melanesian affinity from North America was about equally related to Europeans and Africans, least related to East Asians among present-day people, and most related to the [~]42000 year old Neanderthal Mezmaiskaya-2 from Adygea Russia among ancient Eurasian DNAs. The [~]12700 year old Anzick-1 of the Clovis culture was most related to the [~]18720 year old El Miron of the Magdalenian culture in Spain among ancient DNAs. Amerindian mtDNA haplotypes, unlike their Eurasian sister haplotypes, share informative SNPs with Australo-Melanesians, Africans, or Neanderthals. These results suggest a unifying account of informative findings on the settlement of the Americas.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/152215',\n",
       "  'title': 'Range size dynamics can explain why evolutionarily age and diversification rate correlate with contemporary extinction risk in plants',\n",
       "  'authors': 'Tanentzap, A. J.; Igea, J.; Johnston, M. G.; Larcombe, M. J.',\n",
       "  'author_corresponding': 'Andrew J. Tanentzap',\n",
       "  'author_corresponding_institution': 'University of Cambridge',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/152215.source.xml',\n",
       "  'abstract': 'Extinction threatens many species, yet few factors predict this risk across the plant Tree of Life (ToL). Taxon age is one factor that may associate with extinction if occupancy of geographic and adaptive zones varies with time, but evidence for such an association has been equivocal. Age-dependent occupancy can also influence diversification rates and thus extinction risk where new taxa have small range and population sizes. Here we analysed 509 well-sampled genera from across the plant ToL. We found that a greater proportion of species were threatened by extinction in younger and faster-diversifying genera. Repeating our analyses in two large, well-sampled groups, we found that extinction risk increased with evolutionary age in conifer species but not palms. Potential range size decreased in older, non-threatened conifers more strongly than in threatened taxa, suggesting that range size dynamics may explain differing patterns of extinction risk across the ToL with consequences for biodiversity conservation.',\n",
       "  'published': '10.1086/707207',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/166819',\n",
       "  'title': 'Competition for synaptic building blocks shapes synaptic plasticity',\n",
       "  'authors': 'Triesch, J.; Vo, A. D.; Hafner, A.-S.',\n",
       "  'author_corresponding': 'Jochen  Triesch',\n",
       "  'author_corresponding_institution': 'Frankfurt Institute for Advanced Studies',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/166819.source.xml',\n",
       "  'abstract': 'Changes in the efficacies of synapses are thought to be the neurobiological basis of learning and memory. The efficacy of a synapse depends on its current number of neurotransmitter receptors. Recent experiments have shown that these receptors are highly dynamic, moving back and forth between synapses on time scales of seconds and minutes. This suggests spontaneous fluctuations in synaptic efficacies and a competition of nearby synapses for available receptors. Here we propose a mathematical model of this competition of synapses for neurotransmitter receptors from a local dendritic pool. Using minimal assumptions, the model produces a fast multiplicative scaling behavior of synapses. Furthermore, the model explains a transient form of heterosynaptic plasticity and predicts that its amount is inversely related to the size of the local receptor pool. Overall, our model reveals logistical tradeoffs during the induction of synaptic plasticity due to the rapid exchange of neurotransmitter receptors between synapses.',\n",
       "  'published': '10.7554/elife.37836',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/172221',\n",
       "  'title': 'Global expansion and redistribution of Aedes-borne virus transmission risk with climate change',\n",
       "  'authors': 'Ryan, S. J.; Carlson, C. J.; Mordecai, E. A.; Johnson, L. R.',\n",
       "  'author_corresponding': 'Sadie J. Ryan',\n",
       "  'author_corresponding_institution': 'University of Florida',\n",
       "  'date': '2018-08-26',\n",
       "  'version': '4',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'epidemiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/172221.source.xml',\n",
       "  'abstract': 'AbstractForecasting the impacts of climate change on Aedes-borne viruses--especially dengue, chikungunya, and Zika--is a key component of public health preparedness. We apply an empirically parameterized Bayesian transmission model of Aedes-borne viruses for the two vectors Aedes aegypti and Ae. albopictus as a function of temperature to predict cumulative monthly global transmission risk in current climates, and compare with projected risk in 2050 and 2080 based on general circulation models (GCMs). Our results show that if mosquito range shifts track optimal temperatures for transmission (26-29 {degrees}C), we can expect poleward shifts in Aedes-borne virus distributions. However, the differing thermal niches of the two vectors produce different patterns of shifts under climate change. More severe climate change scenarios produce proportionally worse population exposures from Ae. aegypti, but not from Ae. albopictus in the most extreme cases. Expanding risk of transmission from both mosquitoes will likely be a serious problem, even in the short term, for most of Europe; but significant reductions are also expected for Aedes albopictus, most noticeably in southeast Asia and west Africa. Within the next century, nearly a billion people are threatened with new exposure to both Aedes spp. in the worst-case scenario; but massive net losses in risk are noticeable for Ae. albopictus, especially in terms of year-round transmission, marking a global shift towards more seasonal risk across regions. Many other complicating factors (like mosquito range limits and viral evolution) exist, but overall our results indicate that while climate change will lead to both increased and new exposures to vector-borne disease, the most extreme increases in Ae. albopictus transmission are predicted to occur at intermediate climate change scenarios.\\\\n\\\\nAuthor SummaryThe established scientific consensus indicates that climate change will severely exacerbate the risk and burden of Aedes-transmitted viruses, including dengue, chikungunya, Zika, West Nile virus, and other significant threats to global health security. Here, we show that the story is more complicated, first and foremost due to differences between the more heat-tolerant Aedes aegypti and the more heat-limited Ae. albopictus. Almost a billion people could face their first exposure to viral transmission from either mosquito in the worst-case scenario, especially in Europe and high-elevation tropical and subtropical regions. On the other hand, while year-round transmission potential from Ae. aegypti is likely to expand (especially in south Asia and sub-Saharan Africa), Ae. albopictus loses significant ground in the tropics, marking a global shift towards seasonal risk as the tropics eventually become too hot for transmission by Ae. albopictus. Complete mitigation of climate change to a pre-industrial baseline could protect almost a billion people from arbovirus range expansions; but middle-of-the-road mitigation may actually produce the greatest expansion in the potential for viral transmission by Ae. albopictus. In any scenario, mitigating climate change also shifts the burden of both dengue and chikungunya (and potentially other Aedes transmitted viruses) from higher-income regions back onto the tropics, where transmission might otherwise start to be curbed by rising temperatures.',\n",
       "  'published': '10.1371/journal.pntd.0007213',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/173039',\n",
       "  'title': 'Genetic screen suggests an alternative mechanism for azide-mediated inhibition of SecA',\n",
       "  'authors': 'Jamshad, M.; Chandler, R.; Yule, J.; Robinson, A.; Alam, F.; Dunne, K.; Nabi, N.; Henderson, I. R.; Huber, D.',\n",
       "  'author_corresponding': 'Damon  Huber',\n",
       "  'author_corresponding_institution': 'University of Birmingham',\n",
       "  'date': '2018-08-22',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'microbiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/173039.source.xml',\n",
       "  'abstract': 'Sodium azide prevents bacterial growth by inhibiting the activity of SecA, which is required for translocation of proteins across the cytoplasmic membrane. Azide inhibits ATP turnover in vitro, but its mechanism of action in vivo is unclear. To investigate how azide inhibits SecA in cells, we used transposon directed insertion-site sequencing (TraDIS) to screen a library of transposon insertion mutants for mutations that affect the susceptibility of E. coli to azide. Insertions disrupting components of the Sec machinery generally increased susceptibility to azide, but insertions truncating the C-terminal tail (CTT) of SecA decreased susceptibility of E. coli to azide. Treatment of cells with azide caused increased aggregation of the CTT, suggesting that azide disrupts its structure. Analysis of the metal-ion content of the CTT indicated that SecA binds to iron and the azide disrupts the interaction of the CTT with iron. Azide also disrupted binding of SecA to membrane phospholipids, as did alanine substitutions in the metal-coordinating amino acids. Furthermore, treating purified phospholipid-bound SecA with azide in the absence of added nucleotide disrupted binding of SecA to phospholipids. Our results suggest that azide does not inhibit SecA by inhibiting the rate of ATP turnover in vivo. Rather, azide inhibits SecA by causing it to \\\\\"backtrack\\\\\" from the ADP-bound to the ATP-bound conformation, which disrupts the interaction of SecA with the cytoplasmic membrane.\\\\n\\\\nSignificance statementSecA is a bacterial ATPase that is required for the translocation of a subset of secreted proteins across the cytoplasmic membrane. Sodium azide is a well-known inhibitor of SecA, but its mechanism of action in vivo is poorly understood. To investigate this mechanism, we examined the effect of azide on the growth of a library of [~]1 million transposon insertion mutations. Our results suggest that azide causes SecA to backtrack in its ATPase cycle, which disrupts binding of SecA to the membrane and to its metal cofactor, which is iron. Our results provide insight into the molecular mechanism by which SecA drives protein translocation and how this essential biological process can be disrupted.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/176057',\n",
       "  'title': 'Prestimulus Activity in the Cingulo-Opercular Network Predicts Memory for Naturalistic Episodic Experience',\n",
       "  'authors': 'Cohen, N.; Ben-Yakov, A.; Weber, J.; Edelson, M.; Paz, R.; Dudai, Y.',\n",
       "  'author_corresponding': 'Noga  Cohen',\n",
       "  'author_corresponding_institution': 'Columbia University',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/176057.source.xml',\n",
       "  'abstract': 'Human memory is strongly influenced by brain states occurring before an event, yet we know little about the underlying mechanisms. We found that activity in the cingulo-opercular network (including bilateral anterior insula and anterior prefrontal cortex) seconds before an event begins can predict whether this event will subsequently be remembered. We then tested how activity in the cingulo-opercular network shapes memory performance. Our findings indicate that prestimulus cingulo-opercular activity affects memory performance by opposingly modulating subsequent activity in two sets of regions previously linked to encoding and retrieval of episodic information. Specifically, higher prestimulus cingulo-opercular activity was associated with a subsequent increase in activity in temporal regions previously linked to encoding and with a subsequent reduction in activity within a set of regions thought to play a role in retrieval and self-referential processing. Together, these findings suggest that prestimulus attentional states modulate memory for real-life events by enhancing encoding and possibly by dampening interference from competing memory substrates.',\n",
       "  'published': '10.1093/cercor/bhz212',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/195768',\n",
       "  'title': 'How genetic disease risks can be misestimated across global populations',\n",
       "  'authors': 'Kim, M. S.; Patel, K. P.; Teng, A. K.; Berens, A. J.; Lachance, J.',\n",
       "  'author_corresponding': 'Joseph  Lachance',\n",
       "  'author_corresponding_institution': 'Georgia Institute of Technology',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/195768.source.xml',\n",
       "  'abstract': 'BackgroundAccurate assessment of health disparities requires unbiased knowledge of genetic risks in different populations. Unfortunately, most genome-wide association studies use genotyping arrays and European samples. Here, we integrate whole genome sequence data from global populations, results from thousands of GWAS, and extensive computer simulations to identify how genetic disease risks can be misestimated.\\\\n\\\\nResultsIn contrast to null expectations, we find that risk allele frequencies at known disease loci are significantly different for African populations compared to other continents. Strikingly, ancestral risk alleles are found at 9.51% higher frequency in Africa and derived risk alleles are found at 5.40% lower frequency in Africa. By simulating GWAS with different study populations, we find that non-African cohorts yield disease associations that have biased allele frequencies and that African cohorts yield disease associations that are relatively free of bias. We also find empirical evidence that genotyping arrays and SNP ascertainment bias contribute to continental differences in risk allele frequencies. Because of these causes, polygenic risk scores can be grossly misestimated for individuals of African descent. Importantly, continental differences in risk allele frequencies are only moderately reduced if GWAS use whole genome sequences and hundreds of thousands of cases and controls. Finally, comparisons between uncorrected and corrected genetic risk scores reveal the benefits of considering whether risk alleles are ancestral or derived.\\\\n\\\\nConclusionsOur results imply that caution must be taken when extrapolating GWAS results from one population to predict disease risks in another population.',\n",
       "  'published': '10.1186/s13059-018-1561-7',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/196972',\n",
       "  'title': '90-90-90, epidemic control, and ending AIDS: review of global situation and recommendations',\n",
       "  'authors': 'Granich, R.; Williams, B. G.; Gupta, S.; Zuniga, J. M.',\n",
       "  'author_corresponding': 'Reuben  Granich',\n",
       "  'author_corresponding_institution': 'Independent Public Health Consultant',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'epidemiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/196972.source.xml',\n",
       "  'abstract': 'HIV remains a devastating pandemic HIV remains a devastating... Ending AIDS The prevention vs. treatment... Economics of ending AIDS Major innovations accelerating... HIV self-tests Universal treatment regimen Comprehensive community-based... Standardized M and E... Unification of global leadership... Last mile strategies,... Conclusion References Although human immunodeficiency virus (HIV) first came to our attention thirty-seven years ago, the acquired immunodeficiency syndrome (AIDS) which it causes is, without treatment, 100% fatal with devastating consequences for millions of people1. More than 35 million people have already died of ...',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/201137',\n",
       "  'title': 'Transcriptomic Dynamics of a non-coding trinucleotide repeat expansion disorder SCA12 in iPSC derived neuronal cells: signatures of interferon induced response',\n",
       "  'authors': 'Kumar, D.; Dhapola, P.; Hussain, A.; Kutum, R.; Srivastava, A. K.; Mukerji, M.; Mukherjee, O.; Faruq, M.',\n",
       "  'author_corresponding': 'Mohammed  Faruq',\n",
       "  'author_corresponding_institution': 'CSIR- Institute of Genomics and Integrative Biology',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'molecular biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/201137.source.xml',\n",
       "  'abstract': 'Spinocerebellar ataxia type-12 (SCA12) is a neurological disorder that exhibits a unique progressive tremor/ataxia syndrome induced by triplet (CAG) repeat expansion in 5 UTR of PPP2R2B. SCA12 is one of the most prominent SCA-subtype in India and till date no appropriate disease models have been described. Our aim was to establish human iPSC derived neuronal cell lines of SCA12 and study transcriptomic level alterations induced by CAG expansion. For translational application, peripheral blood transcriptomics of SCA12 patients was also performed. Lymphoblastoid cell lines of three SCA12 patients were reprogrammed to iPSCs and then re-differentiated into pan-neuronal lineage. RNA-sequencing based comparative transcriptomics was performed for disease and control cell lineages. Microarray based transcriptomic profiling of peripheral blood of SCA12 patients was performed in a case/control (n=15/9) design. We have successfully created human neuronal cell lines of SCA12 patient as exhibited by their molecular profiling. Differential expression analysis of RNA-Seq data has shown enrichment for type-I interferon signaling and other relevant cellular processes in SCA12-neurons. At the splice-isoform level, we observed an upregulation of expanded CAG containing non-coding transcript of PPP2R2B. Peripheral blood transcriptomics analysis and targeted validation of RNA-Seq data has allowed us to identify inflammatory signatures as potential markers of molecular pathology in SCA12. Our study has allowed us to establish first iPSC based neuronal cell lines of SCA12. We have identified pro-inflammatory signatures in SCA12-neurons suggestive of a dsRNA mediated activation of interferon signaling and that corroborates with the emerging evidence of neuronal atrophy due to neuro-inflammation in common neurodegenerative diseases. This study involved development of an iPSCs derived neuronal cells of SCA12 and look through signatures of neurodegeneration by whole RNA sequencing. This model sheds light upon key role of RNA mediated induced response in Interferon signaling for neurodegeneration.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/201285',\n",
       "  'title': 'Electrophysiological Signature Reveals Laminar Structure of the Porcine Hippocampus',\n",
       "  'authors': 'Ulyanova, A. V.; Koch, P. F.; Cottone, C.; Grovola, M. R.; Browne, K. D.; Russo, R. J.; Adam, C. D.; Smith, D. H.; Chen, H. I.; Johnson, V. E.; Cullen, D. K.; Wolf, J. A.',\n",
       "  'author_corresponding': 'John A Wolf',\n",
       "  'author_corresponding_institution': 'University of Pennsylvania, Corporal Michael J. Crescenz Veterans Affair Medical Center',\n",
       "  'date': '2018-08-26',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/201285.source.xml',\n",
       "  'abstract': 'The hippocampus is integral to working and episodic memory, and is a central region of interest in diseases affecting these processes. Pig models are widely used in translational research, and may provide an excellent bridge between rodents and non-human primates for CNS disease models due to their gyrencephalic neuroanatomy and significant white matter composition. However, the laminar structure of the pig hippocampus has not been well characterized. Therefore, we histologically characterized the dorsal hippocampus of Yucatan miniature pigs and quantified the cytoarchitecture of the hippocampal layers. We then utilized stereotaxis combined with single unit electrophysiological mapping to precisely place multichannel laminar silicon probes into the dorsal hippocampus without the need for image guidance. We used in vivo electrophysiological recordings of simultaneous laminar field potentials and single unit activity in multiple layers of the dorsal hippocampus to physiologically identify and quantify these layers under anesthesia. Consistent with previous reports, we found the porcine hippocampus to have the expected archicortical laminar structure with some anatomical and histological features comparable to the rodent and others to the primate hippocampus. Importantly, we found these distinct features to be reflected in the laminar electrophysiology. This characterization, as well as our electrophysiology-based methodology targeting the porcine hippocampal lamina combined with high channel count silicon probes will allow for analysis of spike-field interactions during normal and disease states in both anesthetized and future awake behaving neurophysiology in this large animal.\\\\n\\\\nSignificance StatementThe hippocampus is central to working and episodic memory and is critically affected by diverse disease processes. In order to investigate hippocampal electrophysiology in translational large animal models, we developed an imaging-free stereotaxis and intraoperative electrophysiology methodology with custom silicon probes to precisely localize probe placement within the hippocampal laminar structure. We report for the first time the profile of single units and local field potentials in the pig dorsal hippocampus and relate them to a histological description. This characterization forms the basis for accessible translational pig models to study diseases of the central nervous system affecting hippocampal circuitry in the large animal gyrencephalic brain, as well as the groundwork for potential awake behaving neurophysiology of the porcine hippocampus.\\\\n\\\\nFunding SourcesThe Department of Veterans Affairs, IK2-RX001479, I01-RX001097. The National Institutes of Health, NINDS R01-NS-101108-01, T32-NS043126. CURE Foundation, Taking Flight Award. DoD ERP CDMRP, W81XWH-16-1-0675.',\n",
       "  'published': '10.1523/eneuro.0102-18.2018',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/213330',\n",
       "  'title': 'The Fission Yeast S-Phase Cyclin Cig2 Can Drive Mitosis',\n",
       "  'authors': 'Magner, M.; Pickering, M.; Keifenheim, D.; Rhind, N.',\n",
       "  'author_corresponding': 'Nicholas  Rhind',\n",
       "  'author_corresponding_institution': 'University of Massachusetts Medical School',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc',\n",
       "  'category': 'cell biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/213330.source.xml',\n",
       "  'abstract': 'Commitment to mitosis is regulated by cyclin-dependent kinase (CDK) activity. In the fission yeast Schizosaccharomyces pombe, the major B-type cyclin, Cdc13, is necessary and sufficient to drive mitotic entry. Furthermore, Cdc13 is also sufficient to drive S phase, demonstrating that a single cyclin can regulate alternating rounds of replication and mitosis and providing the foundation of the quantitative model of CDK function. It has been assumed that Cig2, a B-type cyclin expressed only during S-phase and incapable of driving mitosis in wild-type cells, was specialized for S-phase regulation. Here, we show that Cig2 is capable of driving mitosis. Cig2/CDK activity drives mitotic catastrophe--lethal mitosis in inviably small cells--in cells that lack CDK inhibition by tyrosine-phosphorylation. Moreover, Cig2/CDK can drive mitosis in the absence of Cdc13/CDK activity and constitutive expression of Cig2 can rescue loss of Cdc13 activity. These results demonstrate that in fission yeast, not only can the presumptive M-phase cyclin drive S phase, but the presumptive S-phase cyclin can drive M phase, further supporting the quantitative model of CDK function. Furthermore, these results provide an explanation, previously proposed on the basis of computational analyses, for the surprising observation that cells expressing a single-chain Cdc13-Cdc2 CDK do not require Y15 phosphorylation for viability. Their viability is due to the fact that in such cells, which lack Cig2/CDK complexes, Cdc13/CDK activity is unable to drive mitotic catastrophe.',\n",
       "  'published': '10.1093/genetics/iyaa002',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/216598',\n",
       "  'title': 'The cost of enforcing a marine protected area to achieve ecological targets for the recovery of fish biomass',\n",
       "  'authors': 'Brown, C.; Parker, B.; Ahmadia, G. N.; Ardiwijaya, R.; Purwanto, P.; Game, E. T.',\n",
       "  'author_corresponding': 'Christopher  Brown',\n",
       "  'author_corresponding_institution': 'Griffith University',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nd',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/216598.source.xml',\n",
       "  'abstract': 'Protected areas are the primary management tool for conserving ecosystems, yet their intended outcomes may often be compromised by poaching. Consequently, many protected areas are ineffective  paper parks that contribute little towards conserving ecosystems. Poaching can be prevented through enforcement and engaging with community members so they support protected areas. It is not clear how much needs to be spent on enforcement and engagement to ensure they are frequent enough to be effective at conserving biodiversity. We develop models of enforcement against illegal fishing in marine protected areas. We apply the models to data on fishing rates and fish biomass from a marine protected area in Raja Ampat, Indonesia and explore how frequent enforcement patrols need to be to achieve targets for coral reef fish biomass. Achieving pristine levels of reef fish biomass required almost year-round enforcement of the protected area. Surveillance of the protected area may also be enhanced if local fishers who support the reserve report on poaching. The opportunity for local fishing boats to participate in surveillance was too small for it to have much benefit for total reef fish biomass, which increases slowly. However, specific functional groups of fish have much higher population growth rates and their biomass was predicted to increase markedly with community surveillance. We conclude that budgets for park management must balance the cost of conducting frequent patrols against supporting alternative activities, like education to build community support. Optimized budgets will be much more likely to achieve ecological targets for recovering fish biomasses and will contribute to fiscal sustainability of protected areas.',\n",
       "  'published': '10.1016/j.biocon.2018.09.021',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/221432',\n",
       "  'title': 'GMSE: an R package for generalised management strategy evaluation',\n",
       "  'authors': 'Duthie, A. B.; Cusack, J. J.; Jones, I. L.; Minderman, J.; Nilsen, E. B.; Pozo, R. A.; Rakotonarivo, O. S.; Van Moorter, B.; Bunnefeld, N.',\n",
       "  'author_corresponding': 'Alexander B Duthie',\n",
       "  'author_corresponding_institution': 'University of Stirling',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/221432.source.xml',\n",
       "  'abstract': 'O_LIManagement strategy evaluation (MSE) is a powerful tool for simulating all key aspects of natural resource management under conditions of uncertainty.\\\\nC_LIO_LIWe present the R package GMSE, which applies genetic algorithms to provide a generalised tool for simulating adaptive decision-making management scenarios between stakeholders with competing objectives under complex social-ecological interactions and uncertainty.\\\\nC_LIO_LIGMSE models can be agent-based and spatially explicit, incorporating a high degree of realism through mechanistic modelling of links and feedbacks among stakeholders and with the ecosystem; additionally, user-defined sub-models can also be incorporated as functions into the broader GMSE framework.\\\\nC_LIO_LIWe show how GMSE simulates a social-ecological system using the example of an adaptively managed waterfowl population on an agricultural landscape; simulated waterfowl exploit agricultural land, causing conflict between conservation interests and the interest of food producers maximising their crop yield.\\\\nC_LIO_LIThe R package GMSE is open source under GNU Public License; source code and documents are freely available on GitHub.\\\\nC_LI',\n",
       "  'published': '10.1111/2041-210x.13091',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/222810',\n",
       "  'title': 'Experimental evidence that female rhesus macaques (Macaca mulatta) perceive variation in male facial masculinity',\n",
       "  'authors': 'Rosenfield, K. A.; Semple, S.; Georgiev, A. V.; Maestripieri, D.; Higham, J. P.; Dubuc, C.',\n",
       "  'author_corresponding': 'Kevin A Rosenfield',\n",
       "  'author_corresponding_institution': 'Penn State University',\n",
       "  'date': '2018-08-26',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nd',\n",
       "  'category': 'animal behavior and cognition',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/222810.source.xml',\n",
       "  'abstract': 'Among many primate species, face shape is sexually dimorphic, and male facial masculinity has been proposed to influence female mate choice and male-male competition by signalling competitive ability. However, whether conspecifics pay attention to facial masculinity has only been assessed in humans. In a study of free-ranging rhesus macaques, Macaca mulatta, we used a two-alternative look-time experiment to test whether females perceive male facial masculinity. We presented 107 females with pairs of images of male faces - one with a more masculine shape and one more feminine - and recorded their looking behaviour. Females looked at the masculine face longer than at the feminine face in more trials than predicted by chance. Although there was no overall difference in average look-time between masculine and feminine faces across all trials, females looked significantly longer at masculine faces in a subset of trials for which the within-pair difference in masculinity was most pronounced. Additionally, the proportion of time subjects looked toward the masculine face increased as the within-pair difference in masculinity increased. This study provides evidence that female macaques perceive variation in male facial shape, a necessary condition for intersexual selection to operate on such a trait. It also highlights the potential impact of perceptual thresholds on look-time experiments.',\n",
       "  'published': '10.1098/rsos.181415',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/224089',\n",
       "  'title': 'Habitat complexity and benthic predator-prey interactions in Chesapeake Bay',\n",
       "  'authors': 'Glaspie, C. N.; Seitz, R. D.',\n",
       "  'author_corresponding': 'Cassandra N Glaspie',\n",
       "  'author_corresponding_institution': 'Oregon State University',\n",
       "  'date': '2018-08-25',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/224089.source.xml',\n",
       "  'abstract': 'In Chesapeake Bay, the soft-shell clam Mya arenaria (thin-shelled, deep-burrowing) exhibits population declines when predators are active and persists at low densities. In contrast, the hard clam Mercenaria mercenaria (thick-shelled, shallow-burrowing) has a stable population and age distribution. We examined the potential for habitat and predators to control densities and distributions of bivalves in a field caging experiment (Mya only) and laboratory mesocosm experiments (both species). In the field, clams exposed to predators experienced 76.3% greater mortality as compared to caged individuals, and blue crabs were likely responsible for most of the mortality of juvenile Mya. In mesocosm experiments, Mya had lower survival in sand and seagrass than in shell hash or oyster shell habitats. However, crabs often missed one or more prey in seagrass, shell, and oyster shell habitats. Predator search times and encounter rates declined when prey were at low densities, likely due to the added cost of inefficient foraging; however, this effect was more pronounced for Mya than for Mercenaria. Mercenaria had higher survival than Mya in mesocosm experiments, likely because predators feeding on Mercenaria spent less time foraging than those feeding on Mya. Mya may retain a low-density refuge from predation even with the loss of structurally complex habitats, though a loss of habitat refuge may result in clam densities that are not sustainable. A better understanding of density-dependent predator-prey interactions is necessary to prevent loss of food-web integrity and to conserve marine resources.',\n",
       "  'published': '10.1371/journal.pone.0205162',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/224733',\n",
       "  'title': 'Combining accurate tumour genome simulation with crowd-sourcing to benchmark somatic structural variant detection',\n",
       "  'authors': 'Lee, A. Y.-W.; Ewing, A. D.; Ellrott, K.; Hu, Y.; Houlahan, K. E.; Bare, J. C.; Espiritu, S. M. G.; Huang, V.; Dang, K.; Chong, Z.; Caloian, C.; Yamaguchi, T. N.; ICGC-TCGA DREAM Somatic Mutation Calling Challenge Participants,  ; Kellen, M. R.; Chen, K.; Norman, T. C.; Friend, S. H.; Guinney, J.; Stolovitzky, G.; Haussler, D.; Margolin, A. A.; Stuart, J. M.; Boutros, P. C.',\n",
       "  'author_corresponding': 'Paul C. Boutros',\n",
       "  'author_corresponding_institution': 'Ontario Institute for Cancer Research',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/224733.source.xml',\n",
       "  'abstract': 'BackgroundThe phenotypes of cancer cells are driven in part by somatic structural variants. Structural variants can initiate tumors, enhance their aggressiveness and provide unique therapeutic opportunities. Whole-genome sequencing of tumors can allow exhaustive identification of the specific structural variants present in an individual cancer, facilitating both clinical diagnostics and the discovery of novel mutagenic mechanisms. A plethora of somatic structural variant detection algorithms have been created to enable these discoveries, however there are no systematic benchmarks of them. Rigorous performance evaluation of somatic structural variant detection methods has been challenged by the lack of gold-standards, extensive resource requirements and difficulties arising from the need to share personal genomic information.\\\\n\\\\nResultsTo facilitate structural variant detection algorithm evaluations, we create a robust simulation framework for somatic structural variants by extending the BAMSurgeon algorithm. We then organize and enable a crowd-sourced benchmarking within the ICGC-TCGA DREAM Somatic Mutation Calling Challenge (SMC-DNA). We report here the results of structural variant benchmarking on three different tumors, comprising 204 submissions from 15 teams. In addition to ranking methods, we identify characteristic error-profiles of individual algorithms and general trends across them. Surprisingly, we find that ensembles of analysis pipelines do not always outperform the best individual method, indicating a need for new ways to aggregate somatic structural variant detection approaches.\\\\n\\\\nConclusionsThe synthetic tumors and somatic structural variant detection leaderboards remain available as a community benchmarking resource, and BAMSurgeon is available at https://github.com/adamewing/bamsurgeon.',\n",
       "  'published': '10.1186/s13059-018-1539-5',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/230375',\n",
       "  'title': 'A Very Oil Yellow1 modifier of the Oil Yellow1-N1989 allele uncovers a cryptic phenotypic impact of cis-regulatory variation in maize',\n",
       "  'authors': 'Khangura, R. S.; Marla, S.; Venkata, B. P.; Heller, N.; Johal, G. S.; Dilkes, B. P.',\n",
       "  'author_corresponding': 'Brian P Dilkes',\n",
       "  'author_corresponding_institution': 'Purdue University',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'genetics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/230375.source.xml',\n",
       "  'abstract': 'Forward genetics determines the function of genes underlying trait variation by identifying the change in DNA responsible for changes in phenotype. Detecting phenotypically-relevant variation outside protein coding sequences and distinguishing this from neutral variants is not trivial; partly because the mechanisms by which DNA polymorphisms in the intergenic regions affect gene regulation are poorly understood. Here we utilized a dominant genetic marker with a convenient phenotype to investigate the effect of cis and trans-acting regulatory variation. We performed a forward genetic screen for natural variation that suppress or enhance the semi-dominant mutant allele Oy1-N1989, encoding the magnesium chelatase subunit I of maize. This mutant permits rapid phenotyping of leaf color as a reporter for chlorophyll accumulation, and mapping of natural variation in maize affecting chlorophyll metabolism. We identified a single modifier locus segregating between B73 and Mo17 that was linked to the reporter gene itself, which we call very oil yellow1. Based on the variation in OY1 transcript abundance and genome-wide association data, vey1 is predicted to consist of multiple cis-acting regulatory sequence polymorphisms encoded at the wild-type oy1 alleles. The vey1 allele appears to be a common polymorphism in the maize germplasm that alters the expression level of a key gene in chlorophyll biosynthesis. These vey1 alleles have no discernable impact on leaf chlorophyll in the absence of the Oy1-N1989 reporter. Thus, use of a mutant as a simple and efficient reporter for magnesium chelatase activity resulted in the detection of expression-level polymorphisms not readily visible in the laboratory.',\n",
       "  'published': '10.1534/g3.118.200798',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/237065',\n",
       "  'title': 'Exploring Single-Cell Data with Deep Multitasking Neural Networks',\n",
       "  'authors': 'Amodio, M.; van Dijk, D.; Srinivasan, K.; Chen, W. S.; Mohsen, H.; Moon, K. R.; Campbell, A.; Zhao, Y.; Wang, X.; Venkataswamy, M.; Desai, A.; V., R.; Kumar, P.; Montgomery, R.; Wolf, G.; Krishnaswamy, S.',\n",
       "  'author_corresponding': 'Smita  Krishnaswamy',\n",
       "  'author_corresponding_institution': 'Yale University',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/237065.source.xml',\n",
       "  'abstract': 'Biomedical researchers are generating high-throughput, high-dimensional single-cell data at a staggering rate. As costs of data generation decrease, experimental design is moving towards measurement of many different single-cell samples in the same dataset. These samples can correspond to different patients, conditions, or treatments. While scalability of methods to datasets of these sizes is a challenge on its own, dealing with large-scale experimental design presents a whole new set of problems, including batch effects and sample comparison issues. Currently, there are no computational tools that can both handle large amounts of data in a scalable manner (many cells) and at the same time deal with many samples (many patients or conditions). Moreover, data analysis currently involves the use of different tools that each operate on their own data representation, not guaranteeing a synchronized analysis pipeline. For instance, data visualization methods can be disjoint and mismatched with the clustering method. For this purpose, we present SAUCIE, a deep neural network that leverages the high degree of parallelization and scalability offered by neural networks, as well as the deep representation of data that can be learned by them to perform many single-cell data analysis tasks, all on a unified representation.\\\\n\\\\nA well-known limitation of neural networks is their interpretability. Our key contribution here are newly formulated regularizations (penalties) that render features learned in hidden layers of the neural network interpretable. When large multi-patient datasets are fed into SAUCIE, the various hidden layers contain denoised and batch-corrected data, a low dimensional visualization, unsupervised clustering, as well as other information that can be used to explore the data. We show this capability by analyzing a newly generated 180-sample dataset consisting of T cells from dengue patients in India, measured with mass cytometry. We show that SAUCIE, for the first time, can batch correct and process this 11-million cell data to identify cluster-based signatures of acute dengue infection and create a patient manifold, stratifying immune response to dengue on the basis of single-cell measurements.',\n",
       "  'published': '10.1038/s41592-019-0576-7',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/237065',\n",
       "  'title': 'Exploring Single-Cell Data with Deep Multitasking Neural Networks',\n",
       "  'authors': 'Amodio, M.; van Dijk, D.; Srinivasan, K.; Chen, W. S.; Mohsen, H.; Moon, K. R.; Campbell, A.; Zhao, Y.; Wang, X.; Venkataswamy, M.; Desai, A.; V., R.; Kumar, P.; Montgomery, R.; Wolf, G.; Krishnaswamy, S.',\n",
       "  'author_corresponding': 'Smita  Krishnaswamy',\n",
       "  'author_corresponding_institution': 'Yale University',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/237065.source.xml',\n",
       "  'abstract': 'Biomedical researchers are generating high-throughput, high-dimensional single-cell data at a staggering rate. As costs of data generation decrease, experimental design is moving towards measurement of many different single-cell samples in the same dataset. These samples can correspond to different patients, conditions, or treatments. While scalability of methods to datasets of these sizes is a challenge on its own, dealing with large-scale experimental design presents a whole new set of problems, including batch effects and sample comparison issues. Currently, there are no computational tools that can both handle large amounts of data in a scalable manner (many cells) and at the same time deal with many samples (many patients or conditions). Moreover, data analysis currently involves the use of different tools that each operate on their own data representation, not guaranteeing a synchronized analysis pipeline. For instance, data visualization methods can be disjoint and mismatched with the clustering method. For this purpose, we present SAUCIE, a deep neural network that leverages the high degree of parallelization and scalability offered by neural networks, as well as the deep representation of data that can be learned by them to perform many single-cell data analysis tasks, all on a unified representation.\\\\n\\\\nA well-known limitation of neural networks is their interpretability. Our key contribution here are newly formulated regularizations (penalties) that render features learned in hidden layers of the neural network interpretable. When large multi-patient datasets are fed into SAUCIE, the various hidden layers contain denoised and batch-corrected data, a low dimensional visualization, unsupervised clustering, as well as other information that can be used to explore the data. We show this capability by analyzing a newly generated 180-sample dataset consisting of T cells from dengue patients in India, measured with mass cytometry. We show that SAUCIE, for the first time, can batch correct and process this 11-million cell data to identify cluster-based signatures of acute dengue infection and create a patient manifold, stratifying immune response to dengue on the basis of single-cell measurements.',\n",
       "  'published': '10.1038/s41592-019-0576-7',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/240713',\n",
       "  'title': 'Scaling of G1 duration with population doubling time by a cyclin in Saccharomyces cerevisiae',\n",
       "  'authors': 'Blank, H. M.; Callahan, M.; Pistikopoulos, I. P. E.; Polymenis, A. O.; Polymenis, M.',\n",
       "  'author_corresponding': 'Michael  Polymenis',\n",
       "  'author_corresponding_institution': 'Texas A&M University',\n",
       "  'date': '2018-08-25',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'cell biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/240713.source.xml',\n",
       "  'abstract': 'The longer cells stay in particular phases of the cell cycle, the longer it will take these cell populations to increase. However, the above qualitative description has very little predictive value, unless it can be codified mathematically. A quantitative relation that defines the population doubling time (Td) as a function of the time eukaryotic cells spend in specific cell cycle phases would be instrumental for estimating rates of cell proliferation and for evaluating introduced perturbations. Here, we show that in human cells the length of the G1 phase (TG1) regressed on Td with a slope of {approx}0.75, while in the yeast Saccharomyces cerevisiae the slope was slightly smaller, at {approx}0.60. On the other hand, cell size was not strongly associated with Td or TG1 in cell cultures that were proliferating at different rates. Furthermore, we show that levels of the yeast G1 cyclin Cln3p were positively associated with rates of cell proliferation over a broad range, at least in part through translational control mediated by a short uORF in the CLN3 transcript. Cln3p was also necessary for the proper scaling between TG1 and Td. In contrast, yeast lacking the Whi5p transcriptional repressor maintained the scaling between TG1 and Td. These data reveal fundamental scaling relations between the duration of eukaryotic cell cycle phases and rates of cell proliferation, point to the necessary role of Cln3p in these relations in yeast and provide a mechanistic basis linking Cln3p levels to proliferation rates and the scaling of G1 with doubling time.',\n",
       "  'published': '10.1534/genetics.118.301507',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/242768',\n",
       "  'title': 'Asymmetric histone incorporation during DNA replication in Drosophila male germline stem cells',\n",
       "  'authors': 'Wooten, M.; Snedeker, J.; Nizami, Z.; Yang, X.; Urban, E.; Feng, X. A.; Kim, J. M.; Gall, J.; Xiao, J.; Chen, X.',\n",
       "  'author_corresponding': 'Xin  Chen',\n",
       "  'author_corresponding_institution': 'Department of Biology, The Johns Hopkins University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'developmental biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/242768.source.xml',\n",
       "  'abstract': 'One Sentence SummaryDNA replication establishes asymmetric epigenomes\\n\\nSummaryOne of the most fundamental questions in developmental biology concerns how cells with identical genomes differentiate into distinct cell types. One important context for understanding cell fate specification is asymmetric cell division, where the two daughter cells establish different cell fates following a single division. Many stem cells undergo asymmetric division to produce both a self-renewing stem cell and a differentiating daughter cell1-5. Here we show that histone H4 is inherited asymmetrically in asymmetrically dividing Drosophila male germline stem cells, similar to H36. In contrast, both H2A and H2B are inherited symmetrically. By combining superresolution microscopy with the chromatin fiber method, we are able to study histone inheritance patterns on newly replicated chromatin fibers. Using this technique, we find asymmetric inheritance patterns for old and new H3, but symmetric inheritance patterns for old and new H2A on replicating sister chromatids. Furthermore, co-localization studies on isolated chromatin fibers and proximity ligation assays on intact nuclei reveal that old H3 are preferentially incorporated by the leading strand while newly synthesized H3 are enriched on the lagging strand. Finally, using a sequential nucleoside analog incorporation assay, we detect a high incidence of unidirectional DNA replication on germline-derived chromatin fibers and DNA fibers. The unidirectional fork movement coupled with the strand preference of histone incorporation could explain how old and new H3 are asymmetrically incorporated by replicating sister chromatids. In summary, our work demonstrates that the intrinsic asymmetries in DNA replication may help construct sister chromatids enriched with distinct populations of histones. Therefore, these results suggest unappreciated roles for DNA replication in asymmetrically dividing cells in multicellular organisms.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/243568',\n",
       "  'title': 'Identifying biomarkers of anti-cancer drug synergy using multi-task learning',\n",
       "  'authors': 'Aben, N.; de Ruiter, J. R.; Bosdriesz, E.; Kim, Y.; Bounova, G.; Vis, D. J.; Wessels, L. F. A.; Michaut, M.',\n",
       "  'author_corresponding': 'Magali  Michaut',\n",
       "  'author_corresponding_institution': 'Netherlands Cancer Institute & Oncode Institute',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/243568.source.xml',\n",
       "  'abstract': 'Combining anti-cancer drugs has the potential to increase treatment efficacy. Because patient responses to drug combinations are highly variable, predictive biomarkers of synergy are required to identify which patients are likely to benefit from a drug combination. To aid biomarker identification, the DREAM challenge consortium has recently released data from a screen containing 85 cell lines and 167 drug combinations. The main challenge of these data is the low sample size: per drug combination, a median of 14 cell lines have been screened. We found that widely used methods in single drug response prediction, such as Elastic Net regression per drug, are not predictive in this setting. Instead, we propose to use multi-task learning: training a single model simultaneously on all drug combinations, which we show results in increased predictive performance. In contrast to other multi-task learning approaches, our approach allows for the identification of biomarkers, by using a modified random forest variable importance score, which we illustrate using artificial data and the DREAM challenge data. Notably, we find that mutations in MYO15A are associated with synergy between ALK / IGFR dual inhibitors and PI3K pathway inhibitors in triple-negative breast cancer.\\\\n\\\\nAuthor summaryCombining drugs is a promising strategy for cancer treatment. However, it is often not known which patients will benefit from a particular drug combination. To identify patients that are likely to benefit, we need to identify biomarkers, such as mutations in the tumors DNA, that are associated with favorable response to the drug combination. In this work, we identified such biomarkers using the drug combination data released by the DREAM challenge consortium, which contain 85 tumor cell lines and 167 drug combinations. The main challenge of these data is the extremely low sample size: a median of 14 cell lines have been screened per drug combination. We found that traditional methods to identify biomarkers for monotherapy response, which analyze each drug separately, are not suitable in this low sample size setting. Instead, we used a technique called multi-task learning to jointly analyze all drug combinations in a single statistical model. In contrast to existing multi-task learning algorithms, which are black-box methods, our method allows for the identification of biomarkers. Notably, we find that, in a subset of breast cancer cell lines, MYO15A mutations associate with response to the combination of ALK / IGFR dual inhibitors and PI3K pathway inhibitors.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/246165',\n",
       "  'title': 'Noise propagation in an integrated model of bacterial gene expression and growth',\n",
       "  'authors': 'Kleijn, I. T.; Krah, L. H. J.; Hermsen, R.',\n",
       "  'author_corresponding': 'Rutger  Hermsen',\n",
       "  'author_corresponding_institution': 'Utrecht University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'systems biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/246165.source.xml',\n",
       "  'abstract': 'In bacterial cells, gene expression, metabolism, and growth are highly interdependent and tightly coordinated. As a result, stochastic fluctuations in expression levels and instantaneous growth rate show intricate cross-correlations. These correlations are shaped by feedback loops, trade-offs and constraints acting at the cellular level; therefore a quantitative understanding requires an integrated approach. To that end, we here present a mathematical model describing a cell that contains multiple proteins that are each expressed stochastically and jointly limit the growth rate. Conversely, metabolism and growth affect protein synthesis and dilution. Thus, expression noise originating in one gene propagates to metabolism, growth, and the expression of all other genes. Nevertheless, under a small-noise approximation many statistical quantities can be calculated analytically. We identify several routes of noise propagation, illustrate their origins and scaling, and establish important connections between noise propagation and the field of metabolic control analysis. We then present a many-protein model containing > 1000 proteins parameterized by previously measured abundance data and demonstrate that the predicted cross-correlations between gene expression and growth rate are in broad agreement with published measurements.',\n",
       "  'published': '10.1371/journal.pcbi.1006386',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/248484',\n",
       "  'title': 'Utility of the SWAN Scale for ADHD Trait-Based Genetic Research: A Validity and Polygenic Risk Study',\n",
       "  'authors': 'Burton, C. L.; Wright, L.; Shan, J.; Xiao, B.; Dupuis, A.; Goodale, T.; Shaheen, S.-M.; Corfield, E. C.; Arnold, P. D.; Schachar, R. J.; Crosbie, J.',\n",
       "  'author_corresponding': 'Jennifer  Crosbie',\n",
       "  'author_corresponding_institution': 'Hospital for Sick Children; University of Toronto',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genetics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/248484.source.xml',\n",
       "  'abstract': 'BackgroundValid and genetically-informative trait measures of psychopathology collected in the general population would provide a powerful complement to case/control genetic designs. We report the convergent, predictive and discriminant validity of the parent- and the self-report versions of the Strengths and Weaknesses of ADHD Symptoms and Normal Behavior Rating Scale (SWAN) for attention-deficit/hyperactivity disorder (ADHD) traits. We tested if SWAN ADHD scores were associated with ADHD diagnosis, ADHD polygenic risk, as well as with traits and polygenic risk for co-occurring disorders such as anxiety and obsessive-compulsive disorder (OCD).\\\\n\\\\nMethodsWe collected parent- and self-report SWAN scores in a community sample (n=15,560; 6-18 years of age) and created norms. Sensitivity-specificity analyses determined SWAN cut-points that discriminated those with a community ADHD diagnosis (n=972) from those without a community diagnosis. We validated cut-points from the community sample in a clinical sample (266 ADHD cases; 36 controls). We tested if SWAN scores were associated with anxiety and obsessive-compulsive (OC) traits and polygenic risk for ADHD, OCD and anxiety disorders.\\\\n\\\\nResultsBoth the parent- and the self-report SWAN measures showed high convergent validity with established ADHD measures and distinguished ADHD participants with high sensitivity and specificity in the community sample. Cut-points established in the community sample discriminated ADHD clinic cases from controls with a sensitivity of 86% and specificity of 94%. High parent- and self-report SWAN scores and scores above the community-based cut-points were associated with polygenic risk for ADHD. High ADHD traits were associated with high anxiety traits, but not OC traits. SWAN scores were not associated with OCD or anxiety disorder polygenic risk.\\\\n\\\\nConclusionThe parent- and self-report SWAN are potentially useful in genetic research because they predict ADHD diagnoses and are associated with ADHD polygenic risk.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/253203',\n",
       "  'title': 'Modelling mitosis with multiple phenotypes: relation to Haeckel’s recapitulation law',\n",
       "  'authors': 'Alexandrov, Y.',\n",
       "  'author_corresponding': 'Yuriy  Alexandrov',\n",
       "  'author_corresponding_institution': 'Imperial College London',\n",
       "  'date': '2018-08-22',\n",
       "  'version': '4',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'physiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/253203.source.xml',\n",
       "  'abstract': 'The article presents a novel stochastic mathematical model of mitosis in heterogeneous (multiple-phenotype), age-dependent cell populations. The developed computational techniques involve flexible use of differentiation tree diagrams. The applicability of the model is discussed in the context of the Haeckelian (biogenetic) paradigm. In particular, the article puts forward the conjecture of generality of Haeckels recapitulation law. The conjecture is briefly collated against relevant scientific evidence and elaborated for the specific case of evolving/mutable cell phenotypes as considered by the model. The feasibility, basic regimes and the convenience of the model are tested on examples and experimental data, and the corresponding open source simulation software is described and demonstrated.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/253518',\n",
       "  'title': 'Priority effects and non-hierarchical competition shape species composition in a complex grassland community',\n",
       "  'authors': 'Uricchio, L. H.; Daws, S. C.; Spear, E. R.; Mordecai, E. A.',\n",
       "  'author_corresponding': 'Erin A Mordecai',\n",
       "  'author_corresponding_institution': 'Stanford University',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '4',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/253518.source.xml',\n",
       "  'abstract': 'Niche and fitness differences control the outcome of competition, but determining their relative importance in invaded communities - which may be far from equilibrium - remains a pressing concern. Moreover, it is unclear whether classic approaches for studying competition, which were developed predominantly for pairs of interacting species, will fully capture dynamics in complex species assemblages. We parameterized a population dynamic model using competition experiments of two native and three exotic species from a grassland community. We found evidence for minimal fitness differences or niche differences between the native species, leading to slow replacement dynamics and priority effects, but large fitness advantages allowed exotics to unconditionally invade natives. Priority effects driven by strong interspecific competition between exotic species drove single-species dominance by one of two exotic species in 80% of model outcomes, while a complex mixture of non-hierarchical competition and coexistence between native and exotic species occurred in the remaining 20%. Fungal infection, a commonly hypothesized coexistence mechanism, had weak fitness effects, and is unlikely to substantially affect coexistence. In contrast to previous work on pairwise outcomes in largely native-dominated communities, our work supports a role for nearly-neutral dynamics and priority effects as drivers of species composition in invaded communities.',\n",
       "  'published': '10.1086/701434',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/257105',\n",
       "  'title': 'Intron-mediated enhancement boosts Rtn4 circRNA expression: A robust method for exploring circRNA function',\n",
       "  'authors': 'Mo, D.; Li, X.; Cui, D.; Vollmar, J.-F.',\n",
       "  'author_corresponding': 'Dingding  Mo',\n",
       "  'author_corresponding_institution': 'Max Planck Institute for Biology of Ageing',\n",
       "  'date': '2018-08-22',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'biochemistry',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/257105.source.xml',\n",
       "  'abstract': 'CircRNAs are expressed in many important biological processes. Studying their function requires an effective expression method. When we used intron-mediated enhancement (IME) to improve circRNA expression of mouse Rtn4 (Nogo, a key protein in Nogo-Rho pathways) circRNA as a test case, we achieved a 4-6-fold improvement compared to an existing method. We further developed this approach into a general circRNA expression vector pCircRNA-DMo. An unexpected feature of our approach is its ability to promote translation of circRNA into detectable amounts of proteins. Intriguingly, both monomer and multimer repeating peptides can be observed as a result of rolling circle translation of RTN4 circRNA. We also confirmed the presence of both peptide forms in human and mouse brains, highlighting the significance of circRNA translation in vivo. In summary, we demonstrate the significant advantage of IME in enhancing circRNA biogenesis and hence our vector offers a robust platform for exploring potential circRNA peptide-encoding functions.',\n",
       "  'published': '10.1038/s41598-019-48224-y',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/259507',\n",
       "  'title': 'The Rate of Molecular Evolution When Mutation May Not Be Weak',\n",
       "  'authors': 'de Koning, A. P. J.; De Sanctis, B. D.',\n",
       "  'author_corresponding': 'A.P. Jason  de Koning',\n",
       "  'author_corresponding_institution': 'University of Calgary',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genetics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/259507.source.xml',\n",
       "  'abstract': 'One of the most fundamental rules of molecular evolution is that the rate of neutral evolution equals the mutation rate and is independent of effective population size. This result lies at the heart of the Neutral Theory, and is the basis for numerous analytic approaches that are widely applied to infer the action of natural selection across the genome and through time, and for dating divergence events using the molecular clock. However, this result was derived under the assumption that evolution is strongly mutation-limited, and it has not been known whether it generalizes across the range of mutation pressures or the spectrum of mutation types observed in natural populations. Validated by both simulations and exact computational analyses, we present a direct and transparent theoretical analysis of the Wright-Fisher model of population genetics, which shows that some of the most important rules of molecular evolution are fundamentally changed by considering recurrent mutations full effect. Surprisingly, the rate of the neutral molecular clock is found to have population-size dependence and to not equal the mutation rate in general. This is because, for increasing values of the population mutation rate parameter ({theta}), the time spent waiting for mutations quickly becomes smaller than the cumulative time mutants spend segregating before a substitution, resulting in a net deceleration compared to classical theory that depends on the population mutation rate. Furthermore, selection exacerbates this effect such that more adaptive alleles experience a greater deceleration than less adaptive alleles, introducing systematic bias in a wide variety of methods for inferring the strength and direction of natural selection from across-species sequence comparisons. Critically, the classical weak mutation approximation performs well only when{theta} < 0.1, a threshold that many biological populations seem to exceed.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/260950',\n",
       "  'title': 'The interplay of synaptic plasticity and scaling enables the self-organized allocation of multiple memory representations',\n",
       "  'authors': 'Auth, J. M.; Nachstedt, T.; Tetzlaff, C.',\n",
       "  'author_corresponding': 'Christian  Tetzlaff',\n",
       "  'author_corresponding_institution': 'Third Institute of Physics, Georg-August-University Goettingen, Germany',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/260950.source.xml',\n",
       "  'abstract': 'It is commonly assumed that memories about experienced stimuli are represented by groups of highly interconnected neurons called cell assemblies. This requires allocating and storing information in the neural circuitry, which happens through synaptic weight adaptation. It remains, however, largely unknown how memory allocation and storage can be achieved and coordinated to allow for a faithful representation of multiple memories without disruptive interference between them. In this theoretical study, we show that the interplay between conventional synaptic plasticity and homeostatic synaptic scaling organizes synaptic weight adaptations such that a new stimulus forms a new memory and where different stimuli are assigned to distinct cell assemblies. The resulting dynamics can reproduce experimental in-vivo data, focusing on how diverse factors as neuronal excitability and network connectivity, influence memory formation. Thus, the here presented model suggests that a few fundamental synaptic mechanisms may suffice to implement memory allocation and storage in neural circuitry.',\n",
       "  'published': '10.3389/fncir.2020.541728',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/264580',\n",
       "  'title': 'Competition in biofilms between cystic fibrosis isolates of Pseudomonas aeruginosa is driven by R-pyocins',\n",
       "  'authors': 'Oluyombo, O.; Diggle, S. P.; Penfold, C. N.',\n",
       "  'author_corresponding': 'Stephen P Diggle',\n",
       "  'author_corresponding_institution': 'Georgia Institute of Technology',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'microbiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/264580.source.xml',\n",
       "  'abstract': 'Pseudomonas aeruginosa is an opportunistic pathogen responsible for a number of different human infections and is the leading cause of morbidity and mortality in cystic fibrosis (CF) patients. P. aeruginosa infections are difficult to treat due to a number of antibiotic resistance mechanisms and the organisms propensity to form multicellular biofilms. Epidemic strains of P. aeruginosa often dominate within the lungs of individual CF patients, but how they achieve this is poorly understood. One of the ways strains of P. aeruginosa can compete, is by producing chromosomally encoded bacteriocins, called pyocins. Three major classes of pyocin have been identified in P. aeruginosa: soluble pyocins (S-types) and tailocins (R- and F-types). In this study, we investigated the distribution of S- and R-type pyocins in 24 clinical strains isolated from individual CF patients and then focused on understanding their roles on inter-strain competition. We found that (i) each strain produced only one R-pyocin type, but the number of S-pyocins varied between strains; (ii) R-pyocins were generally important for strain dominance during competition assays in planktonic cultures and biofilm communities in strains with both disparate R and S pyocin sub-types. (iii) purified R-pyocins demonstrated significant antimicrobial activity against established biofilms. Our work provides support for a key role played by R-pyocins in the competition between P. aeruginosa strains, and may help explain why certain strains and lineages of P. aeruginosa dominate and displace others during CF lung infection. Furthermore, we demonstrate the potential of exploiting R-pyocins for therapeutic gains in an era when antibiotic resistance is a global concern.\\\\n\\\\nIMPORTANCEA major clinical problem caused by Pseudomonas aeruginosa, is chronic biofilm infection of the lungs in individuals with cystic fibrosis (CF). Epidemic P. aeruginosa strains dominate and displace others during CF infection, but these intra-species interactions remain poorly understood. Here we demonstrate that R-pyocins (bacterocins) are important factors in driving competitive interactions in biofilms between P. aeruginosa strains isolated from different CF patients. In addition, we found that these phage-like pyocins are inhibitory against mature biofilms of susceptible strains. This highlights the potential of R-pyocins as antimicrobial and antibiofilm agents, at a time when new antimicrobial therapies are desperately needed.',\n",
       "  'published': '10.1128/mBio.01828-18',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/264960',\n",
       "  'title': 'A plant biodiversity effect resolved to a single genetic locus',\n",
       "  'authors': 'Wuest, S. E.; Niklaus, P. A.',\n",
       "  'author_corresponding': 'Samuel E Wuest',\n",
       "  'author_corresponding_institution': 'University of Zurich',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/264960.source.xml',\n",
       "  'abstract': 'Despite extensive evidence that biodiversity promotes plant community productivity, progress towards understanding the mechanistic basis of this effect remains slow, impeding the development of predictive ecological theory and agricultural applications. Here, we analysed non-additive interactions between genetically divergent Arabidopsis accessions in experimental plant communities. By combining methods from ecology and genetics, we identified a major effect locus at which allelic differences between individuals increases above-ground productivity of communities. In experiments with near-isogenic lines, we show that this diversity effect acts independently of other genomic regions and can be resolved to a single region representing less than 0.3% of the genome. Using plant-soil-feedback experiments, we also demonstrate that allelic diversity causes genotype-specific soil legacy responses in a consecutive growing period, even after the original community has disappeared. Our work thus suggests that positive diversity effects can be linked to single Mendelian factors, and that a range of complex community properties can have a simple cause. This may pave the way to a better understanding of diversity effects, and to novel breeding strategies, focussing on phenotypic properties that manifest themselves beyond isolated individuals, i.e. at a higher level of biological organisation.',\n",
       "  'published': '10.1038/s41559-018-0708-y',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/264945',\n",
       "  'title': 'Neo-formation of chromosomes in bacteria',\n",
       "  'authors': 'Poirion, O. B.; Lafay, B.',\n",
       "  'author_corresponding': 'Benedicte  Lafay',\n",
       "  'author_corresponding_institution': 'CNRS',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/264945.source.xml',\n",
       "  'abstract': 'Although the bacterial secondary chromosomes/megaplasmids/chromids, first noticed about forty years ago, are commonly held to originate from stabilized plasmids, their true nature and definition are yet to be resolved. On the premise that the integration of a replicon within the cell cycle is key to deciphering its essential nature, we show that the content in genes involved in the replication, partition and segregation of the replicons and in the cell cycle discriminates the bacterial replicons into chromosomes, plasmids, and another class of essential genomic elements that function as chromosomes. These latter do not derive directly from plasmids. Rather, they arise from the fission of a multi-replicon molecule corresponding to the co-integrated and rearranged ancestral chromosome and plasmid. All essential replicons in a distributed genome are thus neochromosomes. Having a distributed genome appears to extend and accelerate the exploration of the bacterial genome evolutionary landscape, producing complex regulation and leading to novel eco-phenotypes and species diversification.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/264846',\n",
       "  'title': 'Cyfip1 haploinsufficiency increases compulsive-like behavior and modulates palatable food intake: Implications for Prader-Willi Syndrome',\n",
       "  'authors': 'Babbs, R. K.; Ruan, Q. T.; Kelliher, J. C.; Beierle, J. A.; Chen, M. M.; Feng, A. X.; Kirkpatrick, S. L.; Benitez, F. A.; Rodriguez, F. A.; Pierre, J.; Anandakumar, J.; Kumar, V.; Mulligan, M. K.; Bryant, C. D.',\n",
       "  'author_corresponding': 'Camron D Bryant',\n",
       "  'author_corresponding_institution': 'Boston University School of Medicine',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genetics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/264846.source.xml',\n",
       "  'abstract': 'Binge eating (BE) is a heritable trait associated with eating disorders and involves rapid consumption of large quantities of food. We identified cytoplasmic FMRP-interacting protein 2 (Cyfip2) as a major genetic factor underlying BE and concomitant compulsive-like behaviors in mice. CYFIP2 is a gene homolog of CYFIP1 - one of four paternally-deleted genes in patients with the more severe Type I Prader-Willi Syndrome (PWS). PWS is a neurodevelopmental disorder where 70% of cases involve paternal deletion of 15q11-q13. PWS symptoms include hyperphagia, obesity (if untreated), cognitive deficits, and obsessive-compulsive behaviors. We tested whether Cyfip1 haploinsufficiency (+/-) would enhance premorbid compulsive-like behavior and palatable food (PF) intake in a parent-of-origin-selective manner. We tested Cyfip1+/- mice on a C57BL/6N (N) background that were homozygous for the BE-associated missense mutation in Cyfip2 (S968F) as well as mice that we backcrossed to homozygosity for the C57BL/6J (J) allele at Cyfip2 (Cyfip2J/J). Cyfip1+/- mice showed increased compulsive-like behavior on both backgrounds, increased PF consumption on the Cyfip2N/N background in a paternally-enhanced manner, and decreased PF consumption in male Cyfip1+/- mice on the Cyfip2J/J background in a maternally selective manner. In the hypothalamus, there was a maternally-enhanced reduction of Cyfip1 transcription, but a paternally-enhanced reduction in CYFIP1 protein. In the nucleus accumbens, there was a maternally-enhanced reduction in CYFIP1 protein. Together, increased compulsive-like behavior, parent-of-origin-, and genetic background-dependent effects of Cyfip1 haploinsufficiency on PF consumption implicate CYFIP1 in behaviors in neurodevelopmental disorders involving reduced expression of CYFIP1, including PWS, Fragile X Syndrome, and 15q11.2 Microdeletion Syndrome.',\n",
       "  'published': '10.1534/g3.119.400470',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/266700',\n",
       "  'title': 'Metagenomic analysis with strain-level resolution reveals fine-scale variation in the human pregnancy microbiome',\n",
       "  'authors': 'Goltsman, D. S. A.; Sun, C. L.; Proctor, D. M.; DiGiulio, D. B.; Robaczewska, A.; Thomas, B. C.; Shaw, G. M.; Stevenson, D. K.; Holmes, S. P.; Banfield, J. F.; Relman, D. A.',\n",
       "  'author_corresponding': 'David A. Relman',\n",
       "  'author_corresponding_institution': 'Stanford University',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'microbiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/266700.source.xml',\n",
       "  'abstract': 'Recent studies suggest that the microbiome has an impact on gestational health and outcome. However, characterization of the pregnancy-associated microbiome has largely relied on 16S rRNA gene amplicon-based surveys. Here, we describe an assembly-driven, metagenomics-based, longitudinal study of the vaginal, gut, and oral microbiomes in 292 samples from ten subjects sampled every three weeks throughout pregnancy. 1.53 Gb of non-human sequence was assembled into scaffolds, and functional genes were predicted for gene-and pathway-based analyses. Vaginal assemblies were binned into 97 draft quality genomes. Redundancy analysis (RDA) of microbial community composition at all three body sites revealed gestational age to be a significant source of variation in patterns of gene abundance. In addition, health complications were associated with variation in community functional gene composition in the mouth and gut. The diversity of Lactobacillus iners-dominated communities in the vagina, unlike most other vaginal community types, significantly increased with gestational age. The genomes of co-occurring Gardnerella vaginalis strains with predicted distinct functions were recovered in samples from two subjects. In seven subjects, gut samples contained strains of the same Lactobacillus species that dominated the vaginal community of that same subject, and not other Lactobacillus species; however, these within-host strains were divergent. CRISPR spacer analysis suggested shared phage and plasmid populations across body sites and individuals. This work underscores the dynamic behavior of the microbiome during pregnancy and suggests the potential importance of understanding the sources of this behavior for fetal development and gestational outcome.',\n",
       "  'published': '10.1101/gr.236000.118',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/268623',\n",
       "  'title': 'Developing an automated iterative near-term forecasting system for an ecological study',\n",
       "  'authors': 'White, E. P.; Yenni, G. M.; Taylor, S. D.; Christensen, E. M.; Bledsoe, E. K.; Simonis, J. L.; Ernest, S. K. M.',\n",
       "  'author_corresponding': 'Ethan P White',\n",
       "  'author_corresponding_institution': 'University of Florida',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/268623.source.xml',\n",
       "  'abstract': 'O_LIMost forecasts for the future state of ecological systems are conducted once and never updated or assessed. As a result, many available ecological forecasts are not based on the most up-to-date data, and the scientific progress of ecological forecasting models is slowed by a lack of feedback on how well the forecasts perform.\\\\nC_LIO_LIIterative near-term ecological forecasting involves repeated daily to annual scale forecasts of an ecological system as new data becomes available and regular assessment of the resulting forecasts. We demonstrate how automated iterative near-term forecasting systems for ecology can be constructed by building one to conduct monthly forecasts of rodent abundances at the Portal Project, a long-term study with over 40 years of monthly data. This system automates most aspects of the six stages of converting raw data into new forecasts: data collection, data sharing, data manipulation, modeling and forecasting, archiving, and presentation of the forecasts.\\\\nC_LIO_LIThe forecasting system uses R code for working with data, fitting models, making forecasts, and archiving and presenting these forecasts. The resulting pipeline is automated using continuous integration (a software development tool) to run the entire pipeline once a week. The cyberinfrastructure is designed for long-term maintainability and to allow the easy addition of new models. Constructing this forecasting system required a team with expertise ranging from field site experience to software development.\\\\nC_LIO_LIAutomated near-term iterative forecasting systems will allow the science of ecological forecasting to advance more rapidly and provide the most up-to-date forecasts possible for conservation and management. These forecasting systems will also accelerate basic science by allowing new models of natural systems to be quickly implemented and compared to existing models. Using existing technology, and teams with diverse skill sets, it is possible for ecologists to build automated forecasting systems and use them to advance our understanding of natural systems.\\\\nC_LI',\n",
       "  'published': '10.1111/2041-210x.13104',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/267906',\n",
       "  'title': 'Tbx1 regulates extracellular matrix-cell interactions in the second heart field.',\n",
       "  'authors': 'Alfano, D.; Altomonte, A.; Cortes, C.; Bilio, M.; Kelly, R. G.; Baldini, A.',\n",
       "  'author_corresponding': 'Antonio  Baldini',\n",
       "  'author_corresponding_institution': 'University of Naples Federico II',\n",
       "  'date': '2018-08-26',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'developmental biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/267906.source.xml',\n",
       "  'abstract': 'Tbx1, the major candidate gene for DiGeorge or 22q11.2 deletion syndrome, is required for efficient incorporation of cardiac progenitors (CPs) of the second heart field (SHF) into the heart. However, the mechanisms by which TBX1 regulates this process are still unclear. Here, we have used two independent models, mouse embryos and cultured cells, to define the role of TBX1 in establishing morphological and dynamic characteristics of SHF in the mouse. We found that loss of TBX1 impairs extra cellular matrix (ECM)-integrin-focal adhesion (FA) signaling in both models. Mosaic analysis in embryos showed that this function is non-cell autonomous and, in cultured cells, loss of TBX1 impairs cell migration and focal adhesions. Additionally, we found that ECM-mediated outside-in integrin signaling is disrupted upon loss of TBX1. Finally, we show that interfering with the ECM-integrin-FA axis between E8.5 and E9.5 in mouse embryos, corresponding to the time window within which TBX1 is required in the SHF, causes outflow tract dysmorphogenesis. Our results demonstrate that TBX1 is required to maintain the integrity of ECM-cell interactions in the SHF, and that this interaction is critical for cardiac outflow tract development. More broadly, our data identifies a novel TBX1 downstream pathway as an important player in SHF tissue architecture and cardiac morphogenesis.',\n",
       "  'published': '10.1093/hmg/ddz058',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/275545',\n",
       "  'title': 'A 3D model of human skeletal muscle innervated with stem cell-derived motor neurons enables epsilon-subunit targeted myasthenic syndrome studies',\n",
       "  'authors': 'Afshar Bakooshli, M.; Lippmann, E. S.; Mulcahy, B.; Iyer, N.; Nguyen, C. T.; Tung, K.; Stewart, B. A.; Mouly, V.; Pegoraro, E.; Ahn, H.; Ginsberg, H.; Zhen, M.; Ashton, R. S.; Gilbert, P. M.',\n",
       "  'author_corresponding': 'Penney M Gilbert',\n",
       "  'author_corresponding_institution': 'Institute of Biomaterials and Biomedical Engineering, University of Toronto, Toronto, ON, Canada',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'bioengineering',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/275545.source.xml',\n",
       "  'abstract': 'Two-dimensional (2D) human skeletal muscle fiber cultures are ill equipped to support the contractile properties of maturing muscle fibers. This limits their application to the study of adult human neuromuscular junction (NMJ) development, a process requiring maturation of muscle fibers in the presence of motor neuron endplates. Here we describe a three-dimensional (3D) co-culture method whereby human muscle progenitors mixed with human pluripotent stem cell-derived motor neurons self-organize to form functional NMJ connections within two weeks. Functional connectivity between motor neuron endplates and muscle fibers is confirmed with calcium transient imaging and electrophysiological recordings. Notably, we only observed epsilon acetylcholine receptor subunit protein upregulation and activity in 3D co-culture. This demonstrates that the 3D co-culture system supports a developmental shift from the embryonic to adult form of the receptor that does not occur in 2D co-culture. Further, 3D co-culture treatments with myasthenia gravis patient sera shows the ease of studying human disease with the system. This work delivers a simple, reproducible, and adaptable method to model and evaluate adult human NMJ de novo development and disease in culture.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/275545',\n",
       "  'title': 'A 3D model of human skeletal muscle innervated with stem cell-derived motor neurons enables epsilon-subunit targeted myasthenic syndrome studies',\n",
       "  'authors': 'Afshar Bakooshli, M.; Lippmann, E. S.; Mulcahy, B.; Iyer, N.; Nguyen, C. T.; Tung, K.; Stewart, B. A.; Bigot, A.; Pegoraro, E.; Ahn, H.; Ginsberg, H.; Zhen, M.; Ashton, R. S.; Gilbert, P. M.',\n",
       "  'author_corresponding': 'Penney M Gilbert',\n",
       "  'author_corresponding_institution': 'Institute of Biomaterials and Biomedical Engineering, University of Toronto, Toronto, ON, Canada',\n",
       "  'date': '2018-08-26',\n",
       "  'version': '4',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'bioengineering',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/26/275545.source.xml',\n",
       "  'abstract': 'Two-dimensional (2D) human skeletal muscle fiber cultures are ill equipped to support the contractile properties of maturing muscle fibers. This limits their application to the study of adult human neuromuscular junction (NMJ) development, a process requiring maturation of muscle fibers in the presence of motor neuron endplates. Here we describe a three-dimensional (3D) co-culture method whereby human muscle progenitors mixed with human pluripotent stem cell-derived motor neurons self-organize to form functional NMJ connections within two weeks. Functional connectivity between motor neuron endplates and muscle fibers is confirmed with calcium transient imaging and electrophysiological recordings. Notably, we only observed epsilon acetylcholine receptor subunit protein upregulation and activity in 3D co-culture. This demonstrates that the 3D co-culture system supports a developmental shift from the embryonic to adult form of the receptor that does not occur in 2D co-culture. Further, 3D co-culture treatments with myasthenia gravis patient sera shows the ease of studying human disease with the system. This work delivers a simple, reproducible, and adaptable method to model and evaluate adult human NMJ de novo development and disease in culture.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/277020',\n",
       "  'title': 'Reconstructing imagined letters from early visual cortex reveals tight topographic correspondence between visual mental imagery and perception',\n",
       "  'authors': 'Senden, M.; Emmerling, T. C.; Van Hoof, R.; Frost, M.; Goebel, R.',\n",
       "  'author_corresponding': 'Mario  Senden',\n",
       "  'author_corresponding_institution': 'Maastricht University',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nd',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/277020.source.xml',\n",
       "  'abstract': 'Visual mental imagery is the quasi-perceptual experience of \\\\\"seeing in the minds eye\\\\\". While a tight correspondence between imagery and perception in terms of subjective experience is well established, their correspondence in terms of neural representations remains insufficiently understood. In the present study, we exploit the high spatial resolution of functional magnetic resonance imaging (fMRI) at 7T, the retinotopic organization of early visual cortex, and machine learning techniques to investigate whether visual imagery of letter shapes preserves the topographic organization of perceived shapes. Sub-millimeter resolution fMRI images were obtained from early visual cortex in six subjects performing visual imagery of four different letter shapes. Predictions of imagery voxel activation patterns based on a population receptive field encoding model and physical letter stimuli provided first evidence in favor of detailed topographic organization. Subsequent visual field reconstructions of imagery data based on the inversion of the encoding model further showed that visual imagery preserves the geometric profile of letter shapes. These results open new avenues for decoding as we show that a denoising autoencoder can be used to pretrain a classifier purely based on perceptual data before fine-tuning it on imagery data. Finally, we show that the autoencoder can project imagery-related voxel activations onto their perceptual counterpart allowing for visually recognizable reconstructions even at the single-trial level. The latter may eventually be utilized for the development of content-based BCI letter-speller systems.',\n",
       "  'published': '10.1007/s00429-019-01828-6',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/277715',\n",
       "  'title': 'Analysis of novel domain-specific mutations in the zebrafish ndr2/cyclops gene generated using CRISPR-Cas9 RNPs',\n",
       "  'authors': 'Turner, A. N.; Andersen, R. S.; Bookout, I. E.; Brashear, L. N.; Davis, J. C.; Gahan, D. M.; Gotham, J. P.; Hijaz, B. A.; Kaushik, A. S.; McGill, J. B.; Miller, V. L.; Moseley, Z. P.; Nowell, C. L.; Patel, R. K.; Rodgers, M. C.; Shihab, Y. A.; Walker, A. P.; Glover, S. R.; Foster, S. D.; Challa, A. K.',\n",
       "  'author_corresponding': 'Anil Kumar  Challa',\n",
       "  'author_corresponding_institution': 'University of Alabama at Birmingham',\n",
       "  'date': '2018-08-25',\n",
       "  'version': '3',\n",
       "  'type': 'confirmatory results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'genetics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/277715.source.xml',\n",
       "  'abstract': 'Nodal-related protein (ndr2) is a member of the transforming growth factor type {beta} superfamily of factors and is required for ventral midline patterning of the embryonic central nervous system in zebrafish. In humans, mutations in the gene encoding nodal cause holoprosencephaly and heterotaxy. Mutations in the ndr2 gene in the zebrafish (Danio rerio) lead to similar phenotypes, including loss of the medial floor plate, severe deficits in ventral forebrain development, and cyclopia. Alleles of the ndr2 gene have been useful in studying patterning of ventral structures of the central nervous system. Fifteen different ndr2 alleles have been reported in zebrafish, of which eight were generated using chemical mutagenesis, four were radiation-induced, and the remaining alleles were obtained via random insertion, gene targeting (TALEN), or unknown methods. Therefore, most mutation sites were random and could not be predicted a priori. Using the CRISPR-Cas9 system from Streptococcus pyogenes, we targeted distinct regions in all three exons of zebrafish ndr2 and observed cyclopia in the injected (G0) embryos. We show that the use of sgRNA-Cas9 ribonucleoprotein (RNP) complexes can cause penetrant cyclopic phenotypes in injected (G0) embryos. Targeted PCR amplicon analysis using Sanger sequencing showed that most of the alleles had small indels resulting in frameshifts. The sequence information correlates with the loss of ndr2 activity. In this study, we validate multiple CRISPR targets using an in vitro nuclease assay and in vivo analysis using embryos. We describe one specific mutant allele resulting in loss of conserved terminal cysteine-coding sequences. This study is another demonstration of the utility of the CRISPR-Cas9 system in generating domain- specific mutations and provides further insights into the structure-function of the ndr2 gene.',\n",
       "  'published': '10.1007/s12041-018-1033-6',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/279570',\n",
       "  'title': 'Capturing diverse microbial sequence with comprehensive and scalable probe design',\n",
       "  'authors': 'Metsky, H. C.; Siddle, K. J.; Gladden-Young, A.; Qu, J.; Yang, D. K.; Brehio, P.; Goldfarb, A.; Piantadosi, A.; Wohl, S.; Carter, A.; Lin, A. E.; Barnes, K. G.; Tully, D. C.; Corleis, B.; Hennigan, S.; Barbosa-Lima, G.; Vieira, Y. R.; Paul, L. M.; Tan, A. L.; Garcia, K. F.; Parham, L. A.; Odia, I.; Eromon, P.; Folarin, O. A.; Goba, A.; Viral Hemorrhagic Fever Consortium,  ; Simon-Loriere, E.; Hensley, L.; Balmaseda, A.; Harris, E.; Kwon, D.; Allen, T. M.; Runstadler, J. A.; Smole, S.; Bozza, F. A.; Souza, T. M.; Isern, S.; Michael, S. F.; Lorenzana, I.; Gehrke, L.; Bosch, I.; Ebel, G.; Grant, ',\n",
       "  'author_corresponding': 'Hayden C Metsky',\n",
       "  'author_corresponding_institution': 'Broad Institute of MIT and Harvard, USA',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/279570.source.xml',\n",
       "  'abstract': 'Metagenomic sequencing has the potential to transform microbial detection and characteri zation, but new tools are needed to improve its sensitivity. We developed CATCH (Compact Aggregation of Targets for Comprehensive Hybridization), a computational method to enhance nucleic acid capture for enrichment of diverse microbial taxa. CATCH designs compact probe sets that achieve full coverage of known sequence diversity and that scale well with this diversity. To illustrate applications of CATCH, we focused on capturing viral genomes. We designed, synthesized, and validated multiple probe sets, including one that targets whole genomes of the 356 viral species known to infect humans. Capture with these probe sets enriched unique viral content on average 18x and allowed us to assemble genomes that we could not otherwise recover, while accurately preserving within-sample diversity. We used this approach to recover genomes from the 2018 Lassa fever outbreak in Nigeria and to improve detection of viral infections in samples with unknown content. Together, this work demonstrates a path toward more sensitive, cost-effective metagenomic sequencing.',\n",
       "  'published': '10.1038/s41587-018-0006-x',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/282301',\n",
       "  'title': 'Genetic dissection of assortative mating behavior',\n",
       "  'authors': 'Merrill, R. M.; Rastas, P.; Melo, M.-C.; Martin, S. H.; Barker, S.; Davey, J.; McMillan, W. O.; Jiggins, C.',\n",
       "  'author_corresponding': 'Richard M Merrill',\n",
       "  'author_corresponding_institution': 'Ludwig-Maximilians-Universitat, Munich',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/282301.source.xml',\n",
       "  'abstract': 'The evolution of new species is made easier when traits under divergent ecological selection are also mating cues. Such ecological mating cues are now considered more common than previously thought, but we still know little about the genetic changes underlying their evolution, or more generally about the genetic basis for assortative mating behaviors. The warning patterns of Heliconius melpomene and H. cydno are under disruptive selection due to increased predation of non-mimetic hybrids, and are used during mate recognition. We carried out a genome-wide quantitative trait locus (QTL) analysis of preference behaviors between these species and showed that divergent male preference has a simple genetic basis. Three QTLs each explain a large proportion of the differences in preference behavior observed between the parental species. Two of these QTLs are on chromosomes with major color pattern genes, including one that is tightly associated with the gene optix. Different loci influence different aspects of attraction, suggesting that behavioral isolation in Heliconius involves the evolution of independently segregating modules, similar to those for the corresponding wing pattern cues. Hybridization and subsequent sharing of wing pattern loci has played an important role during adaptation and speciation in Heliconius butterflies. The existence of large effect preference loci could similarly assist the evolution of novel behavioral phenotypes through recombination and introgression, and should facilitate rapid speciation.',\n",
       "  'published': '10.1371/journal.pbio.2005902',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/285981',\n",
       "  'title': 'Integrative Cross-species Analyses Suggest Deficits in Habituation Learning as a Widely Affected Mechanism in Intellectual Disability and Autism Spectrum Disorders',\n",
       "  'authors': 'Fenckova, M.; Asztalos, L.; Cizek, P.; Singgih, E. L.; Blok, L. E. R.; Glennon, J. C.; IntHout, J.; Zweier, C.; Eichler, E. E.; Bernier, R.; Asztalos, Z.; Schenck, A.',\n",
       "  'author_corresponding': 'Annette  Schenck',\n",
       "  'author_corresponding_institution': 'Radboud university medical center',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/285981.source.xml',\n",
       "  'abstract': 'BackgroundAlthough habituation is one of the most ancient and fundamental forms of learning, its regulators and relevance for human disease are poorly understood.\\\\n\\\\nMethodsWe manipulated the orthologs of 286 genes implicated in intellectual disability (ID) with or without comorbid autism spectrum disorder (ASD) specifically in Drosophila neurons, and tested these models in light-off jump habituation. We dissected neuronal substrates underlying the identified habituation deficits and integrated genotype-phenotype annotations, gene ontologies and interaction networks to determine the clinical features and molecular processes that are associated with habituation deficits.\\\\n\\\\nResultsWe identified more than 100 genes required for habituation learning. For the vast majority of these, 93 genes, a role in habituation learning was previously unknown. These genes characterize ID disorders with overgrowth/macrocephaly and comorbid ASD. Moreover, ASD individuals from the Simons Simplex Collection carrying disruptive de novo mutations in these genes exhibit increased rates of specific aberrant behaviors including stereotypic speech, hyperactivity and irritability. At the molecular level, ID genes required for normal habituation are enriched in synaptic function and converge on Ras-MAPK signaling. Both increased Ras-MAPK signaling in GABAergic and decreased Ras-MAPK signaling in cholinergic neurons specifically inhibit the adaptive habituation response.\\\\n\\\\nConclusionsOur work demonstrates the relevance of habituation learning to autism, identifies an unprecedented number of novel habituation players, supports an emerging role for inhibitory neurons in habituation and reveals an opposing, circuit-level-based mechanism for Ras-MAPK signaling. This establishes habituation as a possible, widely applicable target for pharmacologic intervention in ID/ASD.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/291088',\n",
       "  'title': 'Intracellular signaling through the comRS system in Streptococcus mutans genetic competence',\n",
       "  'authors': 'Underhill, S. A. M.; Shields, R. C.; Kaspar, J. R.; Haider, M.; Burne, R. A.; Hagen, S. J.',\n",
       "  'author_corresponding': 'Stephen J Hagen',\n",
       "  'author_corresponding_institution': 'University of Florida, Physics Department',\n",
       "  'date': '2018-08-22',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'microbiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/291088.source.xml',\n",
       "  'abstract': 'AbstractEntry into genetic competence in streptococci is controlled by ComX, an alternative sigma factor for genes that enable the import of exogenous DNA. In Streptococcus mutans, the immediate activator of comX is the ComRS signaling system, which consists of the cytosolic receptor ComR and the 7-residue signal peptide XIP, which is derived from ComS. Extracellular XIP imported by an oligopeptide permease interacts with ComR to form a transcriptional activator for both comX and comS. Therefore, extracellular XIP can function as an exogenous signal to trigger S. mutans competence. However, the mechanisms that process ComS and export it as XIP are not fully known in S. mutans. The observation that comX is expressed bimodally under some environmental conditions suggests that ComR may also interact with endogenously produced XIP or ComS, creating an intracellular positive feedback loop in comS transcription. Here we use single cell and microfluidic methods to compare the effects of the native comS gene and extracellular XIP on comX expression. We find that deletion of comS reduces the response of comX to extracellular XIP. We also find that comS-overexpressing cells autoactivate their comX even when their growth medium is rapidly exchanged, although this autoactivation requires an intact copy of comS under control of its own promoter. However comS-overexpressing cells do not activate comS-deficient mutants growing in coculture. These data show that individual cells can activate comX without exporting or importing the XIP or ComS signal, and that endogenously and exogenously produced ComS/XIP have inequivalent effects on comX behavior. These data are fully consistent with a model in which intracellular positive feedback in comS transcription plays a role in ComRS signaling, and is responsible for the bimodal expression of comX.\\\\n\\\\nAuthor SummaryHeterogeneous gene expression in genetically identical populations plays an important role in bacterial persistence and survival under changing environmental conditions. In the oral pathogen Streptococcus mutans, the physiological state of genetic competence can exhibit bimodality, with only some cells becoming competent. S. mutans controls its entry into competence by using the ComRS signaling system to activate comX, a gene encoding the master competence regulator ComX. The ComRS system is understood as a quorum sensing system, in which the extracellular accumulation of the small signal peptide XIP, derived from ComS, induces comX expression. We coupled observation of bacteria that fluoresce when comX is active with mathematical analysis and chemical binding assays to show that activation of comX does not necessarily require extracellular XIP or ComS, and that comX-active cells do not necessarily export XIP. Our experiments and mathematical modeling indicate that a positive feedback loop in comS transcription allows a cell to activate comX in response to its own XIP or ComS in the absence of extracellular XIP, or to amplify its comX response to extracellular XIP if present. Such positive feedback loops are often the cause of bimodal gene expression like that seen in S. mutans competence.',\n",
       "  'published': '10.1128/mSphere.00444-18',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/298356',\n",
       "  'title': 'Science podcasts: analysis of global production and output from 2004 to 2018',\n",
       "  'authors': 'MacKenzie, L. E.',\n",
       "  'author_corresponding': 'Lewis E MacKenzie',\n",
       "  'author_corresponding_institution': 'Durham University, Department of Chemistry',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '8',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc',\n",
       "  'category': 'scientific communication and education',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/298356.source.xml',\n",
       "  'abstract': '1.Since 2004, podcasts have emerged as a decentralised medium for science communication to the global public. However, to-date, there have been no large-scale quantitative studies of the production and dissemination of science podcasts. This study identified 952 English language science podcasts available between January and February 2018 and analysed online textual and visual data related to the podcasts and classified and noted key production parameters. It was found that the total number of science podcast series available grew linearly between 2004 and 2010, and then exponentially between 2010 and 2018. 65% of science podcast series were hosted by scientists and 77% were targeted to public audiences. Although a wide range of primarily single-subject science podcasts series were noted, 34% of science podcast series were not dedicated to a science subject. Compared to biology and physics, chemistry may be under-represented by science podcasts. Only 24% of science podcast series had any overt financial income. 62% of science podcast series were affiliated to an organisation; producing a greater number of episodes (median = 24, average = 96) than independent science podcast series (median = 16, average = 48). This study provides the first  snapshot of how science podcasts are being used to communicate science to public audiences around the globe.',\n",
       "  'published': '10.1098/rsos.180932',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/300251',\n",
       "  'title': 'Persistent coding of outcome-predictive cue features in the rat nucleus accumbens',\n",
       "  'authors': 'Gmaz, J. M.; Carmichael, J. E.; van der Meer, M.',\n",
       "  'author_corresponding': 'Matthijs  van der Meer',\n",
       "  'author_corresponding_institution': 'Dartmouth College',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/300251.source.xml',\n",
       "  'abstract': 'The nucleus accumbens (NAc) is important for learning from feedback, and for biasing and invigorating behavior in response to cues that predict motivationally relevant outcomes. NAc encodes outcome-related cue features such as the magnitude and identity of reward. However, little is known about how features of cues themselves are encoded. We designed a decision making task where rats learned multiple sets of outcome-predictive cues, and recorded single-unit activity in the NAc during performance. We found that coding of cue identity and location occurred alongside coding of expected outcome. Furthermore, this coding persisted both during a delay period, after the rat made a decision and was waiting for an outcome, and after the outcome was revealed. Encoding of cue features in the NAc may enable contextual modulation of ongoing behavior, and provide an eligibility trace of outcome-predictive stimuli for updating stimulus-outcome associations to inform future behavior.',\n",
       "  'published': '10.7554/elife.37275',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/301838',\n",
       "  'title': 'Robust and clinically relevant prediction of response to anti-cancer drugs via network integration of molecular profiles',\n",
       "  'authors': 'Franco, M.; Jeggari, A.; Peuget, S.; Böttger, F.; Selivanova, G.; Alexeyenko, A.',\n",
       "  'author_corresponding': 'Andrey  Alexeyenko',\n",
       "  'author_corresponding_institution': 'Karolinska Institutet',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'cancer biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/301838.source.xml',\n",
       "  'abstract': 'In order to tackle heterogeneity of cancer samples and high data space dimensionality, we propose a method NEAmarker for finding sensitive and robust biomarkers at the pathway level. In this method, scores from network enrichment analysis transform the original space of altered genes into a lower-dimensional space of pathways, which is then correlated with phenotype variables. The analysis was first done on in vitro anti-cancer drug screen datasets and then on clinical data. In parallel, we tested a panel of state-of-the-art enrichment methods. In this comparison, our method proved superior in terms of 1) universal applicability to different data types with a possibility of cross-platform integration, 2) consistency of the discovered correlates between independent drug screens, and 3) ability to explain differential survival of treated patients. Our new in vitro screen validated performance of the discovered multivariate models. Finally, NEAmarker was the only method to discover predictors of both in vitro response and patient survival given administration of the same drug.',\n",
       "  'published': '10.1038/s41598-019-39019-2',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/302679',\n",
       "  'title': 'Draft Aphaenogaster genomes expand our view of ant genome size variation across climate gradients',\n",
       "  'authors': 'Lau, M.; Ellison, A. M.; Nguyen, A. D.; Penick, C.; DeMarco, B.; Gotelli, N. J.; Sanders, N.; Dunn, R.; Helms Cahan, S.',\n",
       "  'author_corresponding': 'Matthew  Lau',\n",
       "  'author_corresponding_institution': 'Harvard University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '4',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/302679.source.xml',\n",
       "  'abstract': 'Given the abundance, broad distribution, and diversity of roles that ants play in many ecosystems, they are an ideal group to serve as ecosystem indicators of climatic change. At present, only a few whole-genome sequences of ants are available (19 of > 16,000 species), mostly from tropical and sub-tropical species. To address this limited sampling, we sequenced genomes of temperate-latitude species from the genus Aphaenogaster, a genus with important seed dispersers. In total, we sampled seven colonies of six species: A. ashmeadi, A. floridana, A. fulva, A. miamiana, A. picea, and A. rudis. The geographic ranges of these species collectively span eastern North America from southern Florida to southern Canada, which encompasses a latitudinal gradient in which many climatic variables are changing rapidly. For the six genomes, we assembled an average of 271,039 contigs into 47,337 scaffolds. The mean genome size was 370.5 Mb, ranging from 310.3 to 429.7, which is comparable to that of other sequenced ant genomes (212.8 to 396.0 Mb) and flow cytometry estimates (210.7 to 690.4 Mb). In an analysis of currently sequenced ant genomes and the new Aphaenogaster sequences, we found that after controlling for both spatial autocorrelation and phylogenetics ant genome size was marginally correlated with sample site climate similarity. Of all examined climate variables, minimum temperature showed the strongest correlation with genome size, with ants from locations with colder minimum temperatures having larger genomes. These results suggest that temperature extremes could be a selective force acting on ant genomes and point to the need for more extensive sequencing of ant genomes.',\n",
       "  'published': '10.7717/peerj.6447',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/304832',\n",
       "  'title': 'Development of the P300 from childhood to adulthood: a multimodal EEG and MRI study',\n",
       "  'authors': 'Overbye, K.; Huster, R. J.; Walhovd, K. B.; Fjell, A. M.; Tamnes, C. K.',\n",
       "  'author_corresponding': 'Knut  Overbye',\n",
       "  'author_corresponding_institution': 'Department of Psychology, University of Oslo',\n",
       "  'date': '2018-08-25',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/304832.source.xml',\n",
       "  'abstract': 'Maturation of attentional processes is central to cognitive development. The electrophysiological P300 is associated with rapid allocation of attention, and bridges stimulus and response processing. P300 is among the most studied and robust electrophysiological components, but how different subcomponents of the P300 develop from childhood to adulthood and relate to structural properties of the cerebral cortex is not well understood. We investigated age-related differences in both early visual and P300 components, and how individual differences in these components related to cortical structure in a cross-sectional sample of participants 8-19 years (n=86). Participants completed a three-stimulus visual oddball task while high-density EEG was recorded. Cortical surface area and thickness were estimated from T1-weighthed MRI. Group-level blind source separation of the EEG data identified two P300-like components, a fronto-central P300 and a parietal P300, as well as a component reflecting N1 and P2. Differences in activity across age were found for the parietal P300, N1 and P2, with the parietal P300 showing stronger activity for older participants, while N1 and P2 were stronger for younger participants. Stronger P300 components were positively associated with task performance, independently of age, while negative associations were found for P2 strength. Parietal P300 strength was age-independently associated with larger surface area in a region in left lateral inferior temporal cortex. We suggest that the age differences in component strength reflect development of attentional mechanisms, with increased brain responses to task-relevant stimuli representing an increasing ability to focus on relevant information and to respond accurately and efficiently.',\n",
       "  'published': '10.1007/s00429-018-1755-5',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/307165',\n",
       "  'title': 'The genome of Ectocarpus subulatus highlights unique mechanisms for stress tolerance in brown algae',\n",
       "  'authors': 'Dittami, S. M.; Corre, E.; Brillet-Gueguen, L.; Pontoizeau, N.; Aite, M.; Avia, K.; Caron, C.; Cho, C. H.; Collen, J.; Cormier, A.; Delage, L.; Doubleau, S.; Frioux, C.; Gobet, A.; Gonzalez-Navarrete, I.; Groisillier, A.; Herve, C.; Jollivet, D.; KleinJan, H.; Leblanc, C.; Lipinska, A. P.; Liu, X.; Marie, D.; Markov, G. V.; Minoche, A. E.; Monsoor, M.; Pericard, P.; Perrineau, M.-M.; Peters, A. F.; Siegel, A.; Simeon, A.; Trottier, C.; Yoon, H. S.; Himmelbauer, H.; Boyen, C.; Tonon, T.',\n",
       "  'author_corresponding': 'Simon M. Dittami',\n",
       "  'author_corresponding_institution': 'Sorbonne Universite, CNRS, Integrative Biology of Marine Models, Station Biologique de Roscoff',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/307165.source.xml',\n",
       "  'abstract': 'Brown algae are multicellular photosynthetic stramenopiles that colonize marine rocky shores worldwide. Ectocarpus sp. Ec32 has been established as a genomic model for brown algae. Here we present the genome and metabolic network of the closely related species, Ectocarpus subulatus Kutzing, which is characterized by high abiotic stress tolerance. Since their separation, both strains show new traces of viral sequences and the activity of large retrotransposons, which may also be related to the expansion of a family of chlorophyll-binding proteins. Further features suspected to contribute to stress tolerance include an expanded family of heat shock proteins, the reduction of genes involved in the production of halogenated defence compounds, and the presence of fewer cell wall polysaccharide-modifying enzymes. Overall, E. subulatus has mainly lost members of gene families down-regulated in low salinities, and conserved those that were up-regulated in the same condition. However, 96% of genes that differed between the two examined Ectocarpus species, as well as all genes under positive selection, were found to encode proteins of unknown function. This underlines the uniqueness of brown algal stress tolerance mechanisms as well as the significance of establishing E. subulatus as a comparative model for future functional studies.',\n",
       "  'published': '10.1016/j.margen.2020.100740',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/307322',\n",
       "  'title': 'When higher carrying capacities lead to faster propagation',\n",
       "  'authors': 'Haond, M.; Morel-Journel, T.; Lombaert, E.; Vercken, E.; Mailleret, L.; Roques, L.',\n",
       "  'author_corresponding': 'Marjorie  Haond',\n",
       "  'author_corresponding_institution': \"Universite Cote d'Azur, INRA, CNRS, ISA, France\",\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/307322.source.xml',\n",
       "  'abstract': 'This preprint has been reviewed and recommended by Peer Community In Ecology (https://dx.doi.org/10.24072/pci.ecology.100004). Finding general patterns in the expansion of natural populations is a major challenge in ecology and invasion biology. Classical spatio-temporal models predict that the carrying capacity (K) of the environment should have no influence on the speed (v) of an expanding population. We tested the generality of this statement with reaction-diffusion equations, stochastic individual-based models, and microcosms experiments with Trichogramma chilonis wasps. We investigated the dependence between K and v under different assumptions: null model (Fisher-KPP-like assumptions), strong Allee effects, and positive density-dependent dispersal. These approaches led to similar and complementary results. Strong Allee effects, positive density-dependent dispersal and demographic stochasticity in small populations lead to a positive dependence between K and v. A positive correlation between carrying capacity and propagation speed might be more frequent than previously expected, and be the rule when individuals at the edge of a population range are not able to fully drive the expansion.',\n",
       "  'published': '10.24072/pcjournal.66',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/307264',\n",
       "  'title': 'The evolution of CHC22 clathrin influences its role in human glucose metabolism',\n",
       "  'authors': 'Fumagalli, M.; Diekmann, Y.; Camus, S. M.; Burke, A.; Camus, M. D.; Norman, P. J.; Joseph, A.-P.; Abi-Rached, L.; Benazzo, A.; Rasteiro, R.; Mathieson, I.; Topf, M.; Parham, P.; Thomas, M. G.; Brodsky, F. M.',\n",
       "  'author_corresponding': 'Frances M Brodsky',\n",
       "  'author_corresponding_institution': 'University College London',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc',\n",
       "  'category': 'genetics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/307264.source.xml',\n",
       "  'abstract': 'CHC22 clathrin plays a key role in intracellular membrane traffic of the insulin-responsive glucose transporter GLUT4 in humans. We performed population genetic and phylogenetic analyses of the CHC22-encoding CLTCL1 gene, revealing independent gene loss in at least two vertebrate lineages, after arising from gene duplication. All vertebrates retained the paralogous CLTC gene encoding CHC17 clathrin, which mediates endocytosis. For vertebrates retaining CLTCL1, strong evidence for purifying selection supports CHC22 functionality. All human populations maintained two high frequency CLTCL1 allelic variants, encoding either methionine or valine at position 1316. Functional studies indicated that CHC22-V1316, which is more frequent in farming populations than in hunter-gatherers, has different cellular dynamics than M1316-CHC22 and is less effective at controlling GLUT4 membrane traffic, attenuating its insulin-regulated response. These analyses suggest that ancestral human dietary change influenced selection of allotypes that affect CHC22s role in metabolism and have potential to differentially influence the human insulin response.',\n",
       "  'published': '10.7554/eLife.41517',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/310227',\n",
       "  'title': 'Molecular Strategy for Blocking Isopeptide Bond Formation in Nascent Pilin Proteins',\n",
       "  'authors': 'Rivas-Pardo, J. A.; Tapia-Rojo, R.; Badilla, C. L.; Alonso Caballero, A.; Fernandez, J. M.',\n",
       "  'author_corresponding': 'Jaime Andres  Rivas-Pardo',\n",
       "  'author_corresponding_institution': 'Columbia University',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'biophysics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/310227.source.xml',\n",
       "  'abstract': 'Bacteria anchor to their host cells through their adhesive pili, which must resist the large mechanical stresses induced by the host as it attempts to dislodge the pathogens. The pili of Gram-positive bacteria are constructed as a single polypeptide made of hundreds of pilin repeats, which contain intramolecular isopeptide bonds strategically located in the structure to prevent their unfolding under force, protecting the pilus from degradation by extant proteases and oxygen radicals. Here, we demonstrate the design of a short peptide that blocks the formation of the isopeptide bond present in the pilin Spy0128 from the human pathogen Streptococcus pyogenes, resulting in mechanically labile pilin domains. We use a combination of protein engineering and AFM force spectroscopy to demonstrate that the peptide blocks the formation of the native isopeptide bond and compromises the mechanics of the domain. While an intact Spy0128 is inextensible at any force, peptide-modified Spy0128 pilins readily unfold at very low forces, marking the abrogation of the intramolecular isopeptide bond as well as the absence of a stable pilin fold. We propose that isopeptide-blocking peptides could be further developed as a novel type of highly-specific anti-adhesive antibiotics to treat Gram-positive pathogens.\\\\n\\\\nSignificanceAt the onset of an infection, Gram-positive bacteria adhere to host cells through their pili, filamentous structures built by hundreds of repeats of pilin proteins. These proteins can withstand large mechanical challenges without unfolding, remaining anchored to the host and resisting cleavage by proteases and oxygen radicals present in the targeted tissues. The key structural component that gives pilins mechanical resilience are internal isopeptide bonds, strategically placed so that pilins become inextensible structures. We target this bond by designing a blocking peptide that interferes with its formation during folding. We demonstrate that peptide-modified pilins lack mechanical stability and extend at low forces. We propose this strategy as a rational design of mechanical antibiotics, targeting the Achilles Heel of bacterial adhesion.',\n",
       "  'published': '10.1073/pnas.1807689115',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/310151',\n",
       "  'title': 'A consistent picture of TRPV1 activation emerges from molecular simulations and experiments',\n",
       "  'authors': 'Kasimova, M.; Yazici, A.; Yudin, Y.; Granata, D.; Klein, M. L.; Rohacs, T.; Carnevale, V.',\n",
       "  'author_corresponding': 'Vincenzo  Carnevale',\n",
       "  'author_corresponding_institution': 'Temple University',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'biophysics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/310151.source.xml',\n",
       "  'abstract': 'Although the structure of TRPV1 has been experimentally determined in both the closed and open states, very little is known about its activation mechanism. In particular, the conformational changes occurring in the pore domain and resulting in ionic conduction have not been identified yet. Here, we suggest a hypothetical molecular mechanism for TRPV1 activation, which involves the rotation of a conserved asparagine in S6 from the S4-S5 linker toward the pore. This rotation is correlated with the dehydration of four peripheral cavities located between S6 and the S4-S5 linker and the hydration of the pore. In light of our hypothesis, we perform bioinformatics analyses of TRP and other evolutionary related ion channels, analyze newly available structures and re-examine previously reported water accessibility and mutagenesis experiments. Overall, we provide several independent lines of evidence that corroborate our hypothesis. Finally, we show that the proposed molecular mechanism is compatible with the currently existing idea that in TRPV1 the selectivity filter acts as a secondary gate.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/311050',\n",
       "  'title': 'Physical activity predicts population-level age-related differences in frontal white matter',\n",
       "  'authors': 'Strommer, J. M.; Davis, S. W.; Henson, R. N.; Tyler, L. K.; Cam-CAN,  ; Campbell, K. L.',\n",
       "  'author_corresponding': 'Juho M. Strommer',\n",
       "  'author_corresponding_institution': 'University of Jyvaskyla',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/311050.source.xml',\n",
       "  'abstract': 'Physical activity has positive effects on brain health and cognitive function throughout the lifespan. Thus far, few studies have examined the effects of physical activity on white matter (WM) microstructure and psychomotor speed within the same, population-based sample (critical if conclusions are to extend to the wider population). Here, using diffusion tensor imaging and a simple reaction time task within a relatively large population-derived sample (N = 399; 18-87 years) from the Cambridge Centre for Ageing and Neuroscience (Cam-CAN), we demonstrate that physical activity mediates the effect of age on white matter integrity, measured with fractional anisotropy. Higher self-reported daily physical activity was associated with greater preservation of WM in several frontal tracts, including the genu of corpus callosum, uncinate fasciculus, external capsule and anterior limb of the internal capsule. We also show that the age-related slowing is mediated by WM integrity in the genu. Our findings contribute to a growing body of work suggesting that a physically active lifestyle may protect against age-related structural disconnection and slowing.',\n",
       "  'published': '10.1093/gerona/gly220',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/247411',\n",
       "  'title': 'Exploring the genetic correlations of antisocial behavior and life history traits',\n",
       "  'authors': 'Tielbeek, J.; Barnes, J. C.; Popma, A.; Polderman, T.; Lee, J. J.; Perry, J. B.; Posthuma, D.; Boutwell, B. B.',\n",
       "  'author_corresponding': 'Brian B Boutwell',\n",
       "  'author_corresponding_institution': 'Saint Louis University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/247411.source.xml',\n",
       "  'abstract': 'Prior evolutionary theory provided reason to suspect that measures of development and reproduction would be correlated with antisocial behaviors in human and non-human species. Behavioral genetics has revealed that most quantitative traits are heritable, suggesting that these phenotypic correlations may share genetic etiologies. We use GWAS data to estimate the genetic correlations between various measures of reproductive development (N= 52,776 - 318,863) and antisocial behavior (N= 31,968). Our genetic correlation analyses demonstrate that alleles associated with higher reproductive output (number of children ever born, rg=0.50, p=.0065) were positively correlated with alleles associated with antisocial behavior, whereas alleles associated with more delayed reproductive onset (age of first birth, rg=-.64, p=.0008) were negatively associated with alleles linked to antisocial behavior. Ultimately, these findings coalesce with evolutionary theories suggesting that increased antisocial behaviors may partly represent a faster life history approach, which may be significantly calibrated by genes.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/312355',\n",
       "  'title': 'Germline genomic landscapes of breast cancer patients significantly predict clinical outcomes',\n",
       "  'authors': 'Milanese, J.-S.; Tibiche, C.; Zaman, N.; Zou, J.; Han, P.; Meng, Z. G.; Nantel, A.; Droit, A.; Wang, E.',\n",
       "  'author_corresponding': 'Edwin  Wang',\n",
       "  'author_corresponding_institution': 'University Of Calgary',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'cancer biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/312355.source.xml',\n",
       "  'abstract': 'Germline genetic variants such as BRCA1/2 play an important role in tumorigenesis and clinical outcomes of cancer patients. However, only a small fraction (i.e., 5-10%) of inherited variants has been associated with clinical outcomes (e.g., BRCA1/2, APC, TP53, PTEN and so on). The challenge remains in using these inherited germline variants to predict clinical outcomes of cancer patient population. In an attempt to solve this issue, we applied our recently developed algorithm, eTumorMetastasis, which constructs predictive models, on exome sequencing data to ER+ breast (n=755) cancer patients. Gene signatures derived from the genes containing functionally germline genetic variants significantly distinguished recurred and non-recurred patients in two ER+ breast cancer independent cohorts (n=200 and 295, P=1.4x10-3). Furthermore, we found that recurred patients possessed a higher rate of germline genetic variants. In addition, the inherited germline variants from these gene signatures were predominately enriched in T cell function, antigen presentation and cytokine interactions, likely impairing the adaptive and innate immune response thus favoring a pro-tumorigenic environment. Hence, germline genomic information could be used for developing non-invasive genomic tests for predicting patients outcomes (or drug response) in breast cancer, other cancer types and even other complex diseases.',\n",
       "  'published': '10.1038/s41698-019-0100-7',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/313296',\n",
       "  'title': 'CHASMplus reveals the scope of somatic missense mutations driving human cancers',\n",
       "  'authors': 'Tokheim, C.; Karchin, R.',\n",
       "  'author_corresponding': 'Rachel  Karchin',\n",
       "  'author_corresponding_institution': 'Johns Hopkins University',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/313296.source.xml',\n",
       "  'abstract': 'Large-scale cancer sequencing studies of patient cohorts have statistically implicated many genes driving cancer growth and progression, and their identification has yielded substantial translational impact. However, a remaining challenge is to increase the resolution of driver prediction from the gene level to the mutation level, because mutation-level predictions are more closely aligned with the goal of precision cancer medicine. Here we present CHASMplus, a computational method, that is uniquely capable of identifying driver missense mutations, including those specific to a cancer type, as evidenced by significantly superior performance on diverse benchmarks. Applied to 8,657 tumor samples across 32 cancer types in The Cancer Genome Atlas, CHASMplus identifies over 4,000 unique driver missense mutations in 240 genes, supporting a prominent role for rare driver mutations. We show which TCGA cancer types are likely to yield discovery of new driver missense mutations by additional sequencing, which has important implications for public policy.\\n\\nSignificanceMissense mutations are the most frequent mutation type in cancers and the most difficult to interpret. While many computational methods have been developed to predict whether genes are cancer drivers or whether missense mutations are generally deleterious or pathogenic, there has not previously been a method to score the oncogenic impact of a missense mutation specifically by cancer type, limiting adoption of computational missense mutation predictors in the clinic. Cancer patients are routinely sequenced with targeted panels of cancer driver genes, but such genes contain a mixture of driver and passenger missense mutations which differ by cancer type. A patients therapeutic response to drugs and optimal assignment to a clinical trial depends on both the specific mutation in the gene of interest and cancer type. We present a new machine learning method honed for each TCGA cancer type, and a resource for fast lookup of the cancer-specific driver propensity of every possible missense mutation in the human exome.',\n",
       "  'published': '10.1016/j.cels.2019.05.005',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/312827',\n",
       "  'title': 'EEG-based auditory attention detection: boundary conditions for background noise and speaker positions',\n",
       "  'authors': 'Das, N.; Bertrand, A.; Francart, T.',\n",
       "  'author_corresponding': 'Neetha  Das',\n",
       "  'author_corresponding_institution': 'KU Leuven',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'bioengineering',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/312827.source.xml',\n",
       "  'abstract': 'ObjectiveA listeners neural responses can be decoded to identify the speaker the person is attending to in a cocktail party environment. Such auditory attention detection methods have the potential to provide noise suppression algorithms in hearing devices with information about the listeners attention. A challenge is the effect of noise and other acoustic conditions that can reduce the attention detection accuracy. Specifically, noise can impact the ability of the person to segregate the sound sources and perform selective attention, as well as the external signal processing necessary to decode the attention effectively. The aim of this work is to systematically analyze the effect of noise level and speaker position on attention decoding accuracy.\\\\n\\\\nApproach28 subjects participated in the experiment. Auditory stimuli consisted of stories narrated by different speakers from 2 different locations, along with surrounding multi-talker background babble. EEG signals of the subjects were recorded while they focused on one story and ignored the other. The strength of the babble noise as well as the spatial separation between the two speakers were varied between presentations. Spatio-temporal decoders were trained for each subject, and applied to decode attention of the subjects from every 30s segment of data. Behavioral speech recognition thresholds were obtained for the different speaker separations.\\\\n\\\\nMain resultsBoth the background noise level and the angular separation between speakers affected attention decoding accuracy. Remarkably, attention decoding performance was seen to increase with the inclusion of moderate background noise (versus no noise), while across the different noise conditions performance dropped significantly with increasing noise level. We also observed that decoding accuracy improved with increasing speaker separation, exhibiting the advantage of spatial release from masking. Furthermore, the effect of speaker separation on the decoding accuracy became stronger when the background noise level increased. A significant correlation between speech intelligibility and attention decoding accuracy was found across conditions.\\\\n\\\\nSignificanceThis work shows how the background noise level and relative positions of competing talkers impact attention decoding accuracy. It indicates in which circumstances a neuro-steered noise suppression system may need to operate, in function of acoustic conditions. It also indicates the boundary conditions for the operation of EEG-based attention detection systems in neuro-steered hearing prostheses.\\\\n\\\\nIndex TermsAuditory attention detection, EEG processing, neuro-steered auditory prostheses, brain-computer interface, cocktail party, acoustic conditions.\\\\n\\\\nThe work is funded by KU Leuven Special Research Fund C14/16/057 and OT/14/119, FWO project nrs. 1.5.123.16N and G0A4918N, the ERC (637424) under the European Unions Horizon 2020 research and innovation programme, and a research gift of Starkey Hearing Technologies. The scientific responsibility is assumed by its authors.',\n",
       "  'published': '10.1088/1741-2552/aae0a6',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/312397',\n",
       "  'title': 'A practical generation interval-based approach to inferring the strength of epidemics from their speed',\n",
       "  'authors': 'Park, S. W.; Champredon, D.; Weitz, J.; Dushoff, J.',\n",
       "  'author_corresponding': 'Jonathan  Dushoff',\n",
       "  'author_corresponding_institution': 'McMaster University',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'epidemiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/312397.source.xml',\n",
       "  'abstract': 'Infectious-disease outbreaks are often characterized by the reproductive number and exponential rate of growth r. provides information about out-break control and predicted final size. Directly estimating is difficult, while r can often be estimated from incidence data. These quantities are linked by the generation interval - the time between when an individual is infected by an infector, and when that infector was infected. It is often infeasible to ob-tain the exact shape of a generation-interval distribution, and to understand how this shape affects estimates of . We show that estimating generation interval mean and variance provides insight into the relationship between and r. We use examples based on Ebola, rabies and measles to explore approximations based on gamma-distributed generation intervals, and find that use of these simple approximations are often sufficient to capture the r- relationship and provide robust estimates of .',\n",
       "  'published': '10.1016/j.epidem.2018.12.002',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/317511',\n",
       "  'title': 'Omitted variable bias in GLMs of neural spiking activity',\n",
       "  'authors': 'Stevenson, I. H.',\n",
       "  'author_corresponding': 'Ian H Stevenson',\n",
       "  'author_corresponding_institution': 'University of Connecticut',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/317511.source.xml',\n",
       "  'abstract': 'Generalized linear models (GLMs) have a wide range of applications in systems neuroscience describing the encoding of stimulus and behavioral variables as well as the dynamics of single neurons. However, in any given experiment, many variables that impact neural activity are not observed or not modeled. Here we demonstrate, in both theory and practice, how these omitted variables can result in biased parameter estimates for the effects that are included. In three case studies, we estimate tuning functions for common experiments in motor cortex, hippocampus, and visual cortex. We find that including traditionally omitted variables changes estimates of the original parameters and that modulation originally attributed to one variable is reduced after new variables are included. In GLMs describing single-neuron dynamics, we then demonstrate how post-spike history effects can also be biased by omitted variables. Here we find that omitted variable bias can lead to mistaken conclusions about the stability of single neuron firing. Omitted variable bias can appear in any model with confounders - where omitted variables modulate neural activity and the effects of the omitted variables covary with the included effects. Understanding how and to what extent omitted variable bias affects parameter estimates is likely to be important for interpreting the parameters and predictions of many neural encoding models.',\n",
       "  'published': '10.1162/neco_a_01138',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/319657',\n",
       "  'title': 'Aurora B is required for programmed variations of cytokinesis during morphogenesis in the C. elegans embryo',\n",
       "  'authors': 'Bai, X.; Lee, P.-Y.; Chen, C.-Y.; Simmons, J. R.; Nebenfuehr, B.; Mitchell, D.; Klebanow, L. R.; Mattson, N.; Sorensen Turpin, C. G.; Chen, B.-C.; Betzig, E.; Bembenek, J. N.',\n",
       "  'author_corresponding': 'Joshua N Bembenek',\n",
       "  'author_corresponding_institution': 'UT Knoxville',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'cell biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/319657.source.xml',\n",
       "  'abstract': 'While cytokinesis has been intensely studied, how it is executed during development is not well understood, despite a long-standing appreciation that various aspects of cytokinesis vary across cell and tissue types. To address this, we investigated cytokinesis during the invariant C. elegans embryo lineage and found several reproducibly altered parameters at different stages. During early divisions, furrow ingression asymmetry and midbody inheritance is consistent, suggesting specific regulation of these events. During morphogenesis, we find several unexpected alterations including migration of midbodies to the apical surface during epithelial polarization in different tissues. Aurora B kinase, which is essential for several aspects of cytokinesis, remains localized to the apical membrane after internalization of other midbody components. Inactivation of Aurora B causes cytokinesis failure, which disrupts polarization and tissue formation. Therefore, cytokinesis shows surprising diversity during development and is required during epithelial polarization to establish cellular architecture during morphogenesis.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/320473',\n",
       "  'title': 'Poor coherence in older people’s speech is explained by impaired semantic and executive processes',\n",
       "  'authors': 'Hoffman, P.; Loginova, E.; Russell, A.',\n",
       "  'author_corresponding': 'Paul  Hoffman',\n",
       "  'author_corresponding_institution': 'University of Edinburgh',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/320473.source.xml',\n",
       "  'abstract': 'The ability to speak coherently is essential for effective communication but declines with age: older people more frequently produce tangential, off-topic speech. The cognitive factors underpinning this decline are poorly understood. We predicted that maintaining coherence relies on effective regulation of activated semantic knowledge about the world, and particularly on the selection of currently relevant semantic representations to drive speech production. To test this, we collected 840 speech samples along with measures of executive and semantic ability from 60 young and older adults, using a novel computational method to quantify coherence. Semantic selection ability predicted coherence, as did level of semantic knowledge and a measure of domain-general executive ability. These factors fully accounted for the age-related coherence deficit. Our results indicate that maintaining coherence in speech becomes more challenging as people age because they accumulate more knowledge but are less able to effectively regulate how it is activated and used.',\n",
       "  'published': '10.7554/eLife.38907',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/321067',\n",
       "  'title': 'RAVEN 2.0: a versatile toolbox for metabolic network reconstruction and a case study on Streptomyces coelicolor',\n",
       "  'authors': 'Wang, H.; Marcisauskas, S.; Sanchez, B. J.; Domenzain, I.; Hermansson, D.; Agren, R.; Nielsen, J.; Kerkhoven, E. J.',\n",
       "  'author_corresponding': 'Eduard Johannes Kerkhoven',\n",
       "  'author_corresponding_institution': 'Chalmers University of Technology',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'systems biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/321067.source.xml',\n",
       "  'abstract': 'RAVEN is a commonly used MATLAB toolbox for genome-scale metabolic model (GEM) reconstruction, curation and constraint-based modelling and simulation. Here we present RAVEN Toolbox 2.0 with major enhancements, including: (i) de novo reconstruction of GEMs based on the MetaCyc pathway database; (ii) a redesigned KEGG-based reconstruction pipeline; (iii) convergence of reconstructions from various sources; (iv) improved performance, usability, and compatibility with the COBRA Toolbox. Capabilities of RAVEN 2.0 are here illustrated through de novo reconstruction of GEMs for the antibiotic-producing bacterium Streptomyces coelicolor. Comparison of the automated de novo reconstructions with the iMK1208 model, a previously published high-quality S. coelicolor GEM, exemplifies that RAVEN 2.0 can capture most of the manually curated model. The generated de novo reconstruction is subsequently used to curate iMK1208 resulting in Sco4, the most comprehensive GEM of S. coelicolor, with increased coverage of both primary and secondary metabolism. This increased coverage allows the use of Sco4 to predict novel genome editing targets for optimized secondary metabolites production. As such, we demonstrate that RAVEN 2.0 can be used not only for de novo GEM reconstruction, but also for curating existing models based on up-to-date databases. Both RAVEN 2.0 and Sco4 are distributed through GitHub to facilitate usage and further development by the community.\\\\n\\\\nAuthor summaryCellular metabolism is a large and complex network. Hence, investigations of metabolic networks are aided by in silico modelling and simulations. Metabolic networks can be derived from whole-genome sequences, through identifying what enzymes are present and connecting these to formalized chemical reactions. To facilitate the reconstruction of genome-scale models of metabolism (GEMs), we have developed RAVEN 2.0. This versatile toolbox can reconstruct GEMs fast, through either metabolic pathway databases KEGG and MetaCyc, or from homology with an existing GEM. We demonstrate RAVENs functionality through generation of a metabolic model of Streptomyces coelicolor, an antibiotic-producing bacterium. Comparison of this de novo generated GEM with a previously manually curated model demonstrates that RAVEN captures most of the previous model, and we subsequently reconstructed an updated model of S. coelicolor: Sco4. Following, we used Sco4 to predict promising targets for genetic engineering, which can be used to increase antibiotic production.',\n",
       "  'published': '10.1371/journal.pcbi.1006541',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/318279',\n",
       "  'title': 'Effects of Attentional Bias Modification on Residual Symptoms in depression. A Randomized Controlled Trial.',\n",
       "  'authors': 'Jonassen, R.; Harmer, C. J.; Hilland, E.; Maglanoc, L. A.; Kraft, B.; Browning, M.; Stiles, T. C.; Haaland, V. O.; Berge, T.; Landro, N. I.',\n",
       "  'author_corresponding': 'Rune  Jonassen',\n",
       "  'author_corresponding_institution': 'Department of Psychology, University of Oslo, Norway',\n",
       "  'date': '2018-08-22',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'clinical trials',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/318279.source.xml',\n",
       "  'abstract': 'BackgroundFollowing treatment, many depressed patients have significant residual symptoms. However, large randomised controlled trials (RCT) in this population are lacking. When Attention bias modification training (ABM) leads to more positive emotional biases, associated changes in clinical symptoms have been reported. A broader and more transparent picture of the true advantage of ABM based on larger and more stringent clinical trials have been requested.\\\\n\\\\nAimsTo evaluate the early effect of two weeks ABM training on blinded clinician-rated and self-reported residual symptoms, and whether changes towards more positive attentional biases (AB) would be associated with symptom reduction.\\\\n\\\\nMethodA total of 321 patients with a history of depression were included in a preregistered randomized controlled double-blinded trial. Patients were randomised to an emotional ABM paradigm over fourteen days or a closely matched control condition. Symptoms based on the Hamilton Rating Scale for Depression (HRSD) and Beck Depression Inventory II (BDI-II) were obtained at baseline and after ABM training.\\\\n\\\\nResultsABM training led to significantly greater decrease in clinician-rated symptoms of depression as compared to the control condition. No differences between ABM and placebo were found for self-reported symptoms. ABM induced a change of AB towards relatively more positive stimuli associated with greater symptom reduction.\\\\n\\\\nConclusionThe current study demonstrates that ABM produces early changes in both AB and blinded clinician-rated depressive symptoms. ABM may have practical potential in the treatment of residual depression.\\\\n\\\\nClinicalTrials.gov ID: NCT02658682',\n",
       "  'published': '10.1186/s12888-019-2105-8',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/322008',\n",
       "  'title': 'Lead-DBS v2: Towards a comprehensive pipeline for deep brain stimulation imaging',\n",
       "  'authors': 'Horn, A.; Li, N.; Dembek, T. A.; Kappel, A.; Boulay, C.; Ewert, S.; Tietze, A.; Husch, A.; Perera, T.; Neumann, W.-J.; Reisert, M.; Si, H.; Oostenveld, R.; Rorden, C.; Yeh, F.-C.; Fang, Q.; Herrington, T. M.; Vorwerk, J.; Kuehn, A. A.',\n",
       "  'author_corresponding': 'Andreas  Horn',\n",
       "  'author_corresponding_institution': 'Department for Neurology, Charite Berlin',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/322008.source.xml',\n",
       "  'abstract': 'Deep brain stimulation (DBS) is a highly efficacious treatment option for movement disorders and a growing number of other indications are investigated in clinical trials. To ensure optimal treatment outcome, exact electrode placement is required. Moreover, to analyze the relationship between electrode location and clinical results, a precise reconstruction of electrode placement is required, posing specific challenges to the field of neuroimaging. Since 2014 the open source toolbox Lead-DBS is available, which aims at facilitating this process. The tool has since become a popular platform for DBS imaging. With support of a broad community of researchers worldwide, methods have been continuously updated and complemented by new tools for tasks such as multispectral nonlinear registration, structural / functional connectivity analyses, brain shift correction, reconstruction of microelectrode recordings and orientation detection of segmented DBS leads. The rapid development and emergence of these methods in DBS data analysis require us to revisit and revise the pipelines introduced in the original methods publication. Here we demonstrate the updated DBS and connectome pipelines of Lead-DBS using a single patient example with state-of-the-art high-field imaging as well as a retrospective cohort of patients scanned in a typical clinical setting at 1.5T. Imaging data of the 3T example patient is co-registered using five algorithms and nonlinearly warped into template space using ten approaches for comparative purposes. After reconstruction of DBS electrodes (which is possible using three methods and a specific refinement tool), the volume of tissue activated is calculated for two DBS settings using four distinct models and various parameters. Finally, four whole-brain tractography algorithms are applied to the patients preoperative diffusion MRI data and structural as well as functional connectivity between the stimulation volume and other brain areas are estimated using a total of eight approaches and datasets. In addition, we demonstrate impact of selected preprocessing strategies on the retrospective sample of 51 PD patients. We compare the amount of variance in clinical improvement that can be explained by the computer model depending on the method of choice.\\\\n\\\\nThis work represents a multi-institutional collaborative effort to develop a comprehensive, open source pipeline for DBS imaging and connectomics, which has already empowered several studies, and may facilitate a variety of future studies in the field.',\n",
       "  'published': '10.1016/j.neuroimage.2018.08.068',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/325399',\n",
       "  'title': 'Common genomic regions underlie natural variation in diverse toxin responses',\n",
       "  'authors': 'Brady, S.; Evans, K.; Bloom, J.; Tanny, R.; Cook, D.; Giuliani, S.; Hippleheuser, S.; Zamanian, M.; Andersen, E.',\n",
       "  'author_corresponding': 'Erik  Andersen',\n",
       "  'author_corresponding_institution': 'Northwestern University',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'genetics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/325399.source.xml',\n",
       "  'abstract': 'Phenotypic complexity is caused by the contributions of environmental factors and multiple genetic loci, interacting or acting independently. Studies of yeast and Arabidopsis found that the majority of natural variation across phenotypes is attributable to independent additive quantitative trait loci (QTL). Detected loci in these organisms explain most of the estimated heritable variation. By contrast, many heritable components underlying phenotypic variation in metazoan models remain undetected. Before the relative impacts of additive and interactive variance components on metazoan phenotypic variation can be dissected, high replication and precise phenotypic measurements are required to obtain sufficient statistical power to detect loci contributing to this missing heritability. Here, we used a panel of 296 recombinant inbred advanced intercross lines of Caenorhabditis elegans and a high-throughput fitness assay to detect loci underlying responses to 16 different toxins, including heavy metals, chemotherapeutic drugs, pesticides, and neuropharmaceuticals. Using linkage mapping, we identified 82 QTL that underlie variation in responses to these toxins and predicted the relative contributions of additive loci and genetic interactions across various growth parameters. Additionally, we identified three genomic regions that impact responses to multiple classes of toxins. These QTL hotspots could represent common factors impacting toxin responses. We went further to generate near-isogenic lines and chromosome-substitution strains and then experimentally validated these QTL hotspots, implicating additive and interactive loci that underlie toxin-response variation.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/326470',\n",
       "  'title': 'The UCSC Xena Platform for cancer genomics data visualization and interpretation',\n",
       "  'authors': 'Goldman, M.; Craft, B.; Kamath, A.; Brooks, A. N.; Zhu, J.; Haussler, D.',\n",
       "  'author_corresponding': 'Mary  Goldman',\n",
       "  'author_corresponding_institution': 'UC Santa Cruz Genomics Institute',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'cancer biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/326470.source.xml',\n",
       "  'abstract': 'UCSC Xena is a visual exploration resource for both public and private omics data, supported through the web-based Xena Browser and multiple turn-key Xena Hubs. This unique archecture allows researchers to view their own data securely, using private Xena Hubs, simultaneously visualizing large public cancer genomics datasets, including TCGA and the GDC. Data integration occurs only within the Xena Browser, keeping private data private. Xena supports virtually any functional genomics data, including SNVs, INDELs, large structural variants, CNV, expression, DNA methylation, ATAC-seq signals, and phenotypic annotations. Browser features include the Visual Spreadsheet, survival analyses, powerful filtering and subgrouping, statistical analyses, genomic signatures, and bookmarks. Xena differentiates itself from other genomics tools, including its predecessor, the UCSC Cancer Genomics Browser, by its ability to easily and securely view public and private data, its high performance, its broad data type support, and many unique features.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/326173',\n",
       "  'title': 'Fast open modification spectral library searching through approximate nearest neighbor indexing',\n",
       "  'authors': 'Bittremieux, W.; Meysman, P.; Noble, W. S.; Laukens, K.',\n",
       "  'author_corresponding': 'Wout  Bittremieux',\n",
       "  'author_corresponding_institution': 'University of Antwerp',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/326173.source.xml',\n",
       "  'abstract': 'Open modification searching (OMS) is a powerful search strategy that identifies peptides carrying any type of modification by allowing a modified spectrum to match against its unmodified variant by using a very wide precursor mass window. A drawback of this strategy, however, is that it leads to a large increase in search time. Although performing an open search can be done using existing spectral library search engines by simply setting a wide precursor mass window, none of these tools have been optimized for OMS, leading to excessive runtimes and suboptimal identification results. Here we present the ANN-SoLo tool for fast and accurate open spectral library searching. ANN-SoLo uses approximate nearest neighbor indexing to speed up OMS by selecting only a limited number of the most relevant library spectra to compare to an unknown query spectrum. This approach is combined with a cascade search strategy to maximize the number of identified unmodified and modified spectra while strictly controlling the false discovery rate, as well as a shifted dot product score to sensitively match modified spectra to their unmodified counterparts. ANN-SoLo achieves state-of-the-art performance in terms of speed and the number of identifications. On a previously published human cell line data set, ANN-SoLo confidently identifies more spectra than SpectraST or MSFragger and achieves a speedup of an order of magnitude compared to SpectraST.\\\\n\\\\nANN-SoLo is implemented in Python and C++. It is freely available under the Apache 2.0 license at https://github.com/bittremieux/ANN-SoLo.',\n",
       "  'published': '10.1021/acs.jproteome.8b00359',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/328328',\n",
       "  'title': 'Simultaneous multiplexed amplicon sequencing and transcriptome profiling in single cells',\n",
       "  'authors': 'Saikia, M.; Burnham, P.; Keshavjee, S. H.; Wang, M. F. Z.; Heyang, M.; Moral-Lopez, P.; Hinchman, M. M.; Danko, C. G.; Parker, J. S. L.; De Vlaminck, I.',\n",
       "  'author_corresponding': 'Iwijn  De Vlaminck',\n",
       "  'author_corresponding_institution': 'Cornell University',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/328328.source.xml',\n",
       "  'abstract': 'We describe Droplet Assisted RNA Targeting by single cell sequencing (DART-seq), a versatile technology that enables multiplexed amplicon sequencing and transcriptome profiling in single cells. We applied DART-seq to simultaneously characterize the non-A-tailed transcripts of a segmented dsRNA virus and the transcriptome of the infected cell. In addition, we used DART-seq to simultaneously determine the natively paired, variable region heavy and light chain amplicons and the transcriptome of B lymphocytes.',\n",
       "  'published': '10.1038/s41592-018-0259-9',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/325290',\n",
       "  'title': 'A systematic method for surveying data visualizations and a resulting genomic epidemiology visualization typology: GEViT',\n",
       "  'authors': 'Crisan, A.; Gardy, J. L.; Munzner, T.',\n",
       "  'author_corresponding': 'Tamara  Munzner',\n",
       "  'author_corresponding_institution': 'University of British Columbia',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/325290.source.xml',\n",
       "  'abstract': 'MotivationData visualization is an important tool for exploring and communicating findings from genomic and healthcare datasets. Yet, without a systematic way of organizing and describing the design space of data visualizations, researchers may not be aware of the breadth of possible visualization design choices or how to distinguish between good and bad options.\\\\n\\\\nResultsWe have developed a method that systematically surveys data visualizations using the analysis of both text and images. Our method supports the construction of a visualization design space that is explorable along two axes: why the visualization was created and how it was constructed. We applied our method to a corpus of scientific research articles from infectious disease genomic epidemiology and derived a Genomic Epidemiology Visualization Typology (GEViT) that describes how visualizations were created from a series of chart types, combinations, and enhancements. We have also implemented an online gallery that allows others to explore our resulting design space of visualizations. Our results have important implications for visualization design and for researchers intending to develop or use data visualization tools. Finally, the method that we introduce is extensible to constructing visualizations design spaces across other research areas.\\\\n\\\\nAvailabilityOur browsable gallery is available at http://gevit.net and all project code can be found at https://github.com/amcrisan/gevitAnalysisRelease',\n",
       "  'published': '10.1093/bioinformatics/bty832',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/329300',\n",
       "  'title': 'Bidirectional intraflagellar transport is restricted to two sets of microtubule doublets in the trypanosome flagellum',\n",
       "  'authors': 'Bertiaux, E.; Mallet, A.; Fort, C.; Blisnick, T.; Bonnefoy, S.; Jung, J.; Lemos, M.; Marco, S.; Vaughan, S.; Trepout, S.; Tinevez, J.-Y.; Bastin, P.',\n",
       "  'author_corresponding': 'Philippe  Bastin',\n",
       "  'author_corresponding_institution': 'Institut Pasteur',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'cell biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/329300.source.xml',\n",
       "  'abstract': 'Intraflagellar transport (IFT) is the rapid bidirectional movement of large protein complexes driven by kinesin and dynein motors along microtubule doublets of cilia and flagella. Here we used a combination of high-resolution electron and light microscopy to investigate how and where these IFT trains move within the flagellum of the protist Trypanosoma brucei. Focused Ion Beam Scanning Electron Microscopy (FIB-SEM) analysis of trypanosomes showed that trains are found almost exclusively along two sets of doublets (3-4 and 7-8) and distribute in two categories according to their length. High-resolution live imaging of cells expressing mNeonGreen::IFT81 or GFP::IFT52 revealed for the first time IFT trafficking on two parallel lines within the flagellum. Anterograde and retrograde IFT occur on each of these lines. At the distal end, a large individual anterograde IFT train is converted in several smaller retrograde trains in the space of 3-4 seconds while remaining on the same side of the axoneme.',\n",
       "  'published': '10.1083/jcb.201805030',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/321216',\n",
       "  'title': 'De novo mutations drive the spread of macrolide resistant Mycoplasma genitalium: a mathematical modelling study',\n",
       "  'authors': 'Cadosch, D.; Garcia, V.; Althaus, C. L.; Jensen, J. S.; Low, N.',\n",
       "  'author_corresponding': 'Nicola  Low',\n",
       "  'author_corresponding_institution': 'University of Bern',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'microbiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/321216.source.xml',\n",
       "  'abstract': 'BackgroundThe rapid spread of azithromycin resistance in sexually transmitted Mycoplasma genitalium infections is a growing concern. It is not yet clear to what degree macrolide resistance in M. genitalium results from the emergence of de novo mutations or the transmission of resistant strains.\\\\n\\\\nMethodsWe analyzed epidemiological data and developed a compartmental model to investigate the contribution of de novo macrolide resistance mutations to the spread of antimicrobial-resistant M. genitalium. We fitted the model to data from France, Denmark and Sweden and estimated treatment rates of infected individuals and the time point of azithromycin introduction.\\\\n\\\\nResultsWe found a high probability of de novo resistance (12%, 95% CI 8-17%), which is responsible for the observed rapid spread of antimicrobial resistant M. genitalium. The estimated per capita treatment rate in France was lower than in Denmark and Sweden but confidence intervals for the three estimates overlap. The estimated dates of introduction of azithromycin in each country are consistent with published reports.\\\\n\\\\nConclusionsSince de novo resistance is the main driver of macrolide resistance in M. genitalium, blind treatment of urethritis with azithromycin is not recommended. Clinical management strategies for M. genitalium should limit the unnecessary use of macrolides.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/330241',\n",
       "  'title': 'Relative value perception in an insect: positive and negative incentive contrasts in ants',\n",
       "  'authors': 'Wendt, S.; Strunk, K. S.; Heinze, J.; Roider, A.; Czaczkes, T. J.',\n",
       "  'author_corresponding': 'Stephanie  Wendt',\n",
       "  'author_corresponding_institution': 'University of Regensburg',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'animal behavior and cognition',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/330241.source.xml',\n",
       "  'abstract': 'Humans tend to value things not on their absolute values, but relative to reference points such as former experience or expectations. People rate the quality of a new salary relative to their previous salary and the salaries of their peers, instead of appreciating its absolute value. Here, we demonstrate a similar effect in an insect: ants, which had previously experienced a low quality food source, showed higher acceptance of medium quality food (e.g. 0.1M then 0.5M; positive contrast) than if they had received the medium food all along (e.g. 0.5M then 0.5M; control), and vice versa for high expectations. Further experiments demonstrate that these contrast effects arise from cognitive rather than mere sensory or pre-cognitive perceptual causes. Pheromone deposition also correlates with perceived reward value, and ants showed successive contrasts in their pheromone deposition. Relative value perception can therefore be expected to have strong effects not only on individual behaviour, but also on collective decision-making. Contrast effects were also social: the quality of food received from other ants affected the perceived value of food found later. Value judgement is a key element in decision making, and thus relative value perception will strongly influence how animals interact with their environment.',\n",
       "  'published': '10.7554/eLife.45450',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/330308',\n",
       "  'title': 'Genome Functional Annotation using Deep Convolutional Neural Networks',\n",
       "  'authors': 'Khodabandelou, G.; Routhier, E.; Mozziconacci, J.',\n",
       "  'author_corresponding': 'Ghazaleh  Khodabandelou',\n",
       "  'author_corresponding_institution': 'Sorbonne Universite',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/330308.source.xml',\n",
       "  'abstract': 'Deep neural network application is today a skyrocketing field in many disciplinary domains. In genomics the development of deep neural networks is expected to revolutionize current practice. Several approaches relying on convolutional neural networks have been developed to associate short genomic sequences with a functional role such as promoters, enhancers or protein binding sites along genomes. These approaches rely on the generation of sequences batches with known annotations for learning purpose. While they show good performance to predict annotations from a test subset of these batches, they usually perform poorly when applied genome-wide.\\\\n\\\\nIn this study, we address this issue and propose an optimal strategy to train convolutional neural networks for this specific application. We use as a case study transcription start sites and show that a model trained on one organism can be used to predict transcription start sites in a different specie. This cross-species application of convolutional neural networks trained with genomic sequence data provides a new technique to annotate any genome from previously existing annotations in related species. It also provides a way to determine whether the sequence patterns recognized by chromatin associated proteins in different species are conserved or not.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/339036',\n",
       "  'title': 'A variably imprinted epiallele impacts seed development',\n",
       "  'authors': 'Pignatta, D.; Novitzky, K.; Satyaki, P. R. V.; Gehring, M.',\n",
       "  'author_corresponding': 'Mary  Gehring',\n",
       "  'author_corresponding_institution': 'Whitehead Institute for Biomedical Research',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'plant biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/339036.source.xml',\n",
       "  'abstract': 'The contribution of epigenetic variation to phenotypic variation is unclear. Imprinted genes, because of their strong association with epigenetic modifications, represent an opportunity for the discovery of such phenomena. In mammals and flowering plants, a subset of genes are expressed from only one parental allele in a process called gene imprinting. Imprinting is associated with differential DNA methylation and chromatin modifications between parental alleles. In flowering plants imprinting occurs in a seed tissue - endosperm. Proper endosperm development is essential for the production of viable seeds. We previously showed that in Arabidopsis thaliana intraspecific imprinting variation is correlated with naturally occurring DNA methylation polymorphisms. Here, we investigated the mechanisms and function of allele-specific imprinting of the class IV homeodomain-Leucine zipper (HD-ZIP) transcription factor HDG3. In imprinted strains, HDG3 is expressed primarily from the methylated paternally inherited allele. We manipulated the methylation state of endogenous HDG3 in a non-imprinted strain and demonstrated that methylation of a proximal transposable element is sufficient to promote HDG3 expression and imprinting. Gain of HDG3 imprinting was associated with earlier endosperm cellularization and changes in seed weight. These results indicate that epigenetic variation alone is sufficient to explain imprinting variation and demonstrate that epialleles can underlie variation in seed development phenotypes.\\\\n\\\\nAuthor SummaryThe contribution of genetic variation to phenotypic variation is well-established. By contrast, it is unknown how frequently epigenetic variation causes differences in organismal phenotypes. Epigenetic information is closely associated with but not encoded in the DNA sequence. In practice, it is challenging to disentangle genetic variation from epigenetic variation, as what appears to be epigenetic variation might have an underlying genetic basis. DNA methylation is one form of epigenetic information. HDG3 encodes an endosperm specific transcription factor that exists in two states in A. thaliana natural populations: methylated and expressed and hypomethylated and repressed. We show that pure epigenetic variation is sufficient to explain expression variation of HDG3 - a naturally lowly expressed allele can be switched to a higher expressed state by adding DNA methylation. We also show that expression of HDG3 in strains where it is normally hypomethylated and relatively repressed causes a seed development phenotype. These data indicate that naturally circulating epialleles have consequences for seed phenotypic variation.',\n",
       "  'published': '10.1371/journal.pgen.1007469',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/336222',\n",
       "  'title': 'The Evolution of Bequeathal in Stable Habitats',\n",
       "  'authors': 'Clarke, P. M. R.; McElreath, M. B.; Barrett, B. J.; Mabry, K. E.; McElreath, R. L.',\n",
       "  'author_corresponding': 'Mary Brooke  McElreath',\n",
       "  'author_corresponding_institution': 'Max Planck Institute for Ornithology',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/336222.source.xml',\n",
       "  'abstract': 'Adults sometimes disperse while philopatric offspring inherit the natal site, a pattern known as bequeathal. Despite a decades-old empirical literature, little theoretical work has explored when natural selection may favor bequeathal. We present a simple mathematical model of the evolution of bequeathal in a stable environment, under both global and local dispersal. We find that natural selection favors bequeathal when adults are competitively advantaged over juveniles, baseline mortality is high, the environment is unsaturated, and when juveniles experience high dispersal mortality. However, frequently bequeathal may not evolve, because the fitness cost for the adult is too large relative to inclusive fitness benefits. Additionally, there are many situations for which bequeathal is an ESS, yet cannot invade the population. As bequeathal in real populations appears to be facultative, yet-to-be-modeled factors like timing of birth in the breeding season may strongly influence the patterns seen in natural populations.',\n",
       "  'published': '10.1002/ece3.4549',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/338624',\n",
       "  'title': 'Automatic Removal of False Connections in Diffusion MRI Tractography Using Topology-Informed Pruning (TIP)',\n",
       "  'authors': 'Yeh, F.-C.; Panesar, S.; Barrios, J.; Fernandes, D.; Abhinav, K.; Meola, A.; Fernandez-Miranda, J. C.',\n",
       "  'author_corresponding': 'Fang-Cheng  Yeh',\n",
       "  'author_corresponding_institution': 'University of Pittsburgh',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/338624.source.xml',\n",
       "  'abstract': 'Diffusion MRI fiber tracking provides a non-invasive method for mapping the trajectories of human brain connections, but its false connection problem has been a major challenge. This study introduces topology-informed pruning (TIP), a method that automatically identifies singular tracts and eliminates them to improve the tracking accuracy. The accuracy of the tractography with and without TIP was evaluated by a team of 6 neuroanatomists in a blinded setting to examine whether TIP could improve the accuracy. The results showed that TIP improved the tracking accuracy by 11.93% in the single-shell scheme and by 3.47% in the grid scheme. The improvement is significantly different from a random pruning (p-value < 0.001). The diagnostic agreement between TIP and neuroanatomists was comparable to the agreement between neuroanatomists. The proposed TIP algorithm can be used to automatically clean up noisy fibers in deterministic tractography, with a potential to confirm the existence of a fiber connection in basic neuroanatomical studies or clinical neurosurgical planning.',\n",
       "  'published': '10.1007/s13311-018-0663-y',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/339630',\n",
       "  'title': 'Weakly supervised classification of rare aortic valve malformations using unlabeled cardiac MRI sequences',\n",
       "  'authors': 'Fries, J. A.; Varma, P.; Chen, V. S.; Xiao, K.; Tejeda, H.; Saha, P.; Dunnmon, J.; Chubb, H.; Maskatia, S.; Fiterau, M.; Delp, S.; Ashley, E.; Re, C.; Priest, J.',\n",
       "  'author_corresponding': 'Jason A Fries',\n",
       "  'author_corresponding_institution': 'Stanford University',\n",
       "  'date': '2018-08-22',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'bioinformatics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/22/339630.source.xml',\n",
       "  'abstract': 'Biomedical repositories such as the UK Biobank provide increasing access to prospectively collected cardiac imaging, however these data are unlabeled which creates barriers to their use in supervised machine learning. We develop a weakly supervised deep learning model for classification of aortic valve malformations using up to 4,000 unlabeled cardiac MRI sequences. Instead of requiring highly curated training data, weak supervision relies on noisy heuristics defined by domain experts to programmatically generate large-scale, imperfect training labels. For aortic valve classification, models trained with imperfect labels substantially outperform a supervised model trained on hand-labeled MRIs. In an orthogonal validation experiment using health outcomes data, our model identifies individuals with a 1.8-fold increase in risk of a major adverse cardiac event. This work formalizes a learning baseline for aortic valve classification and outlines a general strategy for using weak supervision to train machine learning models using unlabeled medical images at scale.',\n",
       "  'published': '10.1038/s41467-019-11012-3',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/339770',\n",
       "  'title': 'Longitudinal studies at birth and age 7 reveal strong effects of genetic variation on ancestry-associated DNA methylation patterns in blood cells from ethnically admixed children',\n",
       "  'authors': \"McKennan, C.; Naughton, K.; Stanhope, C.; Kattan, M.; O'Connor, G.; Sandel, M.; Visness, C.; Wood, R.; Bacharier, L.; Beigelman, A.; Lovisky-Desir, S.; Togias, A.; Gern, J.; Nicolae, D.; Ober, C.\",\n",
       "  'author_corresponding': 'Chris  McKennan',\n",
       "  'author_corresponding_institution': 'Department of Human Genetics, University of Chicago',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/339770.source.xml',\n",
       "  'abstract': 'Epigenetic architecture is influenced by genetic and environmental factors, but little is known about their relative contributions or longitudinal dynamics. Here, we studied DNA methylation (DNAm) at over 750,000 CpG sites in mononuclear blood cells collected at birth and age 7 from 196 children of primarily self-reported Black and Hispanic ethnicities to study race-associated DNAm patterns. We developed a novel Bayesian method for high dimensional longitudinal data and showed that race-associated DNAm patterns at birth and age 7 are nearly identical. Additionally, we estimated that up to 51% of all self-reported race-associated CpGs had race-dependent DNAm levels that were mediated through local genotype and, quite surprisingly, found that genetic factors explained an overwhelming majority of the variation in DNAm levels at other, previously identified, environmentally-associated CpGs. These results not only indicate that race-associated DNAm patterns in blood are present at birth and are primarily genetically, and not environmentally, determined, but also that DNAm in blood cells overall is robust to many environmental exposures during the first 7 years of life.',\n",
       "  'published': '10.1080/15592294.2020.1817290',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/346007',\n",
       "  'title': 'Cerebrovascular Injuries Induce Lymphatic Invasion into Brain Parenchyma to Guide Vascular Regeneration in Zebrafish',\n",
       "  'authors': 'Chen, J.; He, J.; Yang, Q.; Zhang, Y.; Luo, L.',\n",
       "  'author_corresponding': 'Lingfei  Luo',\n",
       "  'author_corresponding_institution': 'School of Life Sciences, Southwest University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'developmental biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/346007.source.xml',\n",
       "  'abstract': 'Damage to regional cerebrovascular network and neuronal tissues occurs during acute cerebrovascular diseases, such as ischemic stroke. The promotion of vascular regeneration is the most promising therapeutic approach. To understand cellular and molecular mechanisms underlying brain vascular regeneration, we developed two zebrafish cerebrovascular injury models using genetic ablation and photochemical thrombosis. Although brain parenchyma is physiologically devoid of lymphatic vasculature, we found that cerebrovascular injuries induce rapid ingrowth of meningeal lymphatics into the injured parenchyma. The ingrown lymphatics on one hand become lumenized drain interstitial fluid to resolve brain edema, on the other hand act as \\\\\"growing tracks\\\\\" for nascent blood vessels. The ingrown lymphatic vessels undergo apoptosis and clearance after cerebrovascular regeneration. This study reveals a pathological function of meningeal lymphatics, through previously unexpected ingrowth into brain parenchyma and a newly identified lymphatic function as vascular \\\\\"growing tracks\\\\\".\\\\n\\\\nHIGHLIGHTSCerebrovascular injuries induce lymphatic ingrowth into the injured brain parenchyma The ingrown lymphatics drain interstitial fluid to resolve brain edema Nascent blood vessels use the ingrown lymphatic vessels as \\\\\"growing tracks\\\\\" The ingrown lymphatic vessels undergo apoptosis after vascular regeneration completes',\n",
       "  'published': '10.1016/j.devcel.2019.03.022',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/352393',\n",
       "  'title': 'Biologically realistic mean-field models of conductance-based networks of spiking neurons with adaptation',\n",
       "  'authors': 'di Volo, M.; Romagnoni, A.; Capone, C.; Destexhe, A.',\n",
       "  'author_corresponding': 'Matteo  di Volo',\n",
       "  'author_corresponding_institution': 'Unite de Neuroscience, Information et Complexite (UNIC), CNRS FRE 3693, 1 avenue de la Terrasse',\n",
       "  'date': '2018-08-25',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'neuroscience',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/25/352393.source.xml',\n",
       "  'abstract': 'Accurate population models are needed to build very large scale neural models, but their derivation is difficult for realistic networks of neurons, in particular when nonlinear properties are involved such as conductance-based interactions and spike-frequency adaptation. Here, we consider such models based on networks of Adaptive exponential Integrate and fire excitatory and inhibitory neurons. Using a Master Equation formalism, we derive a mean-field model of such networks and compare it to the full network dynamics. The mean-field model is capable to correctly predict the average spontaneous activity levels in asynchronous irregular regimes similar to in vivo activity. It also captures the transient temporal response of the network to complex external inputs. Finally, the mean-field model is also able to quantitatively describe regimes where high and low activity states alternate (UP-DOWN state dynamics), leading to slow oscillations. We conclude that such mean-field models are \\\\\"biologically realistic\\\\\" in the sense that they can capture both spontaneous and evoked activity, and they naturally appear as candidates to build very large scale models involving multiple brain areas.',\n",
       "  'published': '10.1162/neco_a_01173',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/356196',\n",
       "  'title': 'A Fasciclin 2 functional switch controls organ size in Drosophila',\n",
       "  'authors': 'Velasquez, E.; Gomez-Sanchez, J. A.; Donier, E.; Grijota-Martinez, C.; Cabedo, H.; Garcia-Alonso, L. A.',\n",
       "  'author_corresponding': 'Luis A Garcia-Alonso',\n",
       "  'author_corresponding_institution': 'Instituto de Neurociencias CSIC-UMH',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nd',\n",
       "  'category': 'developmental biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/356196.source.xml',\n",
       "  'abstract': 'How cell to cell interactions control local tissue growth to attain a species-specific pattern and organ size is a central question in developmental biology. The Drosophila Neural Cell Adhesion Molecule, Fasciclin 2 (Drosophila NCAM), is expressed during the development of neural and epithelial organs. Genetic mosaic analysis of Fasciclin 2 reveals two complementary and opposing functions during imaginal disc growth, a cell autonomous requirement to promote growth and an opposite non-cell autonomous function to restrain growth at high expression levels. This non-cell autonomous function is mediated by the Fasciclin 2 heterophilic-binding partners CG15630 and CG33543. We show that EGFR physically interacts with Fasciclin 2 and mediates both the cell autonomous and the non-cell autonomous function. We further show that EGFR activity in turn promotes the cell autonomous expression of Fasciclin 2. We suggest that the auto-stimulatory loop between EGFR and Fasciclin 2 operates until reaching a threshold where the Fasciclin 2 non-cell autonomous function counteracts the growth-promoting activity of the homophilic interaction to terminate imaginal disc growth. Accordingly, we have found that Fasciclin 2 limits imaginal disc growth by the end of larval development. Cellular integration of Fasciclin 2 autonomous and non-cell autonomous signaling from neighbor cells may be a key regulator component to orchestrate the rate of intercalary cell proliferation and the final size of an organ.\\\\n\\\\nAuthor SummaryOne of the key unsolved problems in Biology is how a species-specific size is attained during animal development. During development cells should compute the amount of intercalary tissue growth to stop cell proliferation when reaching a correct pattern and size. Classic studies demonstrated that local cell interactions are key in controlling organ growth to reach a correct size and pattern in vertebrates and invertebrates. We present evidence strongly suggesting that Fasciclin 2 (the ortholog of NCAM in Drosophila) functions as a growth level switch to control pattern and organ size. First, we use genetic mosaic analyses to show that Fasciclin 2 promotes organ growth in a cell autonomous manner. Then we show that Fasciclin 2 restrains growth at high expression levels in a non-cell autonomous manner, and that there is a requirement for Fasciclin 2 to limit growth by the end of larval development. This function is dependent on Fasciclin 2 heterophilic binding partners CG15630 and CG33543. The Epidermal Growth Factor receptor mediates both functional facets of Fasciclin 2 and its activity in turn increases Fasciclin 2 cell autonomous expression, suggesting the existence of a functional auto-stimulatory loop. We also show that the Epidermal Growth Factor receptor and Fasciclin 2 physically interact. Our results show that the amount of Fasciclin 2 between cells determines organ size by acting as an expression level switch for EGFR function, and suggest that other specific CAM interactions may integrate similar expression level switches acting as a code for cells to compute local growth in attaining a species-specific organ size and shape.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/356246',\n",
       "  'title': 'Gut bacterial tyrosine decarboxylases restrict the bioavailability of levodopa, the primary treatment in Parkinson’s disease',\n",
       "  'authors': 'van Kessel, S. P.; Frye, A. K.; El-Gendy, A. O.; Castejon, M.; Keshavarzian, A.; van Dijk, G.; El Aidy, S.',\n",
       "  'author_corresponding': 'Sahar  El Aidy',\n",
       "  'author_corresponding_institution': 'Groningen Biomolecular Sciences and Biotechnology Institute (GBB)',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'molecular biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/356246.source.xml',\n",
       "  'abstract': 'Human gut bacteria play a critical role in the regulation of immune and metabolic systems, as well as in the function of the nervous system. The microbiota senses its environment and responds by releasing metabolites, some of which are key regulators of human health and disease. In this study, we identify and characterize gut-associated bacteria in their ability to decarboxylate L-DOPA (also known as Levodopa or L-3,4-dihydroxyphenylalanine) to dopamine via the tyrosine decarboxylases, which are mainly present in the class Bacilli. Although the bacterial tyrosine decarboxylases have a higher affinity for tyrosine compared to L-DOPA, this does not affect their ability to decarboxylate L-DOPA, nor does any inhibitor of the human decarboxylase. This study indicates that in situ bioavailability of L-DOPA is compromised by the gut bacterial tyrosine decarboxylase abundance in Parkinsons patients. Finally, we show that the tyrosine decarboxylase abundance in the microbiota at the site of L-DOPA absorption, the proximal small intestine, significantly influences L-DOPA bioavailability in the plasma of rats. Our results highlight the role of microbial metabolism in drug bioavailability, and specifically, that small intestinal abundance of bacterial tyrosine decarboxylase can explain the highly variable L-DOPA dosage regimens required in the treatment of individual Parkinsons patients.\\\\n\\\\nHighlightsO_LISmall intestinal bacteria is able to convert L-DOPA to dopamine\\\\nC_LIO_LIL-DOPA metabolism by gut bacteria reduce the bioavailability of L-DOPA in the body, thus is a significant explanatory factor of the highly variable L-DOPA dosage regimens required in the treatment of individual Parkinsons patients.\\\\nC_LIO_LIInhibitors of the human DOPA decarboxylase are not potent inhibitors for bacterial tyrosine decarboxylases\\\\nC_LI',\n",
       "  'published': '10.1038/s41467-019-08294-y',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/357467',\n",
       "  'title': 'DataRemix: a universal data transformation for optimal inference from gene expression datasets',\n",
       "  'authors': 'Mao, W.; Hausler, R.; Chikina, M.',\n",
       "  'author_corresponding': 'Maria  Chikina',\n",
       "  'author_corresponding_institution': 'University of Pittsburgh',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/357467.source.xml',\n",
       "  'abstract': 'MotivationRNAseq technology provides unprecedented power in the assesment of the transcription abundance and can be used to perform a variety of downstream tasks such as inference of gene-correlation network and eQTL discovery. However, raw gene expression values have to be normalized for nuisance biological variation and technical covariates, and different normalization strategies can lead to dramatically different results in the downstream study.\\n\\nResultsWe describe a generalization of SVD-based reconstruction for which the common techniques of whitening, rank-k approximation, and removing the top k principle components are special cases. Our simple three-parameter transformation, DataRemix, can be tuned to reweight the contribution of hidden factors and reveal otherwise hidden biological signals. In particular, we demonstrate that the method can effectively prioritize biological signals over noise without leveraging external dataset-specific knowledge, and can outperform normalization methods that make explicit use of known technical factors. We also show that DataRemix can be efficiently optimized via Thompson Sampling approach, which makes it feasible for computationally expensive objectives such as eQTL analysis. Finally we reanalyze the Depression Gene Networks (DGN) dataset, and we highlight new trans-eQTL networks which were not reported in the initial study.\\n\\nAvailabilityDataRemix is an R package which is freely available at GitHub (https://github.com/wgmao/DataRemix).\\n\\nContactmchikina@pitt.edu',\n",
       "  'published': '10.1093/bioinformatics/btaa745',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/358044',\n",
       "  'title': 'Phylogeographic analysis and identification of factors impacting the diffusion of Foot-and-Mouth disease virus in Africa',\n",
       "  'authors': 'Duchatel, F.; Bronsvoort, M.; Lycett, S.',\n",
       "  'author_corresponding': 'Samantha  Lycett',\n",
       "  'author_corresponding_institution': 'Roslin institute, Edinburgh University',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/358044.source.xml',\n",
       "  'abstract': 'Foot and mouth disease (FMD) is endemic in sub-Saharan Africa and can lead to important and continuous economic losses for affected countries. Due to the complexity of the disease epidemiology and the lack of data there is a need to use inferential computational approaches to fill the gaps in our understanding of the circulation of FMD virus on this continent. Using a phylogeographic approach we reconstructed the circulation of FMD virus serotypes A, O and SAT2 in Africa and evaluated the influence of potential environmental and anthropological predictors of virus diffusion. Our results show that over the last hundred year the continental circulation of the tree serotypes was mainly driven by livestock trade. Whilst our analyses show that the serotypes A and O were introduced in Africa trough livestock trades, the SAT2 serotype probably originates from African wildlife population. The circulation of serotype O in eastern Africa is impacted by both indirect transmission through persistence in the environment and anthropological activities such as cattle movements.',\n",
       "  'published': '10.3389/fevo.2019.00371',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/361089',\n",
       "  'title': 'Genetic variation in a complex polyploid: unveiling the dynamic allelic features of sugarcane',\n",
       "  'authors': 'Sforca, D. A.; Vautrin, S.; Cardoso-Silva, C. B.; Mancini, M. C.; Cruz, M. V. R.; Pereira, G. d. S.; Conte, M.; Bellec, A.; Dahmer, N.; Fourment, J.; Rodde, N.; Van-Sluys, M.-A.; Vicentini, R.; Garcia, A. A. F.; Forni-Martins, E. R.; Carneiro, M. S.; Hoffmann, H. P.; Pinto, L. R.; Landell, M. G. d. A.; Vincentz, M.; Berges, H.; Souza, A. P.',\n",
       "  'author_corresponding': 'Anete Pereira Souza',\n",
       "  'author_corresponding_institution': 'Universidade Estadual de Campinas (UNICAMP), Campinas, SP, Brazil',\n",
       "  'date': '2018-08-21',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'plant biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/21/361089.source.xml',\n",
       "  'abstract': 'BackgroundSugarcane (Saccharum spp.) is highly polyploid and aneuploid. Modern cultivars are derived from hybridization between S. officinarum and S. spontaneum. This combination results in a genome exhibiting variable ploidy among different loci, a huge genome size (approximately 10 Gb) and a high content of repetitive regions. Gene expression mechanisms are poorly understood in these cultivars. An approach using genomic, transcriptomic and genetic mapping can improve our knowledge of the behavior of genetics in sugarcane.\\\\n\\\\nResultsThe hypothetical HP600 and centromere protein C (CENP-C) genes from sugarcane were used to elucidate the allelic expression and genomic and genetic behavior of this complex polyploid. The genomically side-by-side genes HP600 and CENP-C were found in two different homeologous chromosome groups with ploidies of eight and ten. The first region (Region01) was a Sorghum bicolor ortholog with all haplotypes of HP600 and CENP- C expressed, but HP600 exhibited an unbalanced haplotype expression. The second region (Region02) was a scrambled sugarcane sequence formed from different noncollinear genes containing duplications of HP600 and CENP-C (paralogs). This duplication occurred before the Saccharum genus formation and after the separation of sorghum and sugarcane, resulting in a nonexpressed HP600 pseudogene and a recombined fusion version of CENP-C and orthologous gene Sobic.003G299500 with at least two chimerical gene haplotypes expressed. The genetic map construction supported the difficulty of mapping markers located in duplicated regions of complex polyploid genomes.\\\\n\\\\nConclusionAll these findings describe a low synteny region in sugarcane, formed by events occurring in all members of the Saccharum genus. Additionally, evidence of duplicated and truncate gene expression and the behavior of genetic markers in a duplicated region was found. Thus, we describe the complexity involved in sugarcane genetics and genomics and allelic dynamics, which can be useful for understanding the complex polyploid genome.',\n",
       "  'published': '10.3389/fpls.2019.00553',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/361758',\n",
       "  'title': 'Continual exit of human cutaneous resident memory CD4 T cells that seed distant skin sites',\n",
       "  'authors': 'Klicznik, M. M.; Morawski, P. A.; Höllbacher, B.; Varkhande, S.; Motley, S.; Rosenblum, M. D.; Long, S. A.; Brachtl, G.; Duhen, T.; Campbell, D.; Gratz, I. K.',\n",
       "  'author_corresponding': 'Iris K Gratz',\n",
       "  'author_corresponding_institution': 'University of Salzburg',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'immunology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/361758.source.xml',\n",
       "  'abstract': 'Tissue-resident memory T cells (TRM) persist locally in non-lymphoid tissues where they provide front-line defense against recurring insults. TRM at barrier surfaces express the markers CD103 and/or CD69 which function to retain them in epithelial tissues. In humans, neither the long-term migratory behavior of TRM nor their ability to re-enter the circulation and potentially migrate to distant tissue sites have been investigated. Using tissue explant cultures, we found that CD4+CD69+CD103+ TRM in human skin can downregulate CD69 and exit the tissue.\\n\\nAdditionally, we identified a skin-tropic CD4+CD69-CD103+ population in human lymph and blood that is transcriptionally, functionally and clonally related to the CD4+CD69+CD103+ TRM population in the skin. Using a skin xenograft model, we confirmed that a fraction of the human cutaneous CD4+CD103+ TRM population can re-enter circulation, and migrate to secondary human skin sites where they re-assume a TRM phenotype. Thus, our data challenge current concepts regarding the strict tissue compartmentalization of CD4+ T cell memory in humans.\\n\\nOne Sentence SummaryHuman CD4+CD103+ cutaneous resident memory T cells are found in the circulation of healthy subjects, and these cells can seed distant skin sites.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/362186',\n",
       "  'title': 'Using the Weibull accelerated failure time regression model to predict time to health events',\n",
       "  'authors': 'Liu, E.; Lim, K.',\n",
       "  'author_corresponding': 'Enwu  Liu',\n",
       "  'author_corresponding_institution': 'MaryMckillop Institue for Health Research',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'epidemiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/362186.source.xml',\n",
       "  'abstract': 'We describe a statistical method protocol to use a Weibull accelerated failure time (AFT) model to predict time to a health-related event. This prediction method is quite common in engineering reliability research but rarely used for medical predictions such as survival time. A worked example for how to perform the prediction using a published dataset is included.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/362129',\n",
       "  'title': 'Parallel adaptations to different host plants despite clonal reproduction in the world most devastating nematode pest',\n",
       "  'authors': 'Koutsovoulos, G.; Marques, E.; Arguel, M.-J.; Machado, A. C.; Carneiro, R. M.; Castagnone-Sereno, P.; Danchin, E. G.; Albuquerque, E. V.',\n",
       "  'author_corresponding': 'Etienne GJ Danchin',\n",
       "  'author_corresponding_institution': \"Institut Sophia Agrobiotech, INRA, CNRS, Universite Cote d'Azur, France\",\n",
       "  'date': '2018-08-23',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nd',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/362129.source.xml',\n",
       "  'abstract': 'The most devastating nematodes to worldwide agriculture are the root-knot nematodes with Meloidogyne incognita being the most widely distributed and damaging species. This parasitic and ecological success seem surprising given its supposed obligatory clonal reproduction. Clonal reproduction has been suspected based on cytological observations but, so far, never confirmed by population genomics data. At the species level, M. incognita is highly polyphagous with thousands of host plants. However, the host range varies among different M. incognita isolates that may present distinct and more restricted host compatibilities. Historically, four  host races had been defined as a function of ranges of compatible and incompatible plants. We sequenced the genomes of 11 isolates across Brazil, covering these four distinct races to assess (i) how clonal reproduction is and (ii) how the level of genome variability associates with biological traits such as the host races, affected agronomic culture, and geographical distribution. By aligning the genomic reads of the isolates to the M. incognita reference genome assembly, we identified SNV and small-scale insertions/deletions. Analysis of linkage disequilibrium and 4-gametes test, showed no sign of recombination, confirming the clonal mode of reproduction of M. incognita. We showed that there are relatively few point variations between the different isolates, and these variations show no significant association with either the host races, the geographical origin of the samples or the host plant on which they have been collected. Due to the lack of phylogenetic signal underlying their existence, we recommend the discontinuation of the terminology  race. Overall, these results suggest that multiple gains and losses of parasitic abilities and adaptations to different environmental conditions account for the broad host spectrum and wide geographic distribution of M. incognita. Hence, this nematode constitutes a model species to study adaptability without sexual recombination and overall low genomic variations in animals.',\n",
       "  'published': '10.1111/eva.12881',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/361840',\n",
       "  'title': 'Detection of instability in 18S and 5.8S rDNA copy numbers and 18S:5.8S ratios in longitudinally-collected human DNA samples detected by monochrome multiplex qPCR',\n",
       "  'authors': 'Ligerman, C. B.; Baird, L.; Leppert, M. F.; Cawthon, R. M.',\n",
       "  'author_corresponding': 'Richard M Cawthon',\n",
       "  'author_corresponding_institution': 'University of Utah, Dept. of Human Genetics',\n",
       "  'date': '2018-08-27',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/27/361840.source.xml',\n",
       "  'abstract': 'Ribosomal DNA encodes the structural RNAs of the ribosomes. Ribosomal DNA instability is a major contributor to aging in yeast, but its role in human aging and longevity is largely unknown. Human 45S rDNA encodes the 18S, 5.8S, and 28S ribosomal RNAs; ranges in copy number from 60 to > 800 per cell, distributed as tandem repeats along the short arms of the five acrocentric chromosomes (p13, p14, p15, p21, and p22); and is prone to frequent homologous and non-homologous recombination. Here we present two multiplex quantitative PCR assays, one for 18S rDNA normalized to the single copy gene beta-globin (HBB), and the other for 5.8S rDNA normalized to the single copy gene albumin (ALB). Longitudinally-collected pairs of DNAs from bloods drawn approximately 16 years apart from 40 females and 39 males, aged < 1 to 77 years at first blood draw, from the Utah CEPH families were assayed. Ribosomal DNA copy number varied over a four-fold range between subjects and increased approximately 14% across the lifespan. Longitudinal within-individual gains in copy number up to +68% and losses down to -25% were observed, while repeated assays of single DNA samples varied approximately +/- 10%. While 18S and 5.8S rDNA tended to be gained and lost together, the 18S:5.8S ratio was also unstable longitudinally, with increases up to +19% and decreases down to -15% observed. The 18S:5.8S ratio at second draw, relative to that at first draw, increased significantly across the lifespan in males, but not in females. To our knowledge this is the first report of within-individual longitudinal changes in the human 18S and 5.8S rDNA copy numbers. These assays will facilitate investigations of the biology of ribosomal RNA genes and their roles in health and disease across the human lifespan.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/361840',\n",
       "  'title': 'Instability in 18S and 5.8S rDNA copy numbers and 18S:5.8S ratios in longitudinally-collected human DNA samples detected by monochrome multiplex qPCR',\n",
       "  'authors': 'Ligerman, C. B.; Baird, L.; Leppert, M. F.; Cawthon, R. M.',\n",
       "  'author_corresponding': 'Richard M Cawthon',\n",
       "  'author_corresponding_institution': 'University of Utah, Dept. of Human Genetics',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_no',\n",
       "  'category': 'genomics',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/361840.source.xml',\n",
       "  'abstract': 'Ribosomal DNA encodes the structural RNAs of the ribosomes. Ribosomal DNA instability is a major contributor to aging in yeast, but its role in human aging and longevity is largely unknown. Human 45S rDNA encodes the 18S, 5.8S, and 28S ribosomal RNAs; ranges in copy number from 60 to > 800 per cell, distributed as tandem repeats along the short arms of the five acrocentric chromosomes (p13, p14, p15, p21, and p22); and is prone to frequent homologous and non-homologous recombination. Here we present two multiplex quantitative PCR assays, one for 18S rDNA normalized to the single copy gene beta-globin (HBB), and the other for 5.8S rDNA normalized to the single copy gene albumin (ALB). Longitudinally-collected pairs of DNAs from bloods drawn approximately 16 years apart from 40 females and 39 males, aged < 1 to 77 years at first blood draw, from the Utah CEPH families were assayed. Ribosomal DNA copy number varied over a four-fold range between subjects and increased approximately 14% across the lifespan. Longitudinal within-individual gains in copy number up to +68% and losses down to -25% were observed, while repeated assays of single DNA samples varied approximately +/- 10%. While 18S and 5.8S rDNA tended to be gained and lost together, the 18S:5.8S ratio was also unstable longitudinally, with increases up to +19% and decreases down to -15% observed. The 18S:5.8S ratio at second draw, relative to that at first draw, increased significantly across the lifespan in males, but not in females. To our knowledge this is the first report of within-individual longitudinal changes in the human 18S and 5.8S rDNA copy numbers. These assays will facilitate investigations of the biology of ribosomal RNA genes and their roles in health and disease across the human lifespan.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/363622',\n",
       "  'title': 'New tools for diet analyses: nanopore sequencing of metagenomic DNA from rat stomach contents to quantify diet',\n",
       "  'authors': 'Pearman, W.; Smith, A. N. H.; Breckell, G.; Dale, J.; Freed, N. E.; Silander, O. K.',\n",
       "  'author_corresponding': 'Olin K. Silander',\n",
       "  'author_corresponding_institution': 'Institute of Natural and Mathematical Sciences, Massey University, Auckland 0745, New Zealand',\n",
       "  'date': '2018-08-28',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'ecology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/28/363622.source.xml',\n",
       "  'abstract': 'BackgroundUsing metagenomics to determine animal diet offers a new and promising alternative to current methods. Here we show that rapid and inexpensive diet quantification is possible through metagenomic sequencing with the portable Oxford Nanopore Technologies (ONT) MinION. Using an amplification-free approach, we profiled the stomach contents from wild-caught rats.\\n\\nResultsWe conservatively identified diet items from over 50 taxonomic orders, ranging across nine phyla that include plants, vertebrates, invertebrates, and fungi. This highlights the wide range of taxa that can be identified using this simple approach. We calibrate the accuracy of this method by comparing the characteristics of reads matching the ground-truth host genome (rat) to those matching diet items, and show that at the family-level, false positive taxon assignments are approximately 97.5% accurate. We also suggest a way to mitigate for database biases in metagenomic approaches. Finally, we implement a constrained ordination analysis and show that we can identify the sampling location of an individual rat within tens of kilometres based on diet content alone.\\n\\nConclusionsThis work establishes proof-of-principle for long-read metagenomic methods in quantitative diet analysis. We show that diet content can be quantified even with limited expertise, using a simple, amplification free workflow and a relatively inexpensive and accessible next generation sequencing method. Continued increases in the accuracy and throughput of ONT sequencing, along with improved genomic databases, suggests that a metagenomic approach to quantification of animal diets will become an important method in the future.',\n",
       "  'published': 'NA',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/363077',\n",
       "  'title': 'Systems Biology Understanding of the Effects of Lithium on Cancer.',\n",
       "  'authors': 'Ge, W.; Jakobsson, E.',\n",
       "  'author_corresponding': 'Eric  Jakobsson',\n",
       "  'author_corresponding_institution': 'University of Illinois at Urbana-Champaign',\n",
       "  'date': '2018-08-24',\n",
       "  'version': '2',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nd',\n",
       "  'category': 'cancer biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/24/363077.source.xml',\n",
       "  'abstract': 'Lithium has many widely varying biochemical and phenomenological effects, suggesting that a systems biology approach is required to understand its action. Multiple lines of evidence point to lithium as a significant factor in development of cancer, showing that understanding lithium action is of high importance. In this paper we undertake first steps towards a systems approach by analyzing mutual enrichment between the interactomes of lithium-sensitive enzymes and the pathways associated with cancer. This work integrates information from two important databases, STRING and KEGG pathways. We find that for the majority of cancer pathways the mutual enrichment is many times greater than chance, reinforcing previous lines of evidence that lithium is an important influence on cancer.',\n",
       "  'published': '10.3389/fonc.2019.00296',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/364331',\n",
       "  'title': 'Resource limitation modulates the fate of dissimilated nitrogen in a dual-pathway Actinobacterium',\n",
       "  'authors': 'Vuono, D. C.; Read, R. W.; Hemp, J.; Sullivan, B. W.; Arnone, J. A.; Neveux, I.; Blank, B.; Staub, C.; Loney, E.; Miceli, D.; Winkler, M.; Chakraborty, R.; Stahl, D. A.; Grzymski, J. J.',\n",
       "  'author_corresponding': 'Joseph J Grzymski',\n",
       "  'author_corresponding_institution': 'Desert Research Institute',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '4',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by_nc_nd',\n",
       "  'category': 'microbiology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/364331.source.xml',\n",
       "  'abstract': 'Respiratory ammonification and denitrification are two evolutionarily unrelated dissimilatory nitrogen (N) processes central to the global N cycle, the activity of which is thought to be controlled by carbon (C) to nitrate (NO3-) ratio. Here we find that Intrasporangium calvum C5, a novel menaquinone-based dual-pathway denitrifier/respiratory ammonifier, disproportionately utilizes ammonification rather than denitrification when grown under carbon or nitrate limitation, not C:NO3- ratio. Higher growth rates are promoted by ammonification and metabolite and transcriptional profiles during growth show that the bacterium produces its own formate from a fermentable carbon source (lactate) to further generate a proton motive force for the ammonification pathway. Transcript abundances encoding for nitrite reducing enzymes, NrfAH and NirK, also significantly increase in response to nitrite production. Mechanistically, our results suggest that pathway selection is driven by intracellular redox potential (redox poise), which may be lowered during resource limitation, thereby decreasing catalytic activity of upstream electron transport steps needed for denitrification enzymes. Our work advances our understanding of the biogeochemical flexibility of N-cycling organisms and pathway evolution.',\n",
       "  'published': '10.3389/fmicb.2019.00003',\n",
       "  'server': 'biorxiv'},\n",
       " {'doi': '10.1101/364190',\n",
       "  'title': 'Intragenomic Conflict over Bet-Hedging',\n",
       "  'authors': 'Wilkins, J. F.; Bhattacharya, T.',\n",
       "  'author_corresponding': 'Jon F Wilkins',\n",
       "  'author_corresponding_institution': 'Ronin Institute',\n",
       "  'date': '2018-08-23',\n",
       "  'version': '3',\n",
       "  'type': 'new results',\n",
       "  'license': 'cc_by',\n",
       "  'category': 'evolutionary biology',\n",
       "  'jatsxml': 'https://www.biorxiv.org/content/early/2018/08/23/364190.source.xml',\n",
       "  'abstract': 'Genomic imprinting, where an alleles expression pattern depends on its parental origin, is thought to result primarily from an intragenomic evolutionary conflict. Imprinted genes are widely expressed in the brain and have been linked to various phenotypes, including behaviors related to risk tolerance. In this paper, we analyze a model of evolutionary bet-hedging in a system with imprinted gene expression. Previous analyses of bet-hedging have shown that natural selection may favor alleles and traits that reduce reproductive variance, even at the expense of reducing mean reproductive success, with the trade-off between mean and variance depending on the population size. In species where the sexes have different reproductive variances, this bet-hedging trade-off differs between maternally and paternally inherited alleles. Where males have the higher reproductive variance, alleles are more strongly selected to reduce variance when paternally inherited than when maternally inherited. We connect this result to phenotypes connected with specific imprinted genes, including delay discounting and social dominance. The empirical patterns are consistent with paternally expressed imprinted genes promoting risk-averse behaviors that reduce reproductive variance. Conversely, maternally expressed imprinted genes promote risk-tolerant, variance-increasing behaviors. We indicate how future research might further test the hypotheses suggested by our analysis.',\n",
       "  'published': '10.1098/rstb.2018.0142',\n",
       "  'server': 'biorxiv'}]"
      ]
     },
     "execution_count": 27,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "json_info[\"collection\"]"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 24,
   "id": "77a59f13",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "[['10.1101/066688',\n",
       "  'Cross-scale dynamics and the evolutionary emergence of infectious diseases',\n",
       "  'Schreiber, S. J.; Ke, R.; Loverdo, C.; Park, M.; Ahsan, P.; Lloyd-Smith, J. O.',\n",
       "  'Sebastian J. Schreiber',\n",
       "  'University of California, Davis',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'epidemiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/066688.source.xml',\n",
       "  '10.1093/ve/veaa105'],\n",
       " ['10.1101/079954',\n",
       "  'iCARE: An R Package to Build, Validate and Apply Absolute Risk Models',\n",
       "  'Pal Choudhury, P.; Maas, P.; Wilcox, A.; Wheeler, W.; Brook, M.; Check, D.; Garcia-Closas, M.; Chatterjee, N.',\n",
       "  'Nilanjan  Chatterjee',\n",
       "  'Johns Hopkins University',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/079954.source.xml',\n",
       "  '10.1371/journal.pone.0228198'],\n",
       " ['10.1101/093120',\n",
       "  'Darwinian selection of host and bacteria supports emergence of Lamarckian-like adaptation of the system as a whole',\n",
       "  'Osmanovic, D.; Kessler, D. A.; Rabin, Y.; Soen, Y.',\n",
       "  'Yoav  Soen',\n",
       "  'Weizmann Institute of Science',\n",
       "  '2018-08-21',\n",
       "  '5',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/093120.source.xml',\n",
       "  '10.1186/s13062-018-0224-7'],\n",
       " ['10.1101/097816',\n",
       "  'Voltage-gated Proton Channel Hv1 Supports Insulin Secretion in Pancreatic β Cells through calcium entry, depolarization and intracellular pH regulation',\n",
       "  'Pang, H.; Wang, X.; Xi, W.; Zhao, Q.; Zhang, S.; Qin, J.; Lv, J.; Che, Y.; Zuo, W.; Li, S. J.',\n",
       "  'Shu Jie  Li',\n",
       "  'Nankai University',\n",
       "  '2018-08-24',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'cell biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/097816.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/130989',\n",
       "  'On The Peopling Of The Americas: Molecular Evidence For The Paleoamerican And The Solutrean Models',\n",
       "  'Yuan, D.; Huang, S.',\n",
       "  'Shi  Huang',\n",
       "  'Central South University',\n",
       "  '2018-08-23',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/130989.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/152215',\n",
       "  'Range size dynamics can explain why evolutionarily age and diversification rate correlate with contemporary extinction risk in plants',\n",
       "  'Tanentzap, A. J.; Igea, J.; Johnston, M. G.; Larcombe, M. J.',\n",
       "  'Andrew J. Tanentzap',\n",
       "  'University of Cambridge',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/152215.source.xml',\n",
       "  '10.1086/707207'],\n",
       " ['10.1101/166819',\n",
       "  'Competition for synaptic building blocks shapes synaptic plasticity',\n",
       "  'Triesch, J.; Vo, A. D.; Hafner, A.-S.',\n",
       "  'Jochen  Triesch',\n",
       "  'Frankfurt Institute for Advanced Studies',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/166819.source.xml',\n",
       "  '10.7554/elife.37836'],\n",
       " ['10.1101/172221',\n",
       "  'Global expansion and redistribution of Aedes-borne virus transmission risk with climate change',\n",
       "  'Ryan, S. J.; Carlson, C. J.; Mordecai, E. A.; Johnson, L. R.',\n",
       "  'Sadie J. Ryan',\n",
       "  'University of Florida',\n",
       "  '2018-08-26',\n",
       "  '4',\n",
       "  'new results',\n",
       "  'epidemiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/26/172221.source.xml',\n",
       "  '10.1371/journal.pntd.0007213'],\n",
       " ['10.1101/173039',\n",
       "  'Genetic screen suggests an alternative mechanism for azide-mediated inhibition of SecA',\n",
       "  'Jamshad, M.; Chandler, R.; Yule, J.; Robinson, A.; Alam, F.; Dunne, K.; Nabi, N.; Henderson, I. R.; Huber, D.',\n",
       "  'Damon  Huber',\n",
       "  'University of Birmingham',\n",
       "  '2018-08-22',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'microbiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/22/173039.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/176057',\n",
       "  'Prestimulus Activity in the Cingulo-Opercular Network Predicts Memory for Naturalistic Episodic Experience',\n",
       "  'Cohen, N.; Ben-Yakov, A.; Weber, J.; Edelson, M.; Paz, R.; Dudai, Y.',\n",
       "  'Noga  Cohen',\n",
       "  'Columbia University',\n",
       "  '2018-08-24',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/176057.source.xml',\n",
       "  '10.1093/cercor/bhz212'],\n",
       " ['10.1101/195768',\n",
       "  'How genetic disease risks can be misestimated across global populations',\n",
       "  'Kim, M. S.; Patel, K. P.; Teng, A. K.; Berens, A. J.; Lachance, J.',\n",
       "  'Joseph  Lachance',\n",
       "  'Georgia Institute of Technology',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/195768.source.xml',\n",
       "  '10.1186/s13059-018-1561-7'],\n",
       " ['10.1101/196972',\n",
       "  '90-90-90, epidemic control, and ending AIDS: review of global situation and recommendations',\n",
       "  'Granich, R.; Williams, B. G.; Gupta, S.; Zuniga, J. M.',\n",
       "  'Reuben  Granich',\n",
       "  'Independent Public Health Consultant',\n",
       "  '2018-08-21',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'epidemiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/196972.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/201137',\n",
       "  'Transcriptomic Dynamics of a non-coding trinucleotide repeat expansion disorder SCA12 in iPSC derived neuronal cells: signatures of interferon induced response',\n",
       "  'Kumar, D.; Dhapola, P.; Hussain, A.; Kutum, R.; Srivastava, A. K.; Mukerji, M.; Mukherjee, O.; Faruq, M.',\n",
       "  'Mohammed  Faruq',\n",
       "  'CSIR- Institute of Genomics and Integrative Biology',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'molecular biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/201137.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/201285',\n",
       "  'Electrophysiological Signature Reveals Laminar Structure of the Porcine Hippocampus',\n",
       "  'Ulyanova, A. V.; Koch, P. F.; Cottone, C.; Grovola, M. R.; Browne, K. D.; Russo, R. J.; Adam, C. D.; Smith, D. H.; Chen, H. I.; Johnson, V. E.; Cullen, D. K.; Wolf, J. A.',\n",
       "  'John A Wolf',\n",
       "  'University of Pennsylvania, Corporal Michael J. Crescenz Veterans Affair Medical Center',\n",
       "  '2018-08-26',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/26/201285.source.xml',\n",
       "  '10.1523/eneuro.0102-18.2018'],\n",
       " ['10.1101/213330',\n",
       "  'The Fission Yeast S-Phase Cyclin Cig2 Can Drive Mitosis',\n",
       "  'Magner, M.; Pickering, M.; Keifenheim, D.; Rhind, N.',\n",
       "  'Nicholas  Rhind',\n",
       "  'University of Massachusetts Medical School',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'cell biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/213330.source.xml',\n",
       "  '10.1093/genetics/iyaa002'],\n",
       " ['10.1101/216598',\n",
       "  'The cost of enforcing a marine protected area to achieve ecological targets for the recovery of fish biomass',\n",
       "  'Brown, C.; Parker, B.; Ahmadia, G. N.; Ardiwijaya, R.; Purwanto, P.; Game, E. T.',\n",
       "  'Christopher  Brown',\n",
       "  'Griffith University',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/216598.source.xml',\n",
       "  '10.1016/j.biocon.2018.09.021'],\n",
       " ['10.1101/221432',\n",
       "  'GMSE: an R package for generalised management strategy evaluation',\n",
       "  'Duthie, A. B.; Cusack, J. J.; Jones, I. L.; Minderman, J.; Nilsen, E. B.; Pozo, R. A.; Rakotonarivo, O. S.; Van Moorter, B.; Bunnefeld, N.',\n",
       "  'Alexander B Duthie',\n",
       "  'University of Stirling',\n",
       "  '2018-08-23',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/221432.source.xml',\n",
       "  '10.1111/2041-210x.13091'],\n",
       " ['10.1101/222810',\n",
       "  'Experimental evidence that female rhesus macaques (Macaca mulatta) perceive variation in male facial masculinity',\n",
       "  'Rosenfield, K. A.; Semple, S.; Georgiev, A. V.; Maestripieri, D.; Higham, J. P.; Dubuc, C.',\n",
       "  'Kevin A Rosenfield',\n",
       "  'Penn State University',\n",
       "  '2018-08-26',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'animal behavior and cognition',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/26/222810.source.xml',\n",
       "  '10.1098/rsos.181415'],\n",
       " ['10.1101/224089',\n",
       "  'Habitat complexity and benthic predator-prey interactions in Chesapeake Bay',\n",
       "  'Glaspie, C. N.; Seitz, R. D.',\n",
       "  'Cassandra N Glaspie',\n",
       "  'Oregon State University',\n",
       "  '2018-08-25',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/25/224089.source.xml',\n",
       "  '10.1371/journal.pone.0205162'],\n",
       " ['10.1101/224733',\n",
       "  'Combining accurate tumour genome simulation with crowd-sourcing to benchmark somatic structural variant detection',\n",
       "  'Lee, A. Y.-W.; Ewing, A. D.; Ellrott, K.; Hu, Y.; Houlahan, K. E.; Bare, J. C.; Espiritu, S. M. G.; Huang, V.; Dang, K.; Chong, Z.; Caloian, C.; Yamaguchi, T. N.; ICGC-TCGA DREAM Somatic Mutation Calling Challenge Participants,  ; Kellen, M. R.; Chen, K.; Norman, T. C.; Friend, S. H.; Guinney, J.; Stolovitzky, G.; Haussler, D.; Margolin, A. A.; Stuart, J. M.; Boutros, P. C.',\n",
       "  'Paul C. Boutros',\n",
       "  'Ontario Institute for Cancer Research',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/224733.source.xml',\n",
       "  '10.1186/s13059-018-1539-5'],\n",
       " ['10.1101/230375',\n",
       "  'A Very Oil Yellow1 modifier of the Oil Yellow1-N1989 allele uncovers a cryptic phenotypic impact of cis-regulatory variation in maize',\n",
       "  'Khangura, R. S.; Marla, S.; Venkata, B. P.; Heller, N.; Johal, G. S.; Dilkes, B. P.',\n",
       "  'Brian P Dilkes',\n",
       "  'Purdue University',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genetics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/230375.source.xml',\n",
       "  '10.1534/g3.118.200798'],\n",
       " ['10.1101/237065',\n",
       "  'Exploring Single-Cell Data with Deep Multitasking Neural Networks',\n",
       "  'Amodio, M.; van Dijk, D.; Srinivasan, K.; Chen, W. S.; Mohsen, H.; Moon, K. R.; Campbell, A.; Zhao, Y.; Wang, X.; Venkataswamy, M.; Desai, A.; V., R.; Kumar, P.; Montgomery, R.; Wolf, G.; Krishnaswamy, S.',\n",
       "  'Smita  Krishnaswamy',\n",
       "  'Yale University',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/237065.source.xml',\n",
       "  '10.1038/s41592-019-0576-7'],\n",
       " ['10.1101/237065',\n",
       "  'Exploring Single-Cell Data with Deep Multitasking Neural Networks',\n",
       "  'Amodio, M.; van Dijk, D.; Srinivasan, K.; Chen, W. S.; Mohsen, H.; Moon, K. R.; Campbell, A.; Zhao, Y.; Wang, X.; Venkataswamy, M.; Desai, A.; V., R.; Kumar, P.; Montgomery, R.; Wolf, G.; Krishnaswamy, S.',\n",
       "  'Smita  Krishnaswamy',\n",
       "  'Yale University',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/237065.source.xml',\n",
       "  '10.1038/s41592-019-0576-7'],\n",
       " ['10.1101/240713',\n",
       "  'Scaling of G1 duration with population doubling time by a cyclin in Saccharomyces cerevisiae',\n",
       "  'Blank, H. M.; Callahan, M.; Pistikopoulos, I. P. E.; Polymenis, A. O.; Polymenis, M.',\n",
       "  'Michael  Polymenis',\n",
       "  'Texas A&M University',\n",
       "  '2018-08-25',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'cell biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/25/240713.source.xml',\n",
       "  '10.1534/genetics.118.301507'],\n",
       " ['10.1101/242768',\n",
       "  'Asymmetric histone incorporation during DNA replication in Drosophila male germline stem cells',\n",
       "  'Wooten, M.; Snedeker, J.; Nizami, Z.; Yang, X.; Urban, E.; Feng, X. A.; Kim, J. M.; Gall, J.; Xiao, J.; Chen, X.',\n",
       "  'Xin  Chen',\n",
       "  'Department of Biology, The Johns Hopkins University',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'developmental biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/242768.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/243568',\n",
       "  'Identifying biomarkers of anti-cancer drug synergy using multi-task learning',\n",
       "  'Aben, N.; de Ruiter, J. R.; Bosdriesz, E.; Kim, Y.; Bounova, G.; Vis, D. J.; Wessels, L. F. A.; Michaut, M.',\n",
       "  'Magali  Michaut',\n",
       "  'Netherlands Cancer Institute & Oncode Institute',\n",
       "  '2018-08-23',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/243568.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/246165',\n",
       "  'Noise propagation in an integrated model of bacterial gene expression and growth',\n",
       "  'Kleijn, I. T.; Krah, L. H. J.; Hermsen, R.',\n",
       "  'Rutger  Hermsen',\n",
       "  'Utrecht University',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'systems biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/246165.source.xml',\n",
       "  '10.1371/journal.pcbi.1006386'],\n",
       " ['10.1101/248484',\n",
       "  'Utility of the SWAN Scale for ADHD Trait-Based Genetic Research: A Validity and Polygenic Risk Study',\n",
       "  'Burton, C. L.; Wright, L.; Shan, J.; Xiao, B.; Dupuis, A.; Goodale, T.; Shaheen, S.-M.; Corfield, E. C.; Arnold, P. D.; Schachar, R. J.; Crosbie, J.',\n",
       "  'Jennifer  Crosbie',\n",
       "  'Hospital for Sick Children; University of Toronto',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genetics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/248484.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/253203',\n",
       "  'Modelling mitosis with multiple phenotypes: relation to Haeckel’s recapitulation law',\n",
       "  'Alexandrov, Y.',\n",
       "  'Yuriy  Alexandrov',\n",
       "  'Imperial College London',\n",
       "  '2018-08-22',\n",
       "  '4',\n",
       "  'new results',\n",
       "  'physiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/22/253203.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/253518',\n",
       "  'Priority effects and non-hierarchical competition shape species composition in a complex grassland community',\n",
       "  'Uricchio, L. H.; Daws, S. C.; Spear, E. R.; Mordecai, E. A.',\n",
       "  'Erin A Mordecai',\n",
       "  'Stanford University',\n",
       "  '2018-08-24',\n",
       "  '4',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/253518.source.xml',\n",
       "  '10.1086/701434'],\n",
       " ['10.1101/257105',\n",
       "  'Intron-mediated enhancement boosts Rtn4 circRNA expression: A robust method for exploring circRNA function',\n",
       "  'Mo, D.; Li, X.; Cui, D.; Vollmar, J.-F.',\n",
       "  'Dingding  Mo',\n",
       "  'Max Planck Institute for Biology of Ageing',\n",
       "  '2018-08-22',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'biochemistry',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/22/257105.source.xml',\n",
       "  '10.1038/s41598-019-48224-y'],\n",
       " ['10.1101/259507',\n",
       "  'The Rate of Molecular Evolution When Mutation May Not Be Weak',\n",
       "  'de Koning, A. P. J.; De Sanctis, B. D.',\n",
       "  'A.P. Jason  de Koning',\n",
       "  'University of Calgary',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genetics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/259507.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/260950',\n",
       "  'The interplay of synaptic plasticity and scaling enables the self-organized allocation of multiple memory representations',\n",
       "  'Auth, J. M.; Nachstedt, T.; Tetzlaff, C.',\n",
       "  'Christian  Tetzlaff',\n",
       "  'Third Institute of Physics, Georg-August-University Goettingen, Germany',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/260950.source.xml',\n",
       "  '10.3389/fncir.2020.541728'],\n",
       " ['10.1101/264580',\n",
       "  'Competition in biofilms between cystic fibrosis isolates of Pseudomonas aeruginosa is driven by R-pyocins',\n",
       "  'Oluyombo, O.; Diggle, S. P.; Penfold, C. N.',\n",
       "  'Stephen P Diggle',\n",
       "  'Georgia Institute of Technology',\n",
       "  '2018-08-21',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'microbiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/264580.source.xml',\n",
       "  '10.1128/mBio.01828-18'],\n",
       " ['10.1101/264960',\n",
       "  'A plant biodiversity effect resolved to a single genetic locus',\n",
       "  'Wuest, S. E.; Niklaus, P. A.',\n",
       "  'Samuel E Wuest',\n",
       "  'University of Zurich',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/264960.source.xml',\n",
       "  '10.1038/s41559-018-0708-y'],\n",
       " ['10.1101/264945',\n",
       "  'Neo-formation of chromosomes in bacteria',\n",
       "  'Poirion, O. B.; Lafay, B.',\n",
       "  'Benedicte  Lafay',\n",
       "  'CNRS',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/264945.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/264846',\n",
       "  'Cyfip1 haploinsufficiency increases compulsive-like behavior and modulates palatable food intake: Implications for Prader-Willi Syndrome',\n",
       "  'Babbs, R. K.; Ruan, Q. T.; Kelliher, J. C.; Beierle, J. A.; Chen, M. M.; Feng, A. X.; Kirkpatrick, S. L.; Benitez, F. A.; Rodriguez, F. A.; Pierre, J.; Anandakumar, J.; Kumar, V.; Mulligan, M. K.; Bryant, C. D.',\n",
       "  'Camron D Bryant',\n",
       "  'Boston University School of Medicine',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genetics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/264846.source.xml',\n",
       "  '10.1534/g3.119.400470'],\n",
       " ['10.1101/266700',\n",
       "  'Metagenomic analysis with strain-level resolution reveals fine-scale variation in the human pregnancy microbiome',\n",
       "  'Goltsman, D. S. A.; Sun, C. L.; Proctor, D. M.; DiGiulio, D. B.; Robaczewska, A.; Thomas, B. C.; Shaw, G. M.; Stevenson, D. K.; Holmes, S. P.; Banfield, J. F.; Relman, D. A.',\n",
       "  'David A. Relman',\n",
       "  'Stanford University',\n",
       "  '2018-08-21',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'microbiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/266700.source.xml',\n",
       "  '10.1101/gr.236000.118'],\n",
       " ['10.1101/268623',\n",
       "  'Developing an automated iterative near-term forecasting system for an ecological study',\n",
       "  'White, E. P.; Yenni, G. M.; Taylor, S. D.; Christensen, E. M.; Bledsoe, E. K.; Simonis, J. L.; Ernest, S. K. M.',\n",
       "  'Ethan P White',\n",
       "  'University of Florida',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/268623.source.xml',\n",
       "  '10.1111/2041-210x.13104'],\n",
       " ['10.1101/267906',\n",
       "  'Tbx1 regulates extracellular matrix-cell interactions in the second heart field.',\n",
       "  'Alfano, D.; Altomonte, A.; Cortes, C.; Bilio, M.; Kelly, R. G.; Baldini, A.',\n",
       "  'Antonio  Baldini',\n",
       "  'University of Naples Federico II',\n",
       "  '2018-08-26',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'developmental biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/26/267906.source.xml',\n",
       "  '10.1093/hmg/ddz058'],\n",
       " ['10.1101/275545',\n",
       "  'A 3D model of human skeletal muscle innervated with stem cell-derived motor neurons enables epsilon-subunit targeted myasthenic syndrome studies',\n",
       "  'Afshar Bakooshli, M.; Lippmann, E. S.; Mulcahy, B.; Iyer, N.; Nguyen, C. T.; Tung, K.; Stewart, B. A.; Mouly, V.; Pegoraro, E.; Ahn, H.; Ginsberg, H.; Zhen, M.; Ashton, R. S.; Gilbert, P. M.',\n",
       "  'Penney M Gilbert',\n",
       "  'Institute of Biomaterials and Biomedical Engineering, University of Toronto, Toronto, ON, Canada',\n",
       "  '2018-08-23',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'bioengineering',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/275545.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/275545',\n",
       "  'A 3D model of human skeletal muscle innervated with stem cell-derived motor neurons enables epsilon-subunit targeted myasthenic syndrome studies',\n",
       "  'Afshar Bakooshli, M.; Lippmann, E. S.; Mulcahy, B.; Iyer, N.; Nguyen, C. T.; Tung, K.; Stewart, B. A.; Bigot, A.; Pegoraro, E.; Ahn, H.; Ginsberg, H.; Zhen, M.; Ashton, R. S.; Gilbert, P. M.',\n",
       "  'Penney M Gilbert',\n",
       "  'Institute of Biomaterials and Biomedical Engineering, University of Toronto, Toronto, ON, Canada',\n",
       "  '2018-08-26',\n",
       "  '4',\n",
       "  'new results',\n",
       "  'bioengineering',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/26/275545.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/277020',\n",
       "  'Reconstructing imagined letters from early visual cortex reveals tight topographic correspondence between visual mental imagery and perception',\n",
       "  'Senden, M.; Emmerling, T. C.; Van Hoof, R.; Frost, M.; Goebel, R.',\n",
       "  'Mario  Senden',\n",
       "  'Maastricht University',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/277020.source.xml',\n",
       "  '10.1007/s00429-019-01828-6'],\n",
       " ['10.1101/277715',\n",
       "  'Analysis of novel domain-specific mutations in the zebrafish ndr2/cyclops gene generated using CRISPR-Cas9 RNPs',\n",
       "  'Turner, A. N.; Andersen, R. S.; Bookout, I. E.; Brashear, L. N.; Davis, J. C.; Gahan, D. M.; Gotham, J. P.; Hijaz, B. A.; Kaushik, A. S.; McGill, J. B.; Miller, V. L.; Moseley, Z. P.; Nowell, C. L.; Patel, R. K.; Rodgers, M. C.; Shihab, Y. A.; Walker, A. P.; Glover, S. R.; Foster, S. D.; Challa, A. K.',\n",
       "  'Anil Kumar  Challa',\n",
       "  'University of Alabama at Birmingham',\n",
       "  '2018-08-25',\n",
       "  '3',\n",
       "  'confirmatory results',\n",
       "  'genetics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/25/277715.source.xml',\n",
       "  '10.1007/s12041-018-1033-6'],\n",
       " ['10.1101/279570',\n",
       "  'Capturing diverse microbial sequence with comprehensive and scalable probe design',\n",
       "  'Metsky, H. C.; Siddle, K. J.; Gladden-Young, A.; Qu, J.; Yang, D. K.; Brehio, P.; Goldfarb, A.; Piantadosi, A.; Wohl, S.; Carter, A.; Lin, A. E.; Barnes, K. G.; Tully, D. C.; Corleis, B.; Hennigan, S.; Barbosa-Lima, G.; Vieira, Y. R.; Paul, L. M.; Tan, A. L.; Garcia, K. F.; Parham, L. A.; Odia, I.; Eromon, P.; Folarin, O. A.; Goba, A.; Viral Hemorrhagic Fever Consortium,  ; Simon-Loriere, E.; Hensley, L.; Balmaseda, A.; Harris, E.; Kwon, D.; Allen, T. M.; Runstadler, J. A.; Smole, S.; Bozza, F. A.; Souza, T. M.; Isern, S.; Michael, S. F.; Lorenzana, I.; Gehrke, L.; Bosch, I.; Ebel, G.; Grant, ',\n",
       "  'Hayden C Metsky',\n",
       "  'Broad Institute of MIT and Harvard, USA',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/279570.source.xml',\n",
       "  '10.1038/s41587-018-0006-x'],\n",
       " ['10.1101/282301',\n",
       "  'Genetic dissection of assortative mating behavior',\n",
       "  'Merrill, R. M.; Rastas, P.; Melo, M.-C.; Martin, S. H.; Barker, S.; Davey, J.; McMillan, W. O.; Jiggins, C.',\n",
       "  'Richard M Merrill',\n",
       "  'Ludwig-Maximilians-Universitat, Munich',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/282301.source.xml',\n",
       "  '10.1371/journal.pbio.2005902'],\n",
       " ['10.1101/285981',\n",
       "  'Integrative Cross-species Analyses Suggest Deficits in Habituation Learning as a Widely Affected Mechanism in Intellectual Disability and Autism Spectrum Disorders',\n",
       "  'Fenckova, M.; Asztalos, L.; Cizek, P.; Singgih, E. L.; Blok, L. E. R.; Glennon, J. C.; IntHout, J.; Zweier, C.; Eichler, E. E.; Bernier, R.; Asztalos, Z.; Schenck, A.',\n",
       "  'Annette  Schenck',\n",
       "  'Radboud university medical center',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/285981.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/291088',\n",
       "  'Intracellular signaling through the comRS system in Streptococcus mutans genetic competence',\n",
       "  'Underhill, S. A. M.; Shields, R. C.; Kaspar, J. R.; Haider, M.; Burne, R. A.; Hagen, S. J.',\n",
       "  'Stephen J Hagen',\n",
       "  'University of Florida, Physics Department',\n",
       "  '2018-08-22',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'microbiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/22/291088.source.xml',\n",
       "  '10.1128/mSphere.00444-18'],\n",
       " ['10.1101/298356',\n",
       "  'Science podcasts: analysis of global production and output from 2004 to 2018',\n",
       "  'MacKenzie, L. E.',\n",
       "  'Lewis E MacKenzie',\n",
       "  'Durham University, Department of Chemistry',\n",
       "  '2018-08-28',\n",
       "  '8',\n",
       "  'new results',\n",
       "  'scientific communication and education',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/298356.source.xml',\n",
       "  '10.1098/rsos.180932'],\n",
       " ['10.1101/300251',\n",
       "  'Persistent coding of outcome-predictive cue features in the rat nucleus accumbens',\n",
       "  'Gmaz, J. M.; Carmichael, J. E.; van der Meer, M.',\n",
       "  'Matthijs  van der Meer',\n",
       "  'Dartmouth College',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/300251.source.xml',\n",
       "  '10.7554/elife.37275'],\n",
       " ['10.1101/301838',\n",
       "  'Robust and clinically relevant prediction of response to anti-cancer drugs via network integration of molecular profiles',\n",
       "  'Franco, M.; Jeggari, A.; Peuget, S.; Böttger, F.; Selivanova, G.; Alexeyenko, A.',\n",
       "  'Andrey  Alexeyenko',\n",
       "  'Karolinska Institutet',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'cancer biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/301838.source.xml',\n",
       "  '10.1038/s41598-019-39019-2'],\n",
       " ['10.1101/302679',\n",
       "  'Draft Aphaenogaster genomes expand our view of ant genome size variation across climate gradients',\n",
       "  'Lau, M.; Ellison, A. M.; Nguyen, A. D.; Penick, C.; DeMarco, B.; Gotelli, N. J.; Sanders, N.; Dunn, R.; Helms Cahan, S.',\n",
       "  'Matthew  Lau',\n",
       "  'Harvard University',\n",
       "  '2018-08-23',\n",
       "  '4',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/302679.source.xml',\n",
       "  '10.7717/peerj.6447'],\n",
       " ['10.1101/304832',\n",
       "  'Development of the P300 from childhood to adulthood: a multimodal EEG and MRI study',\n",
       "  'Overbye, K.; Huster, R. J.; Walhovd, K. B.; Fjell, A. M.; Tamnes, C. K.',\n",
       "  'Knut  Overbye',\n",
       "  'Department of Psychology, University of Oslo',\n",
       "  '2018-08-25',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/25/304832.source.xml',\n",
       "  '10.1007/s00429-018-1755-5'],\n",
       " ['10.1101/307165',\n",
       "  'The genome of Ectocarpus subulatus highlights unique mechanisms for stress tolerance in brown algae',\n",
       "  'Dittami, S. M.; Corre, E.; Brillet-Gueguen, L.; Pontoizeau, N.; Aite, M.; Avia, K.; Caron, C.; Cho, C. H.; Collen, J.; Cormier, A.; Delage, L.; Doubleau, S.; Frioux, C.; Gobet, A.; Gonzalez-Navarrete, I.; Groisillier, A.; Herve, C.; Jollivet, D.; KleinJan, H.; Leblanc, C.; Lipinska, A. P.; Liu, X.; Marie, D.; Markov, G. V.; Minoche, A. E.; Monsoor, M.; Pericard, P.; Perrineau, M.-M.; Peters, A. F.; Siegel, A.; Simeon, A.; Trottier, C.; Yoon, H. S.; Himmelbauer, H.; Boyen, C.; Tonon, T.',\n",
       "  'Simon M. Dittami',\n",
       "  'Sorbonne Universite, CNRS, Integrative Biology of Marine Models, Station Biologique de Roscoff',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/307165.source.xml',\n",
       "  '10.1016/j.margen.2020.100740'],\n",
       " ['10.1101/307322',\n",
       "  'When higher carrying capacities lead to faster propagation',\n",
       "  'Haond, M.; Morel-Journel, T.; Lombaert, E.; Vercken, E.; Mailleret, L.; Roques, L.',\n",
       "  'Marjorie  Haond',\n",
       "  \"Universite Cote d'Azur, INRA, CNRS, ISA, France\",\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/307322.source.xml',\n",
       "  '10.24072/pcjournal.66'],\n",
       " ['10.1101/307264',\n",
       "  'The evolution of CHC22 clathrin influences its role in human glucose metabolism',\n",
       "  'Fumagalli, M.; Diekmann, Y.; Camus, S. M.; Burke, A.; Camus, M. D.; Norman, P. J.; Joseph, A.-P.; Abi-Rached, L.; Benazzo, A.; Rasteiro, R.; Mathieson, I.; Topf, M.; Parham, P.; Thomas, M. G.; Brodsky, F. M.',\n",
       "  'Frances M Brodsky',\n",
       "  'University College London',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genetics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/307264.source.xml',\n",
       "  '10.7554/eLife.41517'],\n",
       " ['10.1101/310227',\n",
       "  'Molecular Strategy for Blocking Isopeptide Bond Formation in Nascent Pilin Proteins',\n",
       "  'Rivas-Pardo, J. A.; Tapia-Rojo, R.; Badilla, C. L.; Alonso Caballero, A.; Fernandez, J. M.',\n",
       "  'Jaime Andres  Rivas-Pardo',\n",
       "  'Columbia University',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'biophysics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/310227.source.xml',\n",
       "  '10.1073/pnas.1807689115'],\n",
       " ['10.1101/310151',\n",
       "  'A consistent picture of TRPV1 activation emerges from molecular simulations and experiments',\n",
       "  'Kasimova, M.; Yazici, A.; Yudin, Y.; Granata, D.; Klein, M. L.; Rohacs, T.; Carnevale, V.',\n",
       "  'Vincenzo  Carnevale',\n",
       "  'Temple University',\n",
       "  '2018-08-24',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'biophysics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/310151.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/311050',\n",
       "  'Physical activity predicts population-level age-related differences in frontal white matter',\n",
       "  'Strommer, J. M.; Davis, S. W.; Henson, R. N.; Tyler, L. K.; Cam-CAN,  ; Campbell, K. L.',\n",
       "  'Juho M. Strommer',\n",
       "  'University of Jyvaskyla',\n",
       "  '2018-08-24',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/311050.source.xml',\n",
       "  '10.1093/gerona/gly220'],\n",
       " ['10.1101/247411',\n",
       "  'Exploring the genetic correlations of antisocial behavior and life history traits',\n",
       "  'Tielbeek, J.; Barnes, J. C.; Popma, A.; Polderman, T.; Lee, J. J.; Perry, J. B.; Posthuma, D.; Boutwell, B. B.',\n",
       "  'Brian B Boutwell',\n",
       "  'Saint Louis University',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/247411.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/312355',\n",
       "  'Germline genomic landscapes of breast cancer patients significantly predict clinical outcomes',\n",
       "  'Milanese, J.-S.; Tibiche, C.; Zaman, N.; Zou, J.; Han, P.; Meng, Z. G.; Nantel, A.; Droit, A.; Wang, E.',\n",
       "  'Edwin  Wang',\n",
       "  'University Of Calgary',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'cancer biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/312355.source.xml',\n",
       "  '10.1038/s41698-019-0100-7'],\n",
       " ['10.1101/313296',\n",
       "  'CHASMplus reveals the scope of somatic missense mutations driving human cancers',\n",
       "  'Tokheim, C.; Karchin, R.',\n",
       "  'Rachel  Karchin',\n",
       "  'Johns Hopkins University',\n",
       "  '2018-08-24',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/313296.source.xml',\n",
       "  '10.1016/j.cels.2019.05.005'],\n",
       " ['10.1101/312827',\n",
       "  'EEG-based auditory attention detection: boundary conditions for background noise and speaker positions',\n",
       "  'Das, N.; Bertrand, A.; Francart, T.',\n",
       "  'Neetha  Das',\n",
       "  'KU Leuven',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'bioengineering',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/312827.source.xml',\n",
       "  '10.1088/1741-2552/aae0a6'],\n",
       " ['10.1101/312397',\n",
       "  'A practical generation interval-based approach to inferring the strength of epidemics from their speed',\n",
       "  'Park, S. W.; Champredon, D.; Weitz, J.; Dushoff, J.',\n",
       "  'Jonathan  Dushoff',\n",
       "  'McMaster University',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'epidemiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/312397.source.xml',\n",
       "  '10.1016/j.epidem.2018.12.002'],\n",
       " ['10.1101/317511',\n",
       "  'Omitted variable bias in GLMs of neural spiking activity',\n",
       "  'Stevenson, I. H.',\n",
       "  'Ian H Stevenson',\n",
       "  'University of Connecticut',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/317511.source.xml',\n",
       "  '10.1162/neco_a_01138'],\n",
       " ['10.1101/319657',\n",
       "  'Aurora B is required for programmed variations of cytokinesis during morphogenesis in the C. elegans embryo',\n",
       "  'Bai, X.; Lee, P.-Y.; Chen, C.-Y.; Simmons, J. R.; Nebenfuehr, B.; Mitchell, D.; Klebanow, L. R.; Mattson, N.; Sorensen Turpin, C. G.; Chen, B.-C.; Betzig, E.; Bembenek, J. N.',\n",
       "  'Joshua N Bembenek',\n",
       "  'UT Knoxville',\n",
       "  '2018-08-27',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'cell biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/319657.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/320473',\n",
       "  'Poor coherence in older people’s speech is explained by impaired semantic and executive processes',\n",
       "  'Hoffman, P.; Loginova, E.; Russell, A.',\n",
       "  'Paul  Hoffman',\n",
       "  'University of Edinburgh',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/320473.source.xml',\n",
       "  '10.7554/eLife.38907'],\n",
       " ['10.1101/321067',\n",
       "  'RAVEN 2.0: a versatile toolbox for metabolic network reconstruction and a case study on Streptomyces coelicolor',\n",
       "  'Wang, H.; Marcisauskas, S.; Sanchez, B. J.; Domenzain, I.; Hermansson, D.; Agren, R.; Nielsen, J.; Kerkhoven, E. J.',\n",
       "  'Eduard Johannes Kerkhoven',\n",
       "  'Chalmers University of Technology',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'systems biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/321067.source.xml',\n",
       "  '10.1371/journal.pcbi.1006541'],\n",
       " ['10.1101/318279',\n",
       "  'Effects of Attentional Bias Modification on Residual Symptoms in depression. A Randomized Controlled Trial.',\n",
       "  'Jonassen, R.; Harmer, C. J.; Hilland, E.; Maglanoc, L. A.; Kraft, B.; Browning, M.; Stiles, T. C.; Haaland, V. O.; Berge, T.; Landro, N. I.',\n",
       "  'Rune  Jonassen',\n",
       "  'Department of Psychology, University of Oslo, Norway',\n",
       "  '2018-08-22',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'clinical trials',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/22/318279.source.xml',\n",
       "  '10.1186/s12888-019-2105-8'],\n",
       " ['10.1101/322008',\n",
       "  'Lead-DBS v2: Towards a comprehensive pipeline for deep brain stimulation imaging',\n",
       "  'Horn, A.; Li, N.; Dembek, T. A.; Kappel, A.; Boulay, C.; Ewert, S.; Tietze, A.; Husch, A.; Perera, T.; Neumann, W.-J.; Reisert, M.; Si, H.; Oostenveld, R.; Rorden, C.; Yeh, F.-C.; Fang, Q.; Herrington, T. M.; Vorwerk, J.; Kuehn, A. A.',\n",
       "  'Andreas  Horn',\n",
       "  'Department for Neurology, Charite Berlin',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/322008.source.xml',\n",
       "  '10.1016/j.neuroimage.2018.08.068'],\n",
       " ['10.1101/325399',\n",
       "  'Common genomic regions underlie natural variation in diverse toxin responses',\n",
       "  'Brady, S.; Evans, K.; Bloom, J.; Tanny, R.; Cook, D.; Giuliani, S.; Hippleheuser, S.; Zamanian, M.; Andersen, E.',\n",
       "  'Erik  Andersen',\n",
       "  'Northwestern University',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genetics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/325399.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/326470',\n",
       "  'The UCSC Xena Platform for cancer genomics data visualization and interpretation',\n",
       "  'Goldman, M.; Craft, B.; Kamath, A.; Brooks, A. N.; Zhu, J.; Haussler, D.',\n",
       "  'Mary  Goldman',\n",
       "  'UC Santa Cruz Genomics Institute',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'cancer biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/326470.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/326173',\n",
       "  'Fast open modification spectral library searching through approximate nearest neighbor indexing',\n",
       "  'Bittremieux, W.; Meysman, P.; Noble, W. S.; Laukens, K.',\n",
       "  'Wout  Bittremieux',\n",
       "  'University of Antwerp',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/326173.source.xml',\n",
       "  '10.1021/acs.jproteome.8b00359'],\n",
       " ['10.1101/328328',\n",
       "  'Simultaneous multiplexed amplicon sequencing and transcriptome profiling in single cells',\n",
       "  'Saikia, M.; Burnham, P.; Keshavjee, S. H.; Wang, M. F. Z.; Heyang, M.; Moral-Lopez, P.; Hinchman, M. M.; Danko, C. G.; Parker, J. S. L.; De Vlaminck, I.',\n",
       "  'Iwijn  De Vlaminck',\n",
       "  'Cornell University',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/328328.source.xml',\n",
       "  '10.1038/s41592-018-0259-9'],\n",
       " ['10.1101/325290',\n",
       "  'A systematic method for surveying data visualizations and a resulting genomic epidemiology visualization typology: GEViT',\n",
       "  'Crisan, A.; Gardy, J. L.; Munzner, T.',\n",
       "  'Tamara  Munzner',\n",
       "  'University of British Columbia',\n",
       "  '2018-08-24',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/325290.source.xml',\n",
       "  '10.1093/bioinformatics/bty832'],\n",
       " ['10.1101/329300',\n",
       "  'Bidirectional intraflagellar transport is restricted to two sets of microtubule doublets in the trypanosome flagellum',\n",
       "  'Bertiaux, E.; Mallet, A.; Fort, C.; Blisnick, T.; Bonnefoy, S.; Jung, J.; Lemos, M.; Marco, S.; Vaughan, S.; Trepout, S.; Tinevez, J.-Y.; Bastin, P.',\n",
       "  'Philippe  Bastin',\n",
       "  'Institut Pasteur',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'cell biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/329300.source.xml',\n",
       "  '10.1083/jcb.201805030'],\n",
       " ['10.1101/321216',\n",
       "  'De novo mutations drive the spread of macrolide resistant Mycoplasma genitalium: a mathematical modelling study',\n",
       "  'Cadosch, D.; Garcia, V.; Althaus, C. L.; Jensen, J. S.; Low, N.',\n",
       "  'Nicola  Low',\n",
       "  'University of Bern',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'microbiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/321216.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/330241',\n",
       "  'Relative value perception in an insect: positive and negative incentive contrasts in ants',\n",
       "  'Wendt, S.; Strunk, K. S.; Heinze, J.; Roider, A.; Czaczkes, T. J.',\n",
       "  'Stephanie  Wendt',\n",
       "  'University of Regensburg',\n",
       "  '2018-08-24',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'animal behavior and cognition',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/330241.source.xml',\n",
       "  '10.7554/eLife.45450'],\n",
       " ['10.1101/330308',\n",
       "  'Genome Functional Annotation using Deep Convolutional Neural Networks',\n",
       "  'Khodabandelou, G.; Routhier, E.; Mozziconacci, J.',\n",
       "  'Ghazaleh  Khodabandelou',\n",
       "  'Sorbonne Universite',\n",
       "  '2018-08-21',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/330308.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/339036',\n",
       "  'A variably imprinted epiallele impacts seed development',\n",
       "  'Pignatta, D.; Novitzky, K.; Satyaki, P. R. V.; Gehring, M.',\n",
       "  'Mary  Gehring',\n",
       "  'Whitehead Institute for Biomedical Research',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'plant biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/339036.source.xml',\n",
       "  '10.1371/journal.pgen.1007469'],\n",
       " ['10.1101/336222',\n",
       "  'The Evolution of Bequeathal in Stable Habitats',\n",
       "  'Clarke, P. M. R.; McElreath, M. B.; Barrett, B. J.; Mabry, K. E.; McElreath, R. L.',\n",
       "  'Mary Brooke  McElreath',\n",
       "  'Max Planck Institute for Ornithology',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/336222.source.xml',\n",
       "  '10.1002/ece3.4549'],\n",
       " ['10.1101/338624',\n",
       "  'Automatic Removal of False Connections in Diffusion MRI Tractography Using Topology-Informed Pruning (TIP)',\n",
       "  'Yeh, F.-C.; Panesar, S.; Barrios, J.; Fernandes, D.; Abhinav, K.; Meola, A.; Fernandez-Miranda, J. C.',\n",
       "  'Fang-Cheng  Yeh',\n",
       "  'University of Pittsburgh',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/338624.source.xml',\n",
       "  '10.1007/s13311-018-0663-y'],\n",
       " ['10.1101/339630',\n",
       "  'Weakly supervised classification of rare aortic valve malformations using unlabeled cardiac MRI sequences',\n",
       "  'Fries, J. A.; Varma, P.; Chen, V. S.; Xiao, K.; Tejeda, H.; Saha, P.; Dunnmon, J.; Chubb, H.; Maskatia, S.; Fiterau, M.; Delp, S.; Ashley, E.; Re, C.; Priest, J.',\n",
       "  'Jason A Fries',\n",
       "  'Stanford University',\n",
       "  '2018-08-22',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'bioinformatics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/22/339630.source.xml',\n",
       "  '10.1038/s41467-019-11012-3'],\n",
       " ['10.1101/339770',\n",
       "  'Longitudinal studies at birth and age 7 reveal strong effects of genetic variation on ancestry-associated DNA methylation patterns in blood cells from ethnically admixed children',\n",
       "  \"McKennan, C.; Naughton, K.; Stanhope, C.; Kattan, M.; O'Connor, G.; Sandel, M.; Visness, C.; Wood, R.; Bacharier, L.; Beigelman, A.; Lovisky-Desir, S.; Togias, A.; Gern, J.; Nicolae, D.; Ober, C.\",\n",
       "  'Chris  McKennan',\n",
       "  'Department of Human Genetics, University of Chicago',\n",
       "  '2018-08-24',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/339770.source.xml',\n",
       "  '10.1080/15592294.2020.1817290'],\n",
       " ['10.1101/346007',\n",
       "  'Cerebrovascular Injuries Induce Lymphatic Invasion into Brain Parenchyma to Guide Vascular Regeneration in Zebrafish',\n",
       "  'Chen, J.; He, J.; Yang, Q.; Zhang, Y.; Luo, L.',\n",
       "  'Lingfei  Luo',\n",
       "  'School of Life Sciences, Southwest University',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'developmental biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/346007.source.xml',\n",
       "  '10.1016/j.devcel.2019.03.022'],\n",
       " ['10.1101/352393',\n",
       "  'Biologically realistic mean-field models of conductance-based networks of spiking neurons with adaptation',\n",
       "  'di Volo, M.; Romagnoni, A.; Capone, C.; Destexhe, A.',\n",
       "  'Matteo  di Volo',\n",
       "  'Unite de Neuroscience, Information et Complexite (UNIC), CNRS FRE 3693, 1 avenue de la Terrasse',\n",
       "  '2018-08-25',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'neuroscience',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/25/352393.source.xml',\n",
       "  '10.1162/neco_a_01173'],\n",
       " ['10.1101/356196',\n",
       "  'A Fasciclin 2 functional switch controls organ size in Drosophila',\n",
       "  'Velasquez, E.; Gomez-Sanchez, J. A.; Donier, E.; Grijota-Martinez, C.; Cabedo, H.; Garcia-Alonso, L. A.',\n",
       "  'Luis A Garcia-Alonso',\n",
       "  'Instituto de Neurociencias CSIC-UMH',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'developmental biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/356196.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/356246',\n",
       "  'Gut bacterial tyrosine decarboxylases restrict the bioavailability of levodopa, the primary treatment in Parkinson’s disease',\n",
       "  'van Kessel, S. P.; Frye, A. K.; El-Gendy, A. O.; Castejon, M.; Keshavarzian, A.; van Dijk, G.; El Aidy, S.',\n",
       "  'Sahar  El Aidy',\n",
       "  'Groningen Biomolecular Sciences and Biotechnology Institute (GBB)',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'molecular biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/356246.source.xml',\n",
       "  '10.1038/s41467-019-08294-y'],\n",
       " ['10.1101/357467',\n",
       "  'DataRemix: a universal data transformation for optimal inference from gene expression datasets',\n",
       "  'Mao, W.; Hausler, R.; Chikina, M.',\n",
       "  'Maria  Chikina',\n",
       "  'University of Pittsburgh',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/357467.source.xml',\n",
       "  '10.1093/bioinformatics/btaa745'],\n",
       " ['10.1101/358044',\n",
       "  'Phylogeographic analysis and identification of factors impacting the diffusion of Foot-and-Mouth disease virus in Africa',\n",
       "  'Duchatel, F.; Bronsvoort, M.; Lycett, S.',\n",
       "  'Samantha  Lycett',\n",
       "  'Roslin institute, Edinburgh University',\n",
       "  '2018-08-23',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/358044.source.xml',\n",
       "  '10.3389/fevo.2019.00371'],\n",
       " ['10.1101/361089',\n",
       "  'Genetic variation in a complex polyploid: unveiling the dynamic allelic features of sugarcane',\n",
       "  'Sforca, D. A.; Vautrin, S.; Cardoso-Silva, C. B.; Mancini, M. C.; Cruz, M. V. R.; Pereira, G. d. S.; Conte, M.; Bellec, A.; Dahmer, N.; Fourment, J.; Rodde, N.; Van-Sluys, M.-A.; Vicentini, R.; Garcia, A. A. F.; Forni-Martins, E. R.; Carneiro, M. S.; Hoffmann, H. P.; Pinto, L. R.; Landell, M. G. d. A.; Vincentz, M.; Berges, H.; Souza, A. P.',\n",
       "  'Anete Pereira Souza',\n",
       "  'Universidade Estadual de Campinas (UNICAMP), Campinas, SP, Brazil',\n",
       "  '2018-08-21',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'plant biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/21/361089.source.xml',\n",
       "  '10.3389/fpls.2019.00553'],\n",
       " ['10.1101/361758',\n",
       "  'Continual exit of human cutaneous resident memory CD4 T cells that seed distant skin sites',\n",
       "  'Klicznik, M. M.; Morawski, P. A.; Höllbacher, B.; Varkhande, S.; Motley, S.; Rosenblum, M. D.; Long, S. A.; Brachtl, G.; Duhen, T.; Campbell, D.; Gratz, I. K.',\n",
       "  'Iris K Gratz',\n",
       "  'University of Salzburg',\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'immunology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/361758.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/362186',\n",
       "  'Using the Weibull accelerated failure time regression model to predict time to health events',\n",
       "  'Liu, E.; Lim, K.',\n",
       "  'Enwu  Liu',\n",
       "  'MaryMckillop Institue for Health Research',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'epidemiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/362186.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/362129',\n",
       "  'Parallel adaptations to different host plants despite clonal reproduction in the world most devastating nematode pest',\n",
       "  'Koutsovoulos, G.; Marques, E.; Arguel, M.-J.; Machado, A. C.; Carneiro, R. M.; Castagnone-Sereno, P.; Danchin, E. G.; Albuquerque, E. V.',\n",
       "  'Etienne GJ Danchin',\n",
       "  \"Institut Sophia Agrobiotech, INRA, CNRS, Universite Cote d'Azur, France\",\n",
       "  '2018-08-23',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/362129.source.xml',\n",
       "  '10.1111/eva.12881'],\n",
       " ['10.1101/361840',\n",
       "  'Detection of instability in 18S and 5.8S rDNA copy numbers and 18S:5.8S ratios in longitudinally-collected human DNA samples detected by monochrome multiplex qPCR',\n",
       "  'Ligerman, C. B.; Baird, L.; Leppert, M. F.; Cawthon, R. M.',\n",
       "  'Richard M Cawthon',\n",
       "  'University of Utah, Dept. of Human Genetics',\n",
       "  '2018-08-27',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/27/361840.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/361840',\n",
       "  'Instability in 18S and 5.8S rDNA copy numbers and 18S:5.8S ratios in longitudinally-collected human DNA samples detected by monochrome multiplex qPCR',\n",
       "  'Ligerman, C. B.; Baird, L.; Leppert, M. F.; Cawthon, R. M.',\n",
       "  'Richard M Cawthon',\n",
       "  'University of Utah, Dept. of Human Genetics',\n",
       "  '2018-08-28',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'genomics',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/361840.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/363622',\n",
       "  'New tools for diet analyses: nanopore sequencing of metagenomic DNA from rat stomach contents to quantify diet',\n",
       "  'Pearman, W.; Smith, A. N. H.; Breckell, G.; Dale, J.; Freed, N. E.; Silander, O. K.',\n",
       "  'Olin K. Silander',\n",
       "  'Institute of Natural and Mathematical Sciences, Massey University, Auckland 0745, New Zealand',\n",
       "  '2018-08-28',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'ecology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/28/363622.source.xml',\n",
       "  'NA'],\n",
       " ['10.1101/363077',\n",
       "  'Systems Biology Understanding of the Effects of Lithium on Cancer.',\n",
       "  'Ge, W.; Jakobsson, E.',\n",
       "  'Eric  Jakobsson',\n",
       "  'University of Illinois at Urbana-Champaign',\n",
       "  '2018-08-24',\n",
       "  '2',\n",
       "  'new results',\n",
       "  'cancer biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/24/363077.source.xml',\n",
       "  '10.3389/fonc.2019.00296'],\n",
       " ['10.1101/364331',\n",
       "  'Resource limitation modulates the fate of dissimilated nitrogen in a dual-pathway Actinobacterium',\n",
       "  'Vuono, D. C.; Read, R. W.; Hemp, J.; Sullivan, B. W.; Arnone, J. A.; Neveux, I.; Blank, B.; Staub, C.; Loney, E.; Miceli, D.; Winkler, M.; Chakraborty, R.; Stahl, D. A.; Grzymski, J. J.',\n",
       "  'Joseph J Grzymski',\n",
       "  'Desert Research Institute',\n",
       "  '2018-08-23',\n",
       "  '4',\n",
       "  'new results',\n",
       "  'microbiology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/364331.source.xml',\n",
       "  '10.3389/fmicb.2019.00003'],\n",
       " ['10.1101/364190',\n",
       "  'Intragenomic Conflict over Bet-Hedging',\n",
       "  'Wilkins, J. F.; Bhattacharya, T.',\n",
       "  'Jon F Wilkins',\n",
       "  'Ronin Institute',\n",
       "  '2018-08-23',\n",
       "  '3',\n",
       "  'new results',\n",
       "  'evolutionary biology',\n",
       "  'https://www.biorxiv.org/content/early/2018/08/23/364190.source.xml',\n",
       "  '10.1098/rstb.2018.0142']]"
      ]
     },
     "execution_count": 24,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "journal_list = []\n",
    "\n",
    "# `license`,`abstract`, and `server` are excluded from the metrics. \n",
    "for journal in response_info[\"collection\"]:\n",
    "    journal_list.append([journal[\"doi\"], journal[\"title\"], journal[\"authors\"],\n",
    "                         journal[\"author_corresponding\"], \n",
    "                         journal[\"author_corresponding_institution\"],\n",
    "                         journal[\"date\"], journal[\"version\"], journal[\"type\"],\n",
    "                         journal[\"category\"], journal[\"jatsxml\"], journal[\"published\"]])\n",
    "    \n",
    "journal_list"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 25,
   "id": "27bbfb07",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>DOI</th>\n",
       "      <th>Title</th>\n",
       "      <th>Authors</th>\n",
       "      <th>Corresponding Authors</th>\n",
       "      <th>Institution</th>\n",
       "      <th>Date</th>\n",
       "      <th>Version</th>\n",
       "      <th>Type</th>\n",
       "      <th>Category</th>\n",
       "      <th>Xml</th>\n",
       "      <th>Published</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>10.1101/066688</td>\n",
       "      <td>Cross-scale dynamics and the evolutionary emer...</td>\n",
       "      <td>Schreiber, S. J.; Ke, R.; Loverdo, C.; Park, M...</td>\n",
       "      <td>Sebastian J. Schreiber</td>\n",
       "      <td>University of California, Davis</td>\n",
       "      <td>2018-08-21</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>epidemiology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1093/ve/veaa105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>10.1101/079954</td>\n",
       "      <td>iCARE: An R Package to Build, Validate and App...</td>\n",
       "      <td>Pal Choudhury, P.; Maas, P.; Wilcox, A.; Wheel...</td>\n",
       "      <td>Nilanjan  Chatterjee</td>\n",
       "      <td>Johns Hopkins University</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>bioinformatics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1371/journal.pone.0228198</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>10.1101/093120</td>\n",
       "      <td>Darwinian selection of host and bacteria suppo...</td>\n",
       "      <td>Osmanovic, D.; Kessler, D. A.; Rabin, Y.; Soen...</td>\n",
       "      <td>Yoav  Soen</td>\n",
       "      <td>Weizmann Institute of Science</td>\n",
       "      <td>2018-08-21</td>\n",
       "      <td>5</td>\n",
       "      <td>new results</td>\n",
       "      <td>evolutionary biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1186/s13062-018-0224-7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>10.1101/097816</td>\n",
       "      <td>Voltage-gated Proton Channel Hv1 Supports Insu...</td>\n",
       "      <td>Pang, H.; Wang, X.; Xi, W.; Zhao, Q.; Zhang, S...</td>\n",
       "      <td>Shu Jie  Li</td>\n",
       "      <td>Nankai University</td>\n",
       "      <td>2018-08-24</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>cell biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>10.1101/130989</td>\n",
       "      <td>On The Peopling Of The Americas: Molecular Evi...</td>\n",
       "      <td>Yuan, D.; Huang, S.</td>\n",
       "      <td>Shi  Huang</td>\n",
       "      <td>Central South University</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>evolutionary biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>10.1101/361840</td>\n",
       "      <td>Instability in 18S and 5.8S rDNA copy numbers ...</td>\n",
       "      <td>Ligerman, C. B.; Baird, L.; Leppert, M. F.; Ca...</td>\n",
       "      <td>Richard M Cawthon</td>\n",
       "      <td>University of Utah, Dept. of Human Genetics</td>\n",
       "      <td>2018-08-28</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>genomics</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>10.1101/363622</td>\n",
       "      <td>New tools for diet analyses: nanopore sequenci...</td>\n",
       "      <td>Pearman, W.; Smith, A. N. H.; Breckell, G.; Da...</td>\n",
       "      <td>Olin K. Silander</td>\n",
       "      <td>Institute of Natural and Mathematical Sciences...</td>\n",
       "      <td>2018-08-28</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>ecology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>NA</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>10.1101/363077</td>\n",
       "      <td>Systems Biology Understanding of the Effects o...</td>\n",
       "      <td>Ge, W.; Jakobsson, E.</td>\n",
       "      <td>Eric  Jakobsson</td>\n",
       "      <td>University of Illinois at Urbana-Champaign</td>\n",
       "      <td>2018-08-24</td>\n",
       "      <td>2</td>\n",
       "      <td>new results</td>\n",
       "      <td>cancer biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.3389/fonc.2019.00296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>10.1101/364331</td>\n",
       "      <td>Resource limitation modulates the fate of diss...</td>\n",
       "      <td>Vuono, D. C.; Read, R. W.; Hemp, J.; Sullivan,...</td>\n",
       "      <td>Joseph J Grzymski</td>\n",
       "      <td>Desert Research Institute</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>4</td>\n",
       "      <td>new results</td>\n",
       "      <td>microbiology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.3389/fmicb.2019.00003</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>10.1101/364190</td>\n",
       "      <td>Intragenomic Conflict over Bet-Hedging</td>\n",
       "      <td>Wilkins, J. F.; Bhattacharya, T.</td>\n",
       "      <td>Jon F Wilkins</td>\n",
       "      <td>Ronin Institute</td>\n",
       "      <td>2018-08-23</td>\n",
       "      <td>3</td>\n",
       "      <td>new results</td>\n",
       "      <td>evolutionary biology</td>\n",
       "      <td>https://www.biorxiv.org/content/early/2018/08/...</td>\n",
       "      <td>10.1098/rstb.2018.0142</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>100 rows × 11 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "               DOI                                              Title  \\\n",
       "0   10.1101/066688  Cross-scale dynamics and the evolutionary emer...   \n",
       "1   10.1101/079954  iCARE: An R Package to Build, Validate and App...   \n",
       "2   10.1101/093120  Darwinian selection of host and bacteria suppo...   \n",
       "3   10.1101/097816  Voltage-gated Proton Channel Hv1 Supports Insu...   \n",
       "4   10.1101/130989  On The Peopling Of The Americas: Molecular Evi...   \n",
       "..             ...                                                ...   \n",
       "95  10.1101/361840  Instability in 18S and 5.8S rDNA copy numbers ...   \n",
       "96  10.1101/363622  New tools for diet analyses: nanopore sequenci...   \n",
       "97  10.1101/363077  Systems Biology Understanding of the Effects o...   \n",
       "98  10.1101/364331  Resource limitation modulates the fate of diss...   \n",
       "99  10.1101/364190             Intragenomic Conflict over Bet-Hedging   \n",
       "\n",
       "                                              Authors   Corresponding Authors  \\\n",
       "0   Schreiber, S. J.; Ke, R.; Loverdo, C.; Park, M...  Sebastian J. Schreiber   \n",
       "1   Pal Choudhury, P.; Maas, P.; Wilcox, A.; Wheel...    Nilanjan  Chatterjee   \n",
       "2   Osmanovic, D.; Kessler, D. A.; Rabin, Y.; Soen...              Yoav  Soen   \n",
       "3   Pang, H.; Wang, X.; Xi, W.; Zhao, Q.; Zhang, S...             Shu Jie  Li   \n",
       "4                                 Yuan, D.; Huang, S.              Shi  Huang   \n",
       "..                                                ...                     ...   \n",
       "95  Ligerman, C. B.; Baird, L.; Leppert, M. F.; Ca...       Richard M Cawthon   \n",
       "96  Pearman, W.; Smith, A. N. H.; Breckell, G.; Da...        Olin K. Silander   \n",
       "97                              Ge, W.; Jakobsson, E.         Eric  Jakobsson   \n",
       "98  Vuono, D. C.; Read, R. W.; Hemp, J.; Sullivan,...       Joseph J Grzymski   \n",
       "99                   Wilkins, J. F.; Bhattacharya, T.           Jon F Wilkins   \n",
       "\n",
       "                                          Institution        Date Version  \\\n",
       "0                     University of California, Davis  2018-08-21       2   \n",
       "1                            Johns Hopkins University  2018-08-23       2   \n",
       "2                       Weizmann Institute of Science  2018-08-21       5   \n",
       "3                                   Nankai University  2018-08-24       3   \n",
       "4                            Central South University  2018-08-23       3   \n",
       "..                                                ...         ...     ...   \n",
       "95        University of Utah, Dept. of Human Genetics  2018-08-28       3   \n",
       "96  Institute of Natural and Mathematical Sciences...  2018-08-28       2   \n",
       "97         University of Illinois at Urbana-Champaign  2018-08-24       2   \n",
       "98                          Desert Research Institute  2018-08-23       4   \n",
       "99                                    Ronin Institute  2018-08-23       3   \n",
       "\n",
       "           Type              Category  \\\n",
       "0   new results          epidemiology   \n",
       "1   new results        bioinformatics   \n",
       "2   new results  evolutionary biology   \n",
       "3   new results          cell biology   \n",
       "4   new results  evolutionary biology   \n",
       "..          ...                   ...   \n",
       "95  new results              genomics   \n",
       "96  new results               ecology   \n",
       "97  new results        cancer biology   \n",
       "98  new results          microbiology   \n",
       "99  new results  evolutionary biology   \n",
       "\n",
       "                                                  Xml  \\\n",
       "0   https://www.biorxiv.org/content/early/2018/08/...   \n",
       "1   https://www.biorxiv.org/content/early/2018/08/...   \n",
       "2   https://www.biorxiv.org/content/early/2018/08/...   \n",
       "3   https://www.biorxiv.org/content/early/2018/08/...   \n",
       "4   https://www.biorxiv.org/content/early/2018/08/...   \n",
       "..                                                ...   \n",
       "95  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "96  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "97  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "98  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "99  https://www.biorxiv.org/content/early/2018/08/...   \n",
       "\n",
       "                       Published  \n",
       "0             10.1093/ve/veaa105  \n",
       "1   10.1371/journal.pone.0228198  \n",
       "2      10.1186/s13062-018-0224-7  \n",
       "3                             NA  \n",
       "4                             NA  \n",
       "..                           ...  \n",
       "95                            NA  \n",
       "96                            NA  \n",
       "97       10.3389/fonc.2019.00296  \n",
       "98      10.3389/fmicb.2019.00003  \n",
       "99        10.1098/rstb.2018.0142  \n",
       "\n",
       "[100 rows x 11 columns]"
      ]
     },
     "execution_count": 25,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "journal_df = pd.DataFrame(data=journal_list,\n",
    "                         columns=[\"DOI\", \"Title\", \"Authors\", \"Corresponding Authors\",\n",
    "                                  \"Institution\",\n",
    "                                  \"Date\", \"Version\", \"Type\", \"Category\", \"Xml\", \"Published\"])\n",
    "journal_df"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.9.12"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
